
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>GCF_028745735.1_Roseinatronobacter_sp._HJB301_HJB301 - 4 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/bacteria.css">
<meta property="og:title" content="GCF_028745735.1_Roseinatronobacter_sp._HJB301_HJB301 - 4 record(s) - 4 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-556dc918-5a32-4dcd-b701-ec6a84e33c93/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://antismash.secondarymetabolites.org/">
<img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://antismash.secondarymetabolites.org/">
antiSMASH version 7.0.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="GCF_028745735.1_Roseinatronobacter_sp._HJB301_HJB301.zip">Download all results</a></li>

<li><a href="GCF_028745735.1_Roseinatronobacter_sp._HJB301_HJB301.gbk">Download GenBank summary file</a></li>

<li><a href="bacteria-556dc918-5a32-4dcd-b701-ec6a84e33c93.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton other hserlactone r44c1">
<a href="#r44c1">44.1</a>
</div>



<div class="regbutton terpene terpene r77c1">
<a href="#r77c1">77.1</a>
</div>



<div class="regbutton hybrid r99c1">
<a href="#r99c1">99.1</a>
</div>



<div class="regbutton other ectoine r110c1">
<a href="#r110c1">110.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>NZ_JAQZSM010000003.1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-44" data-filename="NZ_JAQZSM010000003.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-44" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r44c1">
<td class="regbutton other hserlactone r44c1">
<a href="#r44c1">Region&nbsp44.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">58,141</td>
<td class="digits table-split-left">78,992</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_JAQZSM010000006.1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-77" data-filename="NZ_JAQZSM010000006.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-77" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r77c1">
<td class="regbutton terpene terpene r77c1">
<a href="#r77c1">Region&nbsp77.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">176,586</td>
<td class="digits table-split-left">197,356</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_JAQZSM010000008.1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-99" data-filename="NZ_JAQZSM010000008.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-99" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r99c1">
<td class="regbutton hybrid r99c1">
<a href="#r99c1">Region&nbsp99.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">24,390</td>
<td class="digits table-split-left">76,264</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_JAQZSM010000009.1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-110" data-filename="NZ_JAQZSM010000009.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-110" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r110c1">
<td class="regbutton other ectoine r110c1">
<a href="#r110c1">Region&nbsp110.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
</td>
<td class="digits">60,785</td>
<td class="digits table-split-left">71,171</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000860/1" target="_blank">ectoine</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 80%, #ffffff00 80%)">80%</td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r44c1">
<td class="regbutton other hserlactone r44c1">
<a href="#r44c1">Region&nbsp44.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">58,141</td>
<td class="digits table-split-left">78,992</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r77c1">
<td class="regbutton terpene terpene r77c1">
<a href="#r77c1">Region&nbsp77.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">176,586</td>
<td class="digits table-split-left">197,356</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r99c1">
<td class="regbutton hybrid r99c1">
<a href="#r99c1">Region&nbsp99.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">24,390</td>
<td class="digits table-split-left">76,264</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r110c1">
<td class="regbutton other ectoine r110c1">
<a href="#r110c1">Region&nbsp110.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
</td>
<td class="digits">60,785</td>
<td class="digits table-split-left">71,171</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000860/1" target="_blank">ectoine</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 80%, #ffffff00 80%)">80%</td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>NZ_JAQZSM010000001.1</strong></dd>

<dd><strong>NZ_JAQZSM010000010.1</strong></dd>

<dd><strong>NZ_JAQZSM010000100.1</strong></dd>

<dd><strong>NZ_JAQZSM010000101.1</strong></dd>

<dd><strong>NZ_JAQZSM010000102.1</strong></dd>

<dd><strong>NZ_JAQZSM010000103.1</strong></dd>

<dd><strong>NZ_JAQZSM010000104.1</strong></dd>

<dd><strong>NZ_JAQZSM010000105.1</strong></dd>

<dd><strong>NZ_JAQZSM010000106.1</strong></dd>

<dd><strong>NZ_JAQZSM010000107.1</strong></dd>

<dd><strong>NZ_JAQZSM010000108.1</strong></dd>

<dd><strong>NZ_JAQZSM010000109.1</strong></dd>

<dd><strong>NZ_JAQZSM010000011.1</strong></dd>

<dd><strong>NZ_JAQZSM010000110.1</strong></dd>

<dd><strong>NZ_JAQZSM010000111.1</strong></dd>

<dd><strong>NZ_JAQZSM010000112.1</strong></dd>

<dd><strong>NZ_JAQZSM010000113.1</strong></dd>

<dd><strong>NZ_JAQZSM010000114.1</strong></dd>

<dd><strong>NZ_JAQZSM010000115.1</strong></dd>

<dd><strong>NZ_JAQZSM010000116.1</strong></dd>

<dd><strong>NZ_JAQZSM010000117.1</strong></dd>

<dd><strong>NZ_JAQZSM010000118.1</strong></dd>

<dd><strong>NZ_JAQZSM010000119.1</strong></dd>

<dd><strong>NZ_JAQZSM010000012.1</strong></dd>

<dd><strong>NZ_JAQZSM010000120.1</strong></dd>

<dd><strong>NZ_JAQZSM010000013.1</strong></dd>

<dd><strong>NZ_JAQZSM010000014.1</strong></dd>

<dd><strong>NZ_JAQZSM010000015.1</strong></dd>

<dd><strong>NZ_JAQZSM010000016.1</strong></dd>

<dd><strong>NZ_JAQZSM010000017.1</strong></dd>

<dd><strong>NZ_JAQZSM010000018.1</strong></dd>

<dd><strong>NZ_JAQZSM010000019.1</strong></dd>

<dd><strong>NZ_JAQZSM010000002.1</strong></dd>

<dd><strong>NZ_JAQZSM010000020.1</strong></dd>

<dd><strong>NZ_JAQZSM010000021.1</strong></dd>

<dd><strong>NZ_JAQZSM010000022.1</strong></dd>

<dd><strong>NZ_JAQZSM010000023.1</strong></dd>

<dd><strong>NZ_JAQZSM010000024.1</strong></dd>

<dd><strong>NZ_JAQZSM010000025.1</strong></dd>

<dd><strong>NZ_JAQZSM010000026.1</strong></dd>

<dd><strong>NZ_JAQZSM010000027.1</strong></dd>

<dd><strong>NZ_JAQZSM010000028.1</strong></dd>

<dd><strong>NZ_JAQZSM010000029.1</strong></dd>

<dd><strong>NZ_JAQZSM010000030.1</strong></dd>

<dd><strong>NZ_JAQZSM010000031.1</strong></dd>

<dd><strong>NZ_JAQZSM010000032.1</strong></dd>

<dd><strong>NZ_JAQZSM010000033.1</strong></dd>

<dd><strong>NZ_JAQZSM010000034.1</strong></dd>

<dd><strong>NZ_JAQZSM010000035.1</strong></dd>

<dd><strong>NZ_JAQZSM010000036.1</strong></dd>

<dd><strong>NZ_JAQZSM010000037.1</strong></dd>

<dd><strong>NZ_JAQZSM010000038.1</strong></dd>

<dd><strong>NZ_JAQZSM010000039.1</strong></dd>

<dd><strong>NZ_JAQZSM010000004.1</strong></dd>

<dd><strong>NZ_JAQZSM010000040.1</strong></dd>

<dd><strong>NZ_JAQZSM010000041.1</strong></dd>

<dd><strong>NZ_JAQZSM010000042.1</strong></dd>

<dd><strong>NZ_JAQZSM010000043.1</strong></dd>

<dd><strong>NZ_JAQZSM010000044.1</strong></dd>

<dd><strong>NZ_JAQZSM010000045.1</strong></dd>

<dd><strong>NZ_JAQZSM010000046.1</strong></dd>

<dd><strong>NZ_JAQZSM010000047.1</strong></dd>

<dd><strong>NZ_JAQZSM010000048.1</strong></dd>

<dd><strong>NZ_JAQZSM010000049.1</strong></dd>

<dd><strong>NZ_JAQZSM010000005.1</strong></dd>

<dd><strong>NZ_JAQZSM010000050.1</strong></dd>

<dd><strong>NZ_JAQZSM010000051.1</strong></dd>

<dd><strong>NZ_JAQZSM010000052.1</strong></dd>

<dd><strong>NZ_JAQZSM010000053.1</strong></dd>

<dd><strong>NZ_JAQZSM010000054.1</strong></dd>

<dd><strong>NZ_JAQZSM010000055.1</strong></dd>

<dd><strong>NZ_JAQZSM010000056.1</strong></dd>

<dd><strong>NZ_JAQZSM010000057.1</strong></dd>

<dd><strong>NZ_JAQZSM010000058.1</strong></dd>

<dd><strong>NZ_JAQZSM010000059.1</strong></dd>

<dd><strong>NZ_JAQZSM010000060.1</strong></dd>

<dd><strong>NZ_JAQZSM010000061.1</strong></dd>

<dd><strong>NZ_JAQZSM010000062.1</strong></dd>

<dd><strong>NZ_JAQZSM010000063.1</strong></dd>

<dd><strong>NZ_JAQZSM010000064.1</strong></dd>

<dd><strong>NZ_JAQZSM010000065.1</strong></dd>

<dd><strong>NZ_JAQZSM010000066.1</strong></dd>

<dd><strong>NZ_JAQZSM010000067.1</strong></dd>

<dd><strong>NZ_JAQZSM010000068.1</strong></dd>

<dd><strong>NZ_JAQZSM010000069.1</strong></dd>

<dd><strong>NZ_JAQZSM010000007.1</strong></dd>

<dd><strong>NZ_JAQZSM010000070.1</strong></dd>

<dd><strong>NZ_JAQZSM010000071.1</strong></dd>

<dd><strong>NZ_JAQZSM010000072.1</strong></dd>

<dd><strong>NZ_JAQZSM010000073.1</strong></dd>

<dd><strong>NZ_JAQZSM010000074.1</strong></dd>

<dd><strong>NZ_JAQZSM010000075.1</strong></dd>

<dd><strong>NZ_JAQZSM010000076.1</strong></dd>

<dd><strong>NZ_JAQZSM010000077.1</strong></dd>

<dd><strong>NZ_JAQZSM010000078.1</strong></dd>

<dd><strong>NZ_JAQZSM010000079.1</strong></dd>

<dd><strong>NZ_JAQZSM010000080.1</strong></dd>

<dd><strong>NZ_JAQZSM010000081.1</strong></dd>

<dd><strong>NZ_JAQZSM010000082.1</strong></dd>

<dd><strong>NZ_JAQZSM010000083.1</strong></dd>

<dd><strong>NZ_JAQZSM010000084.1</strong></dd>

<dd><strong>NZ_JAQZSM010000085.1</strong></dd>

<dd><strong>NZ_JAQZSM010000086.1</strong></dd>

<dd><strong>NZ_JAQZSM010000087.1</strong></dd>

<dd><strong>NZ_JAQZSM010000088.1</strong></dd>

<dd><strong>NZ_JAQZSM010000089.1</strong></dd>

<dd><strong>NZ_JAQZSM010000090.1</strong></dd>

<dd><strong>NZ_JAQZSM010000091.1</strong></dd>

<dd><strong>NZ_JAQZSM010000092.1</strong></dd>

<dd><strong>NZ_JAQZSM010000093.1</strong></dd>

<dd><strong>NZ_JAQZSM010000094.1</strong></dd>

<dd><strong>NZ_JAQZSM010000095.1</strong></dd>

<dd><strong>NZ_JAQZSM010000096.1</strong></dd>

<dd><strong>NZ_JAQZSM010000097.1</strong></dd>

<dd><strong>NZ_JAQZSM010000098.1</strong></dd>

<dd><strong>NZ_JAQZSM010000099.1</strong></dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>







<div class="page" id='r44c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_JAQZSM010000003.1 - Region 1 - hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-41"></div> <span class="help-tooltip" id="region_svg-help-41">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_JAQZSM010000003.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r44c1-svg" data-filename="NZ_JAQZSM010000003.1_r44c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 58,141 - 78,992 nt. (total: 20,852 nt)

<a class="cluster-rules-header" id="r44c1-rules-header" href="#r44c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r44c1-rules">hserlactone: Autoind_synth</div>

<div class="region-svg-container">
<div id='r44c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-42"></div> <span class="help-tooltip" id="focus-panel-help-42">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r44c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r44c1-gene-table-details" data-name="r44c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r44c1-MIBiG-cluster-compare" data-name="r44c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r44c1-clusterblast" data-name="r44c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r44c1-knownclusterblast" data-name="r44c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r44c1-subclusterblast" data-name="r44c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r44c1-tfbs-finder" data-name="r44c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r44c1-pfam-details" data-name="r44c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r44c1-tigrfam-details" data-name="r44c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r44c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-7"></div> <span class="help-tooltip" id="gt-help-tooltip-help-7">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r44c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r44c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r44c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r44c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r44c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-1"></div> <span class="help-tooltip" id="MIBiG-help-1">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r44c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r44c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r44c1"></div>
<div id="comparison-MIBiG-r44c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">hserlactone</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002324: 0-17497">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002324/index.html#r1c1" target="_new">BGC0002324</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.44, order:0.39, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.06</td>
<td>NRP, Polyketide</td>

<td>myxochelin B, myxochelin N, myxochelin O, myxochelin P, myxochelin Q, myxochelin A</td>


<td>Archangium violaceum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001437: 451-19435">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001437/index.html#r1c1" target="_new">BGC0001437</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.43, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>NRP</td>

<td>obafluorin</td>


<td>Pseudomonas fluorescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000329: 485-21536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000329/index.html#r1c1" target="_new">BGC0000329</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.42, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>NRP</td>

<td>corpeptin A, corpeptin B</td>


<td>Pseudomonas corrugata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001591: 66-30223">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001591/index.html#r1c1" target="_new">BGC0001591</a></td>





<td class="cc-heat-cell" title="0.22 (id:0.42, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>


<td>0.05</td>
<td>Other</td>

<td>fatty acid enol ester, N-tetradecanoyl tyrosine</td>


<td>uncultured bacterium CSLC2</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000610: 0-34546">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000610/index.html#r1c1" target="_new">BGC0000610</a></td>





<td class="cc-heat-cell" title="0.21 (id:0.45, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>


<td>0.05</td>
<td>RiPP</td>

<td>nosiheptide</td>


<td>Streptomyces actuosus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001480: 24-19784">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001480/index.html#r1c1" target="_new">BGC0001480</a></td>





<td class="cc-heat-cell" title="0.21 (id:0.44, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>


<td>0.05</td>
<td>Other</td>

<td>tundrenone</td>


<td>Methylobacter tundripaludum 21/22</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000864: 742-41396">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000864/index.html#r1c1" target="_new">BGC0000864</a></td>





<td class="cc-heat-cell" title="0.20 (id:0.43, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>


<td>0.04</td>
<td>Other</td>

<td>eicosapentaenoic acid-like compound</td>


<td>Moritella marina ATCC 15381</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000961: 0-36595">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000961/index.html#r1c1" target="_new">BGC0000961</a></td>





<td class="cc-heat-cell" title="0.19 (id:0.44, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>


<td>0.04</td>
<td>NRP, Polyketide</td>

<td>bactobolin</td>


<td>Burkholderia thailandensis E264</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001435: 200-36599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001435/index.html#r1c1" target="_new">BGC0001435</a></td>





<td class="cc-heat-cell" title="0.19 (id:0.42, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>


<td>0.04</td>
<td>NRP, Polyketide</td>

<td>ikarugamycin</td>


<td>Streptomyces sp. ZJ306</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000865: 0-33099">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000865/index.html#r1c1" target="_new">BGC0000865</a></td>





<td class="cc-heat-cell" title="0.18 (id:0.44, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>


<td>0.04</td>
<td>Other</td>

<td>eicoseicosapentaenoic acid</td>


<td>Photobacterium profundum SS9</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r44c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002324: 0-17497">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002324/index.html#r1c1" target="_new">BGC0002324</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.44, order:0.39, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>NRP, Polyketide</td>
<td>myxochelin B, myxochelin N, myxochelin O, myxochelin P, myxochelin Q, myxochelin A</td>
<td>Archangium violaceum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001437: 451-19435">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001437/index.html#r1c1" target="_new">BGC0001437</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.43, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>NRP</td>
<td>obafluorin</td>
<td>Pseudomonas fluorescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000329: 485-21536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000329/index.html#r1c1" target="_new">BGC0000329</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.42, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>NRP</td>
<td>corpeptin A, corpeptin B</td>
<td>Pseudomonas corrugata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001591: 66-30223">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001591/index.html#r1c1" target="_new">BGC0001591</a></td>


<td class="cc-heat-cell" title="0.22 (id:0.42, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
<td>0.22</td>
<td>Other</td>
<td>fatty acid enol ester, N-tetradecanoyl tyrosine</td>
<td>uncultured bacterium CSLC2</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000610: 0-34546">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000610/index.html#r1c1" target="_new">BGC0000610</a></td>


<td class="cc-heat-cell" title="0.21 (id:0.45, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
<td>0.21</td>
<td>RiPP</td>
<td>nosiheptide</td>
<td>Streptomyces actuosus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001480: 24-19784">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001480/index.html#r1c1" target="_new">BGC0001480</a></td>


<td class="cc-heat-cell" title="0.21 (id:0.44, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
<td>0.21</td>
<td>Other</td>
<td>tundrenone</td>
<td>Methylobacter tundripaludum 21/22</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000864: 742-41396">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000864/index.html#r1c1" target="_new">BGC0000864</a></td>


<td class="cc-heat-cell" title="0.20 (id:0.43, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>
<td>0.20</td>
<td>Other</td>
<td>eicosapentaenoic acid-like compound</td>
<td>Moritella marina ATCC 15381</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000961: 0-36595">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000961/index.html#r1c1" target="_new">BGC0000961</a></td>


<td class="cc-heat-cell" title="0.19 (id:0.44, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
<td>0.19</td>
<td>NRP, Polyketide</td>
<td>bactobolin</td>
<td>Burkholderia thailandensis E264</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001435: 200-36599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001435/index.html#r1c1" target="_new">BGC0001435</a></td>


<td class="cc-heat-cell" title="0.19 (id:0.42, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
<td>0.19</td>
<td>NRP, Polyketide</td>
<td>ikarugamycin</td>
<td>Streptomyces sp. ZJ306</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000865: 0-33099">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000865/index.html#r1c1" target="_new">BGC0000865</a></td>


<td class="cc-heat-cell" title="0.18 (id:0.44, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>
<td>0.18</td>
<td>Other</td>
<td>eicoseicosapentaenoic acid</td>
<td>Photobacterium profundum SS9</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r44c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-2"></div> <span class="help-tooltip" id="cb-general-help-2">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r44c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r44c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r44c1_1.svg">
NZ_CP024899 (2485912-2506734): Rhodobaca barguzinensis strain alga05 chromoso... (33% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r44c1_2.svg">
NZ_MEHT01000009 (229666-250488): Roseinatronobacter thiooxidans strain ALG1 A... (33% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r44c1_3.svg">
NZ_QGKU01000052 (49023-69659): Rhodobacteraceae bacterium TG-679, whole genom... (33% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r44c1_4.svg">
NZ_CP027407 (3725466-3746105): Roseobacter denitrificans strain FDAARGOS 309 ... (31% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r44c1_5.svg">
NC_015730 (3117561-3137797): Roseobacter litoralis Och 149, complete sequence (35% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r44c1_6.svg">
NC_003911 (377211-397282): Ruegeria pomeroyi DSS-3, complete sequence (35% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r44c1_7.svg">
NZ_FOVP01000004 (42991-63618): Roseovarius lutimaris strain DSM 28463, whole ... (30% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r44c1_8.svg">
NZ_JAAJBD010000001 (106948-127581): Boseongicola sp. TAW-CT132 scaffold1, who... (28% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r44c1_9.svg">
NZ_CP016364 (3058721-3079363): Phaeobacter porticola strain P97 chromosome, c... (33% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r44c1_10.svg">
NZ_SLWW01000001 (41952-62597): Rhodovulum euryhalinum strain DSM 4868 Ga03105... (33% of genes show similarity), hserlactone
</option>

</select>
<button id= "clusterblast-r44c1-download">Download graphic</button>
</div>
<div id="clusterblast-r44c1-svg"></div>

</div>
</div>

<div class="body-details-section r44c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-3"></div> <span class="help-tooltip" id="cb-known-help-3">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r44c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-4"></div> <span class="help-tooltip" id="cb-sub-help-4">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r44c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-6"></div> <span class="help-tooltip" id="tfbs-body-help-6">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AbrC3
<div class="collapser collapser-target-AbrC3 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Antibiotic production activator<br>
<span>Score</span>: 18.16 out of 18.16
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r44c1-70234-AbrC3"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
LexA
<div class="collapser collapser-target-LexA collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Repressor of DNA damage response<br>
<span>Score</span>: 21.64 out of 29.65
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r44c1-78366-LexA"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r44c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-8"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-8">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r44c1-pfam-details-svg" data-filename="NZ_JAQZSM010000003.1_r44c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r44c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r44c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-9"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-9">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r44c1-tigrfam-details-svg" data-filename="NZ_JAQZSM010000003.1_r44c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r44c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r44c1-tfbs-finder" data-name="r44c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r44c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-5"></div> <span class="help-tooltip" id="tfbs-side-help-5">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AbrC3" target="_blank">AbrC3</a></td>
<td>Antibiotic production activator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/LexA" target="_blank">LexA</a></td>
<td>Repressor of DNA damage response</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r77c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_JAQZSM010000006.1 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-43"></div> <span class="help-tooltip" id="region_svg-help-43">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_JAQZSM010000006.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r77c1-svg" data-filename="NZ_JAQZSM010000006.1_r77c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 176,586 - 197,356 nt. (total: 20,771 nt)

<a class="cluster-rules-header" id="r77c1-rules-header" href="#r77c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r77c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r77c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-44"></div> <span class="help-tooltip" id="focus-panel-help-44">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r77c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r77c1-gene-table-details" data-name="r77c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r77c1-MIBiG-cluster-compare" data-name="r77c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r77c1-clusterblast" data-name="r77c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r77c1-knownclusterblast" data-name="r77c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r77c1-subclusterblast" data-name="r77c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r77c1-tfbs-finder" data-name="r77c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r77c1-pfam-details" data-name="r77c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r77c1-tigrfam-details" data-name="r77c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r77c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-16"></div> <span class="help-tooltip" id="gt-help-tooltip-help-16">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r77c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r77c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r77c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r77c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r77c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-10"></div> <span class="help-tooltip" id="MIBiG-help-10">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r77c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r77c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r77c1"></div>
<div id="comparison-MIBiG-r77c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000904: 116-14341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000904/index.html#r1c1" target="_new">BGC0000904</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Other</td>

<td>FR-900098</td>


<td>Streptomyces rubellomurinus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000592: 3421-9210">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000592/index.html#r1c1" target="_new">BGC0000592</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>microviridin B</td>


<td>Microcystis aeruginosa NIES-298</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002049: 0-7559">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002049/index.html#r1c1" target="_new">BGC0002049</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>microviridin N9, microviridin N8, microviridin N7, microviridin N6, microviridin N5, microviridin N4, microviridin N3</td>


<td>Nostoc punctiforme PCC 73102</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002630: 0-4291">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002630/index.html#r1c1" target="_new">BGC0002630</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>thatisin</td>


<td>Lysobacter antibioticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001638: 626-5883">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001638/index.html#r1c1" target="_new">BGC0001638</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>klebsazolicin</td>


<td>Klebsiella pneumoniae subsp. ozaenae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001222: 137-3363">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001222/index.html#r1c1" target="_new">BGC0001222</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>acidocin B</td>


<td>Lactobacillus acidophilus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002531: 214-11704">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002531/index.html#r1c1" target="_new">BGC0002531</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>NRP</td>

<td>cepaciachelin</td>


<td>Burkholderia ambifaria AMMD</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000583: 0-6725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000583/index.html#r1c1" target="_new">BGC0000583</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>grisemycin</td>


<td>Streptomyces griseus subsp. griseus NBRC 13350</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001887: 1126-4317">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001887/index.html#r1c1" target="_new">BGC0001887</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>huazacin</td>


<td>Bacillus thuringiensis serovar huazhongensis BGSC 4BD1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001526: 0-14417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001526/index.html#r1c1" target="_new">BGC0001526</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.12</td>
<td>Other</td>

<td>bartoloside E, bartoloside F, bartoloside G, bartoloside H, bartoloside I, bartoloside J, bartoloside K</td>


<td>Synechocystis salina LEGE 06099</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r77c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000904: 116-14341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000904/index.html#r1c1" target="_new">BGC0000904</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other</td>
<td>FR-900098</td>
<td>Streptomyces rubellomurinus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000592: 3421-9210">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000592/index.html#r1c1" target="_new">BGC0000592</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>microviridin B</td>
<td>Microcystis aeruginosa NIES-298</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002049: 0-7559">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002049/index.html#r1c1" target="_new">BGC0002049</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>microviridin N9, microviridin N8, microviridin N7, microviridin N6, microviridin N5, microviridin N4, microviridin N3</td>
<td>Nostoc punctiforme PCC 73102</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002630: 0-4291">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002630/index.html#r1c1" target="_new">BGC0002630</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>thatisin</td>
<td>Lysobacter antibioticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001638: 626-5883">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001638/index.html#r1c1" target="_new">BGC0001638</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>klebsazolicin</td>
<td>Klebsiella pneumoniae subsp. ozaenae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001222: 137-3363">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001222/index.html#r1c1" target="_new">BGC0001222</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>RiPP</td>
<td>acidocin B</td>
<td>Lactobacillus acidophilus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002531: 214-11704">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002531/index.html#r1c1" target="_new">BGC0002531</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>NRP</td>
<td>cepaciachelin</td>
<td>Burkholderia ambifaria AMMD</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000583: 0-6725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000583/index.html#r1c1" target="_new">BGC0000583</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>RiPP</td>
<td>grisemycin</td>
<td>Streptomyces griseus subsp. griseus NBRC 13350</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001887: 1126-4317">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001887/index.html#r1c1" target="_new">BGC0001887</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>RiPP</td>
<td>huazacin</td>
<td>Bacillus thuringiensis serovar huazhongensis BGSC 4BD1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001526: 0-14417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001526/index.html#r1c1" target="_new">BGC0001526</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Other</td>
<td>bartoloside E, bartoloside F, bartoloside G, bartoloside H, bartoloside I, bartoloside J, bartoloside K</td>
<td>Synechocystis salina LEGE 06099</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r77c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-11"></div> <span class="help-tooltip" id="cb-general-help-11">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r77c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r77c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r77c1_1.svg">
NZ_MEHT01000009 (25734-53535): Roseinatronobacter thiooxidans strain ALG1 ALG... (75% of genes show similarity), ectoine,terpene
</option>

<option value = "svg/clusterblast_r77c1_2.svg">
NZ_SNYP01000003 (110595-131363): Rhodobacter veldkampii DSM 11550 Ga0310482 1... (26% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r77c1_3.svg">
NZ_FOGU01000004 (180868-209551): Roseivivax roseus strain DSM 23042, whole ge... (16% of genes show similarity), hserlactone,terpene
</option>

<option value = "svg/clusterblast_r77c1_4.svg">
NZ_CP045410 (632682-652225): Roseovarius sp. THAF8 chromosome, complete genome (19% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r77c1_5.svg">
NZ_ML137218 (1006870-1025367): Falsirhodobacter deserti strain W402 Scaffold1... (31% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r77c1_6.svg">
NZ_SLWW01000004 (119618-140365): Rhodovulum euryhalinum strain DSM 4868 Ga031... (21% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r77c1_7.svg">
NZ_CP038492 (1546062-1566833): Boseongicola sp. CCM32 chromosome, complete ge... (20% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r77c1_8.svg">
NZ_CP031750 (1846464-1867232): Rhodobacter sphaeroides strain EBL0706 chromos... (22% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r77c1_9.svg">
NZ_CP004372 (2915243-2936014): Roseibacterium elongatum DSM 19469, complete g... (21% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r77c1_10.svg">
NZ_CP010869 (1570899-1591667): Confluentimicrobium sp. EMB200-NS6 strain EMBL... (20% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r77c1-download">Download graphic</button>
</div>
<div id="clusterblast-r77c1-svg"></div>

</div>
</div>

<div class="body-details-section r77c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-12"></div> <span class="help-tooltip" id="cb-known-help-12">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r77c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-13"></div> <span class="help-tooltip" id="cb-sub-help-13">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r77c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-15"></div> <span class="help-tooltip" id="tfbs-body-help-15">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
BldD
<div class="collapser collapser-target-BldD collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Development and antibiotic global regulator<br>
<span>Score</span>: 20.24 out of 23.43
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r77c1-189652-BldD"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (2):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 19.04 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r77c1-181483-ZuR"></div>

<div class="tfbs-hit-detail">
NrdR
<div class="collapser collapser-target-NrdR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Represses ribonucleotide reductase encoding genes<br>
<span>Score</span>: 20.85 out of 27.18
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r77c1-196523-NrdR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r77c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-17"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-17">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r77c1-pfam-details-svg" data-filename="NZ_JAQZSM010000006.1_r77c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r77c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r77c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-18"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-18">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r77c1-tigrfam-details-svg" data-filename="NZ_JAQZSM010000006.1_r77c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r77c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r77c1-tfbs-finder" data-name="r77c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r77c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-14"></div> <span class="help-tooltip" id="tfbs-side-help-14">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/BldD" target="_blank">BldD</a></td>
<td>Development and antibiotic global regulator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrdR" target="_blank">NrdR</a></td>
<td>Represses ribonucleotide reductase encoding genes</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r99c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_JAQZSM010000008.1 - Region 1 - NRPS-like,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-45"></div> <span class="help-tooltip" id="region_svg-help-45">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_JAQZSM010000008.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r99c1-svg" data-filename="NZ_JAQZSM010000008.1_r99c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 24,390 - 76,264 nt. (total: 51,875 nt)

<a class="cluster-rules-header" id="r99c1-rules-header" href="#r99c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r99c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r99c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-46"></div> <span class="help-tooltip" id="focus-panel-help-46">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r99c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r99c1-gene-table-details" data-name="r99c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r99c1-nrps_pks" data-name="r99c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r99c1-MIBiG-cluster-compare" data-name="r99c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r99c1-clusterblast" data-name="r99c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r99c1-knownclusterblast" data-name="r99c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r99c1-subclusterblast" data-name="r99c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r99c1-tfbs-finder" data-name="r99c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r99c1-nrps-pks-bubbles" data-name="r99c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r99c1-pfam-details" data-name="r99c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r99c1-tigrfam-details" data-name="r99c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r99c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-29"></div> <span class="help-tooltip" id="gt-help-tooltip-help-29">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r99c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r99c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r99c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r99c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r99c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-19"></div> <span class="help-tooltip" id="nrps-domain-help-19">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r99c1-details-svg" data-filename="NZ_JAQZSM010000008.1_r99c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r99c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r99c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-20"></div> <span class="help-tooltip" id="MIBiG-help-20">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r99c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r99c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r99c1"></div>
<div id="comparison-MIBiG-r99c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002474: 103-7254">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002474/index.html#r1c1" target="_new">BGC0002474</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.50</td>
<td>NRP</td>

<td>agrobactin</td>


<td>Agrobacterium tumefaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.46</td>
<td>Polyketide</td>

<td>HEx-pks16 polyketide</td>


<td>Punctularia strigosozonata HHB-11173 SS5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.44</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.44</td>
<td>Polyketide</td>

<td>strobilurin A</td>


<td>Strobilurus tenacellus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275: 1063-6614">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.42</td>
<td>Polyketide</td>

<td>6-methylsalicyclic acid</td>


<td>Glarea lozoyensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276: 1101-6575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.42</td>
<td>Polyketide</td>

<td>6-methylsalicyclic acid</td>


<td>Aspergillus terreus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.42</td>
<td>Polyketide</td>

<td>citreoviridin</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273: 0-5301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.42</td>
<td>Polyketide</td>

<td>asperlactone</td>


<td>Aspergillus ochraceus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.42</td>
<td>Polyketide</td>

<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000883: 0-30199">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000883/index.html#r1c1" target="_new">BGC0000883</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.46, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.46, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.40</td>
<td>Other</td>

<td>arginomycin</td>


<td>Streptomyces arginensis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r99c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002474: 103-7254">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002474/index.html#r1c1" target="_new">BGC0002474</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>agrobactin</td>
<td>Agrobacterium tumefaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>HEx-pks16 polyketide</td>
<td>Punctularia strigosozonata HHB-11173 SS5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>strobilurin A</td>
<td>Strobilurus tenacellus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275: 1063-6614">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>6-methylsalicyclic acid</td>
<td>Glarea lozoyensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276: 1101-6575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>6-methylsalicyclic acid</td>
<td>Aspergillus terreus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>citreoviridin</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273: 0-5301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>asperlactone</td>
<td>Aspergillus ochraceus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000883: 0-30199">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000883/index.html#r1c1" target="_new">BGC0000883</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.46, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other</td>
<td>arginomycin</td>
<td>Streptomyces arginensis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r99c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-21"></div> <span class="help-tooltip" id="cb-general-help-21">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r99c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r99c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r99c1_1.svg">
NZ_FNOM01000001 (453882-506024): Roseicitreum antarcticum strain CGMCC 1.8894... (34% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r99c1_2.svg">
NZ_CP018076 (1138633-1217190): Sulfitobacter sp. AM1-D1 chromosome, complete ... (16% of genes show similarity), NRPS-like,T1PKS,T3PKS
</option>

<option value = "svg/clusterblast_r99c1_3.svg">
NZ_CP025803 (2887359-2939378): Sulfitobacter sp. SK011 chromosome, complete g... (28% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r99c1_4.svg">
NZ_JACIEI010000001 (517165-569024): Sulfitobacter undariae strain DSM 102234 ... (25% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r99c1_5.svg">
NZ_FNBP01000002 (288918-340949): Sulfitobacter delicatus strain DSM 16477, wh... (20% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r99c1_6.svg">
NZ_PHJF01000003 (149422-201425): Epibacterium ulvae strain U95 Ga0207336 103,... (23% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r99c1_7.svg">
NZ_FODS01000007 (12278-98127): Salinihabitans flavidus strain DSM 27842, whol... (15% of genes show similarity), NRPS-like,T1PKS,phosphonate
</option>

<option value = "svg/clusterblast_r99c1_8.svg">
NZ_CP022303 (2428846-2480268): Thalassococcus sp. S3 chromosome, complete genome (18% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r99c1_9.svg">
NZ_CP039964 (2161038-2212876): Pseudorhodobacter sp. S12M18 chromosome, compl... (18% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r99c1_10.svg">
NZ_KE386893 (29718-81879): Pseudodonghicola xiamenensis DSM 18339 G455DRAFT s... (19% of genes show similarity), NRPS-like,T1PKS
</option>

</select>
<button id= "clusterblast-r99c1-download">Download graphic</button>
</div>
<div id="clusterblast-r99c1-svg"></div>

</div>
</div>

<div class="body-details-section r99c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-22"></div> <span class="help-tooltip" id="cb-known-help-22">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r99c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-23"></div> <span class="help-tooltip" id="cb-sub-help-23">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r99c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-27"></div> <span class="help-tooltip" id="tfbs-body-help-27">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AfsR
<div class="collapser collapser-target-AfsR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Pleiotropic regulatory for antibiotic production<br>
<span>Score</span>: 20.58 out of 23.05
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r99c1-64779-AfsR"></div>

</div>


</div>
</div>



<div class="body-details-section r99c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-28"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-28">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r99c1-domain-bubble-svg" data-filename="NZ_JAQZSM010000008.1_r99c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r99c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (24389 - 76264): neighbouring T1PKS-NRPS-like
</option>



<option value="CC2">
Candidate 2 (24389 - 70737): single T1PKS
</option>



<option value="CC3">
Candidate 3 (31737 - 76264): single NRPS-like
</option>


</select>

<div class="details-svg" id="r99c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r99c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r99c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-30"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-30">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r99c1-pfam-details-svg" data-filename="NZ_JAQZSM010000008.1_r99c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r99c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r99c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-31"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-31">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r99c1-tigrfam-details-svg" data-filename="NZ_JAQZSM010000008.1_r99c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r99c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r99c1-nrps_pks_products" data-name="r99c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r99c1-nrps_pks_monomers" data-name="r99c1-nrps_pks_monomers">
NRPS/PKS monomers
</div>



<div class="sidepanel-details-header r99c1-tfbs-finder" data-name="r99c1-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r99c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-24"></div> <span class="help-tooltip" id="nrps-structures-help-24">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 24389 - 76264:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r99c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r99c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 2, location 24389 - 70737:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r99c1-2" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r99c1-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 3, location 31737 - 76264:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r99c1-3" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r99c1-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r99c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS monomer predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-25"></div> <span class="help-tooltip" id="nrps-monomers-help-25">Shows the predicted monomers for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">ctg99_46</span></strong>: mal
<div class="collapser collapser-target-ctg99_46 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (536..831)</strong>: mal
<div class="collapser collapser-target-ctg99_46 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-ctg99_46-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 62.5%</dd>
<dd>2-Rhyd-Malonyl-CoA: 58.3%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-ctg99_46-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 74.3</dt>

<dd></dd><dt>inactive: 47.9</dt>

<dd></dd><dt>Methylmalonyl-CoA: 43.5</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 37.2</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 23.9</dt>

<dd></dd><dt>Isobutyryl-CoA: 16.6</dt>

<dd></dd><dt>Propionyl-CoA: 14.2</dt>

<dd></dd><dt>fatty_acid: 12.7</dt>

<dd></dd><dt>Benzoyl-CoA: 12.0</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 11.3</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 9.2</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>CHC-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

<strong><span class="serif">ctg99_48</span></strong>: X
<div class="collapser collapser-target-ctg99_48 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>A-OX (529..1303)</strong>: X
<div class="collapser collapser-target-ctg99_48 collapser-level-candidate"> </div><div class="collapser-content">

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r99c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-26"></div> <span class="help-tooltip" id="tfbs-side-help-26">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AfsR" target="_blank">AfsR</a></td>
<td>Pleiotropic regulatory for antibiotic production</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>




<div class="page" id='r110c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_JAQZSM010000009.1 - Region 1 - ectoine <div class="help-container"> <div class="help-icon" data-id="region_svg-help-47"></div> <span class="help-tooltip" id="region_svg-help-47">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_JAQZSM010000009.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r110c1-svg" data-filename="NZ_JAQZSM010000009.1_r110c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 60,785 - 71,171 nt. (total: 10,387 nt)

<a class="cluster-rules-header" id="r110c1-rules-header" href="#r110c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r110c1-rules">ectoine: ectoine_synt</div>

<div class="region-svg-container">
<div id='r110c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-48"></div> <span class="help-tooltip" id="focus-panel-help-48">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r110c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r110c1-gene-table-details" data-name="r110c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r110c1-MIBiG-cluster-compare" data-name="r110c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r110c1-clusterblast" data-name="r110c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r110c1-knownclusterblast" data-name="r110c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r110c1-subclusterblast" data-name="r110c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r110c1-tfbs-finder" data-name="r110c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r110c1-pfam-details" data-name="r110c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r110c1-tigrfam-details" data-name="r110c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r110c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-38"></div> <span class="help-tooltip" id="gt-help-tooltip-help-38">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r110c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r110c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r110c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r110c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r110c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-32"></div> <span class="help-tooltip" id="MIBiG-help-32">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r110c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r110c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r110c1"></div>
<div id="comparison-MIBiG-r110c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">ectoine</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860: 710-5134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860</a></td>





<td class="cc-heat-cell" title="0.81 (id:0.53, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>


<td>0.55</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylarcula marina</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856: 517-4347">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856</a></td>





<td class="cc-heat-cell" title="0.80 (id:0.51, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>


<td>0.52</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylophaga thalassica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859: 258-3305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859</a></td>





<td class="cc-heat-cell" title="0.80 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>


<td>0.52</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium alcaliphilum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857: 300-4871">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857</a></td>





<td class="cc-heat-cell" title="0.79 (id:0.49, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>


<td>0.51</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylophaga alcalica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855: 245-3455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855</a></td>





<td class="cc-heat-cell" title="0.73 (id:0.51, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>


<td>0.41</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium kenyense</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854: 838-6879">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.49, order:0.83, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium alcaliphilum 20Z</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.49, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>Other</td>

<td>ectoine</td>


<td>Sporosarcina pasteurii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853: 8-3345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.48, order:0.76, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Other</td>

<td>ectoine</td>


<td>Streptomyces anulatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052: 188-3620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.48, order:0.76, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Other (Ectoine)</td>

<td>ectoine</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.47, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylobacter marinus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r110c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860: 710-5134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860</a></td>


<td class="cc-heat-cell" title="0.81 (id:0.53, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>
<td>0.81</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylarcula marina</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856: 517-4347">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856</a></td>


<td class="cc-heat-cell" title="0.80 (id:0.51, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
<td>0.80</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylophaga thalassica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859: 258-3305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859</a></td>


<td class="cc-heat-cell" title="0.80 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
<td>0.80</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium alcaliphilum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857: 300-4871">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857</a></td>


<td class="cc-heat-cell" title="0.79 (id:0.49, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
<td>0.79</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylophaga alcalica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855: 245-3455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855</a></td>


<td class="cc-heat-cell" title="0.73 (id:0.51, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
<td>0.73</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium kenyense</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854: 838-6879">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.49, order:0.83, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium alcaliphilum 20Z</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.49, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Other</td>
<td>ectoine</td>
<td>Sporosarcina pasteurii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853: 8-3345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.48, order:0.76, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other</td>
<td>ectoine</td>
<td>Streptomyces anulatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052: 188-3620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.48, order:0.76, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other (Ectoine)</td>
<td>ectoine</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.47, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylobacter marinus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r110c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-33"></div> <span class="help-tooltip" id="cb-general-help-33">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r110c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r110c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r110c1_1.svg">
NZ_FXBQ01000001 (2565371-2575770): Maritimibacter sp. HL-12, whole genome sho... (46% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r110c1_2.svg">
NZ_CP024899 (2098634-2109021): Rhodobaca barguzinensis strain alga05 chromoso... (50% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r110c1_3.svg">
NZ_MEHT01000009 (25734-53535): Roseinatronobacter thiooxidans strain ALG1 ALG... (17% of genes show similarity), ectoine,terpene
</option>

<option value = "svg/clusterblast_r110c1_4.svg">
NZ_LXYH01000056 (3698-14082): Rhodobacteraceae bacterium EhC02 A5 Contig6, wh... (50% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r110c1_5.svg">
NZ_CP049056 (3924672-3935059): Rhodobacteraceae bacterium RR4-56 chromosome, ... (71% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r110c1_6.svg">
NZ_FOQH01000008 (123921-134320): Albimonas pacifica strain CGMCC 1.11030, who... (62% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r110c1_7.svg">
NZ_SLXL01000012 (45020-55416): Rhodovulum adriaticum strain DSM 2781 Ga031049... (55% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r110c1_8.svg">
NZ_CP042265 (437871-448267): Litoreibacter sp. LN3S51 plasmid unnamed4, compl... (55% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r110c1_9.svg">
NZ_SLXP01000018 (18835-29231): Rhodovulum marinum strain DSM 18063 Ga0310484 ... (55% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r110c1_10.svg">
NZ_SLWW01000004 (35317-45707): Rhodovulum euryhalinum strain DSM 4868 Ga03105... (55% of genes show similarity), ectoine
</option>

</select>
<button id= "clusterblast-r110c1-download">Download graphic</button>
</div>
<div id="clusterblast-r110c1-svg"></div>

</div>
</div>

<div class="body-details-section r110c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-34"></div> <span class="help-tooltip" id="cb-known-help-34">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r110c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r110c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r110c1_1.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r110c1_2.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r110c1_3.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r110c1_4.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r110c1_5.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r110c1_6.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r110c1_7.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r110c1_8.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r110c1_9.svg">
ectoine
</option>

</select>
<button id="knownclusterblast-r110c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r110c1-svg"></div>

</div>
</div>

<div class="body-details-section r110c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-35"></div> <span class="help-tooltip" id="cb-sub-help-35">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r110c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-37"></div> <span class="help-tooltip" id="tfbs-body-help-37">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ArgR
<div class="collapser collapser-target-ArgR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Regulator of arginine biosynthesis genes<br>
<span>Score</span>: 17.78 out of 20.01
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r110c1-69448-ArgR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r110c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-39"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-39">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r110c1-pfam-details-svg" data-filename="NZ_JAQZSM010000009.1_r110c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r110c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r110c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-40"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-40">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r110c1-tigrfam-details-svg" data-filename="NZ_JAQZSM010000009.1_r110c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r110c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r110c1-tfbs-finder" data-name="r110c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r110c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-36"></div> <span class="help-tooltip" id="tfbs-side-help-36">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ArgR" target="_blank">ArgR</a></td>
<td>Regulator of arginine biosynthesis genes</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>