
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>GCF_003350345.1_Alkalilacustris_brevis_34079 - 4 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/bacteria.css">
<meta property="og:title" content="GCF_003350345.1_Alkalilacustris_brevis_34079 - 4 record(s) - 4 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-b1cf77c9-cd77-4c8a-90c7-4f0bc01fa774/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://antismash.secondarymetabolites.org/">
<img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://antismash.secondarymetabolites.org/">
antiSMASH version 7.0.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="GCF_003350345.1_Alkalilacustris_brevis_34079.zip">Download all results</a></li>

<li><a href="GCF_003350345.1_Alkalilacustris_brevis_34079.gbk">Download GenBank summary file</a></li>

<li><a href="bacteria-b1cf77c9-cd77-4c8a-90c7-4f0bc01fa774.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton other betalactone r20c1">
<a href="#r20c1">20.1</a>
</div>



<div class="regbutton other ectoine r38c1">
<a href="#r38c1">38.1</a>
</div>



<div class="regbutton other hserlactone r62c1">
<a href="#r62c1">62.1</a>
</div>



<div class="regbutton RiPP RRE-containing r98c1">
<a href="#r98c1">98.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>NZ_QNVJ01000007.1</strong> (Alkalilacustris brevis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-20" data-filename="NZ_QNVJ01000007.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-20" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r20c1">
<td class="regbutton other betalactone r20c1">
<a href="#r20c1">Region&nbsp20.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#betalactone" target="_blank">betalactone</a>
</td>
<td class="digits">9,635</td>
<td class="digits table-split-left">35,999</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002284/1" target="_blank">corynecin III/corynecin I/corynecin II</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 13%, #ffffff00 13%)">13%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_QNVJ01000008.1</strong> (Alkalilacustris brevis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-38" data-filename="NZ_QNVJ01000008.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-38" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r38c1">
<td class="regbutton other ectoine r38c1">
<a href="#r38c1">Region&nbsp38.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
</td>
<td class="digits">3,642</td>
<td class="digits table-split-left">14,037</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000860/1" target="_blank">ectoine</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_QNVJ01000054.1</strong> (Alkalilacustris brevis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-62" data-filename="NZ_QNVJ01000054.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-62" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r62c1">
<td class="regbutton other hserlactone r62c1">
<a href="#r62c1">Region&nbsp62.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">60,685</td>
<td class="digits table-split-left">81,347</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_QNVJ01000053.1</strong> (Alkalilacustris brevis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-98" data-filename="NZ_QNVJ01000053.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-98" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r98c1">
<td class="regbutton RiPP RRE-containing r98c1">
<a href="#r98c1">Region&nbsp98.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#rre-containing" target="_blank">RRE-containing</a>
</td>
<td class="digits">2,446</td>
<td class="digits table-split-left">22,739</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r20c1">
<td class="regbutton other betalactone r20c1">
<a href="#r20c1">Region&nbsp20.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#betalactone" target="_blank">betalactone</a>
</td>
<td class="digits">9,635</td>
<td class="digits table-split-left">35,999</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002284/1" target="_blank">corynecin III/corynecin I/corynecin II</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 13%, #ffffff00 13%)">13%</td>

</tr>


<tr class="linked-row even" data-anchor="#r38c1">
<td class="regbutton other ectoine r38c1">
<a href="#r38c1">Region&nbsp38.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
</td>
<td class="digits">3,642</td>
<td class="digits table-split-left">14,037</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000860/1" target="_blank">ectoine</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r62c1">
<td class="regbutton other hserlactone r62c1">
<a href="#r62c1">Region&nbsp62.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">60,685</td>
<td class="digits table-split-left">81,347</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r98c1">
<td class="regbutton RiPP RRE-containing r98c1">
<a href="#r98c1">Region&nbsp98.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#rre-containing" target="_blank">RRE-containing</a>
</td>
<td class="digits">2,446</td>
<td class="digits table-split-left">22,739</td>

<td colspan="3"></td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>NZ_QNVJ01000078.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000047.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000100.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000075.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000084.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000074.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000091.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000064.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000095.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000102.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000015.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000058.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000098.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000002.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000087.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000081.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000022.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000050.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000094.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000049.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000027.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000099.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000073.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000076.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000005.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000040.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000032.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000038.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000089.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000042.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000069.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000051.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000067.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000043.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000103.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000019.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000021.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000034.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000056.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000020.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000077.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000011.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000052.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000062.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000101.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000090.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000088.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000018.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000045.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000085.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000044.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000057.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000097.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000092.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000041.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000025.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000029.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000072.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000039.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000046.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000028.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000024.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000023.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000048.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000055.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000037.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000013.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000009.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000004.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000059.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000012.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000017.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000080.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000065.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000016.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000066.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000006.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000026.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000003.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000010.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000063.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000083.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000060.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000061.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000068.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000093.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000071.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000030.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000086.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000001.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000036.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000082.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000031.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000014.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000096.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000033.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000035.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000070.1</strong> (Alkalilacustris brevis)</dd>

<dd><strong>NZ_QNVJ01000079.1</strong> (Alkalilacustris brevis)</dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>







<div class="page" id='r20c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_QNVJ01000007.1 - Region 1 - betalactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-35"></div> <span class="help-tooltip" id="region_svg-help-35">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_QNVJ01000007.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r20c1-svg" data-filename="NZ_QNVJ01000007.1_r20c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 9,635 - 35,999 nt. (total: 26,365 nt)

<a class="cluster-rules-header" id="r20c1-rules-header" href="#r20c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r20c1-rules">betalactone: (HMGL-like and AMP-binding)</div>

<div class="region-svg-container">
<div id='r20c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-36"></div> <span class="help-tooltip" id="focus-panel-help-36">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r20c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r20c1-gene-table-details" data-name="r20c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r20c1-MIBiG-cluster-compare" data-name="r20c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r20c1-clusterblast" data-name="r20c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r20c1-knownclusterblast" data-name="r20c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r20c1-subclusterblast" data-name="r20c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r20c1-tfbs-finder" data-name="r20c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r20c1-pfam-details" data-name="r20c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r20c1-tigrfam-details" data-name="r20c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r20c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-7"></div> <span class="help-tooltip" id="gt-help-tooltip-help-7">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r20c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r20c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r20c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r20c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r20c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-1"></div> <span class="help-tooltip" id="MIBiG-help-1">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r20c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r20c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r20c1"></div>
<div id="comparison-MIBiG-r20c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">betalactone</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002405: 17312-42861">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002405/index.html#r1c1" target="_new">BGC0002405</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>Saccharide</td>

<td>mid-chain acyl sugars</td>


<td>Solanum lycopersicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234: 152-6978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.46, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>jadomycin</td>


<td>Streptomyces venezuelae ATCC 10712</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000925: 319-18326">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000925/index.html#r1c1" target="_new">BGC0000925</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.46, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Other</td>

<td>quinolobactin</td>


<td>Pseudomonas fluorescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002284: 0-17900">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002284/index.html#r1c1" target="_new">BGC0002284</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.53, order:0.36, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Other</td>

<td>corynecin III, corynecin I, corynecin II</td>


<td>Rhodococcus sp. H-CA8f</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000913: 1-11028">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000913/index.html#r1c1" target="_new">BGC0000913</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Other</td>

<td>MK-8</td>


<td>Enterobacter cloacae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002476: 0-10330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002476/index.html#r1c1" target="_new">BGC0002476</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Escherichia coli str. K-12 substr. MG1655</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001262: 1986-31560">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001262/index.html#r1c1" target="_new">BGC0001262</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>AK-toxin I</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002711: 0-9402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002711/index.html#r1c1" target="_new">BGC0002711</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Polyketide</td>

<td>nostovalerolactone, 9-dehydronostovalerolactone</td>


<td>Nostoc punctiforme PCC 73102</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554: 0-6546">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>SRO15-3108</td>


<td>Streptomyces filamentosus NRRL 15998</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002701: 0-9977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002701/index.html#r1c1" target="_new">BGC0002701</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>paenithipeptin</td>


<td>Paenibacillus thiaminolyticus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r20c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002405: 17312-42861">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002405/index.html#r1c1" target="_new">BGC0002405</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Saccharide</td>
<td>mid-chain acyl sugars</td>
<td>Solanum lycopersicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234: 152-6978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.46, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>jadomycin</td>
<td>Streptomyces venezuelae ATCC 10712</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000925: 319-18326">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000925/index.html#r1c1" target="_new">BGC0000925</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.46, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Other</td>
<td>quinolobactin</td>
<td>Pseudomonas fluorescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002284: 0-17900">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002284/index.html#r1c1" target="_new">BGC0002284</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.53, order:0.36, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Other</td>
<td>corynecin III, corynecin I, corynecin II</td>
<td>Rhodococcus sp. H-CA8f</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000913: 1-11028">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000913/index.html#r1c1" target="_new">BGC0000913</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Other</td>
<td>MK-8</td>
<td>Enterobacter cloacae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002476: 0-10330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002476/index.html#r1c1" target="_new">BGC0002476</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Escherichia coli str. K-12 substr. MG1655</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001262: 1986-31560">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001262/index.html#r1c1" target="_new">BGC0001262</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Polyketide</td>
<td>AK-toxin I</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002711: 0-9402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002711/index.html#r1c1" target="_new">BGC0002711</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Polyketide</td>
<td>nostovalerolactone, 9-dehydronostovalerolactone</td>
<td>Nostoc punctiforme PCC 73102</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554: 0-6546">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>SRO15-3108</td>
<td>Streptomyces filamentosus NRRL 15998</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002701: 0-9977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002701/index.html#r1c1" target="_new">BGC0002701</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>paenithipeptin</td>
<td>Paenibacillus thiaminolyticus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r20c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-2"></div> <span class="help-tooltip" id="cb-general-help-2">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r20c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r20c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r20c1_1.svg">
NZ_LT629784 (651581-681804): Stappia sp. ES.058 chromosome I (46% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r20c1_2.svg">
NZ_CP020330 (530804-560985): Martelella mediterranea DSM 17316 strain MACL11 ... (40% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r20c1_3.svg">
NZ_FORH01000003 (348122-378356): Celeribacter neptunius strain DSM 26471, who... (48% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r20c1_4.svg">
NZ_CP042261 (1474809-1505000): Litoreibacter sp. LN3S51 chromosome, complete ... (48% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r20c1_5.svg">
NZ_KE557274 (584713-614937): Salipiger mucosus DSM 16094 scaffold2, whole gen... (48% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r20c1_6.svg">
NZ_CP028472 (136599-166806): Celeribacter baekdonensis strain LH4 plasmid pCB... (41% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r20c1_7.svg">
NZ_JAATNG010000001 (1068397-1097412): Pseudoruegeria sp. HB172150 scaffold1, ... (41% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r20c1_8.svg">
NZ_JACIDZ010000008 (92911-123101): Martelella radicis strain DSM 28101 Ga0373... (50% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r20c1_9.svg">
NZ_CP028901 (2068776-2098720): Bordetella sp. HZ20 chromosome, complete genome (36% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r20c1_10.svg">
NZ_LT607803 (2962398-2992676): Variovorax sp. HW608 chromosome I (33% of genes show similarity), betalactone
</option>

</select>
<button id= "clusterblast-r20c1-download">Download graphic</button>
</div>
<div id="clusterblast-r20c1-svg"></div>

</div>
</div>

<div class="body-details-section r20c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-3"></div> <span class="help-tooltip" id="cb-known-help-3">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r20c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r20c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r20c1_1.svg">
corynecin III/corynecin I/corynecin II
</option>

</select>
<button id="knownclusterblast-r20c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r20c1-svg"></div>

</div>
</div>

<div class="body-details-section r20c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-4"></div> <span class="help-tooltip" id="cb-sub-help-4">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r20c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-6"></div> <span class="help-tooltip" id="tfbs-body-help-6">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AbrC3
<div class="collapser collapser-target-AbrC3 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Antibiotic production activator<br>
<span>Score</span>: 18.16 out of 18.16
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r20c1-9917-AbrC3"></div>

<div class="tfbs-hit-detail">
AbrC3
<div class="collapser collapser-target-AbrC3 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Antibiotic production activator<br>
<span>Score</span>: 18.16 out of 18.16
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r20c1-15997-AbrC3"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
HypR
<div class="collapser collapser-target-HypR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: L-hydroxyproline utilization repressor<br>
<span>Score</span>: 18.79 out of 22.40
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r20c1-35913-HypR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r20c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-8"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-8">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r20c1-pfam-details-svg" data-filename="NZ_QNVJ01000007.1_r20c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r20c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r20c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-9"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-9">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r20c1-tigrfam-details-svg" data-filename="NZ_QNVJ01000007.1_r20c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r20c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r20c1-tfbs-finder" data-name="r20c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r20c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-5"></div> <span class="help-tooltip" id="tfbs-side-help-5">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AbrC3" target="_blank">AbrC3</a></td>
<td>Antibiotic production activator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/HypR" target="_blank">HypR</a></td>
<td>L-hydroxyproline utilization repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r38c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_QNVJ01000008.1 - Region 1 - ectoine <div class="help-container"> <div class="help-icon" data-id="region_svg-help-37"></div> <span class="help-tooltip" id="region_svg-help-37">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_QNVJ01000008.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r38c1-svg" data-filename="NZ_QNVJ01000008.1_r38c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,642 - 14,037 nt. (total: 10,396 nt)

<a class="cluster-rules-header" id="r38c1-rules-header" href="#r38c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r38c1-rules">ectoine: ectoine_synt</div>

<div class="region-svg-container">
<div id='r38c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-38"></div> <span class="help-tooltip" id="focus-panel-help-38">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r38c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r38c1-gene-table-details" data-name="r38c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r38c1-MIBiG-cluster-compare" data-name="r38c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r38c1-clusterblast" data-name="r38c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r38c1-knownclusterblast" data-name="r38c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r38c1-subclusterblast" data-name="r38c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r38c1-pfam-details" data-name="r38c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r38c1-tigrfam-details" data-name="r38c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r38c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-14"></div> <span class="help-tooltip" id="gt-help-tooltip-help-14">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r38c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r38c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r38c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r38c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r38c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-10"></div> <span class="help-tooltip" id="MIBiG-help-10">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r38c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r38c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r38c1"></div>
<div id="comparison-MIBiG-r38c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">ectoine</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860: 710-5134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860</a></td>





<td class="cc-heat-cell" title="0.79 (id:0.55, order:0.90, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>


<td>0.51</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylarcula marina</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052: 188-3620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052</a></td>





<td class="cc-heat-cell" title="0.79 (id:0.49, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>


<td>0.51</td>
<td>Other (Ectoine)</td>

<td>ectoine</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853: 8-3345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853</a></td>





<td class="cc-heat-cell" title="0.79 (id:0.49, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>


<td>0.51</td>
<td>Other</td>

<td>ectoine</td>


<td>Streptomyces anulatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854: 838-6879">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854</a></td>





<td class="cc-heat-cell" title="0.77 (id:0.50, order:0.92, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>


<td>0.48</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium alcaliphilum 20Z</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857: 300-4871">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857</a></td>





<td class="cc-heat-cell" title="0.77 (id:0.50, order:0.90, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>


<td>0.47</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylophaga alcalica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856: 517-4347">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856</a></td>





<td class="cc-heat-cell" title="0.76 (id:0.51, order:0.88, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>


<td>0.46</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylophaga thalassica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859: 258-3305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859</a></td>





<td class="cc-heat-cell" title="0.76 (id:0.51, order:0.88, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>


<td>0.46</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium alcaliphilum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855: 245-3455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855</a></td>





<td class="cc-heat-cell" title="0.73 (id:0.51, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>


<td>0.41</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium kenyense</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.49, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>Other</td>

<td>ectoine</td>


<td>Sporosarcina pasteurii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.48, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylobacter marinus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r38c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860: 710-5134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860</a></td>


<td class="cc-heat-cell" title="0.79 (id:0.55, order:0.90, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
<td>0.79</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylarcula marina</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052: 188-3620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052</a></td>


<td class="cc-heat-cell" title="0.79 (id:0.49, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
<td>0.79</td>
<td>Other (Ectoine)</td>
<td>ectoine</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853: 8-3345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853</a></td>


<td class="cc-heat-cell" title="0.79 (id:0.49, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
<td>0.79</td>
<td>Other</td>
<td>ectoine</td>
<td>Streptomyces anulatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854: 838-6879">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854</a></td>


<td class="cc-heat-cell" title="0.77 (id:0.50, order:0.92, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
<td>0.77</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium alcaliphilum 20Z</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857: 300-4871">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857</a></td>


<td class="cc-heat-cell" title="0.77 (id:0.50, order:0.90, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
<td>0.77</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylophaga alcalica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856: 517-4347">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856</a></td>


<td class="cc-heat-cell" title="0.76 (id:0.51, order:0.88, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>
<td>0.76</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylophaga thalassica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859: 258-3305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859</a></td>


<td class="cc-heat-cell" title="0.76 (id:0.51, order:0.88, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>
<td>0.76</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium alcaliphilum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855: 245-3455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855</a></td>


<td class="cc-heat-cell" title="0.73 (id:0.51, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
<td>0.73</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium kenyense</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.49, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Other</td>
<td>ectoine</td>
<td>Sporosarcina pasteurii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.48, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylobacter marinus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r38c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-11"></div> <span class="help-tooltip" id="cb-general-help-11">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r38c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r38c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r38c1_1.svg">
NZ_FOXZ01000004 (136534-146927): Nitratireductor indicus strain CGMCC 1.10953... (54% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r38c1_2.svg">
NZ_AP018711 (2285625-2296039): Sphingosinicella microcystinivorans strain B-9 (70% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r38c1_3.svg">
NC_011144 (1510673-1521066): Phenylobacterium zucineum HLK1, complete sequence (60% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r38c1_4.svg">
NZ_CP005085 (218914-229328): Sphingobium sp. TKS plasmid pTK1, complete sequence (60% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r38c1_5.svg">
NZ_KQ130473 (41239-51653): Sphingobium baderi LL03 25433 scaffold 03, whole g... (60% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r38c1_6.svg">
NZ_CP039250 (62790-73192): Sphingopyxis sp. PAMC25046 chromosome, complete ge... (75% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r38c1_7.svg">
NZ_CP009452 (3129372-3139774): Sphingopyxis sp. 113P3 chromosome, complete ge... (60% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r38c1_8.svg">
NC_015186 (3646945-3657344): Acidiphilium multivorum AIU301, complete genome (66% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r38c1_9.svg">
NZ_JAATIT010000004 (183514-193916): Sphingopyxis italica strain DSM 25229 Ga0... (75% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r38c1_10.svg">
NC_009484 (3288218-3298617): Acidiphilium cryptum JF-5, complete genome (66% of genes show similarity), ectoine
</option>

</select>
<button id= "clusterblast-r38c1-download">Download graphic</button>
</div>
<div id="clusterblast-r38c1-svg"></div>

</div>
</div>

<div class="body-details-section r38c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-12"></div> <span class="help-tooltip" id="cb-known-help-12">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r38c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r38c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r38c1_1.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r38c1_2.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r38c1_3.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r38c1_4.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r38c1_5.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r38c1_6.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r38c1_7.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r38c1_8.svg">
ectoine
</option>

</select>
<button id="knownclusterblast-r38c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r38c1-svg"></div>

</div>
</div>

<div class="body-details-section r38c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-13"></div> <span class="help-tooltip" id="cb-sub-help-13">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r38c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-15"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-15">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r38c1-pfam-details-svg" data-filename="NZ_QNVJ01000008.1_r38c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r38c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r38c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-16"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-16">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r38c1-tigrfam-details-svg" data-filename="NZ_QNVJ01000008.1_r38c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r38c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>




<div class="page" id='r62c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_QNVJ01000054.1 - Region 1 - hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-39"></div> <span class="help-tooltip" id="region_svg-help-39">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_QNVJ01000054.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r62c1-svg" data-filename="NZ_QNVJ01000054.1_r62c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 60,685 - 81,347 nt. (total: 20,663 nt)

<a class="cluster-rules-header" id="r62c1-rules-header" href="#r62c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r62c1-rules">hserlactone: Autoind_synth</div>

<div class="region-svg-container">
<div id='r62c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-40"></div> <span class="help-tooltip" id="focus-panel-help-40">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r62c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r62c1-gene-table-details" data-name="r62c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r62c1-MIBiG-cluster-compare" data-name="r62c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r62c1-clusterblast" data-name="r62c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r62c1-knownclusterblast" data-name="r62c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r62c1-subclusterblast" data-name="r62c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r62c1-tfbs-finder" data-name="r62c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r62c1-pfam-details" data-name="r62c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r62c1-tigrfam-details" data-name="r62c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r62c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-23"></div> <span class="help-tooltip" id="gt-help-tooltip-help-23">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r62c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r62c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r62c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r62c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r62c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-17"></div> <span class="help-tooltip" id="MIBiG-help-17">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r62c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r62c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r62c1"></div>
<div id="comparison-MIBiG-r62c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">hserlactone</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001897: 3-16597">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001897/index.html#r1c1" target="_new">BGC0001897</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.46, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>cepacin A</td>


<td>Burkholderia ambifaria IOP40-10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001633: 0-6215">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001633/index.html#r1c1" target="_new">BGC0001633</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.43, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Other</td>

<td>ketomemicin B3, ketomemicin B4</td>


<td>Streptomyces mobaraensis NBRC 13819 = DSM 40847</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002284: 0-17900">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002284/index.html#r1c1" target="_new">BGC0002284</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.06</td>
<td>Other</td>

<td>corynecin III, corynecin I, corynecin II</td>


<td>Rhodococcus sp. H-CA8f</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000339: 4596-41004">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000339/index.html#r1c1" target="_new">BGC0000339</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.47, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>NRP</td>

<td>echinomycin</td>


<td>Streptomyces lasaliensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000864: 742-41396">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000864/index.html#r1c1" target="_new">BGC0000864</a></td>





<td class="cc-heat-cell" title="0.21 (id:0.46, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>


<td>0.05</td>
<td>Other</td>

<td>eicosapentaenoic acid-like compound</td>


<td>Moritella marina ATCC 15381</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001371: 972-51788">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001371/index.html#r1c1" target="_new">BGC0001371</a></td>





<td class="cc-heat-cell" title="0.18 (id:0.44, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>


<td>0.04</td>
<td>NRP</td>

<td>echinocandin B</td>


<td>Aspergillus nidulans NRRL 8112</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002611: 1682-51377">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002611/index.html#r1c1" target="_new">BGC0002611</a></td>





<td class="cc-heat-cell" title="0.18 (id:0.43, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>


<td>0.04</td>
<td>NRP</td>

<td>mulundocandin</td>


<td>Aspergillus mulundensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001220: 899-59804">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001220/index.html#r1c1" target="_new">BGC0001220</a></td>





<td class="cc-heat-cell" title="0.17 (id:0.43, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>


<td>0.03</td>
<td>NRP</td>

<td>aculeacin A</td>


<td>Aspergillus japonicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000415: 2-35965">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000415/index.html#r1c1" target="_new">BGC0000415</a></td>





<td class="cc-heat-cell" title="0.17 (id:0.46, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>


<td>0.03</td>
<td>NRP</td>

<td>quinomycin</td>


<td>Streptomyces griseovariabilis subsp. bandungensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001732: 2271-63281">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001732/index.html#r1c1" target="_new">BGC0001732</a></td>





<td class="cc-heat-cell" title="0.17 (id:0.41, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>


<td>0.03</td>
<td>Other</td>

<td>paulomycin</td>


<td>Streptomyces sp. YN86</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r62c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001897: 3-16597">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001897/index.html#r1c1" target="_new">BGC0001897</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.46, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Polyketide</td>
<td>cepacin A</td>
<td>Burkholderia ambifaria IOP40-10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001633: 0-6215">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001633/index.html#r1c1" target="_new">BGC0001633</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.43, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>Other</td>
<td>ketomemicin B3, ketomemicin B4</td>
<td>Streptomyces mobaraensis NBRC 13819 = DSM 40847</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002284: 0-17900">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002284/index.html#r1c1" target="_new">BGC0002284</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>Other</td>
<td>corynecin III, corynecin I, corynecin II</td>
<td>Rhodococcus sp. H-CA8f</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000339: 4596-41004">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000339/index.html#r1c1" target="_new">BGC0000339</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.47, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>NRP</td>
<td>echinomycin</td>
<td>Streptomyces lasaliensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000864: 742-41396">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000864/index.html#r1c1" target="_new">BGC0000864</a></td>


<td class="cc-heat-cell" title="0.21 (id:0.46, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
<td>0.21</td>
<td>Other</td>
<td>eicosapentaenoic acid-like compound</td>
<td>Moritella marina ATCC 15381</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001371: 972-51788">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001371/index.html#r1c1" target="_new">BGC0001371</a></td>


<td class="cc-heat-cell" title="0.18 (id:0.44, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>
<td>0.18</td>
<td>NRP</td>
<td>echinocandin B</td>
<td>Aspergillus nidulans NRRL 8112</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002611: 1682-51377">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002611/index.html#r1c1" target="_new">BGC0002611</a></td>


<td class="cc-heat-cell" title="0.18 (id:0.43, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>
<td>0.18</td>
<td>NRP</td>
<td>mulundocandin</td>
<td>Aspergillus mulundensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001220: 899-59804">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001220/index.html#r1c1" target="_new">BGC0001220</a></td>


<td class="cc-heat-cell" title="0.17 (id:0.43, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>
<td>0.17</td>
<td>NRP</td>
<td>aculeacin A</td>
<td>Aspergillus japonicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000415: 2-35965">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000415/index.html#r1c1" target="_new">BGC0000415</a></td>


<td class="cc-heat-cell" title="0.17 (id:0.46, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>
<td>0.17</td>
<td>NRP</td>
<td>quinomycin</td>
<td>Streptomyces griseovariabilis subsp. bandungensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001732: 2271-63281">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001732/index.html#r1c1" target="_new">BGC0001732</a></td>


<td class="cc-heat-cell" title="0.17 (id:0.41, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>
<td>0.17</td>
<td>Other</td>
<td>paulomycin</td>
<td>Streptomyces sp. YN86</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r62c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-18"></div> <span class="help-tooltip" id="cb-general-help-18">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r62c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r62c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r62c1_1.svg">
NZ_JABZEH010000001 (435253-455868): Ralstonia solanacearum strain CCRMRsB7 CC... (20% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r62c1_2.svg">
NZ_CDRX01000001 (4190866-4211481): Ralstonia solanacearum strain RS2, whole g... (17% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r62c1_3.svg">
NZ_CDQJ01000001 (3985488-4006103): Ralstonia solanacearum strain CFBP3858, wh... (15% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r62c1_4.svg">
NZ_CDRJ01000001 (569559-590174): Ralstonia solanacearum strain CFBP7014, whol... (17% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r62c1_5.svg">
NZ_CDLW01000001 (2342443-2363037): Ralstonia solanacearum strain Grenada 9-1,... (14% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r62c1_6.svg">
NZ_CP034194 (3158327-3178942): Ralstonia solanacearum strain UA-1579 chromosome (15% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r62c1_7.svg">
NZ_CDMA01000001 (313982-334597): Ralstonia solanacearum strain B50, whole gen... (18% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r62c1_8.svg">
NZ_CDLZ01000001 (536408-557023): Ralstonia solanacearum strain UW179, whole g... (17% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r62c1_9.svg">
NZ_CP021766 (226613-247228): Ralstonia solanacearum strain RS 489 chromosome,... (17% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r62c1_10.svg">
NZ_MKKZ01000003 (11084-31699): Ralstonia solanacearum strain BBAC-C1 scaffold... (14% of genes show similarity), hserlactone
</option>

</select>
<button id= "clusterblast-r62c1-download">Download graphic</button>
</div>
<div id="clusterblast-r62c1-svg"></div>

</div>
</div>

<div class="body-details-section r62c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-19"></div> <span class="help-tooltip" id="cb-known-help-19">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r62c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-20"></div> <span class="help-tooltip" id="cb-sub-help-20">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r62c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-22"></div> <span class="help-tooltip" id="tfbs-body-help-22">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AbrC3
<div class="collapser collapser-target-AbrC3 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Antibiotic production activator<br>
<span>Score</span>: 18.07 out of 18.16
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r62c1-65095-AbrC3"></div>

<div class="tfbs-hit-detail">
NrdR
<div class="collapser collapser-target-NrdR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Represses ribonucleotide reductase encoding genes<br>
<span>Score</span>: 22.03 out of 27.18
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r62c1-73369-NrdR"></div>

<div class="tfbs-hit-detail">
ArgR
<div class="collapser collapser-target-ArgR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Regulator of arginine biosynthesis genes<br>
<span>Score</span>: 18.25 out of 20.01
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r62c1-76479-ArgR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
HypR
<div class="collapser collapser-target-HypR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: L-hydroxyproline utilization repressor<br>
<span>Score</span>: 18.94 out of 22.40
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r62c1-69746-HypR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r62c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-24"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-24">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r62c1-pfam-details-svg" data-filename="NZ_QNVJ01000054.1_r62c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r62c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r62c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-25"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-25">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r62c1-tigrfam-details-svg" data-filename="NZ_QNVJ01000054.1_r62c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r62c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r62c1-tfbs-finder" data-name="r62c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r62c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-21"></div> <span class="help-tooltip" id="tfbs-side-help-21">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AbrC3" target="_blank">AbrC3</a></td>
<td>Antibiotic production activator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrdR" target="_blank">NrdR</a></td>
<td>Represses ribonucleotide reductase encoding genes</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/HypR" target="_blank">HypR</a></td>
<td>L-hydroxyproline utilization repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ArgR" target="_blank">ArgR</a></td>
<td>Regulator of arginine biosynthesis genes</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r98c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_QNVJ01000053.1 - Region 1 - RRE-containing <div class="help-container"> <div class="help-icon" data-id="region_svg-help-41"></div> <span class="help-tooltip" id="region_svg-help-41">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_QNVJ01000053.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r98c1-svg" data-filename="NZ_QNVJ01000053.1_r98c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,446 - 22,739 nt. (total: 20,294 nt)

<a class="cluster-rules-header" id="r98c1-rules-header" href="#r98c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r98c1-rules">RRE-containing: minimum(1, [Bottromycin_Methyltransferase_RRE, Cyanobactin_D_RRE, Cytolysin_C_RRE, Goadsporin_CD_RRE, Heterocycloanthracin_C_RRE, Lanthipeptide_LanB_RRE, Lasso_Fused_RRE, Mycofactocin_RRE, NHLP_CD_RRE, Other_Sactipeptide_rSAM_RRE, Pantocin_Microcin_RRE, Plantazolicin_C_RRE, PlpY_RRE, PqqD_RRE, Proteusin_Epimerase_RRE, Proteusin_Methyltransferase_RRE, Quinohemoprotein_rSAM_RRE, Ranthipeptide_rSAM_RRE, Stand_Alone_Lasso_RRE, Streptide_RRE, Subtilosin_rSAM_RRE, Thiaglutamate_B_RRE, Thiaglutamate_G_RRE, Thiaglutamate_I_RRE, Thiopeptide_F_RRE, Thuricin_rSAM_RRE, Thurincin_rSAM_RRE, Trifolitoxin_RRE])</div>

<div class="region-svg-container">
<div id='r98c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-42"></div> <span class="help-tooltip" id="focus-panel-help-42">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r98c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r98c1-gene-table-details" data-name="r98c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r98c1-MIBiG-cluster-compare" data-name="r98c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r98c1-clusterblast" data-name="r98c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r98c1-knownclusterblast" data-name="r98c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r98c1-subclusterblast" data-name="r98c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r98c1-tfbs-finder" data-name="r98c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r98c1-pfam-details" data-name="r98c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r98c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-33"></div> <span class="help-tooltip" id="gt-help-tooltip-help-33">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r98c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r98c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r98c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r98c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r98c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-26"></div> <span class="help-tooltip" id="MIBiG-help-26">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r98c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r98c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r98c1"></div>
<div id="comparison-MIBiG-r98c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">RRE-containing</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001587: 6-7413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001587/index.html#r1c1" target="_new">BGC0001587</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>Other</td>

<td>ethylenediaminesuccinic acid hydroxyarginine (EDHA)</td>


<td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002681: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002681/index.html#r1c1" target="_new">BGC0002681</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Other</td>

<td>rhizobactin</td>


<td>Francisella tularensis subsp. tularensis SCHU S4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000585: 202-6207">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000585/index.html#r1c1" target="_new">BGC0000585</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>RiPP</td>

<td>microcin C7</td>


<td>Escherichia coli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000772: 960-13404">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000772/index.html#r1c1" target="_new">BGC0000772</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Saccharide</td>

<td>lipopolysaccharide</td>


<td>Brucella melitensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000707: 0-7321">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000707/index.html#r1c1" target="_new">BGC0000707</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Saccharide</td>

<td>kasugamycin</td>


<td>Streptomyces kasugaensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000777: 356-11114">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000777/index.html#r1c1" target="_new">BGC0000777</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Saccharide</td>

<td>lipopolysaccharide</td>


<td>Escherichia coli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000786: 1-12215">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000786/index.html#r1c1" target="_new">BGC0000786</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.42, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Saccharide</td>

<td>O-antigen</td>


<td>Edwardsiella ictaluri 93-146</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000926: 1-12724">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000926/index.html#r1c1" target="_new">BGC0000926</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Other</td>

<td>rhizocticin A</td>


<td>Bacillus subtilis subsp. spizizenii ATCC 6633</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000025: 159-9297">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000025/index.html#r1c1" target="_new">BGC0000025</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Polyketide</td>

<td>avermectin</td>


<td>Streptomyces avermitilis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000769: 2-26368">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000769/index.html#r1c1" target="_new">BGC0000769</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Saccharide</td>

<td>glycopeptidolipid</td>


<td>Mycobacterium avium subsp. hominissuis A5</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r98c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001587: 6-7413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001587/index.html#r1c1" target="_new">BGC0001587</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Other</td>
<td>ethylenediaminesuccinic acid hydroxyarginine (EDHA)</td>
<td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002681: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002681/index.html#r1c1" target="_new">BGC0002681</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>rhizobactin</td>
<td>Francisella tularensis subsp. tularensis SCHU S4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000585: 202-6207">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000585/index.html#r1c1" target="_new">BGC0000585</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>RiPP</td>
<td>microcin C7</td>
<td>Escherichia coli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000772: 960-13404">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000772/index.html#r1c1" target="_new">BGC0000772</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Saccharide</td>
<td>lipopolysaccharide</td>
<td>Brucella melitensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000707: 0-7321">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000707/index.html#r1c1" target="_new">BGC0000707</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Saccharide</td>
<td>kasugamycin</td>
<td>Streptomyces kasugaensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000777: 356-11114">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000777/index.html#r1c1" target="_new">BGC0000777</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Saccharide</td>
<td>lipopolysaccharide</td>
<td>Escherichia coli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000786: 1-12215">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000786/index.html#r1c1" target="_new">BGC0000786</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.42, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Saccharide</td>
<td>O-antigen</td>
<td>Edwardsiella ictaluri 93-146</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000926: 1-12724">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000926/index.html#r1c1" target="_new">BGC0000926</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Other</td>
<td>rhizocticin A</td>
<td>Bacillus subtilis subsp. spizizenii ATCC 6633</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000025: 159-9297">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000025/index.html#r1c1" target="_new">BGC0000025</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Polyketide</td>
<td>avermectin</td>
<td>Streptomyces avermitilis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000769: 2-26368">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000769/index.html#r1c1" target="_new">BGC0000769</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Saccharide</td>
<td>glycopeptidolipid</td>
<td>Mycobacterium avium subsp. hominissuis A5</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r98c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-27"></div> <span class="help-tooltip" id="cb-general-help-27">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r98c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r98c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r98c1_1.svg">
NC_007802 (2615046-2657830): Jannaschia sp. CCS1, complete sequence (4% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r98c1_2.svg">
NZ_CP014596 (444160-489905): Yangia sp. CCB-MM3 chromosome 2, complete sequence (5% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r98c1_3.svg">
NZ_JAAOXM010000005 (154672-252481): Burkholderia sp. D-99 contig5, whole geno... (3% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r98c1_4.svg">
NZ_CP013465 (605114-738500): Burkholderia ubonensis strain MSMB1471WGS chromo... (2% of genes show similarity), NRPS,T1PKS,arylpolyene,other
</option>

<option value = "svg/clusterblast_r98c1_5.svg">
NZ_MECN01000002 (130314-263694): Burkholderia ubonensis strain MSMB0842 Conti... (2% of genes show similarity), NRPS,T1PKS,arylpolyene,other
</option>

<option value = "svg/clusterblast_r98c1_6.svg">
NC_011666 (3700929-3742012): Methylocella silvestris BL2, complete sequence (5% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r98c1_7.svg">
NZ_CP014859 (4621949-4667892): Actinoalloteichus hymeniacidonis strain HPA177... (5% of genes show similarity), oligosaccharide
</option>

<option value = "svg/clusterblast_r98c1_8.svg">
NZ_FMZV01000023 (35598-56315): Ruegeria marina strain CGMCC 1.9108, whole gen... (11% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r98c1_9.svg">
NZ_FNWJ01000002 (540781-569846): Thermoleophilum album strain ATCC 35263, who... (8% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r98c1_10.svg">
NZ_AP019791 (113557-132727): Rubrobacter xylanophilus strain AA3-22 (10% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r98c1-download">Download graphic</button>
</div>
<div id="clusterblast-r98c1-svg"></div>

</div>
</div>

<div class="body-details-section r98c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-28"></div> <span class="help-tooltip" id="cb-known-help-28">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r98c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-29"></div> <span class="help-tooltip" id="cb-sub-help-29">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r98c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-32"></div> <span class="help-tooltip" id="tfbs-body-help-32">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
CelR
<div class="collapser collapser-target-CelR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Cellobiose uptake repressor<br>
<span>Score</span>: 23.51 out of 30.11
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r98c1-4655-CelR"></div>

<div class="tfbs-hit-detail">
AfsR
<div class="collapser collapser-target-AfsR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Pleiotropic regulatory for antibiotic production<br>
<span>Score</span>: 18.76 out of 23.05
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r98c1-9014-AfsR"></div>

<div class="tfbs-hit-detail">
NrdR
<div class="collapser collapser-target-NrdR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Represses ribonucleotide reductase encoding genes<br>
<span>Score</span>: 23.04 out of 27.18
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r98c1-12138-NrdR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (3):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
LexA
<div class="collapser collapser-target-LexA collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Repressor of DNA damage response<br>
<span>Score</span>: 21.27 out of 29.65
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r98c1-2569-LexA"></div>

<div class="tfbs-hit-detail">
CelR
<div class="collapser collapser-target-CelR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Cellobiose uptake repressor<br>
<span>Score</span>: 23.20 out of 30.11
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r98c1-4654-CelR"></div>

<div class="tfbs-hit-detail">
DmdR1
<div class="collapser collapser-target-DmdR1 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Iron(II)-dependent repressor<br>
<span>Score</span>: 22.70 out of 36.53
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r98c1-17668-DmdR1"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r98c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-34"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-34">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r98c1-pfam-details-svg" data-filename="NZ_QNVJ01000053.1_r98c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r98c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r98c1-RREfinder" data-name="r98c1-RREfinder">
RREFinder
</div>



<div class="sidepanel-details-header r98c1-tfbs-finder" data-name="r98c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r98c1-RREfinder">
<div class="more-details">
<div class="heading">
<span>RRE predictions</span>
<div class="help-container"> <div class="help-icon" data-id="rre-side-help-30"></div> <span class="help-tooltip" id="rre-side-help-30">RREfinder results sidepanel.</span></div>
</div>
<dl class = 'prediction-text'>

<span><strong>RRE-containing protocluster</strong> (2445...22739)</span>
<dt></dt>




<strong>JCPCKFCL_02983</strong> <div class="collapser collapser-target-JCPCKFCL_02983 collapser-level-cds"> </div><div class="collapser-content">

<dd>RRE type: Stand_Alone_Lasso_RRE</dd>
<dd>Location: 21 - 89</dd>
<dd>Score: 35.9</dd>

</div><br>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r98c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-31"></div> <span class="help-tooltip" id="tfbs-side-help-31">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/LexA" target="_blank">LexA</a></td>
<td>Repressor of DNA damage response</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/CelR" target="_blank">CelR</a></td>
<td>Cellobiose uptake repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrdR" target="_blank">NrdR</a></td>
<td>Represses ribonucleotide reductase encoding genes</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AfsR" target="_blank">AfsR</a></td>
<td>Pleiotropic regulatory for antibiotic production</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/DmdR1" target="_blank">DmdR1</a></td>
<td>Iron(II)-dependent repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>