
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>GCF_016653255.1_Rhodobaculum_claviforme_LMG_28126 - 4 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/bacteria.css">
<meta property="og:title" content="GCF_016653255.1_Rhodobaculum_claviforme_LMG_28126 - 4 record(s) - 4 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-a75238f4-1dda-4efb-9fc5-2c7893247a64/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://antismash.secondarymetabolites.org/">
<img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://antismash.secondarymetabolites.org/">
antiSMASH version 7.0.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="GCF_016653255.1_Rhodobaculum_claviforme_LMG_28126.zip">Download all results</a></li>

<li><a href="GCF_016653255.1_Rhodobaculum_claviforme_LMG_28126.gbk">Download GenBank summary file</a></li>

<li><a href="bacteria-a75238f4-1dda-4efb-9fc5-2c7893247a64.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton other ectoine r218c1">
<a href="#r218c1">218.1</a>
</div>



<div class="regbutton terpene terpene r237c1">
<a href="#r237c1">237.1</a>
</div>



<div class="regbutton terpene terpene r273c1">
<a href="#r273c1">273.1</a>
</div>



<div class="regbutton other betalactone r292c1">
<a href="#r292c1">292.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>NZ_NHSD01000218.1</strong> (Rhodobaculum claviforme)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-218" data-filename="NZ_NHSD01000218.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-218" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r218c1">
<td class="regbutton other ectoine r218c1">
<a href="#r218c1">Region&nbsp218.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">9,387</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000860/1" target="_blank">ectoine</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 80%, #ffffff00 80%)">80%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_NHSD01000237.1</strong> (Rhodobaculum claviforme)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-237" data-filename="NZ_NHSD01000237.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-237" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r237c1">
<td class="regbutton terpene terpene r237c1">
<a href="#r237c1">Region&nbsp237.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">3,563</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000647/1" target="_blank">carotenoid</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_NHSD01000273.1</strong> (Rhodobaculum claviforme)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-273" data-filename="NZ_NHSD01000273.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-273" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r273c1">
<td class="regbutton terpene terpene r273c1">
<a href="#r273c1">Region&nbsp273.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">6,312</td>
<td class="digits table-split-left">21,589</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000647/1" target="_blank">carotenoid</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_NHSD01000292.1</strong> (Rhodobaculum claviforme)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-292" data-filename="NZ_NHSD01000292.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-292" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r292c1">
<td class="regbutton other betalactone r292c1">
<a href="#r292c1">Region&nbsp292.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#betalactone" target="_blank">betalactone</a>
</td>
<td class="digits">11,262</td>
<td class="digits table-split-left">36,052</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r218c1">
<td class="regbutton other ectoine r218c1">
<a href="#r218c1">Region&nbsp218.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">9,387</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000860/1" target="_blank">ectoine</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 80%, #ffffff00 80%)">80%</td>

</tr>


<tr class="linked-row even" data-anchor="#r237c1">
<td class="regbutton terpene terpene r237c1">
<a href="#r237c1">Region&nbsp237.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">3,563</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000647/1" target="_blank">carotenoid</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r273c1">
<td class="regbutton terpene terpene r273c1">
<a href="#r273c1">Region&nbsp273.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">6,312</td>
<td class="digits table-split-left">21,589</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000647/1" target="_blank">carotenoid</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>


<tr class="linked-row even" data-anchor="#r292c1">
<td class="regbutton other betalactone r292c1">
<a href="#r292c1">Region&nbsp292.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#betalactone" target="_blank">betalactone</a>
</td>
<td class="digits">11,262</td>
<td class="digits table-split-left">36,052</td>

<td colspan="3"></td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>NZ_NHSD01000001.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000002.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000003.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000004.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000005.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000006.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000007.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000008.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000009.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000010.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000011.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000012.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000013.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000014.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000015.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000016.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000017.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000018.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000019.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000020.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000021.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000022.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000023.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000024.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000025.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000026.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000027.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000028.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000029.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000030.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000031.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000032.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000033.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000034.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000035.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000036.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000037.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000038.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000039.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000040.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000041.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000042.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000043.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000044.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000045.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000046.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000047.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000048.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000049.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000050.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000051.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000052.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000053.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000054.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000055.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000056.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000057.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000058.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000059.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000060.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000061.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000062.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000063.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000064.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000065.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000066.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000067.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000068.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000069.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000070.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000071.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000072.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000073.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000074.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000075.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000076.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000077.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000078.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000079.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000080.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000081.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000082.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000083.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000084.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000085.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000086.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000087.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000088.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000089.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000090.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000091.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000092.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000093.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000094.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000095.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000096.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000097.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000098.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000099.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000100.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000101.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000102.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000103.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000104.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000105.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000106.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000107.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000108.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000109.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000110.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000111.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000112.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000113.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000114.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000115.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000116.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000117.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000118.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000119.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000120.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000121.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000122.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000123.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000124.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000125.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000126.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000127.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000128.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000129.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000130.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000131.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000132.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000133.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000134.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000135.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000136.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000137.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000138.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000139.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000140.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000141.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000142.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000143.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000144.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000145.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000146.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000147.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000148.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000149.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000150.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000151.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000152.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000153.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000154.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000155.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000156.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000157.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000158.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000159.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000160.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000161.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000162.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000163.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000164.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000165.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000166.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000167.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000168.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000169.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000170.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000171.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000172.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000173.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000174.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000175.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000176.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000177.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000178.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000179.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000180.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000181.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000182.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000183.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000184.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000185.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000186.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000187.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000188.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000189.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000190.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000191.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000192.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000193.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000194.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000195.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000196.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000197.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000198.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000199.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000200.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000201.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000202.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000203.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000204.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000205.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000206.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000207.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000208.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000209.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000210.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000211.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000212.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000213.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000214.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000215.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000216.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000217.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000219.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000220.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000221.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000222.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000223.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000224.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000225.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000226.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000227.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000228.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000229.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000230.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000231.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000232.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000233.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000234.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000235.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000236.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000238.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000239.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000240.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000241.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000242.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000243.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000244.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000245.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000246.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000247.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000248.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000249.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000250.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000251.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000252.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000253.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000254.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000255.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000256.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000257.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000258.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000259.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000260.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000261.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000262.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000263.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000264.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000265.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000266.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000267.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000268.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000269.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000270.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000271.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000272.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000274.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000275.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000276.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000277.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000278.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000279.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000280.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000281.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000282.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000283.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000284.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000285.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000286.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000287.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000288.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000289.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000290.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000291.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000293.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000294.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000295.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000296.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000297.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000298.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000299.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000300.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000301.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000302.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000303.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000304.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000305.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000306.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000307.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000308.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000309.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000310.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000311.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000312.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000313.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000314.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000315.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000316.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000317.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000318.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000319.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000320.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000321.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000322.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000323.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000324.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000325.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000326.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000327.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000328.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000329.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000330.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000331.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000332.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000333.1</strong> (Rhodobaculum claviforme)</dd>

<dd><strong>NZ_NHSD01000334.1</strong> (Rhodobaculum claviforme)</dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>







<div class="page" id='r218c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_NHSD01000218.1 - Region 1 - ectoine <div class="help-container"> <div class="help-icon" data-id="region_svg-help-33"></div> <span class="help-tooltip" id="region_svg-help-33">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_NHSD01000218.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r218c1-svg" data-filename="NZ_NHSD01000218.1_r218c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 9,387 nt. (total: 9,387 nt)

<a class="cluster-rules-header" id="r218c1-rules-header" href="#r218c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r218c1-rules">ectoine: ectoine_synt</div>

<div class="region-svg-container">
<div id='r218c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-34"></div> <span class="help-tooltip" id="focus-panel-help-34">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r218c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r218c1-gene-table-details" data-name="r218c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r218c1-MIBiG-cluster-compare" data-name="r218c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r218c1-clusterblast" data-name="r218c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r218c1-knownclusterblast" data-name="r218c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r218c1-subclusterblast" data-name="r218c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r218c1-pfam-details" data-name="r218c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r218c1-tigrfam-details" data-name="r218c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r218c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-5"></div> <span class="help-tooltip" id="gt-help-tooltip-help-5">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r218c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r218c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r218c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r218c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r218c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-1"></div> <span class="help-tooltip" id="MIBiG-help-1">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r218c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r218c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r218c1"></div>
<div id="comparison-MIBiG-r218c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">ectoine</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860: 710-5134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860</a></td>





<td class="cc-heat-cell" title="0.81 (id:0.53, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>


<td>0.54</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylarcula marina</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856: 517-4347">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856</a></td>





<td class="cc-heat-cell" title="0.80 (id:0.51, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>


<td>0.52</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylophaga thalassica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859: 258-3305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859</a></td>





<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>


<td>0.51</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium alcaliphilum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857: 300-4871">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857</a></td>





<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>


<td>0.51</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylophaga alcalica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855: 245-3455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855</a></td>





<td class="cc-heat-cell" title="0.73 (id:0.51, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>


<td>0.41</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium kenyense</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.49, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>Other</td>

<td>ectoine</td>


<td>Sporosarcina pasteurii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854: 838-6879">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.49, order:0.83, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium alcaliphilum 20Z</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.47, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylobacter marinus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853: 8-3345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.49, order:0.76, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>Other</td>

<td>ectoine</td>


<td>Streptomyces anulatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052: 188-3620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.48, order:0.76, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>Other (Ectoine)</td>

<td>ectoine</td>


<td>Streptomyces sp.</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r218c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860: 710-5134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860</a></td>


<td class="cc-heat-cell" title="0.81 (id:0.53, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>
<td>0.81</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylarcula marina</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856: 517-4347">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856</a></td>


<td class="cc-heat-cell" title="0.80 (id:0.51, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
<td>0.80</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylophaga thalassica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859: 258-3305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859</a></td>


<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
<td>0.79</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium alcaliphilum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857: 300-4871">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857</a></td>


<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
<td>0.79</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylophaga alcalica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855: 245-3455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855</a></td>


<td class="cc-heat-cell" title="0.73 (id:0.51, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
<td>0.73</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium kenyense</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.49, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Other</td>
<td>ectoine</td>
<td>Sporosarcina pasteurii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854: 838-6879">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.49, order:0.83, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium alcaliphilum 20Z</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.47, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylobacter marinus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853: 8-3345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.49, order:0.76, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Other</td>
<td>ectoine</td>
<td>Streptomyces anulatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052: 188-3620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.48, order:0.76, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Other (Ectoine)</td>
<td>ectoine</td>
<td>Streptomyces sp.</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r218c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-2"></div> <span class="help-tooltip" id="cb-general-help-2">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r218c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r218c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r218c1_1.svg">
NZ_CP024899 (2098634-2109021): Rhodobaca barguzinensis strain alga05 chromoso... (50% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r218c1_2.svg">
NZ_MEHT01000009 (25734-53535): Roseinatronobacter thiooxidans strain ALG1 ALG... (17% of genes show similarity), ectoine,terpene
</option>

<option value = "svg/clusterblast_r218c1_3.svg">
NZ_LXYH01000056 (3698-14082): Rhodobacteraceae bacterium EhC02 A5 Contig6, wh... (50% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r218c1_4.svg">
NZ_CP049056 (3924672-3935059): Rhodobacteraceae bacterium RR4-56 chromosome, ... (71% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r218c1_5.svg">
NZ_FOQH01000008 (123921-134320): Albimonas pacifica strain CGMCC 1.11030, who... (62% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r218c1_6.svg">
NZ_ML119082 (150587-160980): Histidinibacterium lentulum strain B17 Scaffold2... (71% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r218c1_7.svg">
NZ_VYQE01000003 (480055-490448): Histidinibacterium sp. 176SS1-4 contig3, who... (62% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r218c1_8.svg">
NZ_RWGU01000036 (45420-55816): Rhodovulum robiginosum strain DSM 12329 NODE 1... (55% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r218c1_9.svg">
NZ_CP031841 (2803713-2894138): Breoghania sp. L-A4 chromosome, complete genome (8% of genes show similarity), NRPS,T3PKS,ectoine,transAT-PKS
</option>

<option value = "svg/clusterblast_r218c1_10.svg">
NZ_FOXA01000013 (39786-50173): Tranquillimonas alkanivorans strain DSM 19547,... (62% of genes show similarity), ectoine
</option>

</select>
<button id= "clusterblast-r218c1-download">Download graphic</button>
</div>
<div id="clusterblast-r218c1-svg"></div>

</div>
</div>

<div class="body-details-section r218c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-3"></div> <span class="help-tooltip" id="cb-known-help-3">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r218c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r218c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r218c1_1.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r218c1_2.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r218c1_3.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r218c1_4.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r218c1_5.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r218c1_6.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r218c1_7.svg">
ectoine
</option>

</select>
<button id="knownclusterblast-r218c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r218c1-svg"></div>

</div>
</div>

<div class="body-details-section r218c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-4"></div> <span class="help-tooltip" id="cb-sub-help-4">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r218c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-6"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-6">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r218c1-pfam-details-svg" data-filename="NZ_NHSD01000218.1_r218c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r218c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r218c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-7"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-7">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r218c1-tigrfam-details-svg" data-filename="NZ_NHSD01000218.1_r218c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r218c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>




<div class="page" id='r237c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_NHSD01000237.1 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-35"></div> <span class="help-tooltip" id="region_svg-help-35">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_NHSD01000237.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r237c1-svg" data-filename="NZ_NHSD01000237.1_r237c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 3,563 nt. (total: 3,563 nt)

<a class="cluster-rules-header" id="r237c1-rules-header" href="#r237c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r237c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r237c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-36"></div> <span class="help-tooltip" id="focus-panel-help-36">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r237c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r237c1-gene-table-details" data-name="r237c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r237c1-MIBiG-cluster-compare" data-name="r237c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r237c1-clusterblast" data-name="r237c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r237c1-knownclusterblast" data-name="r237c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r237c1-subclusterblast" data-name="r237c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r237c1-pfam-details" data-name="r237c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r237c1-tigrfam-details" data-name="r237c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r237c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-12"></div> <span class="help-tooltip" id="gt-help-tooltip-help-12">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r237c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r237c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r237c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r237c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r237c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-8"></div> <span class="help-tooltip" id="MIBiG-help-8">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r237c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r237c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r237c1"></div>
<div id="comparison-MIBiG-r237c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647: 251-2872">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647</a></td>





<td class="cc-heat-cell" title="0.80 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>


<td>0.52</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Rhodobacter sphaeroides</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000650: 28-7720">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000650/index.html#r1c1" target="_new">BGC0000650</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Algoriphagus sp. KK10202C</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656: 169-3770">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.49, order:0.69, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>Terpene</td>

<td>zeaxanthin</td>


<td>Xanthobacter autotrophicus Py2</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648: 1057-11443">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Myxococcus xanthus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640: 2399-8466">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.48, order:0.57, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.19</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Enterobacteriaceae bacterium DC404</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630: 389-6051">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.48, order:0.57, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.19</td>
<td>Terpene</td>

<td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>


<td>Paracoccus haeundaensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646: 60-6380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.49, order:0.57, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>β-carotein</td>


<td>uncultured bacterium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000634: 1312-11784">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000634/index.html#r1c1" target="_new">BGC0000634</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.49, order:0.57, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Brevundimonas sp. SD212</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000643: 363-10046">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000643/index.html#r1c1" target="_new">BGC0000643</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.49, order:0.57, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Brevundimonas vesicularis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664: 0-8391">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.44, order:0.57, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Terpene</td>

<td>isorenieratene</td>


<td>Streptomyces griseus subsp. griseus NBRC 13350</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r237c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647: 251-2872">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647</a></td>


<td class="cc-heat-cell" title="0.80 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
<td>0.80</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Rhodobacter sphaeroides</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000650: 28-7720">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000650/index.html#r1c1" target="_new">BGC0000650</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Algoriphagus sp. KK10202C</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656: 169-3770">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.49, order:0.69, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Terpene</td>
<td>zeaxanthin</td>
<td>Xanthobacter autotrophicus Py2</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648: 1057-11443">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Myxococcus xanthus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640: 2399-8466">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.48, order:0.57, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Enterobacteriaceae bacterium DC404</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630: 389-6051">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.48, order:0.57, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Terpene</td>
<td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>
<td>Paracoccus haeundaensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646: 60-6380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.49, order:0.57, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>β-carotein</td>
<td>uncultured bacterium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000634: 1312-11784">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000634/index.html#r1c1" target="_new">BGC0000634</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.49, order:0.57, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Brevundimonas sp. SD212</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000643: 363-10046">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000643/index.html#r1c1" target="_new">BGC0000643</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.49, order:0.57, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Brevundimonas vesicularis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664: 0-8391">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.44, order:0.57, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Terpene</td>
<td>isorenieratene</td>
<td>Streptomyces griseus subsp. griseus NBRC 13350</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r237c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-9"></div> <span class="help-tooltip" id="cb-general-help-9">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r237c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r237c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r237c1_1.svg">
NZ_KI911559 (3189394-3208975): Rhodovibrio salinarum DSM 9154 RHOSADRAFT scaf... (25% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r237c1_2.svg">
NZ_CP029606 (759578-780685): Methylibium sp. Pch-M chromosome, complete genome (25% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r237c1_3.svg">
NZ_CP007268 (2158145-2187304): Ectothiorhodospira haloalkaliphila strain Halo... (18% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r237c1_4.svg">
NC_017082 (1996617-2017018): Bradyrhizobium cosmicum, complete genome (16% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r237c1_5.svg">
NZ_NPBC01000011 (128262-149321): Afifella marina DSM 2698 NODE 11 length 1791... (20% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r237c1_6.svg">
NZ_CP044543 (5135548-5155966): Bradyrhizobium betae strain PL7HG1 chromosome,... (16% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r237c1_7.svg">
NZ_CP029426 (5114355-5135414): Bradyrhizobium amphicarpaeae strain 39S1MB chr... (16% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r237c1_8.svg">
NZ_CP041656 (1513215-1533612): Bradyrhizobium cosmicum strain 58S1 chromosome... (16% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r237c1_9.svg">
NZ_JHAA01000004 (44468-65524): Rhodopseudomonas palustris PSBS contig 4, whol... (18% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r237c1_10.svg">
NZ_JAAMRM010000022 (631385-652441): Bradyrhizobium sp. BL NODE 2, whole genom... (16% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r237c1-download">Download graphic</button>
</div>
<div id="clusterblast-r237c1-svg"></div>

</div>
</div>

<div class="body-details-section r237c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-10"></div> <span class="help-tooltip" id="cb-known-help-10">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r237c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r237c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r237c1_1.svg">
carotenoid
</option>

</select>
<button id="knownclusterblast-r237c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r237c1-svg"></div>

</div>
</div>

<div class="body-details-section r237c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-11"></div> <span class="help-tooltip" id="cb-sub-help-11">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r237c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-13"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-13">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r237c1-pfam-details-svg" data-filename="NZ_NHSD01000237.1_r237c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r237c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r237c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-14"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-14">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r237c1-tigrfam-details-svg" data-filename="NZ_NHSD01000237.1_r237c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r237c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>




<div class="page" id='r273c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_NHSD01000273.1 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-37"></div> <span class="help-tooltip" id="region_svg-help-37">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_NHSD01000273.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r273c1-svg" data-filename="NZ_NHSD01000273.1_r273c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 6,312 - 21,589 nt. (total: 15,278 nt)

<a class="cluster-rules-header" id="r273c1-rules-header" href="#r273c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r273c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r273c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-38"></div> <span class="help-tooltip" id="focus-panel-help-38">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r273c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r273c1-gene-table-details" data-name="r273c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r273c1-MIBiG-cluster-compare" data-name="r273c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r273c1-clusterblast" data-name="r273c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r273c1-knownclusterblast" data-name="r273c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r273c1-subclusterblast" data-name="r273c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r273c1-tfbs-finder" data-name="r273c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r273c1-pfam-details" data-name="r273c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r273c1-tigrfam-details" data-name="r273c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r273c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-21"></div> <span class="help-tooltip" id="gt-help-tooltip-help-21">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r273c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r273c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r273c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r273c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r273c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-15"></div> <span class="help-tooltip" id="MIBiG-help-15">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r273c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r273c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r273c1"></div>
<div id="comparison-MIBiG-r273c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647: 251-2872">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647</a></td>





<td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>


<td>0.58</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Rhodobacter sphaeroides</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000637: 540-9054">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000637/index.html#r1c1" target="_new">BGC0000637</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Corynebacterium glutamicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656: 169-3770">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.47, order:0.69, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Terpene</td>

<td>zeaxanthin</td>


<td>Xanthobacter autotrophicus Py2</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648: 1057-11443">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.45, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Myxococcus xanthus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646: 60-6380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.48, order:0.48, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Terpene</td>

<td>β-carotein</td>


<td>uncultured bacterium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630: 389-6051">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.47, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Terpene</td>

<td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>


<td>Paracoccus haeundaensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640: 2399-8466">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.47, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.19</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Enterobacteriaceae bacterium DC404</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664: 0-8391">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.44, order:0.45, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Terpene</td>

<td>isorenieratene</td>


<td>Streptomyces griseus subsp. griseus NBRC 13350</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000641: 1208-7454">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000641/index.html#r1c1" target="_new">BGC0000641</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.47, order:0.48, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Enterobacteriaceae bacterium DC416</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000639: 324-6564">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000639/index.html#r1c1" target="_new">BGC0000639</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.46, order:0.48, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Enterobacteriaceae bacterium DC260</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r273c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647: 251-2872">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647</a></td>


<td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
<td>0.83</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Rhodobacter sphaeroides</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000637: 540-9054">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000637/index.html#r1c1" target="_new">BGC0000637</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Corynebacterium glutamicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656: 169-3770">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.47, order:0.69, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Terpene</td>
<td>zeaxanthin</td>
<td>Xanthobacter autotrophicus Py2</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648: 1057-11443">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.45, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Myxococcus xanthus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646: 60-6380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.48, order:0.48, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Terpene</td>
<td>β-carotein</td>
<td>uncultured bacterium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630: 389-6051">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.47, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Terpene</td>
<td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>
<td>Paracoccus haeundaensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640: 2399-8466">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.47, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Enterobacteriaceae bacterium DC404</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664: 0-8391">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.44, order:0.45, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Terpene</td>
<td>isorenieratene</td>
<td>Streptomyces griseus subsp. griseus NBRC 13350</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000641: 1208-7454">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000641/index.html#r1c1" target="_new">BGC0000641</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.47, order:0.48, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Enterobacteriaceae bacterium DC416</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000639: 324-6564">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000639/index.html#r1c1" target="_new">BGC0000639</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.46, order:0.48, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Enterobacteriaceae bacterium DC260</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r273c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-16"></div> <span class="help-tooltip" id="cb-general-help-16">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r273c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r273c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r273c1_1.svg">
NZ_CP024899 (619066-640104): Rhodobaca barguzinensis strain alga05 chromosome... (47% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r273c1_2.svg">
NZ_MEHT01000018 (61223-82261): Roseinatronobacter thiooxidans strain ALG1 ALG... (47% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r273c1_3.svg">
NZ_SJZH01000001 (193776-214545): Tabrizicola sp. YIM 78059 Scaffold1, whole g... (47% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r273c1_4.svg">
NZ_JACIEJ010000015 (21148-42186): Sagittula marina strain DSM 102235 Ga037326... (50% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r273c1_5.svg">
NZ_JAAIKE010000001 (123967-144083): Rhodobacter sp. ETT8 contig1, whole genom... (50% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r273c1_6.svg">
NZ_SNYP01000010 (18408-39440): Rhodobacter veldkampii DSM 11550 Ga0310482 110... (50% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r273c1_7.svg">
NZ_CP028918 (1124240-1143571): Gemmobacter sp. HYN0069 chromosome, complete g... (47% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r273c1_8.svg">
NZ_PZZW01000002 (622436-643504): Rhodobacter johrii strain JA192 Ga0191650 10... (47% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r273c1_9.svg">
NZ_CP031750 (893210-913315): Rhodobacter sphaeroides strain EBL0706 chromosom... (44% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r273c1_10.svg">
NC_011963 (1615872-1636940): Rhodobacter sphaeroides KD131 chromosome 1, comp... (47% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r273c1-download">Download graphic</button>
</div>
<div id="clusterblast-r273c1-svg"></div>

</div>
</div>

<div class="body-details-section r273c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-17"></div> <span class="help-tooltip" id="cb-known-help-17">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r273c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r273c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r273c1_1.svg">
carotenoid
</option>

</select>
<button id="knownclusterblast-r273c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r273c1-svg"></div>

</div>
</div>

<div class="body-details-section r273c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-18"></div> <span class="help-tooltip" id="cb-sub-help-18">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r273c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-20"></div> <span class="help-tooltip" id="tfbs-body-help-20">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 18.83 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r273c1-11858-NrtR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r273c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-22"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-22">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r273c1-pfam-details-svg" data-filename="NZ_NHSD01000273.1_r273c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r273c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r273c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-23"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-23">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r273c1-tigrfam-details-svg" data-filename="NZ_NHSD01000273.1_r273c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r273c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r273c1-tfbs-finder" data-name="r273c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r273c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-19"></div> <span class="help-tooltip" id="tfbs-side-help-19">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrtR" target="_blank">NrtR</a></td>
<td>NAD synthesis repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r292c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_NHSD01000292.1 - Region 1 - betalactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-39"></div> <span class="help-tooltip" id="region_svg-help-39">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_NHSD01000292.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r292c1-svg" data-filename="NZ_NHSD01000292.1_r292c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 11,262 - 36,052 nt. (total: 24,791 nt)

<a class="cluster-rules-header" id="r292c1-rules-header" href="#r292c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r292c1-rules">betalactone: (HMGL-like and AMP-binding)</div>

<div class="region-svg-container">
<div id='r292c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-40"></div> <span class="help-tooltip" id="focus-panel-help-40">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r292c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r292c1-gene-table-details" data-name="r292c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r292c1-MIBiG-cluster-compare" data-name="r292c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r292c1-clusterblast" data-name="r292c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r292c1-knownclusterblast" data-name="r292c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r292c1-subclusterblast" data-name="r292c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r292c1-tfbs-finder" data-name="r292c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r292c1-pfam-details" data-name="r292c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r292c1-tigrfam-details" data-name="r292c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r292c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-30"></div> <span class="help-tooltip" id="gt-help-tooltip-help-30">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r292c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r292c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r292c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r292c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r292c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-24"></div> <span class="help-tooltip" id="MIBiG-help-24">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r292c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r292c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r292c1"></div>
<div id="comparison-MIBiG-r292c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">betalactone</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002405: 17312-42861">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002405/index.html#r1c1" target="_new">BGC0002405</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Saccharide</td>

<td>mid-chain acyl sugars</td>


<td>Solanum lycopersicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001137: 0-18476">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001137/index.html#r1c1" target="_new">BGC0001137</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Alkaloid</td>

<td>marinacarboline A, marinacarboline B, marinacarboline C, marinacarboline D</td>


<td>Marinactinospora thermotolerans</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000867: 21-5822">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000867/index.html#r1c1" target="_new">BGC0000867</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Other</td>

<td>polyhydroxyalkanoic acids</td>


<td>Ectothiorhodospira shaposhnikovii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000866: 467-5165">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000866/index.html#r1c1" target="_new">BGC0000866</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Other</td>

<td>polyhydroxyalkanoate</td>


<td>Burkholderia sp. DSM 9242</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000933: 0-5189">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000933/index.html#r1c1" target="_new">BGC0000933</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Other</td>

<td>xiamenmycin A</td>


<td>Streptomyces xiamenensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231: 189-5010">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>griseusin A, griseusin B</td>


<td>Streptomyces griseus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001534: 799-5264">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001534/index.html#r1c1" target="_new">BGC0001534</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Other</td>

<td>branched-chain fatty acids</td>


<td>Streptomyces filamentosus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000888: 0-4342">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000888/index.html#r1c1" target="_new">BGC0000888</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Other</td>

<td>bacilysin</td>


<td>Bacillus sp. CS93</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000649: 148-10747">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000649/index.html#r1c1" target="_new">BGC0000649</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Streptomyces griseus subsp. griseus NBRC 13350</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.45, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>SCO-2138</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r292c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002405: 17312-42861">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002405/index.html#r1c1" target="_new">BGC0002405</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Saccharide</td>
<td>mid-chain acyl sugars</td>
<td>Solanum lycopersicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001137: 0-18476">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001137/index.html#r1c1" target="_new">BGC0001137</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Alkaloid</td>
<td>marinacarboline A, marinacarboline B, marinacarboline C, marinacarboline D</td>
<td>Marinactinospora thermotolerans</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000867: 21-5822">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000867/index.html#r1c1" target="_new">BGC0000867</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Other</td>
<td>polyhydroxyalkanoic acids</td>
<td>Ectothiorhodospira shaposhnikovii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000866: 467-5165">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000866/index.html#r1c1" target="_new">BGC0000866</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Other</td>
<td>polyhydroxyalkanoate</td>
<td>Burkholderia sp. DSM 9242</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000933: 0-5189">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000933/index.html#r1c1" target="_new">BGC0000933</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Other</td>
<td>xiamenmycin A</td>
<td>Streptomyces xiamenensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231: 189-5010">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Polyketide</td>
<td>griseusin A, griseusin B</td>
<td>Streptomyces griseus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001534: 799-5264">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001534/index.html#r1c1" target="_new">BGC0001534</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Other</td>
<td>branched-chain fatty acids</td>
<td>Streptomyces filamentosus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000888: 0-4342">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000888/index.html#r1c1" target="_new">BGC0000888</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Other</td>
<td>bacilysin</td>
<td>Bacillus sp. CS93</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000649: 148-10747">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000649/index.html#r1c1" target="_new">BGC0000649</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Streptomyces griseus subsp. griseus NBRC 13350</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.45, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>SCO-2138</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r292c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-25"></div> <span class="help-tooltip" id="cb-general-help-25">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r292c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r292c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r292c1_1.svg">
NZ_CP042906 (2833920-2874895): Rhodospirillaceae bacterium R5913 chromosome, ... (11% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r292c1_2.svg">
NZ_CP042582 (2388557-2429532): Rhodospirillaceae bacterium R5959 chromosome, ... (11% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r292c1_3.svg">
NZ_KB903227 (11663-54141): Actinomycetospora chiangmaiensis DSM 45062 A3CODRA... (7% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r292c1_4.svg">
NC_015677 (396878-438762): Ramlibacter tataouinensis TTB310, complete sequence (7% of genes show similarity), resorcinol
</option>

<option value = "svg/clusterblast_r292c1_5.svg">
NZ_CP054932 (3704966-3817330): Actinomadura sp. NAK00032 chromosome, complete... (3% of genes show similarity), NRPS,TfuA-related,bacteriocin,other
</option>

<option value = "svg/clusterblast_r292c1_6.svg">
NZ_CP019449 (1417023-1479886): Sphingopyxis sp. QXT-31, complete genome (10% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r292c1_7.svg">
NZ_CP040818 (785612-805584): Paraoceanicella profunda strain D4M1 chromosome,... (15% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r292c1_8.svg">
NZ_FOFR01000004 (148068-233969): Lechevalieria xinjiangensis strain CGMCC 4.3... (4% of genes show similarity), PKS-like,T2PKS
</option>

<option value = "svg/clusterblast_r292c1_9.svg">
NZ_FNET01000005 (185091-271621): Lentzea violacea strain DSM 44796, whole gen... (4% of genes show similarity), PKS-like,T2PKS
</option>

<option value = "svg/clusterblast_r292c1_10.svg">
NZ_FOYL01000007 (226175-310922): Lentzea waywayandensis strain DSM 44232, who... (4% of genes show similarity), PKS-like,T2PKS
</option>

</select>
<button id= "clusterblast-r292c1-download">Download graphic</button>
</div>
<div id="clusterblast-r292c1-svg"></div>

</div>
</div>

<div class="body-details-section r292c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-26"></div> <span class="help-tooltip" id="cb-known-help-26">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r292c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-27"></div> <span class="help-tooltip" id="cb-sub-help-27">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r292c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-29"></div> <span class="help-tooltip" id="tfbs-body-help-29">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AbrC3
<div class="collapser collapser-target-AbrC3 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Antibiotic production activator<br>
<span>Score</span>: 18.16 out of 18.16
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r292c1-12753-AbrC3"></div>

<div class="tfbs-hit-detail">
AfsR
<div class="collapser collapser-target-AfsR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Pleiotropic regulatory for antibiotic production<br>
<span>Score</span>: 21.40 out of 23.05
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r292c1-28313-AfsR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 19.28 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r292c1-22874-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r292c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-31"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-31">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r292c1-pfam-details-svg" data-filename="NZ_NHSD01000292.1_r292c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r292c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r292c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-32"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-32">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r292c1-tigrfam-details-svg" data-filename="NZ_NHSD01000292.1_r292c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r292c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r292c1-tfbs-finder" data-name="r292c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r292c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-28"></div> <span class="help-tooltip" id="tfbs-side-help-28">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AbrC3" target="_blank">AbrC3</a></td>
<td>Antibiotic production activator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AfsR" target="_blank">AfsR</a></td>
<td>Pleiotropic regulatory for antibiotic production</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>