
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>GCF_009908165.1_Frigidibacter_albus_SP32 - 3 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/bacteria.css">
<meta property="og:title" content="GCF_009908165.1_Frigidibacter_albus_SP32 - 3 record(s) - 3 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-8a924a22-0b55-4103-a7be-cce634a77cf9/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://antismash.secondarymetabolites.org/">
<img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://antismash.secondarymetabolites.org/">
antiSMASH version 7.0.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="GCF_009908165.1_Frigidibacter_albus_SP32.zip">Download all results</a></li>

<li><a href="GCF_009908165.1_Frigidibacter_albus_SP32.gbk">Download GenBank summary file</a></li>

<li><a href="bacteria-8a924a22-0b55-4103-a7be-cce634a77cf9.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton terpene terpene r22c1">
<a href="#r22c1">22.1</a>
</div>



<div class="regbutton PKS r39c1">
<a href="#r39c1">39.1</a>
</div>



<div class="regbutton other hserlactone r40c1">
<a href="#r40c1">40.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>NZ_JAAAUC010000002.1</strong> (Frigidibacter albus)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-22" data-filename="NZ_JAAAUC010000002.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-22" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r22c1">
<td class="regbutton terpene terpene r22c1">
<a href="#r22c1">Region&nbsp22.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">403,893</td>
<td class="digits table-split-left">424,669</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_JAAAUC010000008.1</strong> (Frigidibacter albus)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-39" data-filename="NZ_JAAAUC010000008.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-39" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r39c1">
<td class="regbutton PKS r39c1">
<a href="#r39c1">Region&nbsp39.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hgle-ks" target="_blank">hglE-KS</a>
</td>
<td class="digits">107,003</td>
<td class="digits table-split-left">159,934</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_JAAAUC010000009.1</strong> (Frigidibacter albus)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-40" data-filename="NZ_JAAAUC010000009.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-40" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r40c1">
<td class="regbutton other hserlactone r40c1">
<a href="#r40c1">Region&nbsp40.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">6,522</td>
<td class="digits table-split-left">27,112</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r22c1">
<td class="regbutton terpene terpene r22c1">
<a href="#r22c1">Region&nbsp22.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">403,893</td>
<td class="digits table-split-left">424,669</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r39c1">
<td class="regbutton PKS r39c1">
<a href="#r39c1">Region&nbsp39.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hgle-ks" target="_blank">hglE-KS</a>
</td>
<td class="digits">107,003</td>
<td class="digits table-split-left">159,934</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r40c1">
<td class="regbutton other hserlactone r40c1">
<a href="#r40c1">Region&nbsp40.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">6,522</td>
<td class="digits table-split-left">27,112</td>

<td colspan="3"></td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>NZ_JAAAUC010000010.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000011.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000012.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000013.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000014.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000015.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000016.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000017.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000018.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000019.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000001.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000020.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000021.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000022.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000023.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000024.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000025.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000026.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000027.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000028.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000029.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000030.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000031.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000032.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000033.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000034.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000035.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000036.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000037.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000038.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000039.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000003.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000040.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000004.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000005.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000006.1</strong> (Frigidibacter albus)</dd>

<dd><strong>NZ_JAAAUC010000007.1</strong> (Frigidibacter albus)</dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>







<div class="page" id='r22c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_JAAAUC010000002.1 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-32"></div> <span class="help-tooltip" id="region_svg-help-32">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_JAAAUC010000002.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r22c1-svg" data-filename="NZ_JAAAUC010000002.1_r22c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 403,893 - 424,669 nt. (total: 20,777 nt)

<a class="cluster-rules-header" id="r22c1-rules-header" href="#r22c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r22c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r22c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-33"></div> <span class="help-tooltip" id="focus-panel-help-33">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r22c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r22c1-gene-table-details" data-name="r22c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r22c1-MIBiG-cluster-compare" data-name="r22c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r22c1-clusterblast" data-name="r22c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r22c1-knownclusterblast" data-name="r22c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r22c1-subclusterblast" data-name="r22c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r22c1-tfbs-finder" data-name="r22c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r22c1-pfam-details" data-name="r22c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r22c1-tigrfam-details" data-name="r22c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r22c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-7"></div> <span class="help-tooltip" id="gt-help-tooltip-help-7">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r22c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r22c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r22c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r22c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r22c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-1"></div> <span class="help-tooltip" id="MIBiG-help-1">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r22c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r22c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r22c1"></div>
<div id="comparison-MIBiG-r22c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000904: 116-14341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000904/index.html#r1c1" target="_new">BGC0000904</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Other</td>

<td>FR-900098</td>


<td>Streptomyces rubellomurinus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001859: 6609-19144">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001859/index.html#r1c1" target="_new">BGC0001859</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Other (Phosphonate)</td>

<td>fosfomycin</td>


<td>Pseudomonas syringae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002132: 0-8455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002132/index.html#r1c1" target="_new">BGC0002132</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Other (Aminocoumarin)</td>

<td>group 1 methanobactin</td>


<td>Methylosinus sp. LW4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002004: 0-6373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002004/index.html#r1c1" target="_new">BGC0002004</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>methanobactin</td>


<td>Methylosinus trichosporium OB3b</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000043: 0-13097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000043/index.html#r1c1" target="_new">BGC0000043</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>curacin A</td>


<td>Moorea producens 19L</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000668: 125-36568">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000668/index.html#r1c1" target="_new">BGC0000668</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Terpene, Alkaloid</td>

<td>12-epi-hapalindole C isonitrile, 12-epi-hapalindole E, 12-epi-fischerindole U isonitrile, fischerindole L, 12-epi-fischerindole I isonitrile, welwitindolinone A isonitrile, welwitindolinone B isothiocyanate, welwitindolinone C isothiocyanate, N-methylwelwitindolinone C isothiocyanate, N-methylwelwitinsolinone C isonitrile, 3-epi-welwitindolinone B isothiocyanate, 3-(Z-2&#39;-isocyanoethenyl)-indole</td>


<td>Hapalosiphon welwitschii UTEX B 1830</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002007: 584-20653">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002007/index.html#r1c1" target="_new">BGC0002007</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.43, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>Terpene</td>

<td>atolypene A, atolypene B</td>


<td>Amycolatopsis tolypomycina</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002510: 0-11781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002510/index.html#r1c1" target="_new">BGC0002510</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.46, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>youssoufene A1, youssoufene B1, youssoufene B2, youssoufene B3, youssoufene B4</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001184: 0-7298">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001184/index.html#r1c1" target="_new">BGC0001184</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.41, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Other</td>

<td>bacilysin</td>


<td>Bacillus velezensis FZB42</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000188: 0-37455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000188/index.html#r1c1" target="_new">BGC0000188</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.41, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>Alkaloid</td>

<td>saxitoxin, neosaxitoxin, decarbamoylsaxitoxin, gonyautoxin 3, gonyautoxin 2</td>


<td>Cylindrospermopsis raciborskii T3</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r22c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000904: 116-14341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000904/index.html#r1c1" target="_new">BGC0000904</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other</td>
<td>FR-900098</td>
<td>Streptomyces rubellomurinus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001859: 6609-19144">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001859/index.html#r1c1" target="_new">BGC0001859</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Other (Phosphonate)</td>
<td>fosfomycin</td>
<td>Pseudomonas syringae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002132: 0-8455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002132/index.html#r1c1" target="_new">BGC0002132</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Other (Aminocoumarin)</td>
<td>group 1 methanobactin</td>
<td>Methylosinus sp. LW4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002004: 0-6373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002004/index.html#r1c1" target="_new">BGC0002004</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>RiPP</td>
<td>methanobactin</td>
<td>Methylosinus trichosporium OB3b</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000043: 0-13097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000043/index.html#r1c1" target="_new">BGC0000043</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Polyketide</td>
<td>curacin A</td>
<td>Moorea producens 19L</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000668: 125-36568">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000668/index.html#r1c1" target="_new">BGC0000668</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Terpene, Alkaloid</td>
<td>12-epi-hapalindole C isonitrile, 12-epi-hapalindole E, 12-epi-fischerindole U isonitrile, fischerindole L, 12-epi-fischerindole I isonitrile, welwitindolinone A isonitrile, welwitindolinone B isothiocyanate, welwitindolinone C isothiocyanate, N-methylwelwitindolinone C isothiocyanate, N-methylwelwitinsolinone C isonitrile, 3-epi-welwitindolinone B isothiocyanate, 3-(Z-2&#39;-isocyanoethenyl)-indole</td>
<td>Hapalosiphon welwitschii UTEX B 1830</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002007: 584-20653">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002007/index.html#r1c1" target="_new">BGC0002007</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.43, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>Terpene</td>
<td>atolypene A, atolypene B</td>
<td>Amycolatopsis tolypomycina</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002510: 0-11781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002510/index.html#r1c1" target="_new">BGC0002510</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.46, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>Polyketide</td>
<td>youssoufene A1, youssoufene B1, youssoufene B2, youssoufene B3, youssoufene B4</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001184: 0-7298">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001184/index.html#r1c1" target="_new">BGC0001184</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.41, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Other</td>
<td>bacilysin</td>
<td>Bacillus velezensis FZB42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000188: 0-37455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000188/index.html#r1c1" target="_new">BGC0000188</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.41, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Alkaloid</td>
<td>saxitoxin, neosaxitoxin, decarbamoylsaxitoxin, gonyautoxin 3, gonyautoxin 2</td>
<td>Cylindrospermopsis raciborskii T3</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r22c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-2"></div> <span class="help-tooltip" id="cb-general-help-2">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r22c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r22c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r22c1_1.svg">
NZ_WUJE01000002 (403857-424634): Frigidibacter albus strain SP32 Scaffold2, w... (90% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r22c1_2.svg">
NZ_CP012661 (1890757-1911534): Defluviimonas alba strain cai42 chromosome, co... (66% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r22c1_3.svg">
NZ_CP025408 (3460256-3484560): Paracoccus sp. BM15 chromosome, complete genome (35% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r22c1_4.svg">
NZ_OAOQ01000001 (323718-364255): Rhodobacter ovatus strain JA234, whole genom... (12% of genes show similarity), hserlactone,terpene
</option>

<option value = "svg/clusterblast_r22c1_5.svg">
NZ_JAAOIE010000001 (249289-270063): Planktotalea sp. M26A2M contig1, whole ge... (27% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r22c1_6.svg">
NZ_GG704596 (2627920-2648697): Ruegeria lacuscaerulensis ITI-1157 scf 1113211... (20% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r22c1_7.svg">
NZ_FNVD01000001 (228370-249144): Jhaorihella thermophila strain DSM 23413, wh... (23% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r22c1_8.svg">
NZ_KZ304951 (326558-347329): Limimaricola cinnabarinus strain XM1 scaffold1, ... (20% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r22c1_9.svg">
NC_003911 (2251591-2272374): Ruegeria pomeroyi DSS-3, complete sequence (20% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r22c1_10.svg">
NZ_KK088652 (53305-74076): Limimaricola hongkongensis DSM 17492 Scaffold7, wh... (20% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r22c1-download">Download graphic</button>
</div>
<div id="clusterblast-r22c1-svg"></div>

</div>
</div>

<div class="body-details-section r22c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-3"></div> <span class="help-tooltip" id="cb-known-help-3">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r22c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-4"></div> <span class="help-tooltip" id="cb-sub-help-4">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r22c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-6"></div> <span class="help-tooltip" id="tfbs-body-help-6">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AfsQ1
<div class="collapser collapser-target-AfsQ1 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Two-component system AfsQ1-Q2, activator of antibiotic production<br>
<span>Score</span>: 17.68 out of 19.52
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r22c1-404959-AfsQ1"></div>

</div>


</div>
</div>



<div class="body-details-section r22c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-8"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-8">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r22c1-pfam-details-svg" data-filename="NZ_JAAAUC010000002.1_r22c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r22c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r22c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-9"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-9">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r22c1-tigrfam-details-svg" data-filename="NZ_JAAAUC010000002.1_r22c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r22c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r22c1-tfbs-finder" data-name="r22c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r22c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-5"></div> <span class="help-tooltip" id="tfbs-side-help-5">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AfsQ1" target="_blank">AfsQ1</a></td>
<td>Two-component system AfsQ1-Q2, activator of antibiotic production</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r39c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_JAAAUC010000008.1 - Region 1 - T1PKS,hglE-KS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-34"></div> <span class="help-tooltip" id="region_svg-help-34">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_JAAAUC010000008.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r39c1-svg" data-filename="NZ_JAAAUC010000008.1_r39c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 107,003 - 159,934 nt. (total: 52,932 nt)

<a class="cluster-rules-header" id="r39c1-rules-header" href="#r39c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r39c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>hglE-KS: (hglE or hglD)</div>

<div class="region-svg-container">
<div id='r39c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-35"></div> <span class="help-tooltip" id="focus-panel-help-35">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r39c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r39c1-gene-table-details" data-name="r39c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r39c1-nrps_pks" data-name="r39c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r39c1-MIBiG-cluster-compare" data-name="r39c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r39c1-clusterblast" data-name="r39c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r39c1-knownclusterblast" data-name="r39c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r39c1-subclusterblast" data-name="r39c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r39c1-tfbs-finder" data-name="r39c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r39c1-nrps-pks-bubbles" data-name="r39c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r39c1-pfam-details" data-name="r39c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r39c1-tigrfam-details" data-name="r39c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r39c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-20"></div> <span class="help-tooltip" id="gt-help-tooltip-help-20">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r39c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r39c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r39c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r39c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r39c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-10"></div> <span class="help-tooltip" id="nrps-domain-help-10">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r39c1-details-svg" data-filename="NZ_JAAAUC010000008.1_r39c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r39c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r39c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-11"></div> <span class="help-tooltip" id="MIBiG-help-11">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r39c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r39c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r39c1"></div>
<div id="comparison-MIBiG-r39c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">hglE-KS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002648: 0-20487">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002648/index.html#r1c1" target="_new">BGC0002648</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.47, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.47, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.67</td>
<td>Polyketide</td>

<td>eicosapentaenoic acid</td>


<td>Shewanella sp. N2AIL</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002404: 204-23152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002404/index.html#r1c1" target="_new">BGC0002404</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.66</td>
<td>Other</td>

<td>falcarindiol</td>


<td>Solanum lycopersicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000863: 1224-16560">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000863/index.html#r1c1" target="_new">BGC0000863</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.63</td>
<td>Other</td>

<td>eicosapentaenoic acid</td>


<td>Pseudoalteromonas sp. DS-12</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000861: 0-20498">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000861/index.html#r1c1" target="_new">BGC0000861</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.62</td>
<td>Other</td>

<td>eicosapentaenoic acid</td>


<td>Shewanella sp. BR-2</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000869: 0-17945">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000869/index.html#r1c1" target="_new">BGC0000869</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.59</td>
<td>Other</td>

<td>heterocyst glycolipids</td>


<td>Nostoc sp. &#39;Peltigera membranacea cyanobiont&#39;</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000284: 0-11949">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000284/index.html#r1c1" target="_new">BGC0000284</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.46, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.46, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.55</td>
<td>Polyketide</td>

<td>phenolic lipids</td>


<td>Azotobacter vinelandii DJ</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000140: 112-14048">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000140/index.html#r1c1" target="_new">BGC0000140</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.54</td>
<td>Polyketide</td>

<td>4-hexadecanoyl-3-hydroxy-2-(hydroxymethyl)-2H-furan-5-one</td>


<td>Streptomyces sp. 88-682</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001338: 273-42813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001338/index.html#r1c1" target="_new">BGC0001338</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>






<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.50</td>
<td>Polyketide</td>

<td>citrinin</td>


<td>Monascus ruber</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056: 2-6689">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.49</td>
<td>Polyketide</td>

<td>esperamicin</td>


<td>Actinomadura verrucosospora</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.49</td>
<td>Other</td>

<td>citrinin</td>


<td>Monascus aurantiacus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r39c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000866: 467-5165">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000866/index.html#r1c1" target="_new">BGC0000866</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other</td>
<td>polyhydroxyalkanoate</td>
<td>Burkholderia sp. DSM 9242</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002648: 0-20487">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002648/index.html#r1c1" target="_new">BGC0002648</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.47, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>eicosapentaenoic acid</td>
<td>Shewanella sp. N2AIL</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002404: 204-23152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002404/index.html#r1c1" target="_new">BGC0002404</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>falcarindiol</td>
<td>Solanum lycopersicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000863: 1224-16560">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000863/index.html#r1c1" target="_new">BGC0000863</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>eicosapentaenoic acid</td>
<td>Pseudoalteromonas sp. DS-12</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000861: 0-20498">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000861/index.html#r1c1" target="_new">BGC0000861</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Other</td>
<td>eicosapentaenoic acid</td>
<td>Shewanella sp. BR-2</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000869: 0-17945">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000869/index.html#r1c1" target="_new">BGC0000869</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Other</td>
<td>heterocyst glycolipids</td>
<td>Nostoc sp. &#39;Peltigera membranacea cyanobiont&#39;</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000284: 0-11949">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000284/index.html#r1c1" target="_new">BGC0000284</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.46, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>phenolic lipids</td>
<td>Azotobacter vinelandii DJ</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000140: 112-14048">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000140/index.html#r1c1" target="_new">BGC0000140</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>4-hexadecanoyl-3-hydroxy-2-(hydroxymethyl)-2H-furan-5-one</td>
<td>Streptomyces sp. 88-682</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001338: 273-42813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001338/index.html#r1c1" target="_new">BGC0001338</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>citrinin</td>
<td>Monascus ruber</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000887: 1816-30991">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000887/index.html#r1c1" target="_new">BGC0000887</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Alkaloid</td>
<td>saxitoxin, gonyautoxin 2, gonyautoxin 3, toxin C2, toxin C3, gonyautoxin 5, decarbamoylsaxitoxin, decarbamoylgonyautoxin 2, decarbamoylgonyautoxin 3</td>
<td>Dolichospermum circinale AWQC131C</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r39c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-12"></div> <span class="help-tooltip" id="cb-general-help-12">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r39c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r39c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r39c1_1.svg">
NZ_CP045389 (3117653-3170642): Roseivivax sp. THAF30 chromosome, complete genome (45% of genes show similarity), hglE-KS
</option>

<option value = "svg/clusterblast_r39c1_2.svg">
NZ_CP048630 (3552803-3605933): Ancylobacter pratisalsi strain DSM 102029 chro... (52% of genes show similarity), T1PKS,hglE-KS
</option>

<option value = "svg/clusterblast_r39c1_3.svg">
NZ_CP045360 (3158059-3209751): Roseivivax sp. THAF40 chromosome, complete genome (61% of genes show similarity), T1PKS,hglE-KS
</option>

<option value = "svg/clusterblast_r39c1_4.svg">
NZ_CP045318 (3076240-3127932): Roseivivax sp. THAF197b chromosome, complete g... (61% of genes show similarity), T1PKS,hglE-KS
</option>

<option value = "svg/clusterblast_r39c1_5.svg">
NZ_JNLC01000010 (205604-258817): Bosea sp. 117 U742DRAFT scaffold00001.1 C, w... (37% of genes show similarity), T1PKS,hglE-KS
</option>

<option value = "svg/clusterblast_r39c1_6.svg">
NZ_BDSM01000025 (115003-159587): Microcystis sp. 0824, whole genome shotgun s... (23% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r39c1_7.svg">
NZ_CP014784 (3645799-3698560): Pseudomonas alcaligenes strain NEB 585 chromos... (23% of genes show similarity), T1PKS,hglE-KS
</option>

<option value = "svg/clusterblast_r39c1_8.svg">
NZ_CP046122 (1253566-1306362): Microlunatus sp. Gsoil 973 chromosome, complet... (18% of genes show similarity), T1PKS,hglE-KS
</option>

<option value = "svg/clusterblast_r39c1_9.svg">
NZ_JACJFN010000003 (218072-270908): Pseudomonas sp. SR9 contig3, whole genome... (28% of genes show similarity), T1PKS,hglE-KS
</option>

<option value = "svg/clusterblast_r39c1_10.svg">
NZ_CP053697 (3206523-3259359): Pseudomonas sp. S1-A32-2 chromosome, complete ... (30% of genes show similarity), T1PKS,hglE-KS
</option>

</select>
<button id= "clusterblast-r39c1-download">Download graphic</button>
</div>
<div id="clusterblast-r39c1-svg"></div>

</div>
</div>

<div class="body-details-section r39c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-13"></div> <span class="help-tooltip" id="cb-known-help-13">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r39c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-14"></div> <span class="help-tooltip" id="cb-sub-help-14">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r39c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-18"></div> <span class="help-tooltip" id="tfbs-body-help-18">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (3):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
HypR
<div class="collapser collapser-target-HypR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: L-hydroxyproline utilization repressor<br>
<span>Score</span>: 18.59 out of 22.40
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r39c1-145434-HypR"></div>

<div class="tfbs-hit-detail">
OsdR
<div class="collapser collapser-target-OsdR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Development and stress management regulator<br>
<span>Score</span>: 24.03 out of 30.12
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r39c1-147039-OsdR"></div>

<div class="tfbs-hit-detail">
NrdR
<div class="collapser collapser-target-NrdR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Represses ribonucleotide reductase encoding genes<br>
<span>Score</span>: 20.25 out of 27.18
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r39c1-155067-NrdR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r39c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-19"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-19">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r39c1-domain-bubble-svg" data-filename="NZ_JAAAUC010000008.1_r39c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r39c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (107002 - 159934): interleaved T1PKS-hglE-KS
</option>


</select>

<div class="details-svg" id="r39c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r39c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r39c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-21"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-21">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r39c1-pfam-details-svg" data-filename="NZ_JAAAUC010000008.1_r39c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r39c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r39c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-22"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-22">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r39c1-tigrfam-details-svg" data-filename="NZ_JAAAUC010000008.1_r39c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r39c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r39c1-nrps_pks_products" data-name="r39c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r39c1-nrps_pks_monomers" data-name="r39c1-nrps_pks_monomers">
NRPS/PKS monomers
</div>



<div class="sidepanel-details-header r39c1-tfbs-finder" data-name="r39c1-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r39c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-15"></div> <span class="help-tooltip" id="nrps-structures-help-15">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 107002 - 159934:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r39c1-1" class="smiles-canvas" data-smiles="CC(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(mal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r39c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="CC(=O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r39c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS monomer predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-16"></div> <span class="help-tooltip" id="nrps-monomers-help-16">Shows the predicted monomers for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">NGLDBBKK_03670</span></strong>: mxmal
<div class="collapser collapser-target-NGLDBBKK_03670 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (570..875)</strong>: mxmal
<div class="collapser collapser-target-NGLDBBKK_03670 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  (unknown)
<div class="collapser collapser-target-NGLDBBKK_03670-ATSignature collapser-level-none"> </div><div class="collapser-content">
No matches above 50%
</div><br>

minowa:  Methoxymalonyl-CoA
<div class="collapser collapser-target-NGLDBBKK_03670-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methoxymalonyl-CoA: 27.2</dt>

<dd></dd><dt>Methylmalonyl-CoA: 20.1</dt>

<dd></dd><dt>Malonyl-CoA: 11.4</dt>

<dd></dd><dt>Isobutyryl-CoA: 8.5</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>inactive: 0.0</dt>

<dd></dd><dt>fatty_acid: 0.0</dt>

<dd></dd><dt>Propionyl-CoA: 0.0</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 0.0</dt>

<dd></dd><dt>CHC-CoA: 0.0</dt>

<dd></dd><dt>Benzoyl-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

<strong><span class="serif">NGLDBBKK_03671</span></strong>: mal
<div class="collapser collapser-target-NGLDBBKK_03671 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (1250..1544)</strong>: mal
<div class="collapser collapser-target-NGLDBBKK_03671 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-NGLDBBKK_03671-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 70.8%</dd>
<dd>inactive: 62.5%</dd>
<dd>Methylmalonyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-NGLDBBKK_03671-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 63.7</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 29.6</dt>

<dd></dd><dt>Methylmalonyl-CoA: 27.5</dt>

<dd></dd><dt>CHC-CoA: 20.7</dt>

<dd></dd><dt>Propionyl-CoA: 18.8</dt>

<dd></dd><dt>inactive: 17.3</dt>

<dd></dd><dt>fatty_acid: 12.8</dt>

<dd></dd><dt>Isobutyryl-CoA: 9.5</dt>

<dd></dd><dt>Benzoyl-CoA: 9.4</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r39c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-17"></div> <span class="help-tooltip" id="tfbs-side-help-17">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/HypR" target="_blank">HypR</a></td>
<td>L-hydroxyproline utilization repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/OsdR" target="_blank">OsdR</a></td>
<td>Development and stress management regulator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrdR" target="_blank">NrdR</a></td>
<td>Represses ribonucleotide reductase encoding genes</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>




<div class="page" id='r40c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_JAAAUC010000009.1 - Region 1 - hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-36"></div> <span class="help-tooltip" id="region_svg-help-36">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_JAAAUC010000009.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r40c1-svg" data-filename="NZ_JAAAUC010000009.1_r40c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 6,522 - 27,112 nt. (total: 20,591 nt)

<a class="cluster-rules-header" id="r40c1-rules-header" href="#r40c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r40c1-rules">hserlactone: Autoind_synth</div>

<div class="region-svg-container">
<div id='r40c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-37"></div> <span class="help-tooltip" id="focus-panel-help-37">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r40c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r40c1-gene-table-details" data-name="r40c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r40c1-MIBiG-cluster-compare" data-name="r40c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r40c1-clusterblast" data-name="r40c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r40c1-knownclusterblast" data-name="r40c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r40c1-subclusterblast" data-name="r40c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r40c1-tfbs-finder" data-name="r40c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r40c1-pfam-details" data-name="r40c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r40c1-tigrfam-details" data-name="r40c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r40c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-29"></div> <span class="help-tooltip" id="gt-help-tooltip-help-29">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r40c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r40c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r40c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r40c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r40c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-23"></div> <span class="help-tooltip" id="MIBiG-help-23">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r40c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r40c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r40c1"></div>
<div id="comparison-MIBiG-r40c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">hserlactone</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002447: 0-8196">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002447/index.html#r1c1" target="_new">BGC0002447</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>cerecyclin</td>


<td>Bacillus cereus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000487: 82-11698">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000487/index.html#r1c1" target="_new">BGC0000487</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>carnocyclin</td>


<td>Carnobacterium maltaromaticum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001992: 0-4749">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001992/index.html#r1c1" target="_new">BGC0001992</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Other</td>

<td>thioguanine</td>


<td>Erwinia amylovora CFBP1430</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554: 0-6546">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>SRO15-3108</td>


<td>Streptomyces filamentosus NRRL 15998</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002116: 0-6188">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002116/index.html#r1c1" target="_new">BGC0002116</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>darobactin A</td>


<td>Photorhabdus khanii HGB 1456</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000548: 588-10610">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000548/index.html#r1c1" target="_new">BGC0000548</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>salivaricin A</td>


<td>Streptococcus salivarius</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001525: 0-16392">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001525/index.html#r1c1" target="_new">BGC0001525</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Other</td>

<td>bartoloside 2, bartoloside 3, bartoloside 4</td>


<td>Synechocystis salina LEGE 06155</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001526: 0-14417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001526/index.html#r1c1" target="_new">BGC0001526</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Other</td>

<td>bartoloside E, bartoloside F, bartoloside G, bartoloside H, bartoloside I, bartoloside J, bartoloside K</td>


<td>Synechocystis salina LEGE 06099</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002442: 0-8391">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002442/index.html#r1c1" target="_new">BGC0002442</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>toyoncin</td>


<td>Bacillus toyonensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002699: 0-12569">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002699/index.html#r1c1" target="_new">BGC0002699</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>nostolysamide A, nostolysamide B</td>


<td>Nostoc punctiforme PCC 73102</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r40c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002447: 0-8196">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002447/index.html#r1c1" target="_new">BGC0002447</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>RiPP</td>
<td>cerecyclin</td>
<td>Bacillus cereus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000487: 82-11698">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000487/index.html#r1c1" target="_new">BGC0000487</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>RiPP</td>
<td>carnocyclin</td>
<td>Carnobacterium maltaromaticum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001992: 0-4749">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001992/index.html#r1c1" target="_new">BGC0001992</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Other</td>
<td>thioguanine</td>
<td>Erwinia amylovora CFBP1430</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554: 0-6546">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>RiPP</td>
<td>SRO15-3108</td>
<td>Streptomyces filamentosus NRRL 15998</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002116: 0-6188">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002116/index.html#r1c1" target="_new">BGC0002116</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>RiPP</td>
<td>darobactin A</td>
<td>Photorhabdus khanii HGB 1456</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000548: 588-10610">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000548/index.html#r1c1" target="_new">BGC0000548</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>RiPP</td>
<td>salivaricin A</td>
<td>Streptococcus salivarius</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001525: 0-16392">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001525/index.html#r1c1" target="_new">BGC0001525</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Other</td>
<td>bartoloside 2, bartoloside 3, bartoloside 4</td>
<td>Synechocystis salina LEGE 06155</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001526: 0-14417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001526/index.html#r1c1" target="_new">BGC0001526</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Other</td>
<td>bartoloside E, bartoloside F, bartoloside G, bartoloside H, bartoloside I, bartoloside J, bartoloside K</td>
<td>Synechocystis salina LEGE 06099</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002442: 0-8391">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002442/index.html#r1c1" target="_new">BGC0002442</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>toyoncin</td>
<td>Bacillus toyonensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002699: 0-12569">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002699/index.html#r1c1" target="_new">BGC0002699</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>nostolysamide A, nostolysamide B</td>
<td>Nostoc punctiforme PCC 73102</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r40c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-24"></div> <span class="help-tooltip" id="cb-general-help-24">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r40c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r40c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r40c1_1.svg">
NZ_WUJE01000010 (159463-180054): Frigidibacter albus strain SP32 Scaffold10, ... (100% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r40c1_2.svg">
NC_009952 (291164-311300): Dinoroseobacter shibae DFL 12 = DSM 16493, complet... (96% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r40c1_3.svg">
NZ_JABJVX010000004 (234066-254654): Rhodophyticola sp. DY48A3-103 NODE 4 leng... (96% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r40c1_4.svg">
NZ_CP012661 (429342-449930): Defluviimonas alba strain cai42 chromosome, comp... (96% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r40c1_5.svg">
NZ_JAAOIE010000007 (100508-121096): Planktotalea sp. M26A2M contig7, whole ge... (92% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r40c1_6.svg">
NZ_CP038492 (3625933-3646521): Boseongicola sp. CCM32 chromosome, complete ge... (96% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r40c1_7.svg">
NZ_JAAZQQ010000004 (49387-69975): Roseibacterium sp. KMU-115 contig4, whole g... (92% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r40c1_8.svg">
NZ_SNYP01000002 (59722-81693): Rhodobacter veldkampii DSM 11550 Ga0310482 102... (90% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r40c1_9.svg">
NZ_SAVA01000013 (35338-55929): Sinirhodobacter hungdaonensis strain CGMCC 1.1... (92% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r40c1_10.svg">
NZ_AYPY01000003 (103064-123655): Rhodobacter capsulatus YW1 seq0003, whole ge... (92% of genes show similarity), hserlactone
</option>

</select>
<button id= "clusterblast-r40c1-download">Download graphic</button>
</div>
<div id="clusterblast-r40c1-svg"></div>

</div>
</div>

<div class="body-details-section r40c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-25"></div> <span class="help-tooltip" id="cb-known-help-25">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r40c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-26"></div> <span class="help-tooltip" id="cb-sub-help-26">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r40c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-28"></div> <span class="help-tooltip" id="tfbs-body-help-28">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (2):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
NrdR
<div class="collapser collapser-target-NrdR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Represses ribonucleotide reductase encoding genes<br>
<span>Score</span>: 20.33 out of 27.18
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r40c1-13913-NrdR"></div>

<div class="tfbs-hit-detail">
NrdR
<div class="collapser collapser-target-NrdR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Represses ribonucleotide reductase encoding genes<br>
<span>Score</span>: 21.16 out of 27.18
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r40c1-15788-NrdR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r40c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-30"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-30">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r40c1-pfam-details-svg" data-filename="NZ_JAAAUC010000009.1_r40c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r40c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r40c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-31"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-31">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r40c1-tigrfam-details-svg" data-filename="NZ_JAAAUC010000009.1_r40c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r40c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r40c1-tfbs-finder" data-name="r40c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r40c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-27"></div> <span class="help-tooltip" id="tfbs-side-help-27">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrdR" target="_blank">NrdR</a></td>
<td>Represses ribonucleotide reductase encoding genes</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>