
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>GCF_000740785.1_Paenirhodobacter_enshiensis_DW2-9 - 1 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/bacteria.css">
<meta property="og:title" content="GCF_000740785.1_Paenirhodobacter_enshiensis_DW2-9 - 1 record(s) - 1 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-378bbc93-897e-4894-a3aa-5f6af126f022/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://antismash.secondarymetabolites.org/">
<img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://antismash.secondarymetabolites.org/">
antiSMASH version 7.0.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="GCF_000740785.1_Paenirhodobacter_enshiensis_DW2-9.zip">Download all results</a></li>

<li><a href="GCF_000740785.1_Paenirhodobacter_enshiensis_DW2-9.gbk">Download GenBank summary file</a></li>

<li><a href="bacteria-378bbc93-897e-4894-a3aa-5f6af126f022.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton NRPS NRPS-like r6c1">
<a href="#r6c1">6.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>NZ_JFZB01000007.1</strong> (Paenirhodobacter enshiensis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-6" data-filename="NZ_JFZB01000007.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-6" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r6c1">
<td class="regbutton NRPS NRPS-like r6c1">
<a href="#r6c1">Region&nbsp6.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">137,907</td>
<td class="digits table-split-left">181,737</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r6c1">
<td class="regbutton NRPS NRPS-like r6c1">
<a href="#r6c1">Region&nbsp6.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">137,907</td>
<td class="digits table-split-left">181,737</td>

<td colspan="3"></td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>NZ_JFZB01000001.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000002.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000003.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000005.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000006.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000008.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000009.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000010.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000011.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000012.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000013.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000014.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000016.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000017.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000018.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000019.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000020.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000021.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000022.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000023.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000024.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000025.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000026.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000027.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000028.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000029.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000030.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000031.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000032.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000033.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000034.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000035.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000036.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000037.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000038.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000039.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000040.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000042.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000043.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000044.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000045.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000046.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000047.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000048.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000049.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000050.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000051.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000052.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000053.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000054.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000056.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000057.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000058.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000059.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000060.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000061.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000062.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000063.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000064.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000065.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000066.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000067.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000068.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000069.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000070.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000071.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000072.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000073.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000074.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000076.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000077.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000080.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000081.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000082.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000083.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000090.1</strong> (Paenirhodobacter enshiensis)</dd>

<dd><strong>NZ_JFZB01000091.1</strong> (Paenirhodobacter enshiensis)</dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>







<div class="page" id='r6c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_JFZB01000007.1 - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-14"></div> <span class="help-tooltip" id="region_svg-help-14">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_JFZB01000007.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c1-svg" data-filename="NZ_JFZB01000007.1_r6c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 137,907 - 181,737 nt. (total: 43,831 nt)

<a class="cluster-rules-header" id="r6c1-rules-header" href="#r6c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r6c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-15"></div> <span class="help-tooltip" id="focus-panel-help-15">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c1-gene-table-details" data-name="r6c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c1-nrps_pks" data-name="r6c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c1-MIBiG-cluster-compare" data-name="r6c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c1-clusterblast" data-name="r6c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c1-knownclusterblast" data-name="r6c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c1-subclusterblast" data-name="r6c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c1-tfbs-finder" data-name="r6c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r6c1-nrps-pks-bubbles" data-name="r6c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r6c1-pfam-details" data-name="r6c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r6c1-tigrfam-details" data-name="r6c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r6c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-11"></div> <span class="help-tooltip" id="gt-help-tooltip-help-11">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-1"></div> <span class="help-tooltip" id="nrps-domain-help-1">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c1-details-svg" data-filename="NZ_JFZB01000007.1_r6c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-2"></div> <span class="help-tooltip" id="MIBiG-help-2">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c1"></div>
<div id="comparison-MIBiG-r6c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001527: 0-8170">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001527/index.html#r1c1" target="_new">BGC0001527</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>Other</td>

<td>basidioferrin</td>


<td>Gelatoporia subvermispora</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001137: 0-18476">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001137/index.html#r1c1" target="_new">BGC0001137</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Alkaloid</td>

<td>marinacarboline A, marinacarboline B, marinacarboline C, marinacarboline D</td>


<td>Marinactinospora thermotolerans</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000933: 0-5189">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000933/index.html#r1c1" target="_new">BGC0000933</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Other</td>

<td>xiamenmycin A</td>


<td>Streptomyces xiamenensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002002: 0-14395">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002002/index.html#r1c1" target="_new">BGC0002002</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>1,6-phenazinedimethanol, fontizine A</td>


<td>Serratia fonticola</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000328: 0-6254">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000328/index.html#r1c1" target="_new">BGC0000328</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>NRP</td>

<td>coronatine</td>


<td>Pseudomonas savastanoi pv. glycinea</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002476: 0-10330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002476/index.html#r1c1" target="_new">BGC0002476</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Escherichia coli str. K-12 substr. MG1655</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002295: 0-9315">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002295/index.html#r1c1" target="_new">BGC0002295</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>NRP, Saccharide</td>

<td>minimycin</td>


<td>Streptomyces hygroscopicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001627: 0-10806">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001627/index.html#r1c1" target="_new">BGC0001627</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>isonitrile lipopeptides</td>


<td>Mycobacterium tuberculosis H37Rv</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000841: 0-11023">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000841/index.html#r1c1" target="_new">BGC0000841</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Other (Non-NRP beta-lactam)</td>

<td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate, clavulanic acid</td>


<td>Streptomyces clavuligerus ATCC 27064</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000131: 0-6280">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000131/index.html#r1c1" target="_new">BGC0000131</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>pyrrolomycin</td>


<td>Streptomyces sp. UC 11065</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001527: 0-8170">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001527/index.html#r1c1" target="_new">BGC0001527</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Other</td>
<td>basidioferrin</td>
<td>Gelatoporia subvermispora</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001137: 0-18476">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001137/index.html#r1c1" target="_new">BGC0001137</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Alkaloid</td>
<td>marinacarboline A, marinacarboline B, marinacarboline C, marinacarboline D</td>
<td>Marinactinospora thermotolerans</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000933: 0-5189">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000933/index.html#r1c1" target="_new">BGC0000933</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>xiamenmycin A</td>
<td>Streptomyces xiamenensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002002: 0-14395">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002002/index.html#r1c1" target="_new">BGC0002002</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>1,6-phenazinedimethanol, fontizine A</td>
<td>Serratia fonticola</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000328: 0-6254">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000328/index.html#r1c1" target="_new">BGC0000328</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>NRP</td>
<td>coronatine</td>
<td>Pseudomonas savastanoi pv. glycinea</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002476: 0-10330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002476/index.html#r1c1" target="_new">BGC0002476</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Escherichia coli str. K-12 substr. MG1655</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002295: 0-9315">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002295/index.html#r1c1" target="_new">BGC0002295</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP, Saccharide</td>
<td>minimycin</td>
<td>Streptomyces hygroscopicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001627: 0-10806">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001627/index.html#r1c1" target="_new">BGC0001627</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>isonitrile lipopeptides</td>
<td>Mycobacterium tuberculosis H37Rv</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000841: 0-11023">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000841/index.html#r1c1" target="_new">BGC0000841</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Other (Non-NRP beta-lactam)</td>
<td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate, clavulanic acid</td>
<td>Streptomyces clavuligerus ATCC 27064</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000131: 0-6280">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000131/index.html#r1c1" target="_new">BGC0000131</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Polyketide</td>
<td>pyrrolomycin</td>
<td>Streptomyces sp. UC 11065</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-3"></div> <span class="help-tooltip" id="cb-general-help-3">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c1_1.svg">
NZ_SAVA01000002 (127464-171295): Sinirhodobacter hungdaonensis strain CGMCC 1... (6% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r6c1_2.svg">
NC_022049 (230638-327538): Paracoccus aminophilus JCM 7686 plasmid pAMI4, com... (3% of genes show similarity), LAP,NRPS-like,TfuA-related,thiopeptide
</option>

<option value = "svg/clusterblast_r6c1_3.svg">
NZ_CM001161 (1328883-1369933): Rhodobacter sphaeroides WS8N chromosome I, who... (18% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r6c1_4.svg">
NZ_CCAZ020000002 (1005998-1049931): Afipia felis strain 76713, whole genome s... (7% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r6c1_5.svg">
NZ_CP031751 (352424-396264): Rhodobacter sphaeroides strain EBL0706 chromosom... (4% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r6c1_6.svg">
NZ_CP017173 (4242529-4408576): Myxococcus xanthus strain MC3.3.5c16 chromosom... (9% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c1_7.svg">
NZ_CP022540 (3492163-3582077): Antarctobacter heliothermus strain SMS3 chromo... (5% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c1_8.svg">
NZ_CP025583 (1662544-1715115): Paracoccus jeotgali strain CBA4604 chromosome,... (7% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r6c1_9.svg">
NZ_LMGC01000012 (696733-718648): Ensifer sp. Root558 contig 2, whole genome s... (17% of genes show similarity), TfuA-related
</option>

<option value = "svg/clusterblast_r6c1_10.svg">
NZ_CP016174 (3685051-3929323): Amycolatopsis orientalis strain B-37 chromosom... (5% of genes show similarity), NRPS,T1PKS,TfuA-related,other
</option>

</select>
<button id= "clusterblast-r6c1-download">Download graphic</button>
</div>
<div id="clusterblast-r6c1-svg"></div>

</div>
</div>

<div class="body-details-section r6c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-4"></div> <span class="help-tooltip" id="cb-known-help-4">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-5"></div> <span class="help-tooltip" id="cb-sub-help-5">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-9"></div> <span class="help-tooltip" id="tfbs-body-help-9">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ArgR
<div class="collapser collapser-target-ArgR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Regulator of arginine biosynthesis genes<br>
<span>Score</span>: 18.57 out of 20.01
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r6c1-148659-ArgR"></div>

<div class="tfbs-hit-detail">
CelR
<div class="collapser collapser-target-CelR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Cellobiose uptake repressor<br>
<span>Score</span>: 23.40 out of 30.11
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r6c1-156629-CelR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (4):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 19.09 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r6c1-138139-ZuR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 18.06 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r6c1-148107-ZuR"></div>

<div class="tfbs-hit-detail">
CelR
<div class="collapser collapser-target-CelR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Cellobiose uptake repressor<br>
<span>Score</span>: 23.20 out of 30.11
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r6c1-156628-CelR"></div>

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 19.97 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r6c1-171326-NrtR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r6c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-10"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-10">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c1-domain-bubble-svg" data-filename="NZ_JFZB01000007.1_r6c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r6c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (137906 - 181737): single NRPS-like
</option>


</select>

<div class="details-svg" id="r6c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r6c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r6c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-12"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-12">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c1-pfam-details-svg" data-filename="NZ_JFZB01000007.1_r6c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r6c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-13"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-13">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c1-tigrfam-details-svg" data-filename="NZ_JFZB01000007.1_r6c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r6c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c1-nrps_pks_products" data-name="r6c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r6c1-nrps_pks_monomers" data-name="r6c1-nrps_pks_monomers">
NRPS/PKS monomers
</div>



<div class="sidepanel-details-header r6c1-tfbs-finder" data-name="r6c1-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r6c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-6"></div> <span class="help-tooltip" id="nrps-structures-help-6">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 137906 - 181737:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r6c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r6c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r6c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS monomer predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-7"></div> <span class="help-tooltip" id="nrps-monomers-help-7">Shows the predicted monomers for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">LBIMNCPI_01172</span></strong>: X
<div class="collapser collapser-target-LBIMNCPI_01172 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (484..871)</strong>: X
<div class="collapser collapser-target-LBIMNCPI_01172 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-LBIMNCPI_01172-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVGSIAAVEK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Trp <span class="serif">DVSSIGAVEK</span> (65% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r6c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-8"></div> <span class="help-tooltip" id="tfbs-side-help-8">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ArgR" target="_blank">ArgR</a></td>
<td>Regulator of arginine biosynthesis genes</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/CelR" target="_blank">CelR</a></td>
<td>Cellobiose uptake repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrtR" target="_blank">NrtR</a></td>
<td>NAD synthesis repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>