
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>GCF_013415485.1_Rhabdonatronobacter_sediminivivens_IM2376 - 6 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/bacteria.css">
<meta property="og:title" content="GCF_013415485.1_Rhabdonatronobacter_sediminivivens_IM2376 - 5 record(s) - 6 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-fc854d9d-af54-40c4-8b61-5cf81178978b/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://antismash.secondarymetabolites.org/">
<img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://antismash.secondarymetabolites.org/">
antiSMASH version 7.0.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="GCF_013415485.1_Rhabdonatronobacter_sediminivivens_IM2376.zip">Download all results</a></li>

<li><a href="GCF_013415485.1_Rhabdonatronobacter_sediminivivens_IM2376.gbk">Download GenBank summary file</a></li>

<li><a href="bacteria-fc854d9d-af54-40c4-8b61-5cf81178978b.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton hybrid r28c1">
<a href="#r28c1">28.1</a>
</div>



<div class="regbutton terpene terpene r36c1">
<a href="#r36c1">36.1</a>
</div>

<div class="regbutton terpene terpene r36c2">
<a href="#r36c2">36.2</a>
</div>



<div class="regbutton other phosphonate r54c1">
<a href="#r54c1">54.1</a>
</div>



<div class="regbutton other hserlactone r58c1">
<a href="#r58c1">58.1</a>
</div>



<div class="regbutton other ectoine r69c1">
<a href="#r69c1">69.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>NZ_JACBXS010000012.1</strong> (Rhabdonatronobacter sediminivivens)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-28" data-filename="NZ_JACBXS010000012.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-28" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r28c1">
<td class="regbutton hybrid r28c1">
<a href="#r28c1">Region&nbsp28.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">693</td>
<td class="digits table-split-left">52,741</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_JACBXS010000001.1</strong> (Rhabdonatronobacter sediminivivens)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-36" data-filename="NZ_JACBXS010000001.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-36" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r36c1">
<td class="regbutton terpene terpene r36c1">
<a href="#r36c1">Region&nbsp36.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">398</td>
<td class="digits table-split-left">10,906</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r36c2">
<td class="regbutton terpene terpene r36c2">
<a href="#r36c2">Region&nbsp36.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">65,623</td>
<td class="digits table-split-left">86,666</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000647/1" target="_blank">carotenoid</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_JACBXS010000036.1</strong> (Rhabdonatronobacter sediminivivens)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-54" data-filename="NZ_JACBXS010000036.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-54" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r54c1">
<td class="regbutton other phosphonate r54c1">
<a href="#r54c1">Region&nbsp54.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#phosphonate" target="_blank">phosphonate</a>
</td>
<td class="digits">2,878</td>
<td class="digits table-split-left">16,494</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000806/1" target="_blank">phosphonoglycans</a></td>
<td>Saccharide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 3%, #ffffff00 3%)">3%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_JACBXS010000003.1</strong> (Rhabdonatronobacter sediminivivens)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-58" data-filename="NZ_JACBXS010000003.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-58" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r58c1">
<td class="regbutton other hserlactone r58c1">
<a href="#r58c1">Region&nbsp58.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">4,587</td>
<td class="digits table-split-left">25,456</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002669/1" target="_blank">TVA-YJ-2</a></td>
<td>RiPP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_JACBXS010000004.1</strong> (Rhabdonatronobacter sediminivivens)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-69" data-filename="NZ_JACBXS010000004.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-69" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r69c1">
<td class="regbutton other ectoine r69c1">
<a href="#r69c1">Region&nbsp69.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
</td>
<td class="digits">111,141</td>
<td class="digits table-split-left">121,524</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000860/1" target="_blank">ectoine</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 80%, #ffffff00 80%)">80%</td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r28c1">
<td class="regbutton hybrid r28c1">
<a href="#r28c1">Region&nbsp28.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">693</td>
<td class="digits table-split-left">52,741</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r36c1">
<td class="regbutton terpene terpene r36c1">
<a href="#r36c1">Region&nbsp36.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">398</td>
<td class="digits table-split-left">10,906</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r36c2">
<td class="regbutton terpene terpene r36c2">
<a href="#r36c2">Region&nbsp36.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">65,623</td>
<td class="digits table-split-left">86,666</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000647/1" target="_blank">carotenoid</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>


<tr class="linked-row even" data-anchor="#r54c1">
<td class="regbutton other phosphonate r54c1">
<a href="#r54c1">Region&nbsp54.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#phosphonate" target="_blank">phosphonate</a>
</td>
<td class="digits">2,878</td>
<td class="digits table-split-left">16,494</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000806/1" target="_blank">phosphonoglycans</a></td>
<td>Saccharide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 3%, #ffffff00 3%)">3%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r58c1">
<td class="regbutton other hserlactone r58c1">
<a href="#r58c1">Region&nbsp58.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">4,587</td>
<td class="digits table-split-left">25,456</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002669/1" target="_blank">TVA-YJ-2</a></td>
<td>RiPP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>

</tr>


<tr class="linked-row even" data-anchor="#r69c1">
<td class="regbutton other ectoine r69c1">
<a href="#r69c1">Region&nbsp69.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
</td>
<td class="digits">111,141</td>
<td class="digits table-split-left">121,524</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000860/1" target="_blank">ectoine</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 80%, #ffffff00 80%)">80%</td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>NZ_JACBXS010000100.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000101.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000102.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000103.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000104.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000105.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000106.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000107.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000108.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000109.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000010.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000110.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000111.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000112.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000113.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000114.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000115.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000116.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000117.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000118.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000119.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000011.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000120.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000121.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000122.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000123.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000124.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000013.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000014.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000015.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000016.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000017.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000018.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000019.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000020.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000021.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000022.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000023.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000024.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000025.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000026.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000027.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000028.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000029.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000002.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000030.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000031.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000032.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000033.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000034.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000035.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000037.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000038.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000039.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000040.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000041.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000042.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000043.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000044.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000045.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000046.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000047.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000048.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000049.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000050.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000051.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000052.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000053.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000054.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000055.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000056.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000057.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000058.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000059.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000005.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000060.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000061.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000062.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000063.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000064.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000065.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000066.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000067.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000068.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000069.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000006.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000070.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000071.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000072.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000073.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000074.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000075.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000076.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000077.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000078.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000079.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000007.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000080.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000081.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000082.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000083.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000084.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000085.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000086.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000087.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000088.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000089.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000008.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000090.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000091.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000092.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000093.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000094.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000095.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000096.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000097.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000098.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000099.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

<dd><strong>NZ_JACBXS010000009.1</strong> (Rhabdonatronobacter sediminivivens)</dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>







<div class="page" id='r28c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_JACBXS010000012.1 - Region 1 - NRPS-like,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-59"></div> <span class="help-tooltip" id="region_svg-help-59">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_JACBXS010000012.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r28c1-svg" data-filename="NZ_JACBXS010000012.1_r28c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 693 - 52,741 nt. (total: 52,049 nt)

<a class="cluster-rules-header" id="r28c1-rules-header" href="#r28c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r28c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r28c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-60"></div> <span class="help-tooltip" id="focus-panel-help-60">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r28c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r28c1-gene-table-details" data-name="r28c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r28c1-nrps_pks" data-name="r28c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r28c1-MIBiG-cluster-compare" data-name="r28c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r28c1-clusterblast" data-name="r28c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r28c1-knownclusterblast" data-name="r28c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r28c1-subclusterblast" data-name="r28c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r28c1-tfbs-finder" data-name="r28c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r28c1-nrps-pks-bubbles" data-name="r28c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r28c1-pfam-details" data-name="r28c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r28c1-tigrfam-details" data-name="r28c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r28c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-11"></div> <span class="help-tooltip" id="gt-help-tooltip-help-11">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r28c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r28c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r28c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r28c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r28c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-1"></div> <span class="help-tooltip" id="nrps-domain-help-1">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r28c1-details-svg" data-filename="NZ_JACBXS010000012.1_r28c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r28c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r28c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-2"></div> <span class="help-tooltip" id="MIBiG-help-2">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r28c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r28c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r28c1"></div>
<div id="comparison-MIBiG-r28c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002474: 103-7254">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002474/index.html#r1c1" target="_new">BGC0002474</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.47</td>
<td>NRP</td>

<td>agrobactin</td>


<td>Agrobacterium tumefaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.47</td>
<td>Polyketide</td>

<td>HEx-pks16 polyketide</td>


<td>Punctularia strigosozonata HHB-11173 SS5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.44</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001928: 108-8402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001928/index.html#r1c1" target="_new">BGC0001928</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.46, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.44</td>
<td>Other</td>

<td>(2R,3S,4S)-5-fluoro-2,3,4-trihydroxypentanoic acid</td>


<td>Streptomyces sp. MA37</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.44</td>
<td>Polyketide</td>

<td>strobilurin A</td>


<td>Strobilurus tenacellus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000800: 32-15282">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000800/index.html#r1c1" target="_new">BGC0000800</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.43</td>
<td>Saccharide</td>

<td>xanthan</td>


<td>Xanthomonas campestris</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276: 1101-6575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.43</td>
<td>Polyketide</td>

<td>6-methylsalicyclic acid</td>


<td>Aspergillus terreus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.42</td>
<td>Polyketide</td>

<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.42</td>
<td>Polyketide</td>

<td>citreoviridin</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244: 0-5358">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.42</td>
<td>Polyketide</td>

<td>(-)-Mellein</td>


<td>Parastagonospora nodorum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r28c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001928: 108-8402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001928/index.html#r1c1" target="_new">BGC0001928</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Other</td>
<td>(2R,3S,4S)-5-fluoro-2,3,4-trihydroxypentanoic acid</td>
<td>Streptomyces sp. MA37</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002474: 103-7254">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002474/index.html#r1c1" target="_new">BGC0002474</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>agrobactin</td>
<td>Agrobacterium tumefaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000800: 32-15282">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000800/index.html#r1c1" target="_new">BGC0000800</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Saccharide</td>
<td>xanthan</td>
<td>Xanthomonas campestris</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>HEx-pks16 polyketide</td>
<td>Punctularia strigosozonata HHB-11173 SS5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>strobilurin A</td>
<td>Strobilurus tenacellus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001591: 66-30223">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001591/index.html#r1c1" target="_new">BGC0001591</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Other</td>
<td>fatty acid enol ester, N-tetradecanoyl tyrosine</td>
<td>uncultured bacterium CSLC2</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000225: 1562-24998">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000225/index.html#r1c1" target="_new">BGC0000225</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>frenolicin B</td>
<td>Streptomyces roseofulvus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276: 1101-6575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>6-methylsalicyclic acid</td>
<td>Aspergillus terreus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001940: 1-19375">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001940/index.html#r1c1" target="_new">BGC0001940</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>pyxidicycline A, pyxidicycline B</td>
<td>Pyxidicoccus fallax</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r28c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-3"></div> <span class="help-tooltip" id="cb-general-help-3">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r28c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r28c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r28c1_1.svg">
NZ_FOCE01000008 (86958-138943): Gemmobacter aquatilis strain DSM 3857, whole ... (30% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r28c1_2.svg">
NZ_FNOM01000001 (453882-506024): Roseicitreum antarcticum strain CGMCC 1.8894... (21% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r28c1_3.svg">
NZ_CP010629 (2794001-2846188): Phaeobacter inhibens strain P78 chromosome, co... (32% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r28c1_4.svg">
NZ_CP010610 (537911-590098): Phaeobacter inhibens strain P92 chromosome, comp... (32% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r28c1_5.svg">
NZ_CP019307 (2992804-3044991): Phaeobacter inhibens strain DOK1-1 chromosome,... (32% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r28c1_6.svg">
NZ_KI421498 (2000683-2052870): Phaeobacter inhibens DSM 16374 PhainDRAFT scaf... (32% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r28c1_7.svg">
NZ_JSWJ01000012 (16504-68700): Phaeobacter sp. S60 Phaeobacter S60 contig 12,... (32% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r28c1_8.svg">
NZ_CP010735 (520291-572478): Phaeobacter inhibens strain P72 chromosome, comp... (32% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r28c1_9.svg">
NZ_CP010668 (525535-577722): Phaeobacter inhibens strain P57 chromosome, comp... (32% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r28c1_10.svg">
NZ_CP010805 (472759-524955): Phaeobacter piscinae strain P23 chromosome, comp... (32% of genes show similarity), NRPS-like,T1PKS
</option>

</select>
<button id= "clusterblast-r28c1-download">Download graphic</button>
</div>
<div id="clusterblast-r28c1-svg"></div>

</div>
</div>

<div class="body-details-section r28c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-4"></div> <span class="help-tooltip" id="cb-known-help-4">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r28c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-5"></div> <span class="help-tooltip" id="cb-sub-help-5">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r28c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-9"></div> <span class="help-tooltip" id="tfbs-body-help-9">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ArgR
<div class="collapser collapser-target-ArgR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Regulator of arginine biosynthesis genes<br>
<span>Score</span>: 18.06 out of 20.01
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r28c1-13610-ArgR"></div>

<div class="tfbs-hit-detail">
ArgR
<div class="collapser collapser-target-ArgR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Regulator of arginine biosynthesis genes<br>
<span>Score</span>: 18.57 out of 20.01
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r28c1-51707-ArgR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
HypR
<div class="collapser collapser-target-HypR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: L-hydroxyproline utilization repressor<br>
<span>Score</span>: 18.94 out of 22.40
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r28c1-12448-HypR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r28c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-10"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-10">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r28c1-domain-bubble-svg" data-filename="NZ_JACBXS010000012.1_r28c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r28c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (692 - 52741): neighbouring T1PKS-NRPS-like
</option>



<option value="CC2">
Candidate 2 (692 - 47046): single T1PKS
</option>



<option value="CC3">
Candidate 3 (8064 - 52741): single NRPS-like
</option>


</select>

<div class="details-svg" id="r28c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r28c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r28c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-12"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-12">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r28c1-pfam-details-svg" data-filename="NZ_JACBXS010000012.1_r28c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r28c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r28c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-13"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-13">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r28c1-tigrfam-details-svg" data-filename="NZ_JACBXS010000012.1_r28c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r28c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r28c1-nrps_pks_products" data-name="r28c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r28c1-nrps_pks_monomers" data-name="r28c1-nrps_pks_monomers">
NRPS/PKS monomers
</div>



<div class="sidepanel-details-header r28c1-tfbs-finder" data-name="r28c1-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r28c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-6"></div> <span class="help-tooltip" id="nrps-structures-help-6">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 692 - 52741:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r28c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r28c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 2, location 692 - 47046:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r28c1-2" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r28c1-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 3, location 8064 - 52741:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r28c1-3" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r28c1-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r28c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS monomer predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-7"></div> <span class="help-tooltip" id="nrps-monomers-help-7">Shows the predicted monomers for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">LGINICCL_00262</span></strong>: mal
<div class="collapser collapser-target-LGINICCL_00262 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (532..827)</strong>: mal
<div class="collapser collapser-target-LGINICCL_00262 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-LGINICCL_00262-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 62.5%</dd>
<dd>2-Rhyd-Malonyl-CoA: 58.3%</dd>
<dd>inactive: 54.2%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-LGINICCL_00262-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 84.3</dt>

<dd></dd><dt>inactive: 49.1</dt>

<dd></dd><dt>Methylmalonyl-CoA: 46.6</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 41.7</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 37.8</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 31.4</dt>

<dd></dd><dt>fatty_acid: 18.0</dt>

<dd></dd><dt>Propionyl-CoA: 15.6</dt>

<dd></dd><dt>Benzoyl-CoA: 15.5</dt>

<dd></dd><dt>Isobutyryl-CoA: 11.3</dt>

<dd></dd><dt>trans-1,2-CPDA: 9.2</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 8.5</dt>

<dd></dd><dt>CHC-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

<strong><span class="serif">LGINICCL_00264</span></strong>: X
<div class="collapser collapser-target-LGINICCL_00264 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>A-OX (563..1338)</strong>: X
<div class="collapser collapser-target-LGINICCL_00264 collapser-level-candidate"> </div><div class="collapser-content">

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r28c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-8"></div> <span class="help-tooltip" id="tfbs-side-help-8">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/HypR" target="_blank">HypR</a></td>
<td>L-hydroxyproline utilization repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ArgR" target="_blank">ArgR</a></td>
<td>Regulator of arginine biosynthesis genes</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>




<div class="page" id='r36c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_JACBXS010000001.1 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-61"></div> <span class="help-tooltip" id="region_svg-help-61">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_JACBXS010000001.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r36c1-svg" data-filename="NZ_JACBXS010000001.1_r36c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 398 - 10,906 nt. (total: 10,509 nt)

<a class="cluster-rules-header" id="r36c1-rules-header" href="#r36c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r36c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r36c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-62"></div> <span class="help-tooltip" id="focus-panel-help-62">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r36c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r36c1-gene-table-details" data-name="r36c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r36c1-MIBiG-cluster-compare" data-name="r36c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r36c1-clusterblast" data-name="r36c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r36c1-knownclusterblast" data-name="r36c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r36c1-subclusterblast" data-name="r36c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r36c1-tfbs-finder" data-name="r36c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r36c1-pfam-details" data-name="r36c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r36c1-tigrfam-details" data-name="r36c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r36c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-20"></div> <span class="help-tooltip" id="gt-help-tooltip-help-20">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r36c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r36c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r36c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r36c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r36c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-14"></div> <span class="help-tooltip" id="MIBiG-help-14">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r36c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r36c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r36c1"></div>
<div id="comparison-MIBiG-r36c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000904: 116-14341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000904/index.html#r1c1" target="_new">BGC0000904</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Other</td>

<td>FR-900098</td>


<td>Streptomyces rubellomurinus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001859: 6609-19144">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001859/index.html#r1c1" target="_new">BGC0001859</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>Other (Phosphonate)</td>

<td>fosfomycin</td>


<td>Pseudomonas syringae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002440: 0-50466">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002440/index.html#r1c1" target="_new">BGC0002440</a></td>





<td class="cc-heat-cell" title="0.18 (id:0.43, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>


<td>0.04</td>
<td>NRP</td>

<td>longicatenamide B, longicatenamide C, longicatenamide A, longicatenamide D</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000383: 0-54757">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000383/index.html#r1c1" target="_new">BGC0000383</a></td>





<td class="cc-heat-cell" title="0.17 (id:0.42, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>


<td>0.04</td>
<td>NRP, Polyketide</td>

<td>luminmycin A, glidobactin A, cepafungin</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000302: 744-33240">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000302/index.html#r1c1" target="_new">BGC0000302</a></td>





<td class="cc-heat-cell" title="0.17 (id:0.41, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>


<td>0.03</td>
<td>NRP</td>

<td>anabaenopeptin</td>


<td>Anabaena sp. 90</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000240: 44-69778">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000240/index.html#r1c1" target="_new">BGC0000240</a></td>





<td class="cc-heat-cell" title="0.15 (id:0.44, order:N/A, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.15), rgb(0, 0, 0, 0.15) 15%, white 15%)"></td>


<td>0.03</td>
<td>Polyketide, Saccharide</td>

<td>lomaiviticin A, lomaiviticin C, lomaiviticin D, lomaiviticin E</td>


<td>Salinispora pacifica</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r36c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000904: 116-14341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000904/index.html#r1c1" target="_new">BGC0000904</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Other</td>
<td>FR-900098</td>
<td>Streptomyces rubellomurinus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001859: 6609-19144">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001859/index.html#r1c1" target="_new">BGC0001859</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Other (Phosphonate)</td>
<td>fosfomycin</td>
<td>Pseudomonas syringae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002440: 0-50466">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002440/index.html#r1c1" target="_new">BGC0002440</a></td>


<td class="cc-heat-cell" title="0.18 (id:0.43, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>
<td>0.18</td>
<td>NRP</td>
<td>longicatenamide B, longicatenamide C, longicatenamide A, longicatenamide D</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000383: 0-54757">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000383/index.html#r1c1" target="_new">BGC0000383</a></td>


<td class="cc-heat-cell" title="0.17 (id:0.42, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>
<td>0.17</td>
<td>NRP, Polyketide</td>
<td>luminmycin A, glidobactin A, cepafungin</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000302: 744-33240">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000302/index.html#r1c1" target="_new">BGC0000302</a></td>


<td class="cc-heat-cell" title="0.17 (id:0.41, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>
<td>0.17</td>
<td>NRP</td>
<td>anabaenopeptin</td>
<td>Anabaena sp. 90</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000240: 44-69778">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000240/index.html#r1c1" target="_new">BGC0000240</a></td>


<td class="cc-heat-cell" title="0.15 (id:0.44, order:N/A, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.15), rgb(0, 0, 0, 0.15) 15%, white 15%)"></td>
<td>0.15</td>
<td>Polyketide, Saccharide</td>
<td>lomaiviticin A, lomaiviticin C, lomaiviticin D, lomaiviticin E</td>
<td>Salinispora pacifica</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r36c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-15"></div> <span class="help-tooltip" id="cb-general-help-15">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r36c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r36c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r36c1_1.svg">
NZ_MEHT01000009 (25734-53535): Roseinatronobacter thiooxidans strain ALG1 ALG... (21% of genes show similarity), ectoine,terpene
</option>

<option value = "svg/clusterblast_r36c1_2.svg">
NZ_AP014800 (1507906-1528668): Rhodovulum sulfidophilum DNA, complete genome,... (23% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r36c1_3.svg">
NZ_MSYR01000001 (842201-861459): Rhodovulum sulfidophilum strain AB30 Scaffol... (25% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r36c1_4.svg">
NZ_CP015421 (3154326-3175088): Rhodovulum sulfidophilum strain SNK001, comple... (23% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r36c1_5.svg">
NZ_JWLJ01000001 (1120971-1141748): Ruegeria sp. ANG-R contig 1, whole genome ... (22% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r36c1_6.svg">
NZ_CP022303 (2070891-2091668): Thalassococcus sp. S3 chromosome, complete genome (18% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r36c1_7.svg">
NZ_QOCF01000018 (163224-184007): Ruegeria sp. A3M17 A3M17 contig 18, whole ge... (19% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r36c1_8.svg">
NZ_CP031946 (956656-977433): Ruegeria sp. AD91A chromosome, complete genome (21% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r36c1_9.svg">
NZ_JAAOIE010000001 (249289-270063): Planktotalea sp. M26A2M contig1, whole ge... (22% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r36c1_10.svg">
NZ_VOPH01000001 (339018-359855): Pseudooceanicola sp. CAU 1508 contig1, whole... (19% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r36c1-download">Download graphic</button>
</div>
<div id="clusterblast-r36c1-svg"></div>

</div>
</div>

<div class="body-details-section r36c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-16"></div> <span class="help-tooltip" id="cb-known-help-16">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r36c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-17"></div> <span class="help-tooltip" id="cb-sub-help-17">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r36c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-19"></div> <span class="help-tooltip" id="tfbs-body-help-19">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (2):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
DasR
<div class="collapser collapser-target-DasR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: N-acetylglucosamine dependent repressor<br>
<span>Score</span>: 19.91 out of 27.04
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r36c1-10201-DasR"></div>

<div class="tfbs-hit-detail">
HypR
<div class="collapser collapser-target-HypR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: L-hydroxyproline utilization repressor<br>
<span>Score</span>: 18.94 out of 22.40
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r36c1-10328-HypR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r36c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-21"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-21">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r36c1-pfam-details-svg" data-filename="NZ_JACBXS010000001.1_r36c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r36c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r36c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-22"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-22">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r36c1-tigrfam-details-svg" data-filename="NZ_JACBXS010000001.1_r36c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r36c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r36c1-tfbs-finder" data-name="r36c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r36c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-18"></div> <span class="help-tooltip" id="tfbs-side-help-18">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/DasR" target="_blank">DasR</a></td>
<td>N-acetylglucosamine dependent repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/HypR" target="_blank">HypR</a></td>
<td>L-hydroxyproline utilization repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r36c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_JACBXS010000001.1 - Region 2 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-63"></div> <span class="help-tooltip" id="region_svg-help-63">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_JACBXS010000001.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r36c2-svg" data-filename="NZ_JACBXS010000001.1_r36c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 65,623 - 86,666 nt. (total: 21,044 nt)

<a class="cluster-rules-header" id="r36c2-rules-header" href="#r36c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r36c2-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r36c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-64"></div> <span class="help-tooltip" id="focus-panel-help-64">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r36c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r36c2-gene-table-details" data-name="r36c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r36c2-MIBiG-cluster-compare" data-name="r36c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r36c2-clusterblast" data-name="r36c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r36c2-knownclusterblast" data-name="r36c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r36c2-subclusterblast" data-name="r36c2-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r36c2-tfbs-finder" data-name="r36c2-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r36c2-pfam-details" data-name="r36c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r36c2-tigrfam-details" data-name="r36c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r36c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-29"></div> <span class="help-tooltip" id="gt-help-tooltip-help-29">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r36c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r36c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r36c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r36c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r36c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-23"></div> <span class="help-tooltip" id="MIBiG-help-23">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r36c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r36c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r36c2"></div>
<div id="comparison-MIBiG-r36c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647: 251-2872">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647</a></td>





<td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>


<td>0.57</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Rhodobacter sphaeroides</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000637: 540-9054">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000637/index.html#r1c1" target="_new">BGC0000637</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Corynebacterium glutamicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656: 169-3770">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.48, order:0.69, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Terpene</td>

<td>zeaxanthin</td>


<td>Xanthobacter autotrophicus Py2</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648: 1057-11443">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.44, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Myxococcus xanthus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646: 60-6380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.48, order:0.48, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Terpene</td>

<td>β-carotein</td>


<td>uncultured bacterium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002007: 584-20653">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002007/index.html#r1c1" target="_new">BGC0002007</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Terpene</td>

<td>atolypene A, atolypene B</td>


<td>Amycolatopsis tolypomycina</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630: 389-6051">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.48, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Terpene</td>

<td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>


<td>Paracoccus haeundaensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640: 2399-8466">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.47, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Enterobacteriaceae bacterium DC404</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645: 0-7475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:0.40, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.19</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Halobacillus halophilus DSM 2266</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664: 0-8391">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:0.45, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>isorenieratene</td>


<td>Streptomyces griseus subsp. griseus NBRC 13350</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r36c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647: 251-2872">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647</a></td>


<td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
<td>0.83</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Rhodobacter sphaeroides</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000637: 540-9054">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000637/index.html#r1c1" target="_new">BGC0000637</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Corynebacterium glutamicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656: 169-3770">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.48, order:0.69, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Terpene</td>
<td>zeaxanthin</td>
<td>Xanthobacter autotrophicus Py2</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648: 1057-11443">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.44, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Myxococcus xanthus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646: 60-6380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.48, order:0.48, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Terpene</td>
<td>β-carotein</td>
<td>uncultured bacterium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002007: 584-20653">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002007/index.html#r1c1" target="_new">BGC0002007</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Terpene</td>
<td>atolypene A, atolypene B</td>
<td>Amycolatopsis tolypomycina</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630: 389-6051">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.48, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Terpene</td>
<td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>
<td>Paracoccus haeundaensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640: 2399-8466">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.47, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Enterobacteriaceae bacterium DC404</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645: 0-7475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:0.40, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Halobacillus halophilus DSM 2266</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664: 0-8391">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:0.45, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>isorenieratene</td>
<td>Streptomyces griseus subsp. griseus NBRC 13350</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r36c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-24"></div> <span class="help-tooltip" id="cb-general-help-24">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r36c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r36c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r36c2_1.svg">
NZ_MEHT01000018 (61223-82261): Roseinatronobacter thiooxidans strain ALG1 ALG... (52% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r36c2_2.svg">
NZ_CP024899 (619066-640104): Rhodobaca barguzinensis strain alga05 chromosome... (52% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r36c2_3.svg">
NZ_JAAIKE010000001 (123967-144083): Rhodobacter sp. ETT8 contig1, whole genom... (55% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r36c2_4.svg">
NZ_SJZH01000001 (193776-214545): Tabrizicola sp. YIM 78059 Scaffold1, whole g... (52% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r36c2_5.svg">
NZ_JACIEJ010000015 (21148-42186): Sagittula marina strain DSM 102235 Ga037326... (55% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r36c2_6.svg">
NZ_SNYP01000010 (18408-39440): Rhodobacter veldkampii DSM 11550 Ga0310482 110... (61% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r36c2_7.svg">
NZ_CP028918 (1124240-1143571): Gemmobacter sp. HYN0069 chromosome, complete g... (52% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r36c2_8.svg">
NZ_PZZW01000002 (622436-643504): Rhodobacter johrii strain JA192 Ga0191650 10... (52% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r36c2_9.svg">
NC_011963 (1615872-1636940): Rhodobacter sphaeroides KD131 chromosome 1, comp... (52% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r36c2_10.svg">
NZ_FZOV01000005 (171484-192552): Rhodobacter megalophilus strain DSM 18937, w... (52% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r36c2-download">Download graphic</button>
</div>
<div id="clusterblast-r36c2-svg"></div>

</div>
</div>

<div class="body-details-section r36c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-25"></div> <span class="help-tooltip" id="cb-known-help-25">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r36c2-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r36c2_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r36c2_1.svg">
carotenoid
</option>

</select>
<button id="knownclusterblast-r36c2-download">Download graphic</button>
</div>
<div id="knownclusterblast-r36c2-svg"></div>

</div>
</div>

<div class="body-details-section r36c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-26"></div> <span class="help-tooltip" id="cb-sub-help-26">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r36c2-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-28"></div> <span class="help-tooltip" id="tfbs-body-help-28">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
NuR
<div class="collapser collapser-target-NuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Nickel-responsive regulator<br>
<span>Score</span>: 20.66 out of 30.98
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r36c2-65752-NuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r36c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-30"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-30">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r36c2-pfam-details-svg" data-filename="NZ_JACBXS010000001.1_r36c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r36c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r36c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-31"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-31">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r36c2-tigrfam-details-svg" data-filename="NZ_JACBXS010000001.1_r36c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r36c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r36c2-tfbs-finder" data-name="r36c2-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r36c2-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-27"></div> <span class="help-tooltip" id="tfbs-side-help-27">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NuR" target="_blank">NuR</a></td>
<td>Nickel-responsive regulator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r54c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_JACBXS010000036.1 - Region 1 - phosphonate <div class="help-container"> <div class="help-icon" data-id="region_svg-help-65"></div> <span class="help-tooltip" id="region_svg-help-65">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_JACBXS010000036.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r54c1-svg" data-filename="NZ_JACBXS010000036.1_r54c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,878 - 16,494 nt. (total: 13,617 nt)

<a class="cluster-rules-header" id="r54c1-rules-header" href="#r54c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r54c1-rules">phosphonate: (phosphonates and minimum(1, [2-Hacid_dh_C, Aldedh, Aminotran_1_2, Aminotran_5, AurF, CTP_transf_like, DUF4992, Fe-ADH, HMGL-like, Metallophos_2, NTP_transf_3, NTP_transf_5, PALP, PEP_mutase, TPP_enzyme_C, TPP_enzyme_N, phosphonates]) and not (phosphonatesHydrolase or phosphonatesMeIsocitrateLyase))</div>

<div class="region-svg-container">
<div id='r54c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-66"></div> <span class="help-tooltip" id="focus-panel-help-66">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r54c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r54c1-gene-table-details" data-name="r54c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r54c1-MIBiG-cluster-compare" data-name="r54c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r54c1-clusterblast" data-name="r54c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r54c1-knownclusterblast" data-name="r54c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r54c1-subclusterblast" data-name="r54c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r54c1-tfbs-finder" data-name="r54c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r54c1-pfam-details" data-name="r54c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r54c1-tigrfam-details" data-name="r54c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r54c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-38"></div> <span class="help-tooltip" id="gt-help-tooltip-help-38">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r54c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r54c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r54c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r54c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r54c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-32"></div> <span class="help-tooltip" id="MIBiG-help-32">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r54c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r54c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r54c1"></div>
<div id="comparison-MIBiG-r54c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">phosphonate</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000904: 116-14341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000904/index.html#r1c1" target="_new">BGC0000904</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Other</td>

<td>FR-900098</td>


<td>Streptomyces rubellomurinus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001859: 6609-19144">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001859/index.html#r1c1" target="_new">BGC0001859</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Other (Phosphonate)</td>

<td>fosfomycin</td>


<td>Pseudomonas syringae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000897: 0-20143">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000897/index.html#r1c1" target="_new">BGC0000897</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.45, order:0.48, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Other</td>

<td>dehydrophos</td>


<td>Streptomyces luridus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000937: 0-23647">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000937/index.html#r1c1" target="_new">BGC0000937</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.45, order:0.42, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Other</td>

<td>fosfazinomycin A, fosfazinomycin B</td>


<td>Streptomyces sp. WM6372</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001739: 0-14815">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001739/index.html#r1c1" target="_new">BGC0001739</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.47, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Other</td>

<td>phosphonoacetic Acid</td>


<td>Streptomyces sp. NRRL F-525</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002036: 0-15983">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002036/index.html#r1c1" target="_new">BGC0002036</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.47, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.08</td>
<td>Other</td>

<td>dehydrofosmidomycin</td>


<td>Streptomyces lavendulae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000926: 1-12724">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000926/index.html#r1c1" target="_new">BGC0000926</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.45, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>Other</td>

<td>rhizocticin A</td>


<td>Bacillus subtilis subsp. spizizenii ATCC 6633</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000938: 104-31263">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000938/index.html#r1c1" target="_new">BGC0000938</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.50, order:0.48, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>Other (Aminocoumarin)</td>

<td>fosfomycin</td>


<td>Streptomyces fradiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000406: 116-37823">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000406/index.html#r1c1" target="_new">BGC0000406</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.46, order:0.39, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.07</td>
<td>NRP</td>

<td>phosphinothricintripeptide</td>


<td>Streptomyces viridochromogenes</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000807: 0-36089">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000807/index.html#r1c1" target="_new">BGC0000807</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.50, order:0.48, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.07</td>
<td>Saccharide</td>

<td>phosphonoglycans</td>


<td>Stackebrandtia nassauensis DSM 44728</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r54c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000904: 116-14341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000904/index.html#r1c1" target="_new">BGC0000904</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Other</td>
<td>FR-900098</td>
<td>Streptomyces rubellomurinus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001859: 6609-19144">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001859/index.html#r1c1" target="_new">BGC0001859</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Other (Phosphonate)</td>
<td>fosfomycin</td>
<td>Pseudomonas syringae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000897: 0-20143">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000897/index.html#r1c1" target="_new">BGC0000897</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.45, order:0.48, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Other</td>
<td>dehydrophos</td>
<td>Streptomyces luridus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000937: 0-23647">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000937/index.html#r1c1" target="_new">BGC0000937</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.45, order:0.42, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Other</td>
<td>fosfazinomycin A, fosfazinomycin B</td>
<td>Streptomyces sp. WM6372</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001739: 0-14815">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001739/index.html#r1c1" target="_new">BGC0001739</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.47, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>Other</td>
<td>phosphonoacetic Acid</td>
<td>Streptomyces sp. NRRL F-525</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002036: 0-15983">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002036/index.html#r1c1" target="_new">BGC0002036</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.47, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>Other</td>
<td>dehydrofosmidomycin</td>
<td>Streptomyces lavendulae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000926: 1-12724">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000926/index.html#r1c1" target="_new">BGC0000926</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.45, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>Other</td>
<td>rhizocticin A</td>
<td>Bacillus subtilis subsp. spizizenii ATCC 6633</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000938: 104-31263">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000938/index.html#r1c1" target="_new">BGC0000938</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.50, order:0.48, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>Other (Aminocoumarin)</td>
<td>fosfomycin</td>
<td>Streptomyces fradiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000406: 116-37823">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000406/index.html#r1c1" target="_new">BGC0000406</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.46, order:0.39, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>NRP</td>
<td>phosphinothricintripeptide</td>
<td>Streptomyces viridochromogenes</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000807: 0-36089">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000807/index.html#r1c1" target="_new">BGC0000807</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.50, order:0.48, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>Saccharide</td>
<td>phosphonoglycans</td>
<td>Stackebrandtia nassauensis DSM 44728</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r54c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-33"></div> <span class="help-tooltip" id="cb-general-help-33">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r54c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r54c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r54c1_1.svg">
NZ_VLLL01000006 (917797-959171): Stackebrandtia albiflava strain DSM 45044 LX... (10% of genes show similarity), other
</option>

<option value = "svg/clusterblast_r54c1_2.svg">
NZ_KQ033901 (249939-291253): Nitriliruptor alkaliphilus DSM 45188 NitalDRAFT ... (9% of genes show similarity), other
</option>

<option value = "svg/clusterblast_r54c1_3.svg">
NZ_AKXV01000009 (400591-474257): Streptomyces sp. SS scaffold4 1, whole genom... (4% of genes show similarity), NRPS,other
</option>

<option value = "svg/clusterblast_r54c1_4.svg">
NC_013947 (6001088-6042372): Stackebrandtia nassauensis DSM 44728, complete s... (8% of genes show similarity), other
</option>

<option value = "svg/clusterblast_r54c1_5.svg">
NZ_CP021748 (4803101-4863960): Streptomyces alboflavus strain MDJK44 chromoso... (6% of genes show similarity), NRPS,butyrolactone,phosphonate
</option>

<option value = "svg/clusterblast_r54c1_6.svg">
NZ_KB913022 (3359730-3400621): Salinispora pacifica DSM 45543 = CNS-863 strai... (7% of genes show similarity), phosphonate
</option>

<option value = "svg/clusterblast_r54c1_7.svg">
NZ_RHGC01000005 (392372-453506): Streptomyces alboflavus strain 313 contig005... (6% of genes show similarity), NRPS,butyrolactone,phosphonate
</option>

<option value = "svg/clusterblast_r54c1_8.svg">
NZ_CP023695 (674759-797403): Streptomyces alboniger strain ATCC 12461 chromos... (4% of genes show similarity), NRPS,phosphonate,terpene
</option>

<option value = "svg/clusterblast_r54c1_9.svg">
NZ_KV411304 (1571301-1641368): Nocardia terpenica strain IFM 0406 scaffold2, ... (4% of genes show similarity), LAP,TfuA-related,lassopeptide,phosphonate,thiopeptide
</option>

<option value = "svg/clusterblast_r54c1_10.svg">
NZ_CP023778 (4324588-4395277): Nocardia terpenica strain NC YFY NT001 chromos... (4% of genes show similarity), LAP,TfuA-related,phosphonate,terpene,thiopeptide
</option>

</select>
<button id= "clusterblast-r54c1-download">Download graphic</button>
</div>
<div id="clusterblast-r54c1-svg"></div>

</div>
</div>

<div class="body-details-section r54c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-34"></div> <span class="help-tooltip" id="cb-known-help-34">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r54c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r54c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r54c1_1.svg">
phosphonoglycans
</option>

<option value = "svg/knownclusterblast_r54c1_2.svg">
fosfomycin
</option>

<option value = "svg/knownclusterblast_r54c1_3.svg">
phosphonoglycans
</option>

<option value = "svg/knownclusterblast_r54c1_4.svg">
polysaccharide B
</option>

</select>
<button id="knownclusterblast-r54c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r54c1-svg"></div>

</div>
</div>

<div class="body-details-section r54c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-35"></div> <span class="help-tooltip" id="cb-sub-help-35">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r54c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-37"></div> <span class="help-tooltip" id="tfbs-body-help-37">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AfsR
<div class="collapser collapser-target-AfsR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Pleiotropic regulatory for antibiotic production<br>
<span>Score</span>: 21.40 out of 23.05
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r54c1-5599-AfsR"></div>

<div class="tfbs-hit-detail">
CatR
<div class="collapser collapser-target-CatR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: H2O2-responsive repressor<br>
<span>Score</span>: 18.36 out of 25.31
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r54c1-15085-CatR"></div>

</div>


</div>
</div>



<div class="body-details-section r54c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-39"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-39">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r54c1-pfam-details-svg" data-filename="NZ_JACBXS010000036.1_r54c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r54c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r54c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-40"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-40">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r54c1-tigrfam-details-svg" data-filename="NZ_JACBXS010000036.1_r54c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r54c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r54c1-tfbs-finder" data-name="r54c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r54c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-36"></div> <span class="help-tooltip" id="tfbs-side-help-36">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AfsR" target="_blank">AfsR</a></td>
<td>Pleiotropic regulatory for antibiotic production</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/CatR" target="_blank">CatR</a></td>
<td>H2O2-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r58c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_JACBXS010000003.1 - Region 1 - hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-67"></div> <span class="help-tooltip" id="region_svg-help-67">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_JACBXS010000003.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r58c1-svg" data-filename="NZ_JACBXS010000003.1_r58c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,587 - 25,456 nt. (total: 20,870 nt)

<a class="cluster-rules-header" id="r58c1-rules-header" href="#r58c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r58c1-rules">hserlactone: Autoind_synth</div>

<div class="region-svg-container">
<div id='r58c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-68"></div> <span class="help-tooltip" id="focus-panel-help-68">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r58c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r58c1-gene-table-details" data-name="r58c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r58c1-MIBiG-cluster-compare" data-name="r58c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r58c1-clusterblast" data-name="r58c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r58c1-knownclusterblast" data-name="r58c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r58c1-subclusterblast" data-name="r58c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r58c1-tfbs-finder" data-name="r58c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r58c1-pfam-details" data-name="r58c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r58c1-tigrfam-details" data-name="r58c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r58c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-47"></div> <span class="help-tooltip" id="gt-help-tooltip-help-47">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r58c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r58c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r58c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r58c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r58c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-41"></div> <span class="help-tooltip" id="MIBiG-help-41">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r58c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r58c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r58c1"></div>
<div id="comparison-MIBiG-r58c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">hserlactone</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002711: 0-9402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002711/index.html#r1c1" target="_new">BGC0002711</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.46, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>nostovalerolactone, 9-dehydronostovalerolactone</td>


<td>Nostoc punctiforme PCC 73102</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001897: 3-16597">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001897/index.html#r1c1" target="_new">BGC0001897</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>cepacin A</td>


<td>Burkholderia ambifaria IOP40-10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778: 0-16475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.41, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Other</td>

<td>showdomycin</td>


<td>Streptomyces showdoensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002510: 0-11781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002510/index.html#r1c1" target="_new">BGC0002510</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>youssoufene A1, youssoufene B1, youssoufene B2, youssoufene B3, youssoufene B4</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000196: 953-9991">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000196/index.html#r1c1" target="_new">BGC0000196</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>AQ-256, AQ-270a, AQ-284a</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000204: 4242-15189">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000204/index.html#r1c1" target="_new">BGC0000204</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>benastatin A, benastatin B, BE-43767, TAN1532B</td>


<td>Streptomyces sp. A2991200</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197: 0-12679">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>frankiamicin</td>


<td>Frankia sp. EAN1pec</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000243: 338-14804">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000243/index.html#r1c1" target="_new">BGC0000243</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.42, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Polyketide</td>

<td>nonactin, monactin, dinactin, trinactin, tetranactin</td>


<td>Streptomyces griseus subsp. griseus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001343: 61-19132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001343/index.html#r1c1" target="_new">BGC0001343</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.51)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Polyketide</td>

<td>aurachin A, aurachin D</td>


<td>Stigmatella aurantiaca Sg a15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000044: 0-21257">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000044/index.html#r1c1" target="_new">BGC0000044</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Polyketide</td>

<td>dawenol</td>


<td>Stigmatella aurantiaca DW4/3-1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r58c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002711: 0-9402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002711/index.html#r1c1" target="_new">BGC0002711</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.46, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>nostovalerolactone, 9-dehydronostovalerolactone</td>
<td>Nostoc punctiforme PCC 73102</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001897: 3-16597">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001897/index.html#r1c1" target="_new">BGC0001897</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Polyketide</td>
<td>cepacin A</td>
<td>Burkholderia ambifaria IOP40-10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778: 0-16475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.41, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Other</td>
<td>showdomycin</td>
<td>Streptomyces showdoensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002510: 0-11781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002510/index.html#r1c1" target="_new">BGC0002510</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Polyketide</td>
<td>youssoufene A1, youssoufene B1, youssoufene B2, youssoufene B3, youssoufene B4</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000196: 953-9991">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000196/index.html#r1c1" target="_new">BGC0000196</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Polyketide</td>
<td>AQ-256, AQ-270a, AQ-284a</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000204: 4242-15189">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000204/index.html#r1c1" target="_new">BGC0000204</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Polyketide</td>
<td>benastatin A, benastatin B, BE-43767, TAN1532B</td>
<td>Streptomyces sp. A2991200</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197: 0-12679">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Polyketide</td>
<td>frankiamicin</td>
<td>Frankia sp. EAN1pec</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000243: 338-14804">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000243/index.html#r1c1" target="_new">BGC0000243</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.42, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Polyketide</td>
<td>nonactin, monactin, dinactin, trinactin, tetranactin</td>
<td>Streptomyces griseus subsp. griseus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001343: 61-19132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001343/index.html#r1c1" target="_new">BGC0001343</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.51)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Polyketide</td>
<td>aurachin A, aurachin D</td>
<td>Stigmatella aurantiaca Sg a15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000044: 0-21257">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000044/index.html#r1c1" target="_new">BGC0000044</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Polyketide</td>
<td>dawenol</td>
<td>Stigmatella aurantiaca DW4/3-1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r58c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-42"></div> <span class="help-tooltip" id="cb-general-help-42">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r58c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r58c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r58c1_1.svg">
NZ_JACIBX010000003 (84986-129126): Limimaricola variabilis strain CECT 8572 G... (12% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r58c1_2.svg">
NZ_QQBB01000001 (471073-491727): Microvirga subterranea strain DSM 14364 Ga02... (17% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r58c1_3.svg">
NZ_FOEP01000010 (38667-59543): Litorimicrobium taeanense strain DSM 22007, wh... (8% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r58c1_4.svg">
NZ_CP051181 (673563-692442): Thalassobius gelatinovorus strain NEB572 chromos... (9% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r58c1_5.svg">
NZ_CYSB01000025 (57022-77904): Thalassobius autumnalis strain CECT 5118, whol... (11% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r58c1_6.svg">
NZ_KE557280 (17213-38074): Salipiger mucosus DSM 16094 scaffold8, whole genom... (10% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r58c1_7.svg">
NZ_DS995276 (3218276-3239056): Rhodobacteraceae bacterium HTCC2083 scf 110620... (8% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r58c1_8.svg">
NZ_CP019044 (1036598-1057225): Salaquimonas pukyongi strain RR3-28 chromosome... (10% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r58c1_9.svg">
NZ_CP024942 (767105-787699): Paraburkholderia terricola strain mHS1 chromosom... (8% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r58c1_10.svg">
NZ_MWJQ01000001 (2180494-2201088): Burkholderia sp. A27 scaffold1, whole geno... (8% of genes show similarity), hserlactone
</option>

</select>
<button id= "clusterblast-r58c1-download">Download graphic</button>
</div>
<div id="clusterblast-r58c1-svg"></div>

</div>
</div>

<div class="body-details-section r58c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-43"></div> <span class="help-tooltip" id="cb-known-help-43">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r58c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r58c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r58c1_1.svg">
TVA-YJ-2
</option>

</select>
<button id="knownclusterblast-r58c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r58c1-svg"></div>

</div>
</div>

<div class="body-details-section r58c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-44"></div> <span class="help-tooltip" id="cb-sub-help-44">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r58c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-46"></div> <span class="help-tooltip" id="tfbs-body-help-46">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (3):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 18.67 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r58c1-5341-ZuR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 19.49 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r58c1-5347-ZuR"></div>

<div class="tfbs-hit-detail">
LexA
<div class="collapser collapser-target-LexA collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Repressor of DNA damage response<br>
<span>Score</span>: 21.44 out of 29.65
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r58c1-13303-LexA"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r58c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-48"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-48">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r58c1-pfam-details-svg" data-filename="NZ_JACBXS010000003.1_r58c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r58c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r58c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-49"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-49">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r58c1-tigrfam-details-svg" data-filename="NZ_JACBXS010000003.1_r58c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r58c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r58c1-tfbs-finder" data-name="r58c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r58c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-45"></div> <span class="help-tooltip" id="tfbs-side-help-45">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/LexA" target="_blank">LexA</a></td>
<td>Repressor of DNA damage response</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r69c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_JACBXS010000004.1 - Region 1 - ectoine <div class="help-container"> <div class="help-icon" data-id="region_svg-help-69"></div> <span class="help-tooltip" id="region_svg-help-69">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_JACBXS010000004.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r69c1-svg" data-filename="NZ_JACBXS010000004.1_r69c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 111,141 - 121,524 nt. (total: 10,384 nt)

<a class="cluster-rules-header" id="r69c1-rules-header" href="#r69c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r69c1-rules">ectoine: ectoine_synt</div>

<div class="region-svg-container">
<div id='r69c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-70"></div> <span class="help-tooltip" id="focus-panel-help-70">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r69c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r69c1-gene-table-details" data-name="r69c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r69c1-MIBiG-cluster-compare" data-name="r69c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r69c1-clusterblast" data-name="r69c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r69c1-knownclusterblast" data-name="r69c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r69c1-subclusterblast" data-name="r69c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r69c1-tfbs-finder" data-name="r69c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r69c1-pfam-details" data-name="r69c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r69c1-tigrfam-details" data-name="r69c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r69c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-56"></div> <span class="help-tooltip" id="gt-help-tooltip-help-56">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r69c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r69c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r69c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r69c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r69c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-50"></div> <span class="help-tooltip" id="MIBiG-help-50">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r69c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r69c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r69c1"></div>
<div id="comparison-MIBiG-r69c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">ectoine</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860: 710-5134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860</a></td>





<td class="cc-heat-cell" title="0.81 (id:0.54, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>


<td>0.55</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylarcula marina</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856: 517-4347">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856</a></td>





<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>


<td>0.51</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylophaga thalassica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859: 258-3305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859</a></td>





<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>


<td>0.51</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium alcaliphilum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857: 300-4871">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857</a></td>





<td class="cc-heat-cell" title="0.79 (id:0.49, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>


<td>0.51</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylophaga alcalica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855: 245-3455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855</a></td>





<td class="cc-heat-cell" title="0.73 (id:0.51, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>


<td>0.41</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium kenyense</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854: 838-6879">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.49, order:0.83, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium alcaliphilum 20Z</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.49, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>Other</td>

<td>ectoine</td>


<td>Sporosarcina pasteurii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052: 188-3620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.49, order:0.76, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Other (Ectoine)</td>

<td>ectoine</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853: 8-3345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.48, order:0.76, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Other</td>

<td>ectoine</td>


<td>Streptomyces anulatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.47, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylobacter marinus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r69c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860: 710-5134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860</a></td>


<td class="cc-heat-cell" title="0.81 (id:0.54, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>
<td>0.81</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylarcula marina</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856: 517-4347">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856</a></td>


<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
<td>0.79</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylophaga thalassica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859: 258-3305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859</a></td>


<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
<td>0.79</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium alcaliphilum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857: 300-4871">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857</a></td>


<td class="cc-heat-cell" title="0.79 (id:0.49, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
<td>0.79</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylophaga alcalica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855: 245-3455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855</a></td>


<td class="cc-heat-cell" title="0.73 (id:0.51, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
<td>0.73</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium kenyense</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854: 838-6879">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.49, order:0.83, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium alcaliphilum 20Z</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.49, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Other</td>
<td>ectoine</td>
<td>Sporosarcina pasteurii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052: 188-3620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.49, order:0.76, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other (Ectoine)</td>
<td>ectoine</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853: 8-3345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.48, order:0.76, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other</td>
<td>ectoine</td>
<td>Streptomyces anulatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.47, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylobacter marinus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r69c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-51"></div> <span class="help-tooltip" id="cb-general-help-51">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r69c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r69c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r69c1_1.svg">
NZ_PHNT01000004 (68118-78511): Roseovarius salinarum strain N53 contig4, whol... (60% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r69c1_2.svg">
NZ_CP024899 (2098634-2109021): Rhodobaca barguzinensis strain alga05 chromoso... (50% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r69c1_3.svg">
NZ_MEHT01000009 (25734-53535): Roseinatronobacter thiooxidans strain ALG1 ALG... (17% of genes show similarity), ectoine,terpene
</option>

<option value = "svg/clusterblast_r69c1_4.svg">
NZ_LXYH01000056 (3698-14082): Rhodobacteraceae bacterium EhC02 A5 Contig6, wh... (50% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r69c1_5.svg">
NZ_CP049056 (3924672-3935059): Rhodobacteraceae bacterium RR4-56 chromosome, ... (71% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r69c1_6.svg">
NZ_FOQH01000008 (123921-134320): Albimonas pacifica strain CGMCC 1.11030, who... (62% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r69c1_7.svg">
NZ_SLXP01000018 (18835-29231): Rhodovulum marinum strain DSM 18063 Ga0310484 ... (55% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r69c1_8.svg">
NZ_FNIO01000001 (1183716-1194121): Lutimaribacter pacificus strain CGMCC 1.10... (62% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r69c1_9.svg">
NZ_SLWW01000004 (35317-45707): Rhodovulum euryhalinum strain DSM 4868 Ga03105... (55% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r69c1_10.svg">
NZ_SLXL01000012 (45020-55416): Rhodovulum adriaticum strain DSM 2781 Ga031049... (55% of genes show similarity), ectoine
</option>

</select>
<button id= "clusterblast-r69c1-download">Download graphic</button>
</div>
<div id="clusterblast-r69c1-svg"></div>

</div>
</div>

<div class="body-details-section r69c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-52"></div> <span class="help-tooltip" id="cb-known-help-52">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r69c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r69c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r69c1_1.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r69c1_2.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r69c1_3.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r69c1_4.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r69c1_5.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r69c1_6.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r69c1_7.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r69c1_8.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r69c1_9.svg">
ectoine
</option>

</select>
<button id="knownclusterblast-r69c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r69c1-svg"></div>

</div>
</div>

<div class="body-details-section r69c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-53"></div> <span class="help-tooltip" id="cb-sub-help-53">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r69c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-55"></div> <span class="help-tooltip" id="tfbs-body-help-55">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AfsQ1
<div class="collapser collapser-target-AfsQ1 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Two-component system AfsQ1-Q2, activator of antibiotic production<br>
<span>Score</span>: 19.52 out of 19.52
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r69c1-121098-AfsQ1"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
NuR
<div class="collapser collapser-target-NuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Nickel-responsive regulator<br>
<span>Score</span>: 23.00 out of 30.98
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r69c1-111165-NuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r69c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-57"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-57">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r69c1-pfam-details-svg" data-filename="NZ_JACBXS010000004.1_r69c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r69c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r69c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-58"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-58">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r69c1-tigrfam-details-svg" data-filename="NZ_JACBXS010000004.1_r69c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r69c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r69c1-tfbs-finder" data-name="r69c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r69c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-54"></div> <span class="help-tooltip" id="tfbs-side-help-54">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NuR" target="_blank">NuR</a></td>
<td>Nickel-responsive regulator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AfsQ1" target="_blank">AfsQ1</a></td>
<td>Two-component system AfsQ1-Q2, activator of antibiotic production</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>