
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>GCF_900100045.1_Paracoccus_denitrificans_DSM_413 - 9 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/bacteria.css">
<meta property="og:title" content="GCF_900100045.1_Paracoccus_denitrificans_DSM_413 - 8 record(s) - 9 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-04193cba-accd-4b76-9ad1-8ed151866555/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://antismash.secondarymetabolites.org/">
<img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://antismash.secondarymetabolites.org/">
antiSMASH version 7.0.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="GCF_900100045.1_Paracoccus_denitrificans_DSM_413.zip">Download all results</a></li>

<li><a href="GCF_900100045.1_Paracoccus_denitrificans_DSM_413.gbk">Download GenBank summary file</a></li>

<li><a href="bacteria-04193cba-accd-4b76-9ad1-8ed151866555.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton RiPP thioamitides r37c1">
<a href="#r37c1">37.1</a>
</div>



<div class="regbutton other hserlactone r42c1">
<a href="#r42c1">42.1</a>
</div>

<div class="regbutton hybrid r42c2">
<a href="#r42c2">42.2</a>
</div>



<div class="regbutton RiPP r44c1">
<a href="#r44c1">44.1</a>
</div>



<div class="regbutton RiPP RiPP-like r55c1">
<a href="#r55c1">55.1</a>
</div>



<div class="regbutton NRPS NRPS r57c1">
<a href="#r57c1">57.1</a>
</div>



<div class="regbutton RiPP ranthipeptide r78c1">
<a href="#r78c1">78.1</a>
</div>



<div class="regbutton hybrid r93c1">
<a href="#r93c1">93.1</a>
</div>



<div class="regbutton NRPS r94c1">
<a href="#r94c1">94.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>NZ_FNEA01000020.1</strong> (Paracoccus denitrificans)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-37" data-filename="NZ_FNEA01000020.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-37" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r37c1">
<td class="regbutton RiPP thioamitides r37c1">
<a href="#r37c1">Region&nbsp37.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#thioamitides" target="_blank">thioamitides</a>
</td>
<td class="digits">68,352</td>
<td class="digits table-split-left">90,834</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_FNEA01000014.1</strong> (Paracoccus denitrificans)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-42" data-filename="NZ_FNEA01000014.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-42" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r42c1">
<td class="regbutton other hserlactone r42c1">
<a href="#r42c1">Region&nbsp42.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">7,957</td>
<td class="digits table-split-left">28,547</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r42c2">
<td class="regbutton hybrid r42c2">
<a href="#r42c2">Region&nbsp42.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">51,227</td>
<td class="digits table-split-left">103,125</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_FNEA01000012.1</strong> (Paracoccus denitrificans)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-44" data-filename="NZ_FNEA01000012.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-44" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r44c1">
<td class="regbutton RiPP r44c1">
<a href="#r44c1">Region&nbsp44.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#thiopeptide" target="_blank">thiopeptide</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lap" target="_blank">LAP</a>
</td>
<td class="digits">8,571</td>
<td class="digits table-split-left">38,669</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000802/1" target="_blank">succinoglycan</a></td>
<td>Saccharide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 33%, #ffffff00 33%)">33%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_FNEA01000006.1</strong> (Paracoccus denitrificans)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-55" data-filename="NZ_FNEA01000006.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-55" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r55c1">
<td class="regbutton RiPP RiPP-like r55c1">
<a href="#r55c1">Region&nbsp55.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
</td>
<td class="digits">177,548</td>
<td class="digits table-split-left">188,345</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_FNEA01000007.1</strong> (Paracoccus denitrificans)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-57" data-filename="NZ_FNEA01000007.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-57" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r57c1">
<td class="regbutton NRPS NRPS r57c1">
<a href="#r57c1">Region&nbsp57.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">29,310</td>
<td class="digits table-split-left">73,167</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002679/1" target="_blank">chromobactin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 20%, #ffffff00 20%)">20%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_FNEA01000030.1</strong> (Paracoccus denitrificans)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-78" data-filename="NZ_FNEA01000030.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-78" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r78c1">
<td class="regbutton RiPP ranthipeptide r78c1">
<a href="#r78c1">Region&nbsp78.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ranthipeptide" target="_blank">ranthipeptide</a>
</td>
<td class="digits">3,840</td>
<td class="digits table-split-left">25,342</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000413/1" target="_blank">Pf-5 pyoverdine</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 1%, #ffffff00 1%)">1%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_FNEA01000005.1</strong> (Paracoccus denitrificans)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-93" data-filename="NZ_FNEA01000005.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-93" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r93c1">
<td class="regbutton hybrid r93c1">
<a href="#r93c1">Region&nbsp93.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#redox-cofactor" target="_blank">redox-cofactor</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrp-metallophore" target="_blank">NRP-metallophore</a>
</td>
<td class="digits">195,863</td>
<td class="digits table-split-left">242,048</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002493/1" target="_blank">parabactin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_FNEA01000013.1</strong> (Paracoccus denitrificans)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-94" data-filename="NZ_FNEA01000013.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-94" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r94c1">
<td class="regbutton NRPS r94c1">
<a href="#r94c1">Region&nbsp94.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrp-metallophore" target="_blank">NRP-metallophore</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">2,515</td>
<td class="digits table-split-left">53,620</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001055/1" target="_blank">yersiniabactin</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 2%, #ffffff00 2%)">2%</td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r37c1">
<td class="regbutton RiPP thioamitides r37c1">
<a href="#r37c1">Region&nbsp37.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#thioamitides" target="_blank">thioamitides</a>
</td>
<td class="digits">68,352</td>
<td class="digits table-split-left">90,834</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r42c1">
<td class="regbutton other hserlactone r42c1">
<a href="#r42c1">Region&nbsp42.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">7,957</td>
<td class="digits table-split-left">28,547</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r42c2">
<td class="regbutton hybrid r42c2">
<a href="#r42c2">Region&nbsp42.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">51,227</td>
<td class="digits table-split-left">103,125</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r44c1">
<td class="regbutton RiPP r44c1">
<a href="#r44c1">Region&nbsp44.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#thiopeptide" target="_blank">thiopeptide</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lap" target="_blank">LAP</a>
</td>
<td class="digits">8,571</td>
<td class="digits table-split-left">38,669</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000802/1" target="_blank">succinoglycan</a></td>
<td>Saccharide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 33%, #ffffff00 33%)">33%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r55c1">
<td class="regbutton RiPP RiPP-like r55c1">
<a href="#r55c1">Region&nbsp55.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
</td>
<td class="digits">177,548</td>
<td class="digits table-split-left">188,345</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r57c1">
<td class="regbutton NRPS NRPS r57c1">
<a href="#r57c1">Region&nbsp57.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">29,310</td>
<td class="digits table-split-left">73,167</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002679/1" target="_blank">chromobactin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 20%, #ffffff00 20%)">20%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r78c1">
<td class="regbutton RiPP ranthipeptide r78c1">
<a href="#r78c1">Region&nbsp78.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ranthipeptide" target="_blank">ranthipeptide</a>
</td>
<td class="digits">3,840</td>
<td class="digits table-split-left">25,342</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000413/1" target="_blank">Pf-5 pyoverdine</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 1%, #ffffff00 1%)">1%</td>

</tr>


<tr class="linked-row even" data-anchor="#r93c1">
<td class="regbutton hybrid r93c1">
<a href="#r93c1">Region&nbsp93.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#redox-cofactor" target="_blank">redox-cofactor</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrp-metallophore" target="_blank">NRP-metallophore</a>
</td>
<td class="digits">195,863</td>
<td class="digits table-split-left">242,048</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002493/1" target="_blank">parabactin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r94c1">
<td class="regbutton NRPS r94c1">
<a href="#r94c1">Region&nbsp94.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrp-metallophore" target="_blank">NRP-metallophore</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">2,515</td>
<td class="digits table-split-left">53,620</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001055/1" target="_blank">yersiniabactin</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 2%, #ffffff00 2%)">2%</td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>NZ_FNEA01000094.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000091.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000088.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000085.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000084.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000089.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000078.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000076.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000072.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000079.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000071.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000069.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000093.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000067.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000065.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000064.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000092.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000077.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000063.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000062.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000068.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000060.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000074.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000059.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000058.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000056.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000083.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000055.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000070.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000053.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000087.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000051.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000050.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000049.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000047.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000022.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000057.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000042.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000033.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000017.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000054.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000021.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000095.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000073.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000052.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000011.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000004.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000037.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000086.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000044.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000003.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000016.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000008.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000024.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000018.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000009.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000019.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000032.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000036.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000090.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000015.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000023.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000025.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000082.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000002.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000026.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000027.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000031.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000080.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000029.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000081.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000043.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000075.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000010.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000034.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000048.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000035.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000040.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000038.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000028.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000039.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000066.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000041.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000001.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000045.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000061.1</strong> (Paracoccus denitrificans)</dd>

<dd><strong>NZ_FNEA01000046.1</strong> (Paracoccus denitrificans)</dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>







<div class="page" id='r37c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_FNEA01000020.1 - Region 1 - thioamitides <div class="help-container"> <div class="help-icon" data-id="region_svg-help-98"></div> <span class="help-tooltip" id="region_svg-help-98">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_FNEA01000020.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r37c1-svg" data-filename="NZ_FNEA01000020.1_r37c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 68,352 - 90,834 nt. (total: 22,483 nt)

<a class="cluster-rules-header" id="r37c1-rules-header" href="#r37c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r37c1-rules">thioamitides: ((YcaO or TIGR03604) and TfuA)</div>

<div class="region-svg-container">
<div id='r37c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-99"></div> <span class="help-tooltip" id="focus-panel-help-99">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r37c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r37c1-gene-table-details" data-name="r37c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r37c1-MIBiG-cluster-compare" data-name="r37c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r37c1-clusterblast" data-name="r37c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r37c1-knownclusterblast" data-name="r37c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r37c1-subclusterblast" data-name="r37c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r37c1-tfbs-finder" data-name="r37c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r37c1-pfam-details" data-name="r37c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r37c1-tigrfam-details" data-name="r37c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r37c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-7"></div> <span class="help-tooltip" id="gt-help-tooltip-help-7">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r37c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r37c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r37c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r37c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r37c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-1"></div> <span class="help-tooltip" id="MIBiG-help-1">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r37c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r37c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r37c1"></div>
<div id="comparison-MIBiG-r37c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">thioamitides</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001849: 0-6950">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001849/index.html#r1c1" target="_new">BGC0001849</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.46, order:0.48, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>RiPP</td>

<td>thiovarsolin A, thiovarsolin B, thiovarsolin C, thiovarsolin D</td>


<td>Streptomyces varsoviensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002584: 0-13427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002584/index.html#r1c1" target="_new">BGC0002584</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>RiPP</td>

<td>azolemycin B, azolemycin A, azolemycin D, azolemycin C</td>


<td>Streptomyces sp. FXJ1.264</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002013: 0-11564">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002013/index.html#r1c1" target="_new">BGC0002013</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>RiPP</td>

<td>curacozole</td>


<td>Streptomyces curacoi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002076: 6-3180">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002076/index.html#r1c1" target="_new">BGC0002076</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Other</td>

<td>4,5-didehydroacaterin, acaterin</td>


<td>Pseudomonas jessenii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002560: 0-11740">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002560/index.html#r1c1" target="_new">BGC0002560</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.45, order:0.41, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>thiostreptamide S4</td>


<td>Streptomyces sp. NRRL S-4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001174: 248-6052">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001174/index.html#r1c1" target="_new">BGC0001174</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>RiPP</td>

<td>corynazolicin</td>


<td>Corynebacterium urealyticum DSM 7111</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000625: 1-15454">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000625/index.html#r1c1" target="_new">BGC0000625</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.46, order:0.38, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>RiPP</td>

<td>thioviridamide</td>


<td>Streptomyces olivoviridis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001802: 854-10845">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001802/index.html#r1c1" target="_new">BGC0001802</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.46, order:0.40, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>RiPP</td>

<td>thioholgamide A, thioholgamide B</td>


<td>Streptomyces malaysiense</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001696: 0-9985">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001696/index.html#r1c1" target="_new">BGC0001696</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.46, order:0.40, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>RiPP</td>

<td>neothioviridamide</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000483: 1302-13724">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000483/index.html#r1c1" target="_new">BGC0000483</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.41, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>aeruginosamide B, aeruginosamide C</td>


<td>Microcystis aeruginosa PCC 9432</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r37c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001849: 0-6950">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001849/index.html#r1c1" target="_new">BGC0001849</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.46, order:0.48, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>RiPP</td>
<td>thiovarsolin A, thiovarsolin B, thiovarsolin C, thiovarsolin D</td>
<td>Streptomyces varsoviensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002584: 0-13427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002584/index.html#r1c1" target="_new">BGC0002584</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>RiPP</td>
<td>azolemycin B, azolemycin A, azolemycin D, azolemycin C</td>
<td>Streptomyces sp. FXJ1.264</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002013: 0-11564">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002013/index.html#r1c1" target="_new">BGC0002013</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>RiPP</td>
<td>curacozole</td>
<td>Streptomyces curacoi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002076: 6-3180">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002076/index.html#r1c1" target="_new">BGC0002076</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Other</td>
<td>4,5-didehydroacaterin, acaterin</td>
<td>Pseudomonas jessenii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002560: 0-11740">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002560/index.html#r1c1" target="_new">BGC0002560</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.45, order:0.41, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>RiPP</td>
<td>thiostreptamide S4</td>
<td>Streptomyces sp. NRRL S-4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001174: 248-6052">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001174/index.html#r1c1" target="_new">BGC0001174</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>RiPP</td>
<td>corynazolicin</td>
<td>Corynebacterium urealyticum DSM 7111</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000625: 1-15454">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000625/index.html#r1c1" target="_new">BGC0000625</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.46, order:0.38, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>RiPP</td>
<td>thioviridamide</td>
<td>Streptomyces olivoviridis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001802: 854-10845">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001802/index.html#r1c1" target="_new">BGC0001802</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.46, order:0.40, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>RiPP</td>
<td>thioholgamide A, thioholgamide B</td>
<td>Streptomyces malaysiense</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001696: 0-9985">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001696/index.html#r1c1" target="_new">BGC0001696</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.46, order:0.40, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>RiPP</td>
<td>neothioviridamide</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000483: 1302-13724">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000483/index.html#r1c1" target="_new">BGC0000483</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.41, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>RiPP</td>
<td>aeruginosamide B, aeruginosamide C</td>
<td>Microcystis aeruginosa PCC 9432</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r37c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-2"></div> <span class="help-tooltip" id="cb-general-help-2">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r37c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r37c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r37c1_1.svg">
NC_008687 (605598-628171): Paracoccus denitrificans PD1222 chromosome 2, comp... (100% of genes show similarity), TfuA-related
</option>

<option value = "svg/clusterblast_r37c1_2.svg">
NZ_CP044426 (2049965-2072483): Paracoccus pantotrophus strain DSM 2944 chromo... (35% of genes show similarity), TfuA-related
</option>

<option value = "svg/clusterblast_r37c1_3.svg">
NZ_CP058690 (1975833-1998351): Paracoccus pantotrophus strain ACCC10489 chrom... (35% of genes show similarity), TfuA-related
</option>

<option value = "svg/clusterblast_r37c1_4.svg">
NZ_KQ955209 (227552-250116): Paracoccus aminovorans strain HPD-2 Scaffold2, w... (43% of genes show similarity), TfuA-related
</option>

<option value = "svg/clusterblast_r37c1_5.svg">
NC_022041 (2876081-2898633): Paracoccus aminophilus JCM 7686, complete sequence (29% of genes show similarity), TfuA-related
</option>

<option value = "svg/clusterblast_r37c1_6.svg">
NZ_JHWH01000024 (22835-66013): Paracoccus yeei ATCC BAA-599 BU28DRAFT scaffol... (14% of genes show similarity), NRPS,TfuA-related
</option>

<option value = "svg/clusterblast_r37c1_7.svg">
NZ_CP044081 (3460570-3514208): Paracoccus yeei strain FDAARGOS 643 chromosome... (11% of genes show similarity), NRPS,TfuA-related
</option>

<option value = "svg/clusterblast_r37c1_8.svg">
NZ_CP031078 (444806-497957): Paracoccus yeei strain CCUG 32053 chromosome, co... (11% of genes show similarity), NRPS,TfuA-related
</option>

<option value = "svg/clusterblast_r37c1_9.svg">
NZ_CP024422 (815376-857435): Paracoccus yeei strain TT13 chromosome, complete... (13% of genes show similarity), NRPS,TfuA-related
</option>

<option value = "svg/clusterblast_r37c1_10.svg">
NZ_CP020442 (1258595-1318385): Paracoccus yeei strain FDAARGOS 252 chromosome... (10% of genes show similarity), NRPS,TfuA-related,thiopeptide
</option>

</select>
<button id= "clusterblast-r37c1-download">Download graphic</button>
</div>
<div id="clusterblast-r37c1-svg"></div>

</div>
</div>

<div class="body-details-section r37c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-3"></div> <span class="help-tooltip" id="cb-known-help-3">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r37c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-4"></div> <span class="help-tooltip" id="cb-sub-help-4">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r37c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-6"></div> <span class="help-tooltip" id="tfbs-body-help-6">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (2):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
NuR
<div class="collapser collapser-target-NuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Nickel-responsive regulator<br>
<span>Score</span>: 22.10 out of 30.98
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r37c1-86079-NuR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 18.92 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r37c1-90610-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r37c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-8"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-8">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r37c1-pfam-details-svg" data-filename="NZ_FNEA01000020.1_r37c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r37c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r37c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-9"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-9">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r37c1-tigrfam-details-svg" data-filename="NZ_FNEA01000020.1_r37c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r37c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r37c1-tfbs-finder" data-name="r37c1-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r37c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-5"></div> <span class="help-tooltip" id="tfbs-side-help-5">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NuR" target="_blank">NuR</a></td>
<td>Nickel-responsive regulator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r42c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_FNEA01000014.1 - Region 1 - hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-100"></div> <span class="help-tooltip" id="region_svg-help-100">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_FNEA01000014.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r42c1-svg" data-filename="NZ_FNEA01000014.1_r42c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 7,957 - 28,547 nt. (total: 20,591 nt)

<a class="cluster-rules-header" id="r42c1-rules-header" href="#r42c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r42c1-rules">hserlactone: Autoind_synth</div>

<div class="region-svg-container">
<div id='r42c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-101"></div> <span class="help-tooltip" id="focus-panel-help-101">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r42c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r42c1-gene-table-details" data-name="r42c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r42c1-MIBiG-cluster-compare" data-name="r42c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r42c1-clusterblast" data-name="r42c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r42c1-knownclusterblast" data-name="r42c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r42c1-subclusterblast" data-name="r42c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r42c1-tfbs-finder" data-name="r42c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r42c1-pfam-details" data-name="r42c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r42c1-tigrfam-details" data-name="r42c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r42c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-16"></div> <span class="help-tooltip" id="gt-help-tooltip-help-16">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r42c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r42c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r42c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r42c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r42c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-10"></div> <span class="help-tooltip" id="MIBiG-help-10">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r42c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r42c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r42c1"></div>
<div id="comparison-MIBiG-r42c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">hserlactone</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001992: 0-4749">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001992/index.html#r1c1" target="_new">BGC0001992</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Other</td>

<td>thioguanine</td>


<td>Erwinia amylovora CFBP1430</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000487: 82-11698">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000487/index.html#r1c1" target="_new">BGC0000487</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>carnocyclin</td>


<td>Carnobacterium maltaromaticum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002447: 0-8196">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002447/index.html#r1c1" target="_new">BGC0002447</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>cerecyclin</td>


<td>Bacillus cereus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554: 0-6546">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>SRO15-3108</td>


<td>Streptomyces filamentosus NRRL 15998</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002116: 0-6188">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002116/index.html#r1c1" target="_new">BGC0002116</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>darobactin A</td>


<td>Photorhabdus khanii HGB 1456</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001525: 0-16392">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001525/index.html#r1c1" target="_new">BGC0001525</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Other</td>

<td>bartoloside 2, bartoloside 3, bartoloside 4</td>


<td>Synechocystis salina LEGE 06155</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001526: 0-14417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001526/index.html#r1c1" target="_new">BGC0001526</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Other</td>

<td>bartoloside E, bartoloside F, bartoloside G, bartoloside H, bartoloside I, bartoloside J, bartoloside K</td>


<td>Synechocystis salina LEGE 06099</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002699: 0-12569">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002699/index.html#r1c1" target="_new">BGC0002699</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>nostolysamide A, nostolysamide B</td>


<td>Nostoc punctiforme PCC 73102</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000587: 1762-13637">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000587/index.html#r1c1" target="_new">BGC0000587</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>microcin H47</td>


<td>Escherichia coli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000589: 0-8421">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000589/index.html#r1c1" target="_new">BGC0000589</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>microcin M</td>


<td>Escherichia coli Nissle 1917</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r42c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001992: 0-4749">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001992/index.html#r1c1" target="_new">BGC0001992</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Other</td>
<td>thioguanine</td>
<td>Erwinia amylovora CFBP1430</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000487: 82-11698">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000487/index.html#r1c1" target="_new">BGC0000487</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>RiPP</td>
<td>carnocyclin</td>
<td>Carnobacterium maltaromaticum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002447: 0-8196">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002447/index.html#r1c1" target="_new">BGC0002447</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>RiPP</td>
<td>cerecyclin</td>
<td>Bacillus cereus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554: 0-6546">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>RiPP</td>
<td>SRO15-3108</td>
<td>Streptomyces filamentosus NRRL 15998</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002116: 0-6188">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002116/index.html#r1c1" target="_new">BGC0002116</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>RiPP</td>
<td>darobactin A</td>
<td>Photorhabdus khanii HGB 1456</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001525: 0-16392">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001525/index.html#r1c1" target="_new">BGC0001525</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Other</td>
<td>bartoloside 2, bartoloside 3, bartoloside 4</td>
<td>Synechocystis salina LEGE 06155</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001526: 0-14417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001526/index.html#r1c1" target="_new">BGC0001526</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Other</td>
<td>bartoloside E, bartoloside F, bartoloside G, bartoloside H, bartoloside I, bartoloside J, bartoloside K</td>
<td>Synechocystis salina LEGE 06099</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002699: 0-12569">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002699/index.html#r1c1" target="_new">BGC0002699</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>nostolysamide A, nostolysamide B</td>
<td>Nostoc punctiforme PCC 73102</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000587: 1762-13637">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000587/index.html#r1c1" target="_new">BGC0000587</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>microcin H47</td>
<td>Escherichia coli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000589: 0-8421">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000589/index.html#r1c1" target="_new">BGC0000589</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>microcin M</td>
<td>Escherichia coli Nissle 1917</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r42c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-11"></div> <span class="help-tooltip" id="cb-general-help-11">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r42c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r42c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r42c1_1.svg">
NC_008686 (751304-771895): Paracoccus denitrificans PD1222 chromosome 1, comp... (100% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r42c1_2.svg">
NZ_PPGA01000002 (321275-341866): Paracoccus denitrificans strain ISTOD1 IIFCA... (100% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r42c1_3.svg">
NZ_CP045072 (1775397-1795988): Paracoccus kondratievae strain BJQ0001 chromos... (100% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r42c1_4.svg">
NZ_FXAQ01000014 (77673-98264): Paracoccus sp. J56, whole genome shotgun sequence (100% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r42c1_5.svg">
NZ_JHWH01000007 (104079-124670): Paracoccus yeei ATCC BAA-599 BU28DRAFT scaff... (100% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r42c1_6.svg">
NZ_CP024422 (622071-642662): Paracoccus yeei strain TT13 chromosome, complete... (100% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r42c1_7.svg">
NZ_CP031078 (2642299-2662890): Paracoccus yeei strain CCUG 32053 chromosome, ... (100% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r42c1_8.svg">
NZ_CP044081 (3368589-3389180): Paracoccus yeei strain FDAARGOS 643 chromosome... (100% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r42c1_9.svg">
NZ_CP020442 (1107303-1127894): Paracoccus yeei strain FDAARGOS 252 chromosome... (100% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r42c1_10.svg">
NZ_CP058690 (1435890-1456481): Paracoccus pantotrophus strain ACCC10489 chrom... (96% of genes show similarity), hserlactone
</option>

</select>
<button id= "clusterblast-r42c1-download">Download graphic</button>
</div>
<div id="clusterblast-r42c1-svg"></div>

</div>
</div>

<div class="body-details-section r42c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-12"></div> <span class="help-tooltip" id="cb-known-help-12">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r42c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-13"></div> <span class="help-tooltip" id="cb-sub-help-13">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r42c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-15"></div> <span class="help-tooltip" id="tfbs-body-help-15">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
NrdR
<div class="collapser collapser-target-NrdR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Represses ribonucleotide reductase encoding genes<br>
<span>Score</span>: 20.33 out of 27.18
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r42c1-15377-NrdR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r42c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-17"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-17">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r42c1-pfam-details-svg" data-filename="NZ_FNEA01000014.1_r42c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r42c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r42c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-18"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-18">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r42c1-tigrfam-details-svg" data-filename="NZ_FNEA01000014.1_r42c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r42c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r42c1-tfbs-finder" data-name="r42c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r42c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-14"></div> <span class="help-tooltip" id="tfbs-side-help-14">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrdR" target="_blank">NrdR</a></td>
<td>Represses ribonucleotide reductase encoding genes</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r42c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_FNEA01000014.1 - Region 2 - NRPS-like,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-102"></div> <span class="help-tooltip" id="region_svg-help-102">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_FNEA01000014.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r42c2-svg" data-filename="NZ_FNEA01000014.1_r42c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 51,227 - 103,125 nt. (total: 51,899 nt)

<a class="cluster-rules-header" id="r42c2-rules-header" href="#r42c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r42c2-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r42c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-103"></div> <span class="help-tooltip" id="focus-panel-help-103">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r42c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r42c2-gene-table-details" data-name="r42c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r42c2-nrps_pks" data-name="r42c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r42c2-MIBiG-cluster-compare" data-name="r42c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r42c2-clusterblast" data-name="r42c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r42c2-knownclusterblast" data-name="r42c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r42c2-subclusterblast" data-name="r42c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r42c2-tfbs-finder" data-name="r42c2-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r42c2-nrps-pks-bubbles" data-name="r42c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r42c2-pfam-details" data-name="r42c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r42c2-tigrfam-details" data-name="r42c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r42c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-29"></div> <span class="help-tooltip" id="gt-help-tooltip-help-29">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r42c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r42c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r42c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r42c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r42c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-19"></div> <span class="help-tooltip" id="nrps-domain-help-19">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r42c2-details-svg" data-filename="NZ_FNEA01000014.1_r42c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r42c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r42c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-20"></div> <span class="help-tooltip" id="MIBiG-help-20">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r42c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r42c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r42c2"></div>
<div id="comparison-MIBiG-r42c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000763: 750-18450">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000763/index.html#r1c1" target="_new">BGC0000763</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.50, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.50, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.64</td>
<td>Saccharide</td>

<td>stewartan</td>


<td>Pantoea stewartii subsp. stewartii DC283</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000741: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000741/index.html#r1c1" target="_new">BGC0000741</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.62</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000744: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000744/index.html#r1c1" target="_new">BGC0000744</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.62</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000745: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000745/index.html#r1c1" target="_new">BGC0000745</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.62</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000765: 457-18083">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000765/index.html#r1c1" target="_new">BGC0000765</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.48, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.48, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.56</td>
<td>Saccharide</td>

<td>exopolysaccharide</td>


<td>Lactobacillus johnsonii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000740: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000740/index.html#r1c1" target="_new">BGC0000740</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.56</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000746: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000746/index.html#r1c1" target="_new">BGC0000746</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.56</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000747: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000747/index.html#r1c1" target="_new">BGC0000747</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.56</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000748: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000748/index.html#r1c1" target="_new">BGC0000748</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.56</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000764: 1684-16663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000764/index.html#r1c1" target="_new">BGC0000764</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.55</td>
<td>Saccharide</td>

<td>exopolysaccharide</td>


<td>Lactococcus lactis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r42c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000763: 750-18450">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000763/index.html#r1c1" target="_new">BGC0000763</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.50, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Saccharide</td>
<td>stewartan</td>
<td>Pantoea stewartii subsp. stewartii DC283</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000741: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000741/index.html#r1c1" target="_new">BGC0000741</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000744: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000744/index.html#r1c1" target="_new">BGC0000744</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000745: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000745/index.html#r1c1" target="_new">BGC0000745</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000765: 457-18083">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000765/index.html#r1c1" target="_new">BGC0000765</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.48, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Saccharide</td>
<td>exopolysaccharide</td>
<td>Lactobacillus johnsonii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000740: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000740/index.html#r1c1" target="_new">BGC0000740</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000746: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000746/index.html#r1c1" target="_new">BGC0000746</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000747: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000747/index.html#r1c1" target="_new">BGC0000747</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000748: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000748/index.html#r1c1" target="_new">BGC0000748</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000730: 151-21786">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000730/index.html#r1c1" target="_new">BGC0000730</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.48, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Klebsiella pneumoniae</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r42c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-21"></div> <span class="help-tooltip" id="cb-general-help-21">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r42c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r42c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r42c2_1.svg">
NC_008686 (794574-846001): Paracoccus denitrificans PD1222 chromosome 1, comp... (100% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r42c2_2.svg">
NZ_PPGA01000002 (364542-416444): Paracoccus denitrificans strain ISTOD1 IIFCA... (100% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r42c2_3.svg">
NZ_CP045072 (1700317-1752279): Paracoccus kondratievae strain BJQ0001 chromos... (89% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r42c2_4.svg">
NZ_FXAQ01000014 (2590-54563): Paracoccus sp. J56, whole genome shotgun sequence (86% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r42c2_5.svg">
NZ_KQ955210 (17103-69008): Paracoccus aminovorans strain HPD-2 Scaffold3, who... (88% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r42c2_6.svg">
NZ_JHWH01000007 (30306-82234): Paracoccus yeei ATCC BAA-599 BU28DRAFT scaffol... (85% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r42c2_7.svg">
NZ_CP020442 (1034079-1085484): Paracoccus yeei strain FDAARGOS 252 chromosome... (85% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r42c2_8.svg">
NZ_CP024422 (548819-600233): Paracoccus yeei strain TT13 chromosome, complete... (85% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r42c2_9.svg">
NZ_CP058690 (1371025-1422939): Paracoccus pantotrophus strain ACCC10489 chrom... (75% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r42c2_10.svg">
NZ_CP044426 (1436313-1480948): Paracoccus pantotrophus strain DSM 2944 chromo... (87% of genes show similarity), NRPS-like
</option>

</select>
<button id= "clusterblast-r42c2-download">Download graphic</button>
</div>
<div id="clusterblast-r42c2-svg"></div>

</div>
</div>

<div class="body-details-section r42c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-22"></div> <span class="help-tooltip" id="cb-known-help-22">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r42c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-23"></div> <span class="help-tooltip" id="cb-sub-help-23">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r42c2-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-27"></div> <span class="help-tooltip" id="tfbs-body-help-27">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AbrC3
<div class="collapser collapser-target-AbrC3 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Antibiotic production activator<br>
<span>Score</span>: 18.16 out of 18.16
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r42c2-69668-AbrC3"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (3):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 19.11 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r42c2-51747-ZuR"></div>

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 19.87 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r42c2-58227-NrtR"></div>

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 20.83 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r42c2-86080-NrtR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r42c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-28"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-28">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r42c2-domain-bubble-svg" data-filename="NZ_FNEA01000014.1_r42c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r42c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (51226 - 103125): neighbouring T1PKS-NRPS-like
</option>



<option value="CC3">
Candidate 3 (51226 - 97508): single T1PKS
</option>



<option value="CC4">
Candidate 4 (58499 - 103125): single NRPS-like
</option>


</select>

<div class="details-svg" id="r42c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r42c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r42c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-30"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-30">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r42c2-pfam-details-svg" data-filename="NZ_FNEA01000014.1_r42c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r42c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r42c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-31"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-31">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r42c2-tigrfam-details-svg" data-filename="NZ_FNEA01000014.1_r42c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r42c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r42c2-nrps_pks_products" data-name="r42c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r42c2-nrps_pks_monomers" data-name="r42c2-nrps_pks_monomers">
NRPS/PKS monomers
</div>



<div class="sidepanel-details-header r42c2-tfbs-finder" data-name="r42c2-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r42c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-24"></div> <span class="help-tooltip" id="nrps-structures-help-24">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 51226 - 103125:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r42c2-2" class="smiles-canvas" data-smiles="C=CNC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(ccmal) + (X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal*&amp;nrps2=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r42c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C=CNC([*])C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 3, location 51226 - 97508:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r42c2-3" class="smiles-canvas" data-smiles="C=CNC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(ccmal) + (X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal*&amp;nrps2=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r42c2-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C=CNC([*])C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 4, location 58499 - 103125:
<div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r42c2-4" class="smiles-canvas" data-smiles="C=CNC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(ccmal) + (X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal*&amp;nrps2=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r42c2-4-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C=CNC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r42c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS monomer predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-25"></div> <span class="help-tooltip" id="nrps-monomers-help-25">Shows the predicted monomers for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">GJDDMNIP_00730</span></strong>: mal
<div class="collapser collapser-target-GJDDMNIP_00730 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (540..836)</strong>: mal
<div class="collapser collapser-target-GJDDMNIP_00730 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-GJDDMNIP_00730-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 66.7%</dd>
<dd>inactive: 62.5%</dd>
<dd>2-Rhyd-Malonyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-GJDDMNIP_00730-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 84.9</dt>

<dd></dd><dt>inactive: 53.4</dt>

<dd></dd><dt>Methylmalonyl-CoA: 42.5</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 31.3</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 24.9</dt>

<dd></dd><dt>Isobutyryl-CoA: 21.5</dt>

<dd></dd><dt>Propionyl-CoA: 21.2</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 15.3</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 10.1</dt>

<dd></dd><dt>Benzoyl-CoA: 9.7</dt>

<dd></dd><dt>fatty_acid: 7.3</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>CHC-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

<strong><span class="serif">GJDDMNIP_00732</span></strong>: X
<div class="collapser collapser-target-GJDDMNIP_00732 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>A-OX (544..1326)</strong>: X
<div class="collapser collapser-target-GJDDMNIP_00732 collapser-level-candidate"> </div><div class="collapser-content">

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r42c2-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-26"></div> <span class="help-tooltip" id="tfbs-side-help-26">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrtR" target="_blank">NrtR</a></td>
<td>NAD synthesis repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AbrC3" target="_blank">AbrC3</a></td>
<td>Antibiotic production activator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>




<div class="page" id='r44c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_FNEA01000012.1 - Region 1 - LAP,thiopeptide <div class="help-container"> <div class="help-icon" data-id="region_svg-help-104"></div> <span class="help-tooltip" id="region_svg-help-104">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_FNEA01000012.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r44c1-svg" data-filename="NZ_FNEA01000012.1_r44c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 8,571 - 38,669 nt. (total: 30,099 nt)

<a class="cluster-rules-header" id="r44c1-rules-header" href="#r44c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r44c1-rules">thiopeptide: ((YcaO or TIGR03604) and ((thio_amide and (PF06968 or PF04055 or PF07366)) or Lant_dehydr_C or Lant_dehydr_N or PF07366 or PF06968 or PF04055) or thiostrepton)<br>LAP: ((((goadsporin_like or PF00881 or TIGR03605) and (YcaO or TIGR03604)) or TIGR03882 or TIGR03603 or TIGR03602) and not cyanobactin_synth)</div>

<div class="region-svg-container">
<div id='r44c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-105"></div> <span class="help-tooltip" id="focus-panel-help-105">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r44c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r44c1-gene-table-details" data-name="r44c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r44c1-MIBiG-cluster-compare" data-name="r44c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r44c1-clusterblast" data-name="r44c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r44c1-knownclusterblast" data-name="r44c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r44c1-subclusterblast" data-name="r44c1-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r44c1-thiopeptides" data-name="r44c1-thiopeptides">
Thiopeptides
</div>



<div class="body-details-header r44c1-pfam-details" data-name="r44c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r44c1-tigrfam-details" data-name="r44c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r44c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-38"></div> <span class="help-tooltip" id="gt-help-tooltip-help-38">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r44c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r44c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r44c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r44c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r44c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-32"></div> <span class="help-tooltip" id="MIBiG-help-32">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r44c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r44c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r44c1"></div>
<div id="comparison-MIBiG-r44c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">thiopeptide</span></th>

<th><span class="cc-heat-product">LAP</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001217: 0-1691">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001217/index.html#r1c1" target="_new">BGC0001217</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.70</td>
<td>Other</td>

<td>PreQ0 Base</td>


<td>Streptomyces qinglanensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002150: 0-3220">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002150/index.html#r1c1" target="_new">BGC0002150</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.67</td>
<td>RiPP</td>

<td>triculamin</td>


<td>Streptomyces triculaminicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002151: 0-3226">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002151/index.html#r1c1" target="_new">BGC0002151</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.67</td>
<td>RiPP</td>

<td>alboverticillin</td>


<td>Streptomyces griseocarneus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000917: 216-2477">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000917/index.html#r1c1" target="_new">BGC0000917</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.65</td>
<td>Other</td>

<td>molybdenum cofactor</td>


<td>Rhodobacter capsulatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001638: 626-5883">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001638/index.html#r1c1" target="_new">BGC0001638</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.62</td>
<td>RiPP</td>

<td>klebsazolicin</td>


<td>Klebsiella pneumoniae subsp. ozaenae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001174: 248-6052">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001174/index.html#r1c1" target="_new">BGC0001174</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.55</td>
<td>RiPP</td>

<td>corynazolicin</td>


<td>Corynebacterium urealyticum DSM 7111</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002144: 0-5515">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002144/index.html#r1c1" target="_new">BGC0002144</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.54</td>
<td>RiPP</td>

<td>phazolicin</td>


<td>Rhizobium sp. Pop5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002325: 814-4534">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002325/index.html#r1c1" target="_new">BGC0002325</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.49, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.49, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.54</td>
<td>RiPP</td>

<td>mycetohabin-16</td>


<td>Mycetohabitans rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002312: 3-4377">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002312/index.html#r1c1" target="_new">BGC0002312</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.50</td>
<td>RiPP</td>

<td>ubonodin</td>


<td>Burkholderia ubonensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002116: 0-6188">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002116/index.html#r1c1" target="_new">BGC0002116</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.49</td>
<td>RiPP</td>

<td>darobactin A</td>


<td>Photorhabdus khanii HGB 1456</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r44c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001217: 0-1691">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001217/index.html#r1c1" target="_new">BGC0001217</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Other</td>
<td>PreQ0 Base</td>
<td>Streptomyces qinglanensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002150: 0-3220">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002150/index.html#r1c1" target="_new">BGC0002150</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>RiPP</td>
<td>triculamin</td>
<td>Streptomyces triculaminicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002151: 0-3226">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002151/index.html#r1c1" target="_new">BGC0002151</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>RiPP</td>
<td>alboverticillin</td>
<td>Streptomyces griseocarneus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000917: 216-2477">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000917/index.html#r1c1" target="_new">BGC0000917</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>molybdenum cofactor</td>
<td>Rhodobacter capsulatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001638: 626-5883">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001638/index.html#r1c1" target="_new">BGC0001638</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>RiPP</td>
<td>klebsazolicin</td>
<td>Klebsiella pneumoniae subsp. ozaenae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001174: 248-6052">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001174/index.html#r1c1" target="_new">BGC0001174</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>RiPP</td>
<td>corynazolicin</td>
<td>Corynebacterium urealyticum DSM 7111</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002144: 0-5515">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002144/index.html#r1c1" target="_new">BGC0002144</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>RiPP</td>
<td>phazolicin</td>
<td>Rhizobium sp. Pop5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002325: 814-4534">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002325/index.html#r1c1" target="_new">BGC0002325</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.49, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>RiPP</td>
<td>mycetohabin-16</td>
<td>Mycetohabitans rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002312: 3-4377">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002312/index.html#r1c1" target="_new">BGC0002312</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>RiPP</td>
<td>ubonodin</td>
<td>Burkholderia ubonensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002116: 0-6188">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002116/index.html#r1c1" target="_new">BGC0002116</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>RiPP</td>
<td>darobactin A</td>
<td>Photorhabdus khanii HGB 1456</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r44c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-33"></div> <span class="help-tooltip" id="cb-general-help-33">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r44c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r44c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r44c1_1.svg">
NZ_PPGA01000001 (423189-453474): Paracoccus denitrificans strain ISTOD1 IIFCA... (100% of genes show similarity), LAP,bacteriocin,thiopeptide
</option>

<option value = "svg/clusterblast_r44c1_2.svg">
NC_008687 (1337178-1367277): Paracoccus denitrificans PD1222 chromosome 2, co... (96% of genes show similarity), LAP,bacteriocin,thiopeptide
</option>

<option value = "svg/clusterblast_r44c1_3.svg">
NZ_QOCG01000031 (100519-122574): Loktanella sp. D2R18 D2R18 contig 4, whole g... (22% of genes show similarity), LAP
</option>

<option value = "svg/clusterblast_r44c1_4.svg">
NZ_RSCM01000014 (111074-141581): Trichormus variabilis SAG 1403-4b sequence14... (16% of genes show similarity), LAP,thiopeptide
</option>

<option value = "svg/clusterblast_r44c1_5.svg">
NZ_CP011456 (2234656-2274120): Anabaena sp. WA102, complete genome (12% of genes show similarity), LAP,thiopeptide
</option>

<option value = "svg/clusterblast_r44c1_6.svg">
NZ_VILF01000003 (939156-978240): Dolichospermum flos-aquae UHCC 0037 newAn37 ... (12% of genes show similarity), LAP,thiopeptide
</option>

<option value = "svg/clusterblast_r44c1_7.svg">
NZ_CP043056 (3452256-3474654): Dolichospermum sp. UHCC 0315A chromosome, comp... (26% of genes show similarity), LAP
</option>

<option value = "svg/clusterblast_r44c1_8.svg">
NZ_LZQD01000002 (604513-636098): Trichormus sp. NMC-1 scaffold2, whole genome... (15% of genes show similarity), LAP,thiopeptide
</option>

<option value = "svg/clusterblast_r44c1_9.svg">
NZ_AP018174 (1554240-1610228): Anabaenopsis circularis NIES-21 DNA, nearly co... (8% of genes show similarity), LAP,NRPS-like
</option>

<option value = "svg/clusterblast_r44c1_10.svg">
NZ_CP051206 (4350670-4372769): Dolichospermum flos-aquae CCAP 1403/13F chromo... (23% of genes show similarity), LAP
</option>

</select>
<button id= "clusterblast-r44c1-download">Download graphic</button>
</div>
<div id="clusterblast-r44c1-svg"></div>

</div>
</div>

<div class="body-details-section r44c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-34"></div> <span class="help-tooltip" id="cb-known-help-34">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r44c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r44c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r44c1_1.svg">
succinoglycan
</option>

<option value = "svg/knownclusterblast_r44c1_2.svg">
toyocamycin/sangivamycin
</option>

<option value = "svg/knownclusterblast_r44c1_3.svg">
huimycin
</option>

<option value = "svg/knownclusterblast_r44c1_4.svg">
toyocamycin
</option>

<option value = "svg/knownclusterblast_r44c1_5.svg">
toyocamycin
</option>

</select>
<button id="knownclusterblast-r44c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r44c1-svg"></div>

</div>
</div>

<div class="body-details-section r44c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-35"></div> <span class="help-tooltip" id="cb-sub-help-35">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r44c1-thiopeptides">
<div class="details">
<div class="heading">
<span>Thiopeptide predictions</span>
<div class="help-container"> <div class="help-icon" data-id="thio-body-help-36"></div> <span class="help-tooltip" id="thio-body-help-36">Lists the possible core peptides for each biosynthetic enzyme. Predicted tail sequences are also shown, if present. <br>Includes CompaRiPPson results for any available databases, with a detailed explanation <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/comparippson' target='_blank'>here</a>.</span></div>
</div>
<div class="details-text ripp-body">

No core peptides found.

</div>
</div>
</div>



<div class="body-details-section r44c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-39"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-39">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r44c1-pfam-details-svg" data-filename="NZ_FNEA01000012.1_r44c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r44c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r44c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-40"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-40">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r44c1-tigrfam-details-svg" data-filename="NZ_FNEA01000012.1_r44c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r44c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">












<div class="sidepanel-details-header r44c1-thiopeptides" data-name="r44c1-thiopeptides">
Thiopeptides
</div>




</div>












<div class="sidepanel-details-section r44c1-thiopeptides">
<div class="more-details">
<div class="heading">
<span>Core peptide predictions</span>
<div class="help-container"> <div class="help-icon" data-id="thio-side-help-37"></div> <span class="help-tooltip" id="thio-side-help-37">Lists the possible core peptides in the region. Each core peptide lists its possible molecular weights and the scores for cleavage site prediction and RODEO. If relevant, other features, such as macrocycle and amidation, will also be listed.</span></div>
</div>

<div class="prediction-text">No core peptides found.</div>

</div>
</div>




</div>

</div>

</div>




<div class="page" id='r55c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_FNEA01000006.1 - Region 1 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-106"></div> <span class="help-tooltip" id="region_svg-help-106">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_FNEA01000006.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r55c1-svg" data-filename="NZ_FNEA01000006.1_r55c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 177,548 - 188,345 nt. (total: 10,798 nt)

<a class="cluster-rules-header" id="r55c1-rules-header" href="#r55c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r55c1-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or TIGR01193 or (YcaO or TIGR03604))</div>

<div class="region-svg-container">
<div id='r55c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-107"></div> <span class="help-tooltip" id="focus-panel-help-107">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r55c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r55c1-gene-table-details" data-name="r55c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r55c1-MIBiG-cluster-compare" data-name="r55c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r55c1-clusterblast" data-name="r55c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r55c1-knownclusterblast" data-name="r55c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r55c1-subclusterblast" data-name="r55c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r55c1-tfbs-finder" data-name="r55c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r55c1-pfam-details" data-name="r55c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r55c1-tigrfam-details" data-name="r55c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r55c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-47"></div> <span class="help-tooltip" id="gt-help-tooltip-help-47">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r55c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r55c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r55c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r55c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r55c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-41"></div> <span class="help-tooltip" id="MIBiG-help-41">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r55c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r55c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r55c1"></div>
<div id="comparison-MIBiG-r55c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001122: 0-43663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001122/index.html#r1c1" target="_new">BGC0001122</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.48, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>dihydroisoflavipucine, isoflavipucine</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001591: 66-30223">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001591/index.html#r1c1" target="_new">BGC0001591</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.48, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>Other</td>

<td>fatty acid enol ester, N-tetradecanoyl tyrosine</td>


<td>uncultured bacterium CSLC2</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001882: 0-43357">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001882/index.html#r1c1" target="_new">BGC0001882</a></td>





<td class="cc-heat-cell" title="0.19 (id:0.50, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>


<td>0.04</td>
<td>Polyketide</td>

<td>chrysoxanthone A, chrysoxanthone B, chrysoxanthone C</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002362: 1-24294">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002362/index.html#r1c1" target="_new">BGC0002362</a></td>





<td class="cc-heat-cell" title="0.19 (id:0.49, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>


<td>0.04</td>
<td>Polyketide</td>

<td>loseolamycin A1, loseolamycin A2</td>


<td>Micromonospora endolithica</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r55c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001122: 0-43663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001122/index.html#r1c1" target="_new">BGC0001122</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.48, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>dihydroisoflavipucine, isoflavipucine</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001591: 66-30223">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001591/index.html#r1c1" target="_new">BGC0001591</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.48, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>Other</td>
<td>fatty acid enol ester, N-tetradecanoyl tyrosine</td>
<td>uncultured bacterium CSLC2</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001882: 0-43357">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001882/index.html#r1c1" target="_new">BGC0001882</a></td>


<td class="cc-heat-cell" title="0.19 (id:0.50, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
<td>0.19</td>
<td>Polyketide</td>
<td>chrysoxanthone A, chrysoxanthone B, chrysoxanthone C</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002362: 1-24294">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002362/index.html#r1c1" target="_new">BGC0002362</a></td>


<td class="cc-heat-cell" title="0.19 (id:0.49, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
<td>0.19</td>
<td>Polyketide</td>
<td>loseolamycin A1, loseolamycin A2</td>
<td>Micromonospora endolithica</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r55c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-42"></div> <span class="help-tooltip" id="cb-general-help-42">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r55c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r55c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r55c1_1.svg">
NC_008687 (1157182-1167265): Paracoccus denitrificans PD1222 chromosome 2, co... (100% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r55c1_2.svg">
NZ_PPGA01000001 (253735-263819): Paracoccus denitrificans strain ISTOD1 IIFCA... (100% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r55c1_3.svg">
NZ_SLZI01000011 (56061-66886): Rhizobium sp. BK315 Ga0307681 111, whole genom... (57% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r55c1_4.svg">
NZ_UBQJ01000016 (43895-54720): Rhizobiales bacterium isolate AFS025266, whole... (57% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r55c1_5.svg">
NZ_JACIAF010000017 (128966-139791): Rhizobium sp. BK386 Ga0365287 17, whole g... (57% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r55c1_6.svg">
NZ_UCJJ01000030 (11733-22558): Rhizobiales bacterium isolate AFS086490, whole... (57% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r55c1_7.svg">
NZ_UBOB01000004 (47343-58168): Rhizobiales bacterium isolate AFS017628, whole... (50% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r55c1_8.svg">
NZ_UBRD01000024 (13915-24740): Rhizobiales bacterium isolate AFS025464, whole... (50% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r55c1_9.svg">
NZ_UBXG01000029 (2557-13382): Rhizobiales bacterium isolate AFS049917, whole ... (57% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r55c1_10.svg">
NZ_LN832562 (286837-297635): Paracoccus aminovorans isolate JCM7685 plasmid I... (25% of genes show similarity), bacteriocin
</option>

</select>
<button id= "clusterblast-r55c1-download">Download graphic</button>
</div>
<div id="clusterblast-r55c1-svg"></div>

</div>
</div>

<div class="body-details-section r55c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-43"></div> <span class="help-tooltip" id="cb-known-help-43">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r55c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-44"></div> <span class="help-tooltip" id="cb-sub-help-44">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r55c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-46"></div> <span class="help-tooltip" id="tfbs-body-help-46">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 21.44 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r55c1-184548-ZuR"></div>

</div>


</div>
</div>



<div class="body-details-section r55c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-48"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-48">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r55c1-pfam-details-svg" data-filename="NZ_FNEA01000006.1_r55c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r55c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r55c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-49"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-49">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r55c1-tigrfam-details-svg" data-filename="NZ_FNEA01000006.1_r55c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r55c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r55c1-tfbs-finder" data-name="r55c1-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r55c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-45"></div> <span class="help-tooltip" id="tfbs-side-help-45">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r57c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_FNEA01000007.1 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-108"></div> <span class="help-tooltip" id="region_svg-help-108">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_FNEA01000007.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r57c1-svg" data-filename="NZ_FNEA01000007.1_r57c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 29,310 - 73,167 nt. (total: 43,858 nt)

<a class="cluster-rules-header" id="r57c1-rules-header" href="#r57c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r57c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r57c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-109"></div> <span class="help-tooltip" id="focus-panel-help-109">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r57c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r57c1-gene-table-details" data-name="r57c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r57c1-nrps_pks" data-name="r57c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r57c1-MIBiG-cluster-compare" data-name="r57c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r57c1-clusterblast" data-name="r57c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r57c1-knownclusterblast" data-name="r57c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r57c1-subclusterblast" data-name="r57c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r57c1-tfbs-finder" data-name="r57c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r57c1-nrps-pks-bubbles" data-name="r57c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r57c1-pfam-details" data-name="r57c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r57c1-tigrfam-details" data-name="r57c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r57c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-60"></div> <span class="help-tooltip" id="gt-help-tooltip-help-60">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r57c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r57c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r57c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r57c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r57c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-50"></div> <span class="help-tooltip" id="nrps-domain-help-50">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r57c1-details-svg" data-filename="NZ_FNEA01000007.1_r57c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r57c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r57c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-51"></div> <span class="help-tooltip" id="MIBiG-help-51">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r57c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r57c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r57c1"></div>
<div id="comparison-MIBiG-r57c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075: 0-3894">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>NRP, Alkaloid</td>

<td>pyreudione A, pyreudione B, pyreudione C, pyreudione D, pyreudione E</td>


<td>Pseudomonas fluorescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002689: 0-23151">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002689/index.html#r1c1" target="_new">BGC0002689</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.51, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>2,3-dihydroxybenzoylserine</td>


<td>Stenotrophomonas maltophilia K279a</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002359: 18-27968">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002359/index.html#r1c1" target="_new">BGC0002359</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>dudomycin A</td>


<td>Streptomyces albus subsp. chlorinus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002494: 273-12179">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002494/index.html#r1c1" target="_new">BGC0002494</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>vibriobactin</td>


<td>Vibrio cholerae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002528: 0-12492">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002528/index.html#r1c1" target="_new">BGC0002528</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>aminochelin, azotochelin, protochelin</td>


<td>Azotobacter vinelandii CA</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001890: 1-28356">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001890/index.html#r1c1" target="_new">BGC0001890</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>amphi-enterobactin 1, amphi-enterobactin 2, amphi-enterobactin 3, amphi-enterobactin 4</td>


<td>Vibrio campbellii ATCC BAA-1116</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131: 0-29471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.48, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>ambactin</td>


<td>Xenorhabdus miraniensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002493: 1205-6076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002493/index.html#r1c1" target="_new">BGC0002493</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>parabactin</td>


<td>Paracoccus denitrificans PD1222</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r57c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075: 0-3894">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP, Alkaloid</td>
<td>pyreudione A, pyreudione B, pyreudione C, pyreudione D, pyreudione E</td>
<td>Pseudomonas fluorescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002689: 0-23151">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002689/index.html#r1c1" target="_new">BGC0002689</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.51, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP</td>
<td>2,3-dihydroxybenzoylserine</td>
<td>Stenotrophomonas maltophilia K279a</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002359: 18-27968">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002359/index.html#r1c1" target="_new">BGC0002359</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP</td>
<td>dudomycin A</td>
<td>Streptomyces albus subsp. chlorinus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002494: 273-12179">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002494/index.html#r1c1" target="_new">BGC0002494</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>vibriobactin</td>
<td>Vibrio cholerae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002528: 0-12492">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002528/index.html#r1c1" target="_new">BGC0002528</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>aminochelin, azotochelin, protochelin</td>
<td>Azotobacter vinelandii CA</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001890: 1-28356">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001890/index.html#r1c1" target="_new">BGC0001890</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>amphi-enterobactin 1, amphi-enterobactin 2, amphi-enterobactin 3, amphi-enterobactin 4</td>
<td>Vibrio campbellii ATCC BAA-1116</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131: 0-29471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.48, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>ambactin</td>
<td>Xenorhabdus miraniensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002493: 1205-6076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002493/index.html#r1c1" target="_new">BGC0002493</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>parabactin</td>
<td>Paracoccus denitrificans PD1222</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r57c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-52"></div> <span class="help-tooltip" id="cb-general-help-52">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r57c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r57c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r57c1_1.svg">
NC_008686 (2553772-2597630): Paracoccus denitrificans PD1222 chromosome 1, co... (100% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r57c1_2.svg">
NZ_PPGA01000003 (27840-70634): Paracoccus denitrificans strain ISTOD1 IIFCABN... (97% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r57c1_3.svg">
NZ_FTMK01000009 (26903-70761): Paracoccus thiocyanatus strain ATCC 700171, wh... (90% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r57c1_4.svg">
NZ_LN832559 (382115-425044): Paracoccus aminovorans isolate JCM7685 chromosome I (82% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r57c1_5.svg">
NZ_FXAQ01000007 (86042-129903): Paracoccus sp. J56, whole genome shotgun sequ... (65% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r57c1_6.svg">
NZ_CP045072 (1913225-1957086): Paracoccus kondratievae strain BJQ0001 chromos... (65% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r57c1_7.svg">
NZ_CP044426 (&lt;175832-218794): Paracoccus pantotrophus strain DSM 2944 chromos... (63% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r57c1_8.svg">
NZ_CP058690 (&lt;99449-&gt;142524): Paracoccus pantotrophus strain ACCC10489 chromo... (65% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r57c1_9.svg">
NZ_CP044081 (440210-482181): Paracoccus yeei strain FDAARGOS 643 chromosome, ... (67% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r57c1_10.svg">
NZ_CP024422 (1282006-1325861): Paracoccus yeei strain TT13 chromosome, comple... (65% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r57c1-download">Download graphic</button>
</div>
<div id="clusterblast-r57c1-svg"></div>

</div>
</div>

<div class="body-details-section r57c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-53"></div> <span class="help-tooltip" id="cb-known-help-53">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r57c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r57c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r57c1_1.svg">
chromobactin
</option>

<option value = "svg/knownclusterblast_r57c1_2.svg">
lipopolysaccharide
</option>

</select>
<button id="knownclusterblast-r57c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r57c1-svg"></div>

</div>
</div>

<div class="body-details-section r57c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-54"></div> <span class="help-tooltip" id="cb-sub-help-54">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r57c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-58"></div> <span class="help-tooltip" id="tfbs-body-help-58">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (4):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 17.79 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r57c1-32265-ZuR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 18.04 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r57c1-48190-ZuR"></div>

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 20.29 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r57c1-64586-NrtR"></div>

<div class="tfbs-hit-detail">
NuR
<div class="collapser collapser-target-NuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Nickel-responsive regulator<br>
<span>Score</span>: 20.32 out of 30.98
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r57c1-66261-NuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r57c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-59"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-59">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r57c1-domain-bubble-svg" data-filename="NZ_FNEA01000007.1_r57c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r57c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (29309 - 73167): single NRPS
</option>


</select>

<div class="details-svg" id="r57c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r57c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r57c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-61"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-61">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r57c1-pfam-details-svg" data-filename="NZ_FNEA01000007.1_r57c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r57c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r57c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-62"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-62">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r57c1-tigrfam-details-svg" data-filename="NZ_FNEA01000007.1_r57c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r57c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r57c1-nrps_pks_products" data-name="r57c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r57c1-nrps_pks_monomers" data-name="r57c1-nrps_pks_monomers">
NRPS/PKS monomers
</div>



<div class="sidepanel-details-header r57c1-tfbs-finder" data-name="r57c1-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r57c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-55"></div> <span class="help-tooltip" id="nrps-structures-help-55">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 29309 - 73167:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r57c1-1" class="smiles-canvas" data-smiles="NC(CO)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Ser)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r57c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CO)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r57c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS monomer predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-56"></div> <span class="help-tooltip" id="nrps-monomers-help-56">Shows the predicted monomers for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">GJDDMNIP_02215</span></strong>: Ser
<div class="collapser collapser-target-GJDDMNIP_02215 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (461..847)</strong>: Ser
<div class="collapser collapser-target-GJDDMNIP_02215 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ser
<div class="collapser collapser-target-GJDDMNIP_02215-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain or phenyl group with -OH (Ser, Thr, Dhpg, Hpg)</dd>
<dt>Small clusters prediction:</dt>
<dd>ser (Ser)</dd>
<dt>Single AA prediction:</dt>
<dd>ser (Ser)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVWHFSLVDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ser <span class="serif">DVWHFSLVDK</span> (97% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r57c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-57"></div> <span class="help-tooltip" id="tfbs-side-help-57">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrtR" target="_blank">NrtR</a></td>
<td>NAD synthesis repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NuR" target="_blank">NuR</a></td>
<td>Nickel-responsive regulator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>




<div class="page" id='r78c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_FNEA01000030.1 - Region 1 - ranthipeptide <div class="help-container"> <div class="help-icon" data-id="region_svg-help-110"></div> <span class="help-tooltip" id="region_svg-help-110">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_FNEA01000030.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r78c1-svg" data-filename="NZ_FNEA01000030.1_r78c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 3,840 - 25,342 nt. (total: 21,503 nt)

<a class="cluster-rules-header" id="r78c1-rules-header" href="#r78c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r78c1-rules">ranthipeptide: (TIGR03973 or SCIFF or (cds(PF04055 and (TIGR04085 or SPASM)) and minimum(1, [Bottromycin_Methyltransferase_RRE, Cyanobactin_D_RRE, Cytolysin_C_RRE, Goadsporin_CD_RRE, Heterocycloanthracin_C_RRE, Lanthipeptide_LanB_RRE, Lasso_Fused_RRE, Mycofactocin_RRE, NHLP_CD_RRE, Other_Sactipeptide_rSAM_RRE, Pantocin_Microcin_RRE, Plantazolicin_C_RRE, PlpY_RRE, PqqD_RRE, Proteusin_Epimerase_RRE, Proteusin_Methyltransferase_RRE, Quinohemoprotein_rSAM_RRE, Ranthipeptide_rSAM_RRE, Stand_Alone_Lasso_RRE, Streptide_RRE, Subtilosin_rSAM_RRE, Thiaglutamate_B_RRE, Thiaglutamate_G_RRE, Thiaglutamate_I_RRE, Thiopeptide_F_RRE, Thuricin_rSAM_RRE, Thurincin_rSAM_RRE, Trifolitoxin_RRE]) and not (TIGR01716 or subtilosin or TIGR02111 or TIGR03966 or TIGR04103 or (YcaO or TIGR03604))))</div>

<div class="region-svg-container">
<div id='r78c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-111"></div> <span class="help-tooltip" id="focus-panel-help-111">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r78c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r78c1-gene-table-details" data-name="r78c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r78c1-MIBiG-cluster-compare" data-name="r78c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r78c1-clusterblast" data-name="r78c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r78c1-knownclusterblast" data-name="r78c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r78c1-subclusterblast" data-name="r78c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r78c1-tfbs-finder" data-name="r78c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r78c1-pfam-details" data-name="r78c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r78c1-tigrfam-details" data-name="r78c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r78c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-70"></div> <span class="help-tooltip" id="gt-help-tooltip-help-70">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r78c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r78c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r78c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r78c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r78c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-63"></div> <span class="help-tooltip" id="MIBiG-help-63">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r78c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r78c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r78c1"></div>
<div id="comparison-MIBiG-r78c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">ranthipeptide</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002306: 0-3536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002306/index.html#r1c1" target="_new">BGC0002306</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>RiPP</td>

<td>freyrasin</td>


<td>Paenibacillus polymyxa ATCC 842</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000802: 0-10580">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000802/index.html#r1c1" target="_new">BGC0000802</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Saccharide</td>

<td>succinoglycan</td>


<td>Sinorhizobium meliloti</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000846: 192-22933">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000846/index.html#r1c1" target="_new">BGC0000846</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Other</td>

<td>tabtoxin</td>


<td>Pseudomonas syringae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001775: 2711-20673">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001775/index.html#r1c1" target="_new">BGC0001775</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Terpene</td>

<td>sespendole</td>


<td>Cordana terrestris</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Other</td>

<td>citrinin</td>


<td>Monascus aurantiacus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002151: 0-3226">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002151/index.html#r1c1" target="_new">BGC0002151</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>alboverticillin</td>


<td>Streptomyces griseocarneus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002150: 0-3220">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002150/index.html#r1c1" target="_new">BGC0002150</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>triculamin</td>


<td>Streptomyces triculaminicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.14</td>
<td>Polyketide</td>

<td>HEx-pks16 polyketide</td>


<td>Punctularia strigosozonata HHB-11173 SS5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Polyketide</td>

<td>strobilurin A</td>


<td>Strobilurus tenacellus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.43, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r78c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002306: 0-3536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002306/index.html#r1c1" target="_new">BGC0002306</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>RiPP</td>
<td>freyrasin</td>
<td>Paenibacillus polymyxa ATCC 842</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000802: 0-10580">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000802/index.html#r1c1" target="_new">BGC0000802</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Saccharide</td>
<td>succinoglycan</td>
<td>Sinorhizobium meliloti</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000846: 192-22933">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000846/index.html#r1c1" target="_new">BGC0000846</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Other</td>
<td>tabtoxin</td>
<td>Pseudomonas syringae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001775: 2711-20673">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001775/index.html#r1c1" target="_new">BGC0001775</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Terpene</td>
<td>sespendole</td>
<td>Cordana terrestris</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Other</td>
<td>citrinin</td>
<td>Monascus aurantiacus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002151: 0-3226">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002151/index.html#r1c1" target="_new">BGC0002151</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>alboverticillin</td>
<td>Streptomyces griseocarneus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002150: 0-3220">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002150/index.html#r1c1" target="_new">BGC0002150</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>triculamin</td>
<td>Streptomyces triculaminicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Polyketide</td>
<td>HEx-pks16 polyketide</td>
<td>Punctularia strigosozonata HHB-11173 SS5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Polyketide</td>
<td>strobilurin A</td>
<td>Strobilurus tenacellus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.43, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r78c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-64"></div> <span class="help-tooltip" id="cb-general-help-64">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r78c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r78c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r78c1_1.svg">
NZ_CP039637 (339472-393118): Azospirillum sp. TSH100 chromosome 4, complete s... (22% of genes show similarity), hglE-KS
</option>

<option value = "svg/clusterblast_r78c1_2.svg">
NZ_LT629767 (4080727-4141960): Pseudomonas umsongensis strain LMG 21317 chrom... (22% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r78c1_3.svg">
NZ_CABVHK010000021 (80813-146068): Pseudomonas fluorescens isolate PS662, who... (18% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r78c1_4.svg">
NZ_QAOV01000017 (27233-92492): Pseudomonas sp. GV085 Ga0189755 117, whole gen... (17% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r78c1_5.svg">
NZ_CP051487 (4892264-4953104): Pseudomonas umsongensis strain CY-1 chromosome... (22% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r78c1_6.svg">
NZ_LKBJ01000002 (2588637-2653886): Pseudomonas fluorescens strain GW456-L13 1... (18% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r78c1_7.svg">
NZ_JAAVXC010000007 (305931-368664): Pseudomonas sp. SbOxS1 y03 R1 contig 7, w... (22% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r78c1_8.svg">
NZ_CABVJB010000010 (111347-176606): Pseudomonas fluorescens isolate PS922, wh... (19% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r78c1_9.svg">
NZ_CP019398 (4344491-4408473): Pseudomonas sp. S34 chromosome, complete genome (18% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r78c1_10.svg">
NZ_CABVIR010000005 (360292-425553): Pseudomonas fluorescens isolate PS858, wh... (19% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r78c1-download">Download graphic</button>
</div>
<div id="clusterblast-r78c1-svg"></div>

</div>
</div>

<div class="body-details-section r78c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-65"></div> <span class="help-tooltip" id="cb-known-help-65">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r78c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r78c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r78c1_1.svg">
Pf-5 pyoverdine
</option>

</select>
<button id="knownclusterblast-r78c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r78c1-svg"></div>

</div>
</div>

<div class="body-details-section r78c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-66"></div> <span class="help-tooltip" id="cb-sub-help-66">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r78c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-69"></div> <span class="help-tooltip" id="tfbs-body-help-69">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AbrC3
<div class="collapser collapser-target-AbrC3 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Antibiotic production activator<br>
<span>Score</span>: 18.07 out of 18.16
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r78c1-22367-AbrC3"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 19.34 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r78c1-25083-NrtR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r78c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-71"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-71">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r78c1-pfam-details-svg" data-filename="NZ_FNEA01000030.1_r78c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r78c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r78c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-72"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-72">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r78c1-tigrfam-details-svg" data-filename="NZ_FNEA01000030.1_r78c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r78c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r78c1-RREfinder" data-name="r78c1-RREfinder">
RREFinder
</div>



<div class="sidepanel-details-header r78c1-tfbs-finder" data-name="r78c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r78c1-RREfinder">
<div class="more-details">
<div class="heading">
<span>RRE predictions</span>
<div class="help-container"> <div class="help-icon" data-id="rre-side-help-67"></div> <span class="help-tooltip" id="rre-side-help-67">RREfinder results sidepanel.</span></div>
</div>
<dl class = 'prediction-text'>

<span><strong>ranthipeptide protocluster</strong> (3839...25342)</span>
<dt></dt>




<strong>GJDDMNIP_03905</strong> <div class="collapser collapser-target-GJDDMNIP_03905 collapser-level-cds"> </div><div class="collapser-content">

<dd>RRE type: Quinohemoprotein_rSAM_RRE</dd>
<dd>Location: 18 - 99</dd>
<dd>Score: 84.2</dd>

</div><br>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r78c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-68"></div> <span class="help-tooltip" id="tfbs-side-help-68">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AbrC3" target="_blank">AbrC3</a></td>
<td>Antibiotic production activator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrtR" target="_blank">NrtR</a></td>
<td>NAD synthesis repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r93c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_FNEA01000005.1 - Region 1 - NRP-metallophore,redox-cofactor <div class="help-container"> <div class="help-icon" data-id="region_svg-help-112"></div> <span class="help-tooltip" id="region_svg-help-112">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_FNEA01000005.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r93c1-svg" data-filename="NZ_FNEA01000005.1_r93c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 195,863 - 242,048 nt. (total: 46,186 nt)

<a class="cluster-rules-header" id="r93c1-rules-header" href="#r93c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r93c1-rules">redox-cofactor: (cds(PF04055 and (TIGR04085 or SPASM)) and (TIGR02111 or TIGR03966) and minimum(1, [Bottromycin_Methyltransferase_RRE, Cyanobactin_D_RRE, Cytolysin_C_RRE, Goadsporin_CD_RRE, Heterocycloanthracin_C_RRE, Lanthipeptide_LanB_RRE, Lasso_Fused_RRE, Mycofactocin_RRE, NHLP_CD_RRE, Other_Sactipeptide_rSAM_RRE, Pantocin_Microcin_RRE, Plantazolicin_C_RRE, PlpY_RRE, PqqD_RRE, Proteusin_Epimerase_RRE, Proteusin_Methyltransferase_RRE, Quinohemoprotein_rSAM_RRE, Ranthipeptide_rSAM_RRE, Stand_Alone_Lasso_RRE, Streptide_RRE, Subtilosin_rSAM_RRE, Thiaglutamate_B_RRE, Thiaglutamate_G_RRE, Thiaglutamate_I_RRE, Thiopeptide_F_RRE, Thuricin_rSAM_RRE, Thurincin_rSAM_RRE, Trifolitoxin_RRE]))<br>NRP-metallophore: (VibH_like or Cy_tandem or (cds(Condensation and AMP-binding) and ((IBH_Asp and not SBH_Asp) or IBH_His or TBH_Asp or CyanoBH_Asp1 or CyanoBH_Asp2 or IPL or SalSyn or (EntA and EntC) or (GrbD and GrbE) or (FbnL and FbnM) or PvdO or PvdP or (Orn_monoox and not (KtzT or MetRS-like)) or Lys_monoox or VbsL)))</div>

<div class="region-svg-container">
<div id='r93c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-113"></div> <span class="help-tooltip" id="focus-panel-help-113">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r93c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r93c1-gene-table-details" data-name="r93c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r93c1-nrps_pks" data-name="r93c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r93c1-MIBiG-cluster-compare" data-name="r93c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r93c1-clusterblast" data-name="r93c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r93c1-knownclusterblast" data-name="r93c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r93c1-subclusterblast" data-name="r93c1-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r93c1-tfbs-finder" data-name="r93c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r93c1-pfam-details" data-name="r93c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r93c1-tigrfam-details" data-name="r93c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r93c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-82"></div> <span class="help-tooltip" id="gt-help-tooltip-help-82">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r93c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r93c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r93c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r93c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r93c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-73"></div> <span class="help-tooltip" id="nrps-domain-help-73">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r93c1-details-svg" data-filename="NZ_FNEA01000005.1_r93c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r93c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r93c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-74"></div> <span class="help-tooltip" id="MIBiG-help-74">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r93c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r93c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r93c1"></div>
<div id="comparison-MIBiG-r93c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">redox-cofactor</span></th>

<th><span class="cc-heat-product">NRP-metallophore</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002493: 1205-6076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002493/index.html#r1c1" target="_new">BGC0002493</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.96 (id:1.00, order:0.88, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.96), rgb(0, 0, 0, 0.96) 96%, white 96%)"></td>


<td>0.87</td>
<td>NRP</td>

<td>parabactin</td>


<td>Paracoccus denitrificans PD1222</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001928: 108-8402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001928/index.html#r1c1" target="_new">BGC0001928</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>






<td class="cc-heat-cell" title="0.48 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.33</td>
<td>Other</td>

<td>(2R,3S,4S)-5-fluoro-2,3,4-trihydroxypentanoic acid</td>


<td>Streptomyces sp. MA37</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001940: 1-19375">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001940/index.html#r1c1" target="_new">BGC0001940</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.47, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>pyxidicycline A, pyxidicycline B</td>


<td>Pyxidicoccus fallax</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.50, order:0.60, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002276: 0-3813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002276/index.html#r1c1" target="_new">BGC0002276</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>






<td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>choline</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002474: 103-7254">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002474/index.html#r1c1" target="_new">BGC0002474</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.51, order:0.43, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>agrobactin</td>


<td>Agrobacterium tumefaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001727: 0-13202">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001727/index.html#r1c1" target="_new">BGC0001727</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>






<td class="cc-heat-cell" title="0.37 (id:0.43, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.27</td>
<td>RiPP</td>

<td>paenilan</td>


<td>Paenibacillus polymyxa E681</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001605: 0-9118">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001605/index.html#r1c1" target="_new">BGC0001605</a></td>





<td class="cc-heat-cell" title="0.21 (id:0.43, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.47, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.26</td>
<td>Terpene</td>

<td>gibberellin</td>


<td>Sinorhizobium fredii NGR234</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000934: 146-25006">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000934/index.html#r1c1" target="_new">BGC0000934</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.45, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>






<td class="cc-heat-cell" title="0.41 (id:0.50, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.26</td>
<td>Other (Phenazine)</td>

<td>5-acetyl-5,10-dihydrophenazine-1-carboxylic acid, 5-(2-hydroxyacetyl)-5,10-dihydrophenazine-1-carboxylic acid, endophenazine A1, endophenazine F, endophenazine G</td>


<td>Kitasatospora sp. HKI 714</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000506: 405-13975">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000506/index.html#r1c1" target="_new">BGC0000506</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>






<td class="cc-heat-cell" title="0.35 (id:0.43, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.25</td>
<td>RiPP</td>

<td>entianin</td>


<td>Bacillus subtilis subsp. spizizenii</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r93c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002493: 1205-6076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002493/index.html#r1c1" target="_new">BGC0002493</a></td>


<td class="cc-heat-cell" title="0.96 (id:1.00, order:0.88, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.96), rgb(0, 0, 0, 0.96) 96%, white 96%)"></td>
<td>0.96</td>
<td>NRP</td>
<td>parabactin</td>
<td>Paracoccus denitrificans PD1222</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.50, order:0.60, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001302: 0-25686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001302/index.html#r1c1" target="_new">BGC0001302</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.51, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Other</td>
<td>lomofungin</td>
<td>Streptomyces lomondensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001965: 0-30490">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001965/index.html#r1c1" target="_new">BGC0001965</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.49, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Other</td>
<td>nybomycin</td>
<td>Streptomyces albus subsp. chlorinus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002474: 103-7254">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002474/index.html#r1c1" target="_new">BGC0002474</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.51, order:0.43, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>agrobactin</td>
<td>Agrobacterium tumefaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002496: 894-14801">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002496/index.html#r1c1" target="_new">BGC0002496</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>serratiochelin A, serratiochelin C, serratiochelin B</td>
<td>Serratia plymuthica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001928: 108-8402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001928/index.html#r1c1" target="_new">BGC0001928</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Other</td>
<td>(2R,3S,4S)-5-fluoro-2,3,4-trihydroxypentanoic acid</td>
<td>Streptomyces sp. MA37</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002425: 1265-25473">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002425/index.html#r1c1" target="_new">BGC0002425</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>RiPP</td>
<td>stlassin</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001723: 128-10733">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001723/index.html#r1c1" target="_new">BGC0001723</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>oviedomycin</td>
<td>Streptomyces antibioticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231: 189-5010">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>griseusin A, griseusin B</td>
<td>Streptomyces griseus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r93c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-75"></div> <span class="help-tooltip" id="cb-general-help-75">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r93c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r93c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r93c1_1.svg">
NC_008686 (2365239-2405817): Paracoccus denitrificans PD1222 chromosome 1, co... (61% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r93c1_2.svg">
NZ_PPGA01000003 (192451-235611): Paracoccus denitrificans strain ISTOD1 IIFCA... (60% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r93c1_3.svg">
NZ_CP044426 (1749291-1792447): Paracoccus pantotrophus strain DSM 2944 chromo... (68% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r93c1_4.svg">
NZ_CP020442 (1258595-1318385): Paracoccus yeei strain FDAARGOS 252 chromosome... (44% of genes show similarity), NRPS,TfuA-related,thiopeptide
</option>

<option value = "svg/clusterblast_r93c1_5.svg">
NZ_JAEN01000018 (42174-85327): Paracoccus sp. J39 ParJ39DRAFT c17 C, whole ge... (56% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r93c1_6.svg">
NZ_CP030239 (1249944-1310119): Paracoccus mutanolyticus strain RSP-02 chromos... (46% of genes show similarity), NRPS,TfuA-related,thiopeptide
</option>

<option value = "svg/clusterblast_r93c1_7.svg">
NZ_CP045072 (150837-190381): Paracoccus kondratievae strain BJQ0001 chromosom... (59% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r93c1_8.svg">
NZ_FXAQ01000012 (47999-91173): Paracoccus sp. J56, whole genome shotgun sequence (60% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r93c1_9.svg">
NZ_LN832559 (2866388-2909555): Paracoccus aminovorans isolate JCM7685 chromos... (70% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r93c1_10.svg">
NZ_WMIF01000001 (114297-157470): Paracoccus limosus strain JCM 17370 Scaffold... (60% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r93c1-download">Download graphic</button>
</div>
<div id="clusterblast-r93c1-svg"></div>

</div>
</div>

<div class="body-details-section r93c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-76"></div> <span class="help-tooltip" id="cb-known-help-76">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r93c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r93c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r93c1_1.svg">
parabactin
</option>

<option value = "svg/knownclusterblast_r93c1_2.svg">
chromobactin
</option>

<option value = "svg/knownclusterblast_r93c1_3.svg">
2,3-dihydroxybenzoylserine
</option>

<option value = "svg/knownclusterblast_r93c1_4.svg">
myxochelin A/myxochelin B
</option>

<option value = "svg/knownclusterblast_r93c1_5.svg">
myxochelin B/myxochelin A
</option>

<option value = "svg/knownclusterblast_r93c1_6.svg">
A23187
</option>

<option value = "svg/knownclusterblast_r93c1_7.svg">
mirubactin
</option>

<option value = "svg/knownclusterblast_r93c1_8.svg">
heterobactin A/heterobactin S2
</option>

<option value = "svg/knownclusterblast_r93c1_9.svg">
heterobactin B/heterobactin S2
</option>

<option value = "svg/knownclusterblast_r93c1_10.svg">
diazaquinomycin H/diazaquinomycin J
</option>

</select>
<button id="knownclusterblast-r93c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r93c1-svg"></div>

</div>
</div>

<div class="body-details-section r93c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-77"></div> <span class="help-tooltip" id="cb-sub-help-77">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<select id= "subclusterblast-r93c1-select" class= "clusterblast-selector">
<option value="svg/subclusterblast_r93c1_all.svg">
All hits
</option>

<option value = "svg/subclusterblast_r93c1_1.svg">
AB307968_c2: KS-505a benzoate (50% of genes show similarity), benzoate
</option>

</select>
<button id="subclusterblast-r93c1-download">Download graphic</button>
</div>
<div id="subclusterblast-r93c1-svg"></div>

</div>
</div>







<div class="body-details-section r93c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-81"></div> <span class="help-tooltip" id="tfbs-body-help-81">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 22.16 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r93c1-230674-ZuR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (4):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
LexA
<div class="collapser collapser-target-LexA collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Repressor of DNA damage response<br>
<span>Score</span>: 20.65 out of 29.65
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r93c1-222793-LexA"></div>

<div class="tfbs-hit-detail">
NuR
<div class="collapser collapser-target-NuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Nickel-responsive regulator<br>
<span>Score</span>: 21.50 out of 30.98
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r93c1-230673-NuR"></div>

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 19.15 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r93c1-240304-NrtR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 20.19 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r93c1-241933-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r93c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-83"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-83">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r93c1-pfam-details-svg" data-filename="NZ_FNEA01000005.1_r93c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r93c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r93c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-84"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-84">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r93c1-tigrfam-details-svg" data-filename="NZ_FNEA01000005.1_r93c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r93c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r93c1-nrps_pks_monomers" data-name="r93c1-nrps_pks_monomers">
NRPS/PKS monomers
</div>



<div class="sidepanel-details-header r93c1-RREfinder" data-name="r93c1-RREfinder">
RREFinder
</div>



<div class="sidepanel-details-header r93c1-tfbs-finder" data-name="r93c1-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r93c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS monomer predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-78"></div> <span class="help-tooltip" id="nrps-monomers-help-78">Shows the predicted monomers for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">GJDDMNIP_04961</span></strong>: diOH-Bz
<div class="collapser collapser-target-GJDDMNIP_04961 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=dioh-bz" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=dioh-bz*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (30..438)</strong>: diOH-Bz
<div class="collapser collapser-target-GJDDMNIP_04961 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  diOH-Bz
<div class="collapser collapser-target-GJDDMNIP_04961-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aromatic (Phe, Tyr, diOH-Bz, Ph-Gly, bOH-Tyr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Hydroxy-benzoic acid derivates (diOH-Bz, Sal)</dd>
<dt>Small clusters prediction:</dt>
<dd>Hydroxy-benzoic acid derivates (diOH-Bz, Sal)</dd>
<dt>Single AA prediction:</dt>
<dd>dhb (diOH-Bz)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">TMPPLGVVNK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>diOH-Bz <span class="serif">TMPSQGVVNK</span> (91% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r93c1-RREfinder">
<div class="more-details">
<div class="heading">
<span>RRE predictions</span>
<div class="help-container"> <div class="help-icon" data-id="rre-side-help-79"></div> <span class="help-tooltip" id="rre-side-help-79">RREfinder results sidepanel.</span></div>
</div>
<dl class = 'prediction-text'>

<span><strong>redox-cofactor protocluster</strong> (195862...218095)</span>
<dt></dt>




<strong>GJDDMNIP_04935</strong> <div class="collapser collapser-target-GJDDMNIP_04935 collapser-level-cds"> </div><div class="collapser-content">

<dd>RRE type: PqqD_RRE</dd>
<dd>Location: 5 - 92</dd>
<dd>Score: 88.0</dd>

<dd>RRE type: Stand_Alone_Lasso_RRE</dd>
<dd>Location: 37 - 92</dd>
<dd>Score: 42.5</dd>

</div><br>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r93c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-80"></div> <span class="help-tooltip" id="tfbs-side-help-80">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/LexA" target="_blank">LexA</a></td>
<td>Repressor of DNA damage response</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NuR" target="_blank">NuR</a></td>
<td>Nickel-responsive regulator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrtR" target="_blank">NrtR</a></td>
<td>NAD synthesis repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r94c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_FNEA01000013.1 - Region 1 - NRP-metallophore,NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-114"></div> <span class="help-tooltip" id="region_svg-help-114">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_FNEA01000013.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r94c1-svg" data-filename="NZ_FNEA01000013.1_r94c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,515 - 53,620 nt. (total: 51,106 nt)

<a class="cluster-rules-header" id="r94c1-rules-header" href="#r94c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r94c1-rules">NRP-metallophore: (VibH_like or Cy_tandem or (cds(Condensation and AMP-binding) and ((IBH_Asp and not SBH_Asp) or IBH_His or TBH_Asp or CyanoBH_Asp1 or CyanoBH_Asp2 or IPL or SalSyn or (EntA and EntC) or (GrbD and GrbE) or (FbnL and FbnM) or PvdO or PvdP or (Orn_monoox and not (KtzT or MetRS-like)) or Lys_monoox or VbsL)))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r94c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-115"></div> <span class="help-tooltip" id="focus-panel-help-115">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r94c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r94c1-gene-table-details" data-name="r94c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r94c1-nrps_pks" data-name="r94c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r94c1-MIBiG-cluster-compare" data-name="r94c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r94c1-clusterblast" data-name="r94c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r94c1-knownclusterblast" data-name="r94c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r94c1-subclusterblast" data-name="r94c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r94c1-tfbs-finder" data-name="r94c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r94c1-nrps-pks-bubbles" data-name="r94c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r94c1-pfam-details" data-name="r94c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r94c1-tigrfam-details" data-name="r94c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r94c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-95"></div> <span class="help-tooltip" id="gt-help-tooltip-help-95">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r94c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r94c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r94c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r94c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r94c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-85"></div> <span class="help-tooltip" id="nrps-domain-help-85">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r94c1-details-svg" data-filename="NZ_FNEA01000013.1_r94c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r94c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r94c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-86"></div> <span class="help-tooltip" id="MIBiG-help-86">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r94c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r94c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r94c1"></div>
<div id="comparison-MIBiG-r94c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRP-metallophore</span></th>

<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000945: 129-10811">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000945/index.html#r1c1" target="_new">BGC0000945</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.44, order:0.46, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>






<td class="cc-heat-cell" title="0.59 (id:0.44, order:0.46, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.50</td>
<td>Other (Unknown)</td>

<td>vibriobactin</td>


<td>Vibrio cholerae O1 biovar El Tor str. N16961</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002476: 0-10330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002476/index.html#r1c1" target="_new">BGC0002476</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.46, order:0.46, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.46, order:0.46, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.48</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Escherichia coli str. K-12 substr. MG1655</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002412: 0-12427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002412/index.html#r1c1" target="_new">BGC0002412</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.46</td>
<td>NRP</td>

<td>holomycin</td>


<td>Photobacterium galatheae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002571: 103-67235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002571/index.html#r1c1" target="_new">BGC0002571</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.48, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.48, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.44</td>
<td>NRP</td>

<td>pyoverdine DC3000</td>


<td>Pseudomonas syringae pv. tomato str. DC3000</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002091: 1-17973">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002091/index.html#r1c1" target="_new">BGC0002091</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.44</td>
<td>NRP</td>

<td>holomycin</td>


<td>Yersinia ruckeri ATCC 29473</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000373: 0-18318">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000373/index.html#r1c1" target="_new">BGC0000373</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.44</td>
<td>NRP</td>

<td>holomycin</td>


<td>Streptomyces clavuligerus ATCC 27064</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.43</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.42</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.42</td>
<td>NRP</td>

<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.42</td>
<td>NRP</td>

<td>rhizomide A, rhizomide B, rhizomide C</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r94c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000945: 129-10811">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000945/index.html#r1c1" target="_new">BGC0000945</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.44, order:0.46, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Other (Unknown)</td>
<td>vibriobactin</td>
<td>Vibrio cholerae O1 biovar El Tor str. N16961</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002476: 0-10330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002476/index.html#r1c1" target="_new">BGC0002476</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.46, order:0.46, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Escherichia coli str. K-12 substr. MG1655</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002571: 103-67235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002571/index.html#r1c1" target="_new">BGC0002571</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.48, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>pyoverdine DC3000</td>
<td>Pseudomonas syringae pv. tomato str. DC3000</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002412: 0-12427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002412/index.html#r1c1" target="_new">BGC0002412</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>holomycin</td>
<td>Photobacterium galatheae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000373: 0-18318">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000373/index.html#r1c1" target="_new">BGC0000373</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>holomycin</td>
<td>Streptomyces clavuligerus ATCC 27064</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002091: 1-17973">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002091/index.html#r1c1" target="_new">BGC0002091</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>NRP</td>
<td>holomycin</td>
<td>Yersinia ruckeri ATCC 29473</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561: 0-17578">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP</td>
<td>curacomycin</td>
<td>Streptomyces curacoi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP</td>
<td>bicornutin A1, bicornutin A2</td>
<td>Xenorhabdus budapestensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP</td>
<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r94c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-87"></div> <span class="help-tooltip" id="cb-general-help-87">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r94c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r94c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r94c1_1.svg">
NC_008687 (160895-213671): Paracoccus denitrificans PD1222 chromosome 2, comp... (91% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r94c1_2.svg">
NZ_CP058690 (678989-730972): Paracoccus pantotrophus strain ACCC10489 chromos... (89% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r94c1_3.svg">
NZ_CP044423 (154914-207746): Paracoccus pantotrophus strain DSM 2944 chromoso... (25% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r94c1_4.svg">
NZ_AZVA01000007 (21735-91183): Paracoccus sp. J55 ParJ55DRAFT scaffold 6.7 C,... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r94c1_5.svg">
NZ_JAEN01000008 (85054-154512): Paracoccus sp. J39 ParJ39DRAFT scaffold 7.8 C... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r94c1_6.svg">
NZ_CP025113 (3482363-3554796): Bradyrhizobium sp. SK17 strain CBNU chromosome... (26% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r94c1_7.svg">
NZ_CP022112 (1024911-1087667): Nitrospirillum amazonense CBAmc chromosome 3, ... (28% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r94c1_8.svg">
NZ_KI912577 (2799168-2841252): Methylobacterium sp. 10 K368DRAFT scaffold0000... (28% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r94c1_9.svg">
NZ_QLLD01000012 (48404-100278): Agrobacterium tumefaciens strain HAMBI 105 ID... (20% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r94c1_10.svg">
NZ_CP019702 (528513-580321): Agrobacterium rhizogenes strain K599 (NCPPB2659)... (26% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r94c1-download">Download graphic</button>
</div>
<div id="clusterblast-r94c1-svg"></div>

</div>
</div>

<div class="body-details-section r94c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-88"></div> <span class="help-tooltip" id="cb-known-help-88">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r94c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r94c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r94c1_1.svg">
yersiniabactin
</option>

</select>
<button id="knownclusterblast-r94c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r94c1-svg"></div>

</div>
</div>

<div class="body-details-section r94c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-89"></div> <span class="help-tooltip" id="cb-sub-help-89">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r94c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-93"></div> <span class="help-tooltip" id="tfbs-body-help-93">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (6):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ArgR
<div class="collapser collapser-target-ArgR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Regulator of arginine biosynthesis genes<br>
<span>Score</span>: 16.81 out of 20.01
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r94c1-2891-ArgR"></div>

<div class="tfbs-hit-detail">
OsdR
<div class="collapser collapser-target-OsdR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Development and stress management regulator<br>
<span>Score</span>: 23.72 out of 30.12
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r94c1-8692-OsdR"></div>

<div class="tfbs-hit-detail">
BldD
<div class="collapser collapser-target-BldD collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Development and antibiotic global regulator<br>
<span>Score</span>: 18.70 out of 23.43
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r94c1-10533-BldD"></div>

<div class="tfbs-hit-detail">
LexA
<div class="collapser collapser-target-LexA collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Repressor of DNA damage response<br>
<span>Score</span>: 20.42 out of 29.65
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r94c1-21367-LexA"></div>

<div class="tfbs-hit-detail">
BldD
<div class="collapser collapser-target-BldD collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Development and antibiotic global regulator<br>
<span>Score</span>: 19.33 out of 23.43
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r94c1-51172-BldD"></div>

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 20.83 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r94c1-53497-NrtR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r94c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-94"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-94">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r94c1-domain-bubble-svg" data-filename="NZ_FNEA01000013.1_r94c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r94c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (2514 - 53620): chemical hybrid NRP-metallophore-NRPS
</option>


</select>

<div class="details-svg" id="r94c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r94c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r94c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-96"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-96">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r94c1-pfam-details-svg" data-filename="NZ_FNEA01000013.1_r94c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r94c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r94c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-97"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-97">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r94c1-tigrfam-details-svg" data-filename="NZ_FNEA01000013.1_r94c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r94c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r94c1-nrps_pks_products" data-name="r94c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r94c1-nrps_pks_monomers" data-name="r94c1-nrps_pks_monomers">
NRPS/PKS monomers
</div>



<div class="sidepanel-details-header r94c1-tfbs-finder" data-name="r94c1-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r94c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-90"></div> <span class="help-tooltip" id="nrps-structures-help-90">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 2514 - 53620:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r94c1-1" class="smiles-canvas" data-smiles="NC(C(O)C)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Thr)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r94c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(C(O)C)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r94c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS monomer predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-91"></div> <span class="help-tooltip" id="nrps-monomers-help-91">Shows the predicted monomers for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">GJDDMNIP_04997</span></strong>: Thr
<div class="collapser collapser-target-GJDDMNIP_04997 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*" target="_new">relaxed</a>
</dd>
<br>







<dd><strong>AMP-binding (876..1259)</strong>: Thr
<div class="collapser collapser-target-GJDDMNIP_04997 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Thr
<div class="collapser collapser-target-GJDDMNIP_04997-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain or phenyl group with -OH (Ser, Thr, Dhpg, Hpg)</dd>
<dt>Small clusters prediction:</dt>
<dd>thr (Thr)</dd>
<dt>Single AA prediction:</dt>
<dd>thr (Thr)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DMFCAGLIWK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Thr <span class="serif">DMFCAGLIWK</span> (94% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r94c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-92"></div> <span class="help-tooltip" id="tfbs-side-help-92">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ArgR" target="_blank">ArgR</a></td>
<td>Regulator of arginine biosynthesis genes</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/OsdR" target="_blank">OsdR</a></td>
<td>Development and stress management regulator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/BldD" target="_blank">BldD</a></td>
<td>Development and antibiotic global regulator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/LexA" target="_blank">LexA</a></td>
<td>Repressor of DNA damage response</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrtR" target="_blank">NrtR</a></td>
<td>NAD synthesis repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>