
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>GCF_001294535.1_Cypionkella_psychrotolerans_PAMC_27389 - 6 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/bacteria.css">
<meta property="og:title" content="GCF_001294535.1_Cypionkella_psychrotolerans_PAMC_27389 - 6 record(s) - 6 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-518f4251-e3b9-49b0-a16d-43eafe9a717f/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://antismash.secondarymetabolites.org/">
<img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://antismash.secondarymetabolites.org/">
antiSMASH version 7.0.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="GCF_001294535.1_Cypionkella_psychrotolerans_PAMC_27389.zip">Download all results</a></li>

<li><a href="GCF_001294535.1_Cypionkella_psychrotolerans_PAMC_27389.gbk">Download GenBank summary file</a></li>

<li><a href="bacteria-518f4251-e3b9-49b0-a16d-43eafe9a717f.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton hybrid r31c1">
<a href="#r31c1">31.1</a>
</div>



<div class="regbutton other hserlactone r52c1">
<a href="#r52c1">52.1</a>
</div>



<div class="regbutton other hserlactone r55c1">
<a href="#r55c1">55.1</a>
</div>



<div class="regbutton other hserlactone r65c1">
<a href="#r65c1">65.1</a>
</div>



<div class="regbutton RiPP RiPP-like r68c1">
<a href="#r68c1">68.1</a>
</div>



<div class="regbutton NRPS NRPS-like r90c1">
<a href="#r90c1">90.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>NZ_LGIC01000031.1</strong> (Cypionkella psychrotolerans)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-31" data-filename="NZ_LGIC01000031.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-31" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r31c1">
<td class="regbutton hybrid r31c1">
<a href="#r31c1">Region&nbsp31.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">20,633</td>
<td class="digits table-split-left">72,527</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000759/1" target="_blank">diutan polysaccharide</a></td>
<td>Saccharide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 16%, #ffffff00 16%)">16%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_LGIC01000052.1</strong> (Cypionkella psychrotolerans)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-52" data-filename="NZ_LGIC01000052.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-52" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r52c1">
<td class="regbutton other hserlactone r52c1">
<a href="#r52c1">Region&nbsp52.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">477</td>
<td class="digits table-split-left">21,067</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_LGIC01000055.1</strong> (Cypionkella psychrotolerans)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-55" data-filename="NZ_LGIC01000055.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-55" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r55c1">
<td class="regbutton other hserlactone r55c1">
<a href="#r55c1">Region&nbsp55.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">17,033</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_LGIC01000065.1</strong> (Cypionkella psychrotolerans)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-65" data-filename="NZ_LGIC01000065.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-65" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r65c1">
<td class="regbutton other hserlactone r65c1">
<a href="#r65c1">Region&nbsp65.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">192,341</td>
<td class="digits table-split-left">213,015</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_LGIC01000068.1</strong> (Cypionkella psychrotolerans)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-68" data-filename="NZ_LGIC01000068.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-68" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r68c1">
<td class="regbutton RiPP RiPP-like r68c1">
<a href="#r68c1">Region&nbsp68.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
</td>
<td class="digits">7,417</td>
<td class="digits table-split-left">18,676</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_LGIC01000090.1</strong> (Cypionkella psychrotolerans)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-90" data-filename="NZ_LGIC01000090.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-90" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r90c1">
<td class="regbutton NRPS NRPS-like r90c1">
<a href="#r90c1">Region&nbsp90.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">37,514</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001758/1" target="_blank">rhizomide A/rhizomide B/rhizomide C</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r31c1">
<td class="regbutton hybrid r31c1">
<a href="#r31c1">Region&nbsp31.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">20,633</td>
<td class="digits table-split-left">72,527</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000759/1" target="_blank">diutan polysaccharide</a></td>
<td>Saccharide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 16%, #ffffff00 16%)">16%</td>

</tr>


<tr class="linked-row even" data-anchor="#r52c1">
<td class="regbutton other hserlactone r52c1">
<a href="#r52c1">Region&nbsp52.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">477</td>
<td class="digits table-split-left">21,067</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r55c1">
<td class="regbutton other hserlactone r55c1">
<a href="#r55c1">Region&nbsp55.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">17,033</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r65c1">
<td class="regbutton other hserlactone r65c1">
<a href="#r65c1">Region&nbsp65.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">192,341</td>
<td class="digits table-split-left">213,015</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r68c1">
<td class="regbutton RiPP RiPP-like r68c1">
<a href="#r68c1">Region&nbsp68.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
</td>
<td class="digits">7,417</td>
<td class="digits table-split-left">18,676</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r90c1">
<td class="regbutton NRPS NRPS-like r90c1">
<a href="#r90c1">Region&nbsp90.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">37,514</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001758/1" target="_blank">rhizomide A/rhizomide B/rhizomide C</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>NZ_LGIC01000001.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000002.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000003.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000004.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000005.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000006.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000007.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000008.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000009.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000010.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000011.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000012.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000013.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000014.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000015.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000016.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000017.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000018.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000019.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000020.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000021.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000022.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000023.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000024.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000025.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000026.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000027.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000028.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000029.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000030.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000032.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000033.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000034.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000035.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000036.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000037.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000038.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000039.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000040.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000041.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000042.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000043.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000044.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000045.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000046.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000047.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000048.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000049.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000050.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000051.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000053.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000054.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000056.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000057.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000058.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000059.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000060.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000061.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000062.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000063.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000064.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000066.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000067.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000069.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000070.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000071.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000072.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000073.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000074.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000075.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000076.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000077.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000078.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000079.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000080.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000081.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000082.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000083.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000084.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000085.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000086.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000087.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000088.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000089.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000091.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000092.1</strong> (Cypionkella psychrotolerans)</dd>

<dd><strong>NZ_LGIC01000093.1</strong> (Cypionkella psychrotolerans)</dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>







<div class="page" id='r31c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_LGIC01000031.1 - Region 1 - NRPS-like,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-56"></div> <span class="help-tooltip" id="region_svg-help-56">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_LGIC01000031.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r31c1-svg" data-filename="NZ_LGIC01000031.1_r31c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 20,633 - 72,527 nt. (total: 51,895 nt)

<a class="cluster-rules-header" id="r31c1-rules-header" href="#r31c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r31c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r31c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-57"></div> <span class="help-tooltip" id="focus-panel-help-57">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r31c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r31c1-gene-table-details" data-name="r31c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r31c1-nrps_pks" data-name="r31c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r31c1-MIBiG-cluster-compare" data-name="r31c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r31c1-clusterblast" data-name="r31c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r31c1-knownclusterblast" data-name="r31c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r31c1-subclusterblast" data-name="r31c1-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r31c1-nrps-pks-bubbles" data-name="r31c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r31c1-pfam-details" data-name="r31c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r31c1-tigrfam-details" data-name="r31c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r31c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-9"></div> <span class="help-tooltip" id="gt-help-tooltip-help-9">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r31c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r31c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r31c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r31c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r31c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-1"></div> <span class="help-tooltip" id="nrps-domain-help-1">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r31c1-details-svg" data-filename="NZ_LGIC01000031.1_r31c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r31c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r31c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-2"></div> <span class="help-tooltip" id="MIBiG-help-2">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r31c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r31c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r31c1"></div>
<div id="comparison-MIBiG-r31c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000765: 457-18083">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000765/index.html#r1c1" target="_new">BGC0000765</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.48, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.48, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.66</td>
<td>Saccharide</td>

<td>exopolysaccharide</td>


<td>Lactobacillus johnsonii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000741: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000741/index.html#r1c1" target="_new">BGC0000741</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.48, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.48, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.63</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000744: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000744/index.html#r1c1" target="_new">BGC0000744</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.48, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.48, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.63</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000745: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000745/index.html#r1c1" target="_new">BGC0000745</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.48, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.48, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.63</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000764: 1684-16663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000764/index.html#r1c1" target="_new">BGC0000764</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.61</td>
<td>Saccharide</td>

<td>exopolysaccharide</td>


<td>Lactococcus lactis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001410: 97-16791">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001410/index.html#r1c1" target="_new">BGC0001410</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.59</td>
<td>Saccharide</td>

<td>polysaccharide A</td>


<td>Bacteroides fragilis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000740: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000740/index.html#r1c1" target="_new">BGC0000740</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.48, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.48, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.57</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000746: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000746/index.html#r1c1" target="_new">BGC0000746</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.48, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.48, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.57</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000747: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000747/index.html#r1c1" target="_new">BGC0000747</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.48, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.48, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.57</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000748: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000748/index.html#r1c1" target="_new">BGC0000748</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.48, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.48, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.57</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r31c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000765: 457-18083">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000765/index.html#r1c1" target="_new">BGC0000765</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.48, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Saccharide</td>
<td>exopolysaccharide</td>
<td>Lactobacillus johnsonii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001410: 97-16791">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001410/index.html#r1c1" target="_new">BGC0001410</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Saccharide</td>
<td>polysaccharide A</td>
<td>Bacteroides fragilis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000764: 1684-16663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000764/index.html#r1c1" target="_new">BGC0000764</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Saccharide</td>
<td>exopolysaccharide</td>
<td>Lactococcus lactis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000751: 0-5961">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000751/index.html#r1c1" target="_new">BGC0000751</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.51, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Actinobacillus pleuropneumoniae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000741: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000741/index.html#r1c1" target="_new">BGC0000741</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.48, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000744: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000744/index.html#r1c1" target="_new">BGC0000744</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.48, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000745: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000745/index.html#r1c1" target="_new">BGC0000745</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.48, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000763: 750-18450">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000763/index.html#r1c1" target="_new">BGC0000763</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.50, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Saccharide</td>
<td>stewartan</td>
<td>Pantoea stewartii subsp. stewartii DC283</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000740: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000740/index.html#r1c1" target="_new">BGC0000740</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.48, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000746: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000746/index.html#r1c1" target="_new">BGC0000746</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.48, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r31c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-3"></div> <span class="help-tooltip" id="cb-general-help-3">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r31c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r31c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r31c1_1.svg">
NZ_JABJXT010000001 (262896-354361): Xinfangfangia sp. D13-10-4-6 Scaffold1, w... (32% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r31c1_2.svg">
NZ_QNHG01000001 (278355-332033): Pseudogemmobacter bohemicus strain Cd-10 sca... (55% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r31c1_3.svg">
NZ_VLLH01000011 (68560-120514): Gemmobacter caeni strain CGMCC 1.7745 IQ03DRA... (36% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r31c1_4.svg">
NZ_QXXQ01000003 (278073-330025): Gemmobacter lutimaris strain YJ-T1-11 scaffo... (36% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r31c1_5.svg">
NZ_VKKX01000033 (377101-428527): Gemmobacter caeruleus strain N8 NODE 3, whol... (36% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r31c1_6.svg">
NZ_FOCE01000008 (86958-138943): Gemmobacter aquatilis strain DSM 3857, whole ... (32% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r31c1_7.svg">
NZ_CP039964 (2161038-2212876): Pseudorhodobacter sp. S12M18 chromosome, compl... (36% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r31c1_8.svg">
NC_003911 (865857-917896): Ruegeria pomeroyi DSS-3, complete sequence (45% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r31c1_9.svg">
NZ_FNAT01000010 (64056-115844): Limimaricola pyoseonensis strain DSM 21424, w... (34% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r31c1_10.svg">
NZ_CP041159 (438023-490057): Leisingera aquaemixtae strain R2C4 chromosome, c... (38% of genes show similarity), NRPS-like,T1PKS
</option>

</select>
<button id= "clusterblast-r31c1-download">Download graphic</button>
</div>
<div id="clusterblast-r31c1-svg"></div>

</div>
</div>

<div class="body-details-section r31c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-4"></div> <span class="help-tooltip" id="cb-known-help-4">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r31c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r31c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r31c1_1.svg">
diutan polysaccharide
</option>

<option value = "svg/knownclusterblast_r31c1_2.svg">
sphingan polysaccharide
</option>

<option value = "svg/knownclusterblast_r31c1_3.svg">
gellan polysaccharide
</option>

<option value = "svg/knownclusterblast_r31c1_4.svg">
galactoglucan
</option>

<option value = "svg/knownclusterblast_r31c1_5.svg">
S-layer glycan
</option>

<option value = "svg/knownclusterblast_r31c1_6.svg">
lipopolysaccharide
</option>

<option value = "svg/knownclusterblast_r31c1_7.svg">
glucorhamnan
</option>

<option value = "svg/knownclusterblast_r31c1_8.svg">
exopolysaccharide
</option>

<option value = "svg/knownclusterblast_r31c1_9.svg">
S-layer glycan
</option>

<option value = "svg/knownclusterblast_r31c1_10.svg">
heteropolysaccharide (HePS)
</option>

</select>
<button id="knownclusterblast-r31c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r31c1-svg"></div>

</div>
</div>

<div class="body-details-section r31c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-5"></div> <span class="help-tooltip" id="cb-sub-help-5">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<select id= "subclusterblast-r31c1-select" class= "clusterblast-selector">
<option value="svg/subclusterblast_r31c1_all.svg">
All hits
</option>

<option value = "svg/subclusterblast_r31c1_1.svg">
AB032523_c1: avermectin oleandrose (25% of genes show similarity), sugar
</option>

<option value = "svg/subclusterblast_r31c1_2.svg">
DQ149987_c2: concanamycin 4&#39;-O-carbamoyl-2&#39;-deoxyrhamnose (28% of genes show similarity), sugar
</option>

<option value = "svg/subclusterblast_r31c1_3.svg">
AY509120_c2: chalcomycin NDP-6-deoxy-D-allose (50% of genes show similarity), sugar
</option>

<option value = "svg/subclusterblast_r31c1_4.svg">
AY509120_c1: chalcomycin NDP-chalcose (40% of genes show similarity), sugar
</option>

<option value = "svg/subclusterblast_r31c1_5.svg">
DQ230532_c3: phenalinolactone deoxyhexose L-amicetose (25% of genes show similarity), sugar
</option>

<option value = "svg/subclusterblast_r31c1_6.svg">
AF170880_c2: novobiocin noviose deoxysugar (33% of genes show similarity), sugar
</option>

<option value = "svg/subclusterblast_r31c1_7.svg">
AF235050_c2: coumermycin noviose deoxysugar (33% of genes show similarity), sugar
</option>

<option value = "svg/subclusterblast_r31c1_8.svg">
AF497482_c3: calicheamicin 4 nucleotide sugars (11% of genes show similarity), sugar
</option>

</select>
<button id="subclusterblast-r31c1-download">Download graphic</button>
</div>
<div id="subclusterblast-r31c1-svg"></div>

</div>
</div>







<div class="body-details-section r31c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-8"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-8">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r31c1-domain-bubble-svg" data-filename="NZ_LGIC01000031.1_r31c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r31c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (20632 - 72527): neighbouring NRPS-like-T1PKS
</option>



<option value="CC2">
Candidate 2 (20632 - 65123): single NRPS-like
</option>



<option value="CC3">
Candidate 3 (26143 - 72527): single T1PKS
</option>


</select>

<div class="details-svg" id="r31c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r31c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r31c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-10"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-10">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r31c1-pfam-details-svg" data-filename="NZ_LGIC01000031.1_r31c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r31c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r31c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-11"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-11">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r31c1-tigrfam-details-svg" data-filename="NZ_LGIC01000031.1_r31c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r31c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r31c1-nrps_pks_products" data-name="r31c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r31c1-nrps_pks_monomers" data-name="r31c1-nrps_pks_monomers">
NRPS/PKS monomers
</div>








</div>










<div class="sidepanel-details-section r31c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-6"></div> <span class="help-tooltip" id="nrps-structures-help-6">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 20632 - 72527:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r31c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r31c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 2, location 20632 - 65123:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r31c1-2" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r31c1-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 3, location 26143 - 72527:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r31c1-3" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r31c1-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r31c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS monomer predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-7"></div> <span class="help-tooltip" id="nrps-monomers-help-7">Shows the predicted monomers for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">CDDIDNMG_01075</span></strong>: X
<div class="collapser collapser-target-CDDIDNMG_01075 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>A-OX (532..1314)</strong>: X
<div class="collapser collapser-target-CDDIDNMG_01075 collapser-level-candidate"> </div><div class="collapser-content">

</div>
</dd>




</div>
<br>

<strong><span class="serif">CDDIDNMG_01077</span></strong>: mal
<div class="collapser collapser-target-CDDIDNMG_01077 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (539..822)</strong>: mal
<div class="collapser collapser-target-CDDIDNMG_01077 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-CDDIDNMG_01077-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 62.5%</dd>
<dd>2-Rhyd-Malonyl-CoA: 58.3%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-CDDIDNMG_01077-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 77.1</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 45.5</dt>

<dd></dd><dt>Methylmalonyl-CoA: 41.9</dt>

<dd></dd><dt>inactive: 38.5</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 27.0</dt>

<dd></dd><dt>Benzoyl-CoA: 15.9</dt>

<dd></dd><dt>fatty_acid: 12.2</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 12.2</dt>

<dd></dd><dt>Isobutyryl-CoA: 10.0</dt>

<dd></dd><dt>Propionyl-CoA: 9.1</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>CHC-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>








</div>

</div>

</div>




<div class="page" id='r52c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_LGIC01000052.1 - Region 1 - hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-58"></div> <span class="help-tooltip" id="region_svg-help-58">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_LGIC01000052.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r52c1-svg" data-filename="NZ_LGIC01000052.1_r52c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 477 - 21,067 nt. (total: 20,591 nt)

<a class="cluster-rules-header" id="r52c1-rules-header" href="#r52c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r52c1-rules">hserlactone: Autoind_synth</div>

<div class="region-svg-container">
<div id='r52c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-59"></div> <span class="help-tooltip" id="focus-panel-help-59">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r52c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r52c1-gene-table-details" data-name="r52c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r52c1-MIBiG-cluster-compare" data-name="r52c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r52c1-clusterblast" data-name="r52c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r52c1-knownclusterblast" data-name="r52c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r52c1-subclusterblast" data-name="r52c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r52c1-tfbs-finder" data-name="r52c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r52c1-pfam-details" data-name="r52c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r52c1-tigrfam-details" data-name="r52c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r52c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-18"></div> <span class="help-tooltip" id="gt-help-tooltip-help-18">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r52c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r52c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r52c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r52c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r52c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-12"></div> <span class="help-tooltip" id="MIBiG-help-12">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r52c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r52c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r52c1"></div>
<div id="comparison-MIBiG-r52c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">hserlactone</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000487: 82-11698">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000487/index.html#r1c1" target="_new">BGC0000487</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>carnocyclin</td>


<td>Carnobacterium maltaromaticum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002447: 0-8196">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002447/index.html#r1c1" target="_new">BGC0002447</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>cerecyclin</td>


<td>Bacillus cereus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554: 0-6546">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>SRO15-3108</td>


<td>Streptomyces filamentosus NRRL 15998</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001992: 0-4749">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001992/index.html#r1c1" target="_new">BGC0001992</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Other</td>

<td>thioguanine</td>


<td>Erwinia amylovora CFBP1430</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002116: 0-6188">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002116/index.html#r1c1" target="_new">BGC0002116</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>darobactin A</td>


<td>Photorhabdus khanii HGB 1456</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001525: 0-16392">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001525/index.html#r1c1" target="_new">BGC0001525</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Other</td>

<td>bartoloside 2, bartoloside 3, bartoloside 4</td>


<td>Synechocystis salina LEGE 06155</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001526: 0-14417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001526/index.html#r1c1" target="_new">BGC0001526</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Other</td>

<td>bartoloside E, bartoloside F, bartoloside G, bartoloside H, bartoloside I, bartoloside J, bartoloside K</td>


<td>Synechocystis salina LEGE 06099</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000548: 588-10610">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000548/index.html#r1c1" target="_new">BGC0000548</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>salivaricin A</td>


<td>Streptococcus salivarius</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002442: 0-8391">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002442/index.html#r1c1" target="_new">BGC0002442</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>toyoncin</td>


<td>Bacillus toyonensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002699: 0-12569">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002699/index.html#r1c1" target="_new">BGC0002699</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>nostolysamide A, nostolysamide B</td>


<td>Nostoc punctiforme PCC 73102</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r52c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000487: 82-11698">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000487/index.html#r1c1" target="_new">BGC0000487</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>RiPP</td>
<td>carnocyclin</td>
<td>Carnobacterium maltaromaticum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002447: 0-8196">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002447/index.html#r1c1" target="_new">BGC0002447</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>RiPP</td>
<td>cerecyclin</td>
<td>Bacillus cereus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554: 0-6546">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>RiPP</td>
<td>SRO15-3108</td>
<td>Streptomyces filamentosus NRRL 15998</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001992: 0-4749">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001992/index.html#r1c1" target="_new">BGC0001992</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Other</td>
<td>thioguanine</td>
<td>Erwinia amylovora CFBP1430</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002116: 0-6188">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002116/index.html#r1c1" target="_new">BGC0002116</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>RiPP</td>
<td>darobactin A</td>
<td>Photorhabdus khanii HGB 1456</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001525: 0-16392">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001525/index.html#r1c1" target="_new">BGC0001525</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Other</td>
<td>bartoloside 2, bartoloside 3, bartoloside 4</td>
<td>Synechocystis salina LEGE 06155</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001526: 0-14417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001526/index.html#r1c1" target="_new">BGC0001526</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Other</td>
<td>bartoloside E, bartoloside F, bartoloside G, bartoloside H, bartoloside I, bartoloside J, bartoloside K</td>
<td>Synechocystis salina LEGE 06099</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000548: 588-10610">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000548/index.html#r1c1" target="_new">BGC0000548</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>salivaricin A</td>
<td>Streptococcus salivarius</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002442: 0-8391">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002442/index.html#r1c1" target="_new">BGC0002442</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>toyoncin</td>
<td>Bacillus toyonensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002699: 0-12569">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002699/index.html#r1c1" target="_new">BGC0002699</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>nostolysamide A, nostolysamide B</td>
<td>Nostoc punctiforme PCC 73102</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r52c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-13"></div> <span class="help-tooltip" id="cb-general-help-13">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r52c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r52c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r52c1_1.svg">
NZ_CP028918 (419188-439689): Gemmobacter sp. HYN0069 chromosome, complete genome (84% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r52c1_2.svg">
NZ_CP034328 (1647435-1669657): Tabrizicola piscis strain K13M18 chromosome, c... (73% of genes show similarity), ectoine,hserlactone
</option>

<option value = "svg/clusterblast_r52c1_3.svg">
NZ_SJZH01000004 (247745-269960): Tabrizicola sp. YIM 78059 Scaffold4, whole g... (73% of genes show similarity), ectoine,hserlactone
</option>

<option value = "svg/clusterblast_r52c1_4.svg">
NZ_CP039964 (1135446-1156034): Pseudorhodobacter sp. S12M18 chromosome, compl... (81% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r52c1_5.svg">
NZ_QKZL01000025 (20358-40949): Hwanghaeicola aestuarii strain DSM 22009 LX81D... (81% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r52c1_6.svg">
NZ_SNYP01000002 (59722-81693): Rhodobacter veldkampii DSM 11550 Ga0310482 102... (80% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r52c1_7.svg">
NZ_AYPY01000003 (103064-123655): Rhodobacter capsulatus YW1 seq0003, whole ge... (81% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r52c1_8.svg">
NZ_QKZO01000006 (234697-255288): Rhodobacter capsulatus strain DSM 1710 LY44D... (81% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r52c1_9.svg">
NC_014034 (348241-368035): Rhodobacter capsulatus SB 1003, complete sequence (84% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r52c1_10.svg">
NZ_OBMT01000001 (652848-673439): Rhodobacter maris strain JA276, whole genome... (84% of genes show similarity), hserlactone
</option>

</select>
<button id= "clusterblast-r52c1-download">Download graphic</button>
</div>
<div id="clusterblast-r52c1-svg"></div>

</div>
</div>

<div class="body-details-section r52c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-14"></div> <span class="help-tooltip" id="cb-known-help-14">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r52c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-15"></div> <span class="help-tooltip" id="cb-sub-help-15">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r52c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-17"></div> <span class="help-tooltip" id="tfbs-body-help-17">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (2):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
BldD
<div class="collapser collapser-target-BldD collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Development and antibiotic global regulator<br>
<span>Score</span>: 18.96 out of 23.43
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r52c1-556-BldD"></div>

<div class="tfbs-hit-detail">
BldD
<div class="collapser collapser-target-BldD collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Development and antibiotic global regulator<br>
<span>Score</span>: 19.22 out of 23.43
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r52c1-11117-BldD"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r52c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-19"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-19">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r52c1-pfam-details-svg" data-filename="NZ_LGIC01000052.1_r52c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r52c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r52c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-20"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-20">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r52c1-tigrfam-details-svg" data-filename="NZ_LGIC01000052.1_r52c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r52c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r52c1-tfbs-finder" data-name="r52c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r52c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-16"></div> <span class="help-tooltip" id="tfbs-side-help-16">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/BldD" target="_blank">BldD</a></td>
<td>Development and antibiotic global regulator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r55c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_LGIC01000055.1 - Region 1 - hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-60"></div> <span class="help-tooltip" id="region_svg-help-60">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_LGIC01000055.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r55c1-svg" data-filename="NZ_LGIC01000055.1_r55c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 17,033 nt. (total: 17,033 nt)

<a class="cluster-rules-header" id="r55c1-rules-header" href="#r55c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r55c1-rules">hserlactone: Autoind_synth</div>

<div class="region-svg-container">
<div id='r55c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-61"></div> <span class="help-tooltip" id="focus-panel-help-61">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r55c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r55c1-gene-table-details" data-name="r55c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r55c1-MIBiG-cluster-compare" data-name="r55c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r55c1-clusterblast" data-name="r55c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r55c1-knownclusterblast" data-name="r55c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r55c1-subclusterblast" data-name="r55c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r55c1-tfbs-finder" data-name="r55c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r55c1-pfam-details" data-name="r55c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r55c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-27"></div> <span class="help-tooltip" id="gt-help-tooltip-help-27">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r55c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r55c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r55c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r55c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r55c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-21"></div> <span class="help-tooltip" id="MIBiG-help-21">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r55c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r55c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r55c1"></div>
<div id="comparison-MIBiG-r55c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">hserlactone</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000043: 0-13097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000043/index.html#r1c1" target="_new">BGC0000043</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>curacin A</td>


<td>Moorea producens 19L</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002026: 219-57197">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002026/index.html#r1c1" target="_new">BGC0002026</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>NRP, Polyketide</td>

<td>thaxteramide A1, thaxteramide A2, thaxteramide B1, thaxteramide B2</td>


<td>Jahnella sp. MSr9139</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000527: 0-12324">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000527/index.html#r1c1" target="_new">BGC0000527</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.44, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>RiPP</td>

<td>mersacidin</td>


<td>Bacillus sp. HIL-Y85/54728</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000547: 618-22455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000547/index.html#r1c1" target="_new">BGC0000547</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.42, order:0.34, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>RiPP</td>

<td>salivaricin 9</td>


<td>Streptococcus salivarius</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000556: 252-11336">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000556/index.html#r1c1" target="_new">BGC0000556</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.43, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>RiPP</td>

<td>streptin</td>


<td>Streptococcus pyogenes</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000506: 405-13975">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000506/index.html#r1c1" target="_new">BGC0000506</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.43, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>RiPP</td>

<td>entianin</td>


<td>Bacillus subtilis subsp. spizizenii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000557: 243-11649">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000557/index.html#r1c1" target="_new">BGC0000557</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.42, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>RiPP</td>

<td>streptococcin A-FF22</td>


<td>Streptococcus pyogenes</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000537: 593-14133">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000537/index.html#r1c1" target="_new">BGC0000537</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.47, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.07</td>
<td>RiPP</td>

<td>nisin U</td>


<td>Streptococcus uberis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001789: 336-13567">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001789/index.html#r1c1" target="_new">BGC0001789</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.46, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>RiPP</td>

<td>suicin 90-1330</td>


<td>Streptococcus suis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000559: 494-13831">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000559/index.html#r1c1" target="_new">BGC0000559</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.45, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>RiPP</td>

<td>subtilin</td>


<td>Bacillus subtilis subsp. spizizenii ATCC 6633</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r55c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000043: 0-13097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000043/index.html#r1c1" target="_new">BGC0000043</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Polyketide</td>
<td>curacin A</td>
<td>Moorea producens 19L</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002026: 219-57197">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002026/index.html#r1c1" target="_new">BGC0002026</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>NRP, Polyketide</td>
<td>thaxteramide A1, thaxteramide A2, thaxteramide B1, thaxteramide B2</td>
<td>Jahnella sp. MSr9139</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000527: 0-12324">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000527/index.html#r1c1" target="_new">BGC0000527</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.44, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>RiPP</td>
<td>mersacidin</td>
<td>Bacillus sp. HIL-Y85/54728</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000547: 618-22455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000547/index.html#r1c1" target="_new">BGC0000547</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.42, order:0.34, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>RiPP</td>
<td>salivaricin 9</td>
<td>Streptococcus salivarius</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000556: 252-11336">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000556/index.html#r1c1" target="_new">BGC0000556</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.43, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>RiPP</td>
<td>streptin</td>
<td>Streptococcus pyogenes</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000506: 405-13975">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000506/index.html#r1c1" target="_new">BGC0000506</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.43, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>RiPP</td>
<td>entianin</td>
<td>Bacillus subtilis subsp. spizizenii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000557: 243-11649">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000557/index.html#r1c1" target="_new">BGC0000557</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.42, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>RiPP</td>
<td>streptococcin A-FF22</td>
<td>Streptococcus pyogenes</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000537: 593-14133">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000537/index.html#r1c1" target="_new">BGC0000537</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.47, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>RiPP</td>
<td>nisin U</td>
<td>Streptococcus uberis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001789: 336-13567">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001789/index.html#r1c1" target="_new">BGC0001789</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.46, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>RiPP</td>
<td>suicin 90-1330</td>
<td>Streptococcus suis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000559: 494-13831">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000559/index.html#r1c1" target="_new">BGC0000559</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.45, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>RiPP</td>
<td>subtilin</td>
<td>Bacillus subtilis subsp. spizizenii ATCC 6633</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r55c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-22"></div> <span class="help-tooltip" id="cb-general-help-22">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r55c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r55c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r55c1_1.svg">
NZ_CP021913 (2949333-2969948): Sagittula sp. P11 chromosome, complete genome (27% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r55c1_2.svg">
NZ_KI421513 (2884121-2904748): Leisingera caerulea DSM 24564 PhacaeDRAFT Scaf... (25% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r55c1_3.svg">
NZ_CP021047 (1372259-1391972): Phaeobacter gallaeciensis strain P128 chromoso... (21% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r55c1_4.svg">
NZ_LXYH01000034 (149028-169733): Rhodobacteraceae bacterium EhC02 A5 Contig4,... (15% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r55c1_5.svg">
NZ_SKCI01000001 (1407982-1428732): Marimonas lutisali strain GH1-19 contig1, ... (13% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r55c1_6.svg">
NZ_KI421507 (2216501-2259285): Pseudophaeobacter arcticus DSM 23566 Phaar neu... (15% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r55c1_7.svg">
NZ_FORY01000004 (82176-102860): Celeribacter halophilus strain CGMCC 1.8891, ... (43% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r55c1_8.svg">
NZ_SWKQ01000009 (3753-21562): Maritimibacter sp. LZ-17 NODE 1, whole genome s... (17% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r55c1_9.svg">
NZ_LJSC01000026 (10205-30955): Gemmobacter sp. LW-1 contig330, whole genome s... (16% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r55c1_10.svg">
NZ_VLLH01000008 (176715-197465): Gemmobacter caeni strain CGMCC 1.7745 IQ03DR... (15% of genes show similarity), hserlactone
</option>

</select>
<button id= "clusterblast-r55c1-download">Download graphic</button>
</div>
<div id="clusterblast-r55c1-svg"></div>

</div>
</div>

<div class="body-details-section r55c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-23"></div> <span class="help-tooltip" id="cb-known-help-23">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r55c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-24"></div> <span class="help-tooltip" id="cb-sub-help-24">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r55c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-26"></div> <span class="help-tooltip" id="tfbs-body-help-26">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 22.16 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r55c1-6276-ZuR"></div>

</div>


</div>
</div>



<div class="body-details-section r55c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-28"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-28">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r55c1-pfam-details-svg" data-filename="NZ_LGIC01000055.1_r55c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r55c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r55c1-tfbs-finder" data-name="r55c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r55c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-25"></div> <span class="help-tooltip" id="tfbs-side-help-25">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r65c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_LGIC01000065.1 - Region 1 - hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-62"></div> <span class="help-tooltip" id="region_svg-help-62">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_LGIC01000065.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r65c1-svg" data-filename="NZ_LGIC01000065.1_r65c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 192,341 - 213,015 nt. (total: 20,675 nt)

<a class="cluster-rules-header" id="r65c1-rules-header" href="#r65c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r65c1-rules">hserlactone: Autoind_synth</div>

<div class="region-svg-container">
<div id='r65c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-63"></div> <span class="help-tooltip" id="focus-panel-help-63">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r65c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r65c1-gene-table-details" data-name="r65c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r65c1-MIBiG-cluster-compare" data-name="r65c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r65c1-clusterblast" data-name="r65c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r65c1-knownclusterblast" data-name="r65c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r65c1-subclusterblast" data-name="r65c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r65c1-tfbs-finder" data-name="r65c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r65c1-pfam-details" data-name="r65c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r65c1-tigrfam-details" data-name="r65c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r65c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-35"></div> <span class="help-tooltip" id="gt-help-tooltip-help-35">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r65c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r65c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r65c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r65c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r65c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-29"></div> <span class="help-tooltip" id="MIBiG-help-29">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r65c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r65c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r65c1"></div>
<div id="comparison-MIBiG-r65c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">hserlactone</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000706: 350-14906">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000706/index.html#r1c1" target="_new">BGC0000706</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.54, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>Saccharide</td>

<td>kanamycin</td>


<td>Streptomyces kanamyceticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.44, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>strobilurin A</td>


<td>Strobilurus tenacellus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000255: 796-9440">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000255/index.html#r1c1" target="_new">BGC0000255</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.44, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>pederin</td>


<td>Uncultured bacterium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001559: 32-13103">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001559/index.html#r1c1" target="_new">BGC0001559</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.46, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>RiPP</td>

<td>crocagin A, crocagin B</td>


<td>Chondromyces crocatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843: 0-23666">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.44, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Other (Non-NRP beta-lactam)</td>

<td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>


<td>Streptomyces clavuligerus ATCC 27064</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001768: 0-28114">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001768/index.html#r1c1" target="_new">BGC0001768</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.42, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.09</td>
<td>NRP</td>

<td>sansalvamide</td>


<td>[Nectria] haematococca mpVI 77-13-4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002362: 1-24294">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002362/index.html#r1c1" target="_new">BGC0002362</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.43, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>Polyketide</td>

<td>loseolamycin A1, loseolamycin A2</td>


<td>Micromonospora endolithica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001151: 0-19936">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001151/index.html#r1c1" target="_new">BGC0001151</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Other (Non-NRP beta-lactam)</td>

<td>valclavam, (-)-2-(2-hydroxyethyl)clavam</td>


<td>Streptomyces antibioticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001578: 0-31150">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001578/index.html#r1c1" target="_new">BGC0001578</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.41, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>duclauxin</td>


<td>Talaromyces stipitatus ATCC 10500</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001650: 0-26129">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001650/index.html#r1c1" target="_new">BGC0001650</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.46, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP</td>

<td>Le-pyrrolopyrazines</td>


<td>Lysobacter enzymogenes</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r65c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000706: 350-14906">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000706/index.html#r1c1" target="_new">BGC0000706</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.54, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>Saccharide</td>
<td>kanamycin</td>
<td>Streptomyces kanamyceticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.44, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>Polyketide</td>
<td>strobilurin A</td>
<td>Strobilurus tenacellus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000255: 796-9440">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000255/index.html#r1c1" target="_new">BGC0000255</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.44, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Polyketide</td>
<td>pederin</td>
<td>Uncultured bacterium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001559: 32-13103">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001559/index.html#r1c1" target="_new">BGC0001559</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.46, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>RiPP</td>
<td>crocagin A, crocagin B</td>
<td>Chondromyces crocatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843: 0-23666">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.44, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>Other (Non-NRP beta-lactam)</td>
<td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>
<td>Streptomyces clavuligerus ATCC 27064</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001768: 0-28114">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001768/index.html#r1c1" target="_new">BGC0001768</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.42, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>NRP</td>
<td>sansalvamide</td>
<td>[Nectria] haematococca mpVI 77-13-4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002362: 1-24294">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002362/index.html#r1c1" target="_new">BGC0002362</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.43, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>Polyketide</td>
<td>loseolamycin A1, loseolamycin A2</td>
<td>Micromonospora endolithica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001151: 0-19936">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001151/index.html#r1c1" target="_new">BGC0001151</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Other (Non-NRP beta-lactam)</td>
<td>valclavam, (-)-2-(2-hydroxyethyl)clavam</td>
<td>Streptomyces antibioticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001578: 0-31150">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001578/index.html#r1c1" target="_new">BGC0001578</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.41, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>Polyketide</td>
<td>duclauxin</td>
<td>Talaromyces stipitatus ATCC 10500</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001650: 0-26129">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001650/index.html#r1c1" target="_new">BGC0001650</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.46, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP</td>
<td>Le-pyrrolopyrazines</td>
<td>Lysobacter enzymogenes</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r65c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-30"></div> <span class="help-tooltip" id="cb-general-help-30">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r65c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r65c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r65c1_1.svg">
NZ_CP042261 (250001-270637): Litoreibacter sp. LN3S51 chromosome, complete ge... (30% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r65c1_2.svg">
NZ_CP025430 (1116032-1144286): Paracoccus zhejiangensis strain J6 chromosome,... (18% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r65c1_3.svg">
NZ_CP010869 (1543716-1564355): Confluentimicrobium sp. EMB200-NS6 strain EMBL... (14% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r65c1_4.svg">
NZ_CP019437 (1554382-1575048): Thioclava nitratireducens strain 25B10 4 chrom... (17% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r65c1_5.svg">
NZ_CP012160 (1933181-1953808): Octadecabacter temperatus strain SB1 chromosom... (13% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r65c1_6.svg">
NZ_ML119084 (46366-67047): Histidinibacterium lentulum strain B17 Scaffold4, ... (15% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r65c1_7.svg">
NZ_QBKN01000001 (223211-243850): Roseivivax pacificus strain DSM 29329 Ga0196... (33% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r65c1_8.svg">
NZ_FNPF01000007 (65815-86499): Citreimonas salinaria strain DSM 26880, whole ... (35% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r65c1_9.svg">
NZ_SLWW01000004 (10424-29255): Rhodovulum euryhalinum strain DSM 4868 Ga03105... (20% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r65c1_10.svg">
NZ_VYQE01000002 (604386-625082): Histidinibacterium sp. 176SS1-4 contig2, who... (14% of genes show similarity), hserlactone
</option>

</select>
<button id= "clusterblast-r65c1-download">Download graphic</button>
</div>
<div id="clusterblast-r65c1-svg"></div>

</div>
</div>

<div class="body-details-section r65c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-31"></div> <span class="help-tooltip" id="cb-known-help-31">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r65c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-32"></div> <span class="help-tooltip" id="cb-sub-help-32">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r65c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-34"></div> <span class="help-tooltip" id="tfbs-body-help-34">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
DasR
<div class="collapser collapser-target-DasR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: N-acetylglucosamine dependent repressor<br>
<span>Score</span>: 21.23 out of 27.04
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r65c1-193932-DasR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
NuR
<div class="collapser collapser-target-NuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Nickel-responsive regulator<br>
<span>Score</span>: 21.09 out of 30.98
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r65c1-210930-NuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r65c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-36"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-36">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r65c1-pfam-details-svg" data-filename="NZ_LGIC01000065.1_r65c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r65c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r65c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-37"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-37">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r65c1-tigrfam-details-svg" data-filename="NZ_LGIC01000065.1_r65c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r65c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r65c1-tfbs-finder" data-name="r65c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r65c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-33"></div> <span class="help-tooltip" id="tfbs-side-help-33">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/DasR" target="_blank">DasR</a></td>
<td>N-acetylglucosamine dependent repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NuR" target="_blank">NuR</a></td>
<td>Nickel-responsive regulator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r68c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_LGIC01000068.1 - Region 1 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-64"></div> <span class="help-tooltip" id="region_svg-help-64">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_LGIC01000068.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r68c1-svg" data-filename="NZ_LGIC01000068.1_r68c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 7,417 - 18,676 nt. (total: 11,260 nt)

<a class="cluster-rules-header" id="r68c1-rules-header" href="#r68c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r68c1-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or TIGR01193 or (YcaO or TIGR03604))</div>

<div class="region-svg-container">
<div id='r68c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-65"></div> <span class="help-tooltip" id="focus-panel-help-65">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r68c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r68c1-gene-table-details" data-name="r68c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r68c1-MIBiG-cluster-compare" data-name="r68c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r68c1-clusterblast" data-name="r68c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r68c1-knownclusterblast" data-name="r68c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r68c1-subclusterblast" data-name="r68c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r68c1-tfbs-finder" data-name="r68c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r68c1-pfam-details" data-name="r68c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r68c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-44"></div> <span class="help-tooltip" id="gt-help-tooltip-help-44">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r68c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r68c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r68c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r68c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r68c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-38"></div> <span class="help-tooltip" id="MIBiG-help-38">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r68c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r68c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r68c1"></div>
<div id="comparison-MIBiG-r68c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000483: 1302-13724">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000483/index.html#r1c1" target="_new">BGC0000483</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.43, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>RiPP</td>

<td>aeruginosamide B, aeruginosamide C</td>


<td>Microcystis aeruginosa PCC 9432</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001174: 248-6052">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001174/index.html#r1c1" target="_new">BGC0001174</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.43, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>RiPP</td>

<td>corynazolicin</td>


<td>Corynebacterium urealyticum DSM 7111</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000481: 0-12515">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000481/index.html#r1c1" target="_new">BGC0000481</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.42, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>RiPP</td>

<td>trichamide</td>


<td>Trichodesmium erythraeum IMS101</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000471: 0-12970">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000471/index.html#r1c1" target="_new">BGC0000471</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.43, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.09</td>
<td>RiPP</td>

<td>viridisamide A</td>


<td>Oscillatoria nigro-viridis PCC 7112</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000565: 2-19346">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000565/index.html#r1c1" target="_new">BGC0000565</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.44, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.08</td>
<td>RiPP</td>

<td>goadsporin</td>


<td>Streptomyces sp. TP-A0584</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002486: 0-10474">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002486/index.html#r1c1" target="_new">BGC0002486</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.43, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>RiPP</td>

<td>tolypamide</td>


<td>Tolypothrix sp. PCC 7601</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000477: 0-11073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000477/index.html#r1c1" target="_new">BGC0000477</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.07</td>
<td>RiPP</td>

<td>patellin 2, patellin 3</td>


<td>uncultured Prochloron sp. 06037A</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000475: 226-12455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000475/index.html#r1c1" target="_new">BGC0000475</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.07</td>
<td>RiPP</td>

<td>patellamide A, patellamide C</td>


<td>Prochloron didemni</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000480: 0-11821">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000480/index.html#r1c1" target="_new">BGC0000480</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.06</td>
<td>RiPP</td>

<td>tenuecyclamide A, tenuecyclamide C</td>


<td>Nostoc spongiaeforme var. tenue str. Carmeli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000473: 867-13221">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000473/index.html#r1c1" target="_new">BGC0000473</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.06</td>
<td>RiPP</td>

<td>microcyclamide</td>


<td>Microcystis aeruginosa NIES-298</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r68c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000483: 1302-13724">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000483/index.html#r1c1" target="_new">BGC0000483</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.43, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>RiPP</td>
<td>aeruginosamide B, aeruginosamide C</td>
<td>Microcystis aeruginosa PCC 9432</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001174: 248-6052">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001174/index.html#r1c1" target="_new">BGC0001174</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.43, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>RiPP</td>
<td>corynazolicin</td>
<td>Corynebacterium urealyticum DSM 7111</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000481: 0-12515">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000481/index.html#r1c1" target="_new">BGC0000481</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.42, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>RiPP</td>
<td>trichamide</td>
<td>Trichodesmium erythraeum IMS101</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000471: 0-12970">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000471/index.html#r1c1" target="_new">BGC0000471</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.43, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>RiPP</td>
<td>viridisamide A</td>
<td>Oscillatoria nigro-viridis PCC 7112</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000565: 2-19346">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000565/index.html#r1c1" target="_new">BGC0000565</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.44, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>RiPP</td>
<td>goadsporin</td>
<td>Streptomyces sp. TP-A0584</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002486: 0-10474">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002486/index.html#r1c1" target="_new">BGC0002486</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.43, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>RiPP</td>
<td>tolypamide</td>
<td>Tolypothrix sp. PCC 7601</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000477: 0-11073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000477/index.html#r1c1" target="_new">BGC0000477</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>RiPP</td>
<td>patellin 2, patellin 3</td>
<td>uncultured Prochloron sp. 06037A</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000475: 226-12455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000475/index.html#r1c1" target="_new">BGC0000475</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>RiPP</td>
<td>patellamide A, patellamide C</td>
<td>Prochloron didemni</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000480: 0-11821">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000480/index.html#r1c1" target="_new">BGC0000480</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>RiPP</td>
<td>tenuecyclamide A, tenuecyclamide C</td>
<td>Nostoc spongiaeforme var. tenue str. Carmeli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000473: 867-13221">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000473/index.html#r1c1" target="_new">BGC0000473</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>RiPP</td>
<td>microcyclamide</td>
<td>Microcystis aeruginosa NIES-298</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r68c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-39"></div> <span class="help-tooltip" id="cb-general-help-39">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r68c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r68c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r68c1_1.svg">
NZ_CP023455 (82687-118350): Rhodobacteraceae bacterium QY30 plasmid unnamed, ... (8% of genes show similarity), LAP
</option>

<option value = "svg/clusterblast_r68c1_2.svg">
NZ_KN676113 (5225425-5256532): Belnapia sp. F-4-1 Scaffold1, whole genome sho... (6% of genes show similarity), thiopeptide
</option>

<option value = "svg/clusterblast_r68c1_3.svg">
NZ_KB895288 (203643-262162): Salinispora pacifica CNT124 C555DRAFT scaffold00... (4% of genes show similarity), LAP,NRPS,thiopeptide
</option>

<option value = "svg/clusterblast_r68c1_4.svg">
NZ_KB896760 (50094-128944): Salinispora pacifica CNT584 C582DRAFT scaffold000... (2% of genes show similarity), LAP,NRPS,terpene,thiopeptide
</option>

<option value = "svg/clusterblast_r68c1_5.svg">
NZ_CP045423 (3720300-3742330): Microvirga sp. HR1 chromosome, complete genome (13% of genes show similarity), TfuA-related
</option>

<option value = "svg/clusterblast_r68c1_6.svg">
NZ_LT598496 (248346-289387): Micromonospora krabiensis strain DSM 45344 chrom... (6% of genes show similarity), LAP,thiopeptide
</option>

<option value = "svg/clusterblast_r68c1_7.svg">
NZ_CP045480 (8745436-8858720): Amycolatopsis sp. YIM 10 chromosome, complete ... (2% of genes show similarity), LAP,NRPS,T1PKS,lanthipeptide,other
</option>

<option value = "svg/clusterblast_r68c1_8.svg">
NZ_CP022774 (1578506-1600750): Ralstonia solanacearum strain T12 chromosome, ... (10% of genes show similarity), LAP
</option>

<option value = "svg/clusterblast_r68c1_9.svg">
NZ_CP023016 (1699945-1721922): Ralstonia solanacearum strain SL3022 chromosom... (9% of genes show similarity), LAP
</option>

<option value = "svg/clusterblast_r68c1_10.svg">
NZ_CP013387 (1604511-1645597): Burkholderia sp. BDU6 chromosome 2, complete s... (5% of genes show similarity), other
</option>

</select>
<button id= "clusterblast-r68c1-download">Download graphic</button>
</div>
<div id="clusterblast-r68c1-svg"></div>

</div>
</div>

<div class="body-details-section r68c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-40"></div> <span class="help-tooltip" id="cb-known-help-40">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r68c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-41"></div> <span class="help-tooltip" id="cb-sub-help-41">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r68c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-43"></div> <span class="help-tooltip" id="tfbs-body-help-43">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 20.18 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r68c1-12378-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r68c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-45"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-45">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r68c1-pfam-details-svg" data-filename="NZ_LGIC01000068.1_r68c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r68c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r68c1-tfbs-finder" data-name="r68c1-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r68c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-42"></div> <span class="help-tooltip" id="tfbs-side-help-42">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r90c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_LGIC01000090.1 - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-66"></div> <span class="help-tooltip" id="region_svg-help-66">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_LGIC01000090.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r90c1-svg" data-filename="NZ_LGIC01000090.1_r90c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 37,514 nt. (total: 37,514 nt)

<a class="cluster-rules-header" id="r90c1-rules-header" href="#r90c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r90c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r90c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-67"></div> <span class="help-tooltip" id="focus-panel-help-67">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r90c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r90c1-gene-table-details" data-name="r90c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r90c1-nrps_pks" data-name="r90c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r90c1-MIBiG-cluster-compare" data-name="r90c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r90c1-clusterblast" data-name="r90c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r90c1-knownclusterblast" data-name="r90c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r90c1-subclusterblast" data-name="r90c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r90c1-tfbs-finder" data-name="r90c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r90c1-pfam-details" data-name="r90c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r90c1-tigrfam-details" data-name="r90c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r90c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-53"></div> <span class="help-tooltip" id="gt-help-tooltip-help-53">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r90c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r90c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r90c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r90c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r90c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-46"></div> <span class="help-tooltip" id="nrps-domain-help-46">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r90c1-details-svg" data-filename="NZ_LGIC01000090.1_r90c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r90c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r90c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-47"></div> <span class="help-tooltip" id="MIBiG-help-47">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r90c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r90c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r90c1"></div>
<div id="comparison-MIBiG-r90c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002476: 0-10330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002476/index.html#r1c1" target="_new">BGC0002476</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Escherichia coli str. K-12 substr. MG1655</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>indigoidine</td>


<td>Streptomyces chromofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.49, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>rhizomide A, rhizomide B, rhizomide C</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.49, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>kolossin</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131: 0-29471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.48, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>ambactin</td>


<td>Xenorhabdus miraniensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.48, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.48, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002494: 273-12179">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002494/index.html#r1c1" target="_new">BGC0002494</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>vibriobactin</td>


<td>Vibrio cholerae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.48, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.19</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r90c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002476: 0-10330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002476/index.html#r1c1" target="_new">BGC0002476</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Escherichia coli str. K-12 substr. MG1655</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>indigoidine</td>
<td>Streptomyces chromofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.49, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>rhizomide A, rhizomide B, rhizomide C</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.49, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>kolossin</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131: 0-29471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.48, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>ambactin</td>
<td>Xenorhabdus miraniensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.48, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>bicornutin A1, bicornutin A2</td>
<td>Xenorhabdus budapestensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.48, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002494: 273-12179">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002494/index.html#r1c1" target="_new">BGC0002494</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>vibriobactin</td>
<td>Vibrio cholerae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.48, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>gamexpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r90c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-48"></div> <span class="help-tooltip" id="cb-general-help-48">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r90c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r90c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r90c1_1.svg">
NZ_JHXQ01000007 (35314-94993): Rhizobium undicola ORS 992 = ATCC 700741 T424D... (17% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r90c1_2.svg">
NZ_PVUF01000001 (423788-467661): Epibacterium scottomollicae strain DSM 25328... (22% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r90c1_3.svg">
NZ_JACHYR010000003 (143561-187563): Rhizobium sp. BK007 Ga0365242 03, whole g... (17% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r90c1_4.svg">
NZ_JACIHI010000002 (35270-79263): Rhizobium esperanzae strain SEMIA 414 Ga039... (15% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r90c1_5.svg">
NZ_UCKM01000008 (262662-306643): Rhizobiales bacterium isolate AFS091992, who... (21% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r90c1_6.svg">
NZ_ATYN01000008 (190101-234094): Rhizobium leguminosarum bv. viciae UPM1137 Y... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r90c1_7.svg">
NZ_KB905374 (380224-424217): Rhizobium leguminosarum bv. phaseoli 4292 Rleg18... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r90c1_8.svg">
NZ_SILA01000002 (146129-190122): Rhizobium leguminosarum strain SM45 plasmid ... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r90c1_9.svg">
NZ_SIQY01000002 (16002-59995): Rhizobium leguminosarum strain SM168B plasmid ... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r90c1_10.svg">
NZ_SILH01000002 (547972-591965): Rhizobium leguminosarum strain SM52 plasmid ... (13% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r90c1-download">Download graphic</button>
</div>
<div id="clusterblast-r90c1-svg"></div>

</div>
</div>

<div class="body-details-section r90c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-49"></div> <span class="help-tooltip" id="cb-known-help-49">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r90c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r90c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r90c1_1.svg">
rhizomide A/rhizomide B/rhizomide C
</option>

<option value = "svg/knownclusterblast_r90c1_2.svg">
kolossin
</option>

<option value = "svg/knownclusterblast_r90c1_3.svg">
oryzanaphthopyran A/oryzanaphthopyran B/oryzanaphthopyran C/oryzanthrone A/oryzanthrone B/chlororyzanthrone A/chlororyzanthrone B
</option>

</select>
<button id="knownclusterblast-r90c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r90c1-svg"></div>

</div>
</div>

<div class="body-details-section r90c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-50"></div> <span class="help-tooltip" id="cb-sub-help-50">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r90c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-52"></div> <span class="help-tooltip" id="tfbs-body-help-52">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AbrC3
<div class="collapser collapser-target-AbrC3 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Antibiotic production activator<br>
<span>Score</span>: 18.16 out of 18.16
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r90c1-36491-AbrC3"></div>

<div class="tfbs-hit-detail">
CatR
<div class="collapser collapser-target-CatR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: H2O2-responsive repressor<br>
<span>Score</span>: 19.79 out of 25.31
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r90c1-36513-CatR"></div>

<div class="tfbs-hit-detail">
CatR
<div class="collapser collapser-target-CatR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: H2O2-responsive repressor<br>
<span>Score</span>: 20.17 out of 25.31
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r90c1-36514-CatR"></div>

<div class="tfbs-hit-detail">
AbrC3
<div class="collapser collapser-target-AbrC3 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Antibiotic production activator<br>
<span>Score</span>: 18.07 out of 18.16
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r90c1-37352-AbrC3"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
LexA
<div class="collapser collapser-target-LexA collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Repressor of DNA damage response<br>
<span>Score</span>: 20.59 out of 29.65
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r90c1-1285-LexA"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r90c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-54"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-54">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r90c1-pfam-details-svg" data-filename="NZ_LGIC01000090.1_r90c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r90c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r90c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-55"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-55">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r90c1-tigrfam-details-svg" data-filename="NZ_LGIC01000090.1_r90c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r90c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">












<div class="sidepanel-details-header r90c1-tfbs-finder" data-name="r90c1-tfbs-finder">
TFBS Finder
</div>




</div>












<div class="sidepanel-details-section r90c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-51"></div> <span class="help-tooltip" id="tfbs-side-help-51">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/LexA" target="_blank">LexA</a></td>
<td>Repressor of DNA damage response</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AbrC3" target="_blank">AbrC3</a></td>
<td>Antibiotic production activator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/CatR" target="_blank">CatR</a></td>
<td>H2O2-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>