
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>GCF_002871005.1_Acidimangrovimonas_sediminis_MS2-2 - 5 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/bacteria.css">
<meta property="og:title" content="GCF_002871005.1_Acidimangrovimonas_sediminis_MS2-2 - 5 record(s) - 5 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-8c423f70-912a-4789-8c9d-cbccc86b7f0b/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://antismash.secondarymetabolites.org/">
<img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://antismash.secondarymetabolites.org/">
antiSMASH version 7.0.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="GCF_002871005.1_Acidimangrovimonas_sediminis_MS2-2.zip">Download all results</a></li>

<li><a href="GCF_002871005.1_Acidimangrovimonas_sediminis_MS2-2.gbk">Download GenBank summary file</a></li>

<li><a href="bacteria-8c423f70-912a-4789-8c9d-cbccc86b7f0b.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton hybrid r1c1">
<a href="#r1c1">1.1</a>
</div>



<div class="regbutton PKS T3PKS r5c1">
<a href="#r5c1">5.1</a>
</div>



<div class="regbutton other ectoine r19c1">
<a href="#r19c1">19.1</a>
</div>



<div class="regbutton terpene terpene r29c1">
<a href="#r29c1">29.1</a>
</div>



<div class="regbutton other hserlactone r38c1">
<a href="#r38c1">38.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>NZ_PHSN01000001.1</strong> (Acidimangrovimonas sediminis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-1" data-filename="NZ_PHSN01000001.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-1" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r1c1">
<td class="regbutton hybrid r1c1">
<a href="#r1c1">Region&nbsp1.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">40,561</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_PHSN01000014.1</strong> (Acidimangrovimonas sediminis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-5" data-filename="NZ_PHSN01000014.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-5" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r5c1">
<td class="regbutton PKS T3PKS r5c1">
<a href="#r5c1">Region&nbsp5.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
</td>
<td class="digits">14,966</td>
<td class="digits table-split-left">56,036</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_PHSN01000027.1</strong> (Acidimangrovimonas sediminis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-19" data-filename="NZ_PHSN01000027.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-19" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r19c1">
<td class="regbutton other ectoine r19c1">
<a href="#r19c1">Region&nbsp19.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
</td>
<td class="digits">67,983</td>
<td class="digits table-split-left">78,384</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000860/1" target="_blank">ectoine</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_PHSN01000004.1</strong> (Acidimangrovimonas sediminis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-29" data-filename="NZ_PHSN01000004.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-29" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r29c1">
<td class="regbutton terpene terpene r29c1">
<a href="#r29c1">Region&nbsp29.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">419,330</td>
<td class="digits table-split-left">440,091</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_PHSN01000005.1</strong> (Acidimangrovimonas sediminis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-38" data-filename="NZ_PHSN01000005.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-38" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r38c1">
<td class="regbutton other hserlactone r38c1">
<a href="#r38c1">Region&nbsp38.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">229,956</td>
<td class="digits table-split-left">250,771</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r1c1">
<td class="regbutton hybrid r1c1">
<a href="#r1c1">Region&nbsp1.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">40,561</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r5c1">
<td class="regbutton PKS T3PKS r5c1">
<a href="#r5c1">Region&nbsp5.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
</td>
<td class="digits">14,966</td>
<td class="digits table-split-left">56,036</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r19c1">
<td class="regbutton other ectoine r19c1">
<a href="#r19c1">Region&nbsp19.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
</td>
<td class="digits">67,983</td>
<td class="digits table-split-left">78,384</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000860/1" target="_blank">ectoine</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>


<tr class="linked-row even" data-anchor="#r29c1">
<td class="regbutton terpene terpene r29c1">
<a href="#r29c1">Region&nbsp29.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">419,330</td>
<td class="digits table-split-left">440,091</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r38c1">
<td class="regbutton other hserlactone r38c1">
<a href="#r38c1">Region&nbsp38.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">229,956</td>
<td class="digits table-split-left">250,771</td>

<td colspan="3"></td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>NZ_PHSN01000010.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000012.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000013.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000015.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000016.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000017.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000018.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000019.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000002.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000020.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000021.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000022.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000023.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000024.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000025.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000026.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000028.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000029.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000030.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000031.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000033.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000035.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000037.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000038.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000039.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000040.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000042.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000043.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000044.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000045.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000047.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000048.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000049.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000050.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000051.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000052.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000053.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000054.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000055.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000056.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000057.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000059.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000006.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000060.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000061.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000062.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000063.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000064.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000065.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000066.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000067.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000068.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000069.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000007.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000070.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000073.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000074.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000075.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000008.1</strong> (Acidimangrovimonas sediminis)</dd>

<dd><strong>NZ_PHSN01000009.1</strong> (Acidimangrovimonas sediminis)</dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>







<div class="page" id='r1c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_PHSN01000001.1 - Region 1 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-51"></div> <span class="help-tooltip" id="region_svg-help-51">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_PHSN01000001.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c1-svg" data-filename="NZ_PHSN01000001.1_r1c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 40,561 nt. (total: 40,561 nt)

<a class="cluster-rules-header" id="r1c1-rules-header" href="#r1c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r1c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r1c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-52"></div> <span class="help-tooltip" id="focus-panel-help-52">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c1-gene-table-details" data-name="r1c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c1-nrps_pks" data-name="r1c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c1-MIBiG-cluster-compare" data-name="r1c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c1-clusterblast" data-name="r1c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c1-knownclusterblast" data-name="r1c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c1-subclusterblast" data-name="r1c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c1-tfbs-finder" data-name="r1c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r1c1-nrps-pks-bubbles" data-name="r1c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r1c1-pfam-details" data-name="r1c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r1c1-tigrfam-details" data-name="r1c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r1c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-11"></div> <span class="help-tooltip" id="gt-help-tooltip-help-11">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-1"></div> <span class="help-tooltip" id="nrps-domain-help-1">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c1-details-svg" data-filename="NZ_PHSN01000001.1_r1c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-2"></div> <span class="help-tooltip" id="MIBiG-help-2">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c1"></div>
<div id="comparison-MIBiG-r1c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457: 0-11274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.56</td>
<td>NRP</td>

<td>vicibactin</td>


<td>Rhizobium etli CFN 42</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000741: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000741/index.html#r1c1" target="_new">BGC0000741</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.48, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.48, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.53</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000744: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000744/index.html#r1c1" target="_new">BGC0000744</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.48, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.48, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.53</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000745: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000745/index.html#r1c1" target="_new">BGC0000745</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.48, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.48, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.53</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000742: 0-2815">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000742/index.html#r1c1" target="_new">BGC0000742</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.48, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.48, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.53</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000743: 0-2815">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000743/index.html#r1c1" target="_new">BGC0000743</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.48, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.48, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.53</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.52</td>
<td>NRP</td>

<td>enniatin</td>


<td>Fusarium equiseti</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:1.00, components:0.49)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>






<td class="cc-heat-cell" title="0.59 (id:0.42, order:1.00, components:0.49)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.52</td>
<td>NRP</td>

<td>taxlllaid A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001410: 97-16791">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001410/index.html#r1c1" target="_new">BGC0001410</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>






<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.52</td>
<td>Saccharide</td>

<td>polysaccharide A</td>


<td>Bacteroides fragilis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002017: 0-12514">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002017/index.html#r1c1" target="_new">BGC0002017</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>






<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.51</td>
<td>NRP</td>

<td>potensibactin</td>


<td>Nocardiopsis potens DSM 45234</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457: 0-11274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>vicibactin</td>
<td>Rhizobium etli CFN 42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000741: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000741/index.html#r1c1" target="_new">BGC0000741</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.48, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000744: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000744/index.html#r1c1" target="_new">BGC0000744</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.48, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000745: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000745/index.html#r1c1" target="_new">BGC0000745</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.48, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000742: 0-2815">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000742/index.html#r1c1" target="_new">BGC0000742</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.48, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000743: 0-2815">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000743/index.html#r1c1" target="_new">BGC0000743</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.48, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>enniatin</td>
<td>Fusarium equiseti</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:1.00, components:0.49)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>taxlllaid A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001410: 97-16791">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001410/index.html#r1c1" target="_new">BGC0001410</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Saccharide</td>
<td>polysaccharide A</td>
<td>Bacteroides fragilis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002017: 0-12514">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002017/index.html#r1c1" target="_new">BGC0002017</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>potensibactin</td>
<td>Nocardiopsis potens DSM 45234</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-3"></div> <span class="help-tooltip" id="cb-general-help-3">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c1_1.svg">
NZ_FNOB01000009 (14393-66889): Acidimangrovimonas indica strain DSM 24802, wh... (22% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c1_2.svg">
NZ_LR701544 (75038-126083): Roseovarius sp. EC-HK134 isolate RV134, whole gen... (19% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r1c1_3.svg">
NZ_VNKR01000088 (260711-313238): Pseudoruegeria sp. M32A2M Ps Sc 9, whole gen... (14% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c1_4.svg">
NZ_CP025583 (1662544-1715115): Paracoccus jeotgali strain CBA4604 chromosome,... (11% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c1_5.svg">
NZ_CP024314 (813403-865899): Rhizobium sp. NXC24 plasmid pRspNXC24c, complete... (17% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c1_6.svg">
NZ_UCMW01000014 (30023-82663): Rhizobiales bacterium isolate R1607 A, whole g... (18% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c1_7.svg">
NZ_UBNV01000007 (98050-150690): Rhizobiales bacterium isolate AFS017416, whol... (18% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c1_8.svg">
NZ_CP042826 (1891906-1944543): Rhizobium sp. WL3 chromosome, complete genome (18% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c1_9.svg">
NZ_CP024310 (992021-1044775): Sinorhizobium fredii strain NXT3 plasmid pSfreN... (16% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r1c1_10.svg">
NZ_JABEKT010000004 (411279-464111): Ensifer adhaerens strain Lc04 Scaffold 4,... (13% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r1c1-download">Download graphic</button>
</div>
<div id="clusterblast-r1c1-svg"></div>

</div>
</div>

<div class="body-details-section r1c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-4"></div> <span class="help-tooltip" id="cb-known-help-4">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-5"></div> <span class="help-tooltip" id="cb-sub-help-5">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-9"></div> <span class="help-tooltip" id="tfbs-body-help-9">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AfsR
<div class="collapser collapser-target-AfsR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Pleiotropic regulatory for antibiotic production<br>
<span>Score</span>: 18.76 out of 23.05
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r1c1-32028-AfsR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (4):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
LexA
<div class="collapser collapser-target-LexA collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Repressor of DNA damage response<br>
<span>Score</span>: 20.50 out of 29.65
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r1c1-22885-LexA"></div>

<div class="tfbs-hit-detail">
NuR
<div class="collapser collapser-target-NuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Nickel-responsive regulator<br>
<span>Score</span>: 21.50 out of 30.98
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r1c1-30666-NuR"></div>

<div class="tfbs-hit-detail">
NuR
<div class="collapser collapser-target-NuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Nickel-responsive regulator<br>
<span>Score</span>: 21.60 out of 30.98
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r1c1-32045-NuR"></div>

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 20.03 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r1c1-35848-NrtR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r1c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-10"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-10">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c1-domain-bubble-svg" data-filename="NZ_PHSN01000001.1_r1c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r1c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (0 - 40561): chemical hybrid T1PKS-NRPS
</option>


</select>

<div class="details-svg" id="r1c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r1c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r1c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-12"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-12">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c1-pfam-details-svg" data-filename="NZ_PHSN01000001.1_r1c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r1c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-13"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-13">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c1-tigrfam-details-svg" data-filename="NZ_PHSN01000001.1_r1c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r1c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c1-nrps_pks_products" data-name="r1c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r1c1-nrps_pks_monomers" data-name="r1c1-nrps_pks_monomers">
NRPS/PKS monomers
</div>



<div class="sidepanel-details-header r1c1-tfbs-finder" data-name="r1c1-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r1c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-6"></div> <span class="help-tooltip" id="nrps-structures-help-6">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 0 - 40561:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c1-1" class="smiles-canvas" data-smiles="NC(Cc1ccccc1)C(=O)CC(=O)NCC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Phe - mal - Gly)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe,gly" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*,gly*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(Cc1ccccc1)C(=O)CC(=O)NCC(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r1c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS monomer predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-7"></div> <span class="help-tooltip" id="nrps-monomers-help-7">Shows the predicted monomers for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">IFJPPCDH_00007</span></strong>: Phe - mal - Gly
<div class="collapser collapser-target-IFJPPCDH_00007 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe,gly" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*,gly*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (39..447)</strong>: Phe
<div class="collapser collapser-target-IFJPPCDH_00007 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Phe
<div class="collapser collapser-target-IFJPPCDH_00007-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>Unpolar aromatic ring (Phe, Trp)</dd>
<dt>Single AA prediction:</dt>
<dd>phe (Phe)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DGFLVGVVVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ile <span class="serif">DGFFLGVVYK</span> (53% 8 match)</dd>
<dd>Val <span class="serif">DGFLLGEVCK</span> (50% 8 match)</dd>
<dd>Trp <span class="serif">DVALVGVVQK</span> (44% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>PKS_AT (1166..1457)</strong>: mal
<div class="collapser collapser-target-IFJPPCDH_00007 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-IFJPPCDH_00007-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>inactive: 75.0%</dd>
<dd>Methylmalonyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-IFJPPCDH_00007-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 130.8</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 67.4</dt>

<dd></dd><dt>inactive: 66.0</dt>

<dd></dd><dt>Methylmalonyl-CoA: 59.7</dt>

<dd></dd><dt>Propionyl-CoA: 36.6</dt>

<dd></dd><dt>Isobutyryl-CoA: 32.0</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 31.8</dt>

<dd></dd><dt>fatty_acid: 28.3</dt>

<dd></dd><dt>trans-1,2-CPDA: 22.3</dt>

<dd></dd><dt>Benzoyl-CoA: 21.6</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 19.0</dt>

<dd></dd><dt>CHC-CoA: 18.6</dt>

<dd></dd><dt>Acetyl-CoA: 16.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 15.5</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>









<dd><strong>AMP-binding (2729..3108)</strong>: Gly
<div class="collapser collapser-target-IFJPPCDH_00007 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly
<div class="collapser collapser-target-IFJPPCDH_00007-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Tiny, hydrophilic, transition to aliphatic (Gly, Ala)</dd>
<dt>Single AA prediction:</dt>
<dd>gly (Gly)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DILQLGLIWK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Gly <span class="serif">DILQLGLIWK</span> (91% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r1c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-8"></div> <span class="help-tooltip" id="tfbs-side-help-8">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/LexA" target="_blank">LexA</a></td>
<td>Repressor of DNA damage response</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NuR" target="_blank">NuR</a></td>
<td>Nickel-responsive regulator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AfsR" target="_blank">AfsR</a></td>
<td>Pleiotropic regulatory for antibiotic production</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrtR" target="_blank">NrtR</a></td>
<td>NAD synthesis repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>




<div class="page" id='r5c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_PHSN01000014.1 - Region 1 - T3PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-53"></div> <span class="help-tooltip" id="region_svg-help-53">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_PHSN01000014.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c1-svg" data-filename="NZ_PHSN01000014.1_r5c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 14,966 - 56,036 nt. (total: 41,071 nt)

<a class="cluster-rules-header" id="r5c1-rules-header" href="#r5c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c1-rules">T3PKS: (Chal_sti_synt_C or Chal_sti_synt_N)</div>

<div class="region-svg-container">
<div id='r5c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-54"></div> <span class="help-tooltip" id="focus-panel-help-54">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c1-gene-table-details" data-name="r5c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c1-nrps_pks" data-name="r5c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c1-MIBiG-cluster-compare" data-name="r5c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c1-clusterblast" data-name="r5c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c1-knownclusterblast" data-name="r5c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c1-subclusterblast" data-name="r5c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c1-tfbs-finder" data-name="r5c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r5c1-pfam-details" data-name="r5c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r5c1-tigrfam-details" data-name="r5c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r5c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-21"></div> <span class="help-tooltip" id="gt-help-tooltip-help-21">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-14"></div> <span class="help-tooltip" id="nrps-domain-help-14">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c1-details-svg" data-filename="NZ_PHSN01000014.1_r5c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-15"></div> <span class="help-tooltip" id="MIBiG-help-15">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c1"></div>
<div id="comparison-MIBiG-r5c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T3PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000282: 0-2613">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000282/index.html#r1c1" target="_new">BGC0000282</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.48, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.37</td>
<td>Polyketide</td>

<td>alkylresorcinol</td>


<td>Streptomyces griseus subsp. griseus NBRC 13350</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001938: 0-1819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001938/index.html#r1c1" target="_new">BGC0001938</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>entotheonella</td>


<td>Candidatus Entotheonella serta</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001454: 0-1185">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001454/index.html#r1c1" target="_new">BGC0001454</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Other</td>

<td>germicidin</td>


<td>Streptomyces argillaceus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001928: 108-8402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001928/index.html#r1c1" target="_new">BGC0001928</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Other</td>

<td>(2R,3S,4S)-5-fluoro-2,3,4-trihydroxypentanoic acid</td>


<td>Streptomyces sp. MA37</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000867: 21-5822">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000867/index.html#r1c1" target="_new">BGC0000867</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Other</td>

<td>polyhydroxyalkanoic acids</td>


<td>Ectothiorhodospira shaposhnikovii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002332: 137-13618">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002332/index.html#r1c1" target="_new">BGC0002332</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>tetraketide resorcinols</td>


<td>Candidatus Entotheonella sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001850: 1068-23493">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001850/index.html#r1c1" target="_new">BGC0001850</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.46, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>Other (Shikimate-derived)</td>

<td>diazaquinomycin A, diazaquinomycin E, diazaquinomycin F, diazaquinomycin G</td>


<td>Streptomyces sp. F001</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000866: 467-5165">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000866/index.html#r1c1" target="_new">BGC0000866</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Other</td>

<td>polyhydroxyalkanoate</td>


<td>Burkholderia sp. DSM 9242</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831: 1-6152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.47, order:0.48, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>alkylpyrone-407, alkylpyrone-393</td>


<td>Myxococcus xanthus DK 1622</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002493: 1205-6076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002493/index.html#r1c1" target="_new">BGC0002493</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>NRP</td>

<td>parabactin</td>


<td>Paracoccus denitrificans PD1222</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000282: 0-2613">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000282/index.html#r1c1" target="_new">BGC0000282</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.48, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Polyketide</td>
<td>alkylresorcinol</td>
<td>Streptomyces griseus subsp. griseus NBRC 13350</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001938: 0-1819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001938/index.html#r1c1" target="_new">BGC0001938</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>entotheonella</td>
<td>Candidatus Entotheonella serta</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001454: 0-1185">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001454/index.html#r1c1" target="_new">BGC0001454</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>germicidin</td>
<td>Streptomyces argillaceus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001928: 108-8402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001928/index.html#r1c1" target="_new">BGC0001928</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Other</td>
<td>(2R,3S,4S)-5-fluoro-2,3,4-trihydroxypentanoic acid</td>
<td>Streptomyces sp. MA37</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000867: 21-5822">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000867/index.html#r1c1" target="_new">BGC0000867</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Other</td>
<td>polyhydroxyalkanoic acids</td>
<td>Ectothiorhodospira shaposhnikovii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002332: 137-13618">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002332/index.html#r1c1" target="_new">BGC0002332</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>tetraketide resorcinols</td>
<td>Candidatus Entotheonella sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001850: 1068-23493">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001850/index.html#r1c1" target="_new">BGC0001850</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.46, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other (Shikimate-derived)</td>
<td>diazaquinomycin A, diazaquinomycin E, diazaquinomycin F, diazaquinomycin G</td>
<td>Streptomyces sp. F001</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000866: 467-5165">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000866/index.html#r1c1" target="_new">BGC0000866</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>polyhydroxyalkanoate</td>
<td>Burkholderia sp. DSM 9242</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831: 1-6152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.47, order:0.48, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>alkylpyrone-407, alkylpyrone-393</td>
<td>Myxococcus xanthus DK 1622</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002493: 1205-6076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002493/index.html#r1c1" target="_new">BGC0002493</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>NRP</td>
<td>parabactin</td>
<td>Paracoccus denitrificans PD1222</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-16"></div> <span class="help-tooltip" id="cb-general-help-16">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c1_1.svg">
NZ_CP010859 (110641-151703): Marinovum algicola DG 898 plasmid pMaD4, complet... (16% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r5c1_2.svg">
NZ_FNYY01000009 (73419-114481): Marinovum algicola strain FF3, whole genome s... (16% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r5c1_3.svg">
NZ_JACHGP010000001 (355302-396373): Mycobacterium sp. AZCC 0083 Ga0417319 01,... (20% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r5c1_4.svg">
NZ_JIAW01000021 (41535-82606): Mycobacterium sp. URHB0044 N536DRAFT scaffold0... (17% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r5c1_5.svg">
NZ_FNZQ01000002 (212474-253524): Jannaschia helgolandensis strain DSM 14858, ... (18% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r5c1_6.svg">
NZ_JACHXN010000024 (7800-48853): Phyllobacterium trifolii strain CECT 7015 Ga... (23% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r5c1_7.svg">
NZ_JACJMY010000002 (247578-288631): UNVERIFIED ORG: Mycolicibacterium obuense... (14% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r5c1_8.svg">
NZ_AP022617 (3697637-3738690): Mycolicibacterium monacense strain JCM 15658 (14% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r5c1_9.svg">
NZ_LIPG01000004 (385534-426593): Streptomyces sp. CB02058 scaffold4, whole ge... (16% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r5c1_10.svg">
NZ_SGXK01000002 (690579-731650): Mycobacterium sp. BK558 Ga0307718 102, whole... (13% of genes show similarity), T3PKS
</option>

</select>
<button id= "clusterblast-r5c1-download">Download graphic</button>
</div>
<div id="clusterblast-r5c1-svg"></div>

</div>
</div>

<div class="body-details-section r5c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-17"></div> <span class="help-tooltip" id="cb-known-help-17">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-18"></div> <span class="help-tooltip" id="cb-sub-help-18">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-20"></div> <span class="help-tooltip" id="tfbs-body-help-20">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 21.67 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r5c1-20662-ZuR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (4):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 19.83 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r5c1-21025-ZuR"></div>

<div class="tfbs-hit-detail">
ArgR
<div class="collapser collapser-target-ArgR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Regulator of arginine biosynthesis genes<br>
<span>Score</span>: 17.82 out of 20.01
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r5c1-28073-ArgR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 17.60 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r5c1-32267-ZuR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 18.69 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r5c1-43055-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r5c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-22"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-22">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c1-pfam-details-svg" data-filename="NZ_PHSN01000014.1_r5c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r5c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-23"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-23">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c1-tigrfam-details-svg" data-filename="NZ_PHSN01000014.1_r5c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r5c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">












<div class="sidepanel-details-header r5c1-tfbs-finder" data-name="r5c1-tfbs-finder">
TFBS Finder
</div>




</div>












<div class="sidepanel-details-section r5c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-19"></div> <span class="help-tooltip" id="tfbs-side-help-19">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ArgR" target="_blank">ArgR</a></td>
<td>Regulator of arginine biosynthesis genes</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r19c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_PHSN01000027.1 - Region 1 - ectoine <div class="help-container"> <div class="help-icon" data-id="region_svg-help-55"></div> <span class="help-tooltip" id="region_svg-help-55">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_PHSN01000027.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r19c1-svg" data-filename="NZ_PHSN01000027.1_r19c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 67,983 - 78,384 nt. (total: 10,402 nt)

<a class="cluster-rules-header" id="r19c1-rules-header" href="#r19c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r19c1-rules">ectoine: ectoine_synt</div>

<div class="region-svg-container">
<div id='r19c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-56"></div> <span class="help-tooltip" id="focus-panel-help-56">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r19c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r19c1-gene-table-details" data-name="r19c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r19c1-MIBiG-cluster-compare" data-name="r19c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r19c1-clusterblast" data-name="r19c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r19c1-knownclusterblast" data-name="r19c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r19c1-subclusterblast" data-name="r19c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r19c1-tfbs-finder" data-name="r19c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r19c1-pfam-details" data-name="r19c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r19c1-tigrfam-details" data-name="r19c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r19c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-30"></div> <span class="help-tooltip" id="gt-help-tooltip-help-30">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r19c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r19c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r19c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r19c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r19c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-24"></div> <span class="help-tooltip" id="MIBiG-help-24">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r19c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r19c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r19c1"></div>
<div id="comparison-MIBiG-r19c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">ectoine</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860: 710-5134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860</a></td>





<td class="cc-heat-cell" title="0.83 (id:0.58, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>


<td>0.59</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylarcula marina</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856: 517-4347">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856</a></td>





<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>


<td>0.51</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylophaga thalassica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855: 245-3455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855</a></td>





<td class="cc-heat-cell" title="0.72 (id:0.50, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>


<td>0.40</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium kenyense</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>





<td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>


<td>0.38</td>
<td>Other</td>

<td>ectoine</td>


<td>Sporosarcina pasteurii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.37</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylobacter marinus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857: 300-4871">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.50, order:0.64, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylophaga alcalica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853: 8-3345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.48, order:0.76, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Other</td>

<td>ectoine</td>


<td>Streptomyces anulatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052: 188-3620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.48, order:0.76, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Other (Ectoine)</td>

<td>ectoine</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859: 258-3305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium alcaliphilum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854: 838-6879">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.49, order:0.57, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium alcaliphilum 20Z</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r19c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860: 710-5134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860</a></td>


<td class="cc-heat-cell" title="0.83 (id:0.58, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
<td>0.83</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylarcula marina</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856: 517-4347">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856</a></td>


<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
<td>0.79</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylophaga thalassica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855: 245-3455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855</a></td>


<td class="cc-heat-cell" title="0.72 (id:0.50, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
<td>0.72</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium kenyense</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>


<td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
<td>0.71</td>
<td>Other</td>
<td>ectoine</td>
<td>Sporosarcina pasteurii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylobacter marinus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857: 300-4871">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.50, order:0.64, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylophaga alcalica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853: 8-3345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.48, order:0.76, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other</td>
<td>ectoine</td>
<td>Streptomyces anulatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052: 188-3620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.48, order:0.76, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other (Ectoine)</td>
<td>ectoine</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859: 258-3305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium alcaliphilum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854: 838-6879">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.49, order:0.57, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium alcaliphilum 20Z</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r19c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-25"></div> <span class="help-tooltip" id="cb-general-help-25">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r19c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r19c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r19c1_1.svg">
NZ_FXBQ01000001 (2565371-2575770): Maritimibacter sp. HL-12, whole genome sho... (46% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r19c1_2.svg">
NZ_PHNT01000004 (68118-78511): Roseovarius salinarum strain N53 contig4, whol... (60% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r19c1_3.svg">
NZ_AUIJ01000007 (23263-33656): Sediminimonas qiaohouensis DSM 21189 G568DRAFT... (66% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r19c1_4.svg">
NZ_CP014275 (4111698-4122097): Martelella sp. AD-3, complete genome (50% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r19c1_5.svg">
NZ_JACIDZ010000008 (145999-156398): Martelella radicis strain DSM 28101 Ga037... (55% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r19c1_6.svg">
NZ_QETF01000003 (168626-179022): Rhodobacteraceae bacterium WDS4C29 contig3, ... (55% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r19c1_7.svg">
NZ_CP010855 (74978-85374): Marinovum algicola DG 898 chromosome, complete genome (50% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r19c1_8.svg">
NZ_FNYY01000006 (214033-224429): Marinovum algicola strain FF3, whole genome ... (62% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r19c1_9.svg">
NZ_FTOU01000006 (7281-17677): Paracoccus saliphilus strain DSM 18447, whole g... (62% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r19c1_10.svg">
NZ_SWKO01000011 (644250-654652): Marivita sp. LZ-15-2 NODE 1, whole genome sh... (45% of genes show similarity), ectoine
</option>

</select>
<button id= "clusterblast-r19c1-download">Download graphic</button>
</div>
<div id="clusterblast-r19c1-svg"></div>

</div>
</div>

<div class="body-details-section r19c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-26"></div> <span class="help-tooltip" id="cb-known-help-26">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r19c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r19c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r19c1_1.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r19c1_2.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r19c1_3.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r19c1_4.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r19c1_5.svg">
ectoine
</option>

</select>
<button id="knownclusterblast-r19c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r19c1-svg"></div>

</div>
</div>

<div class="body-details-section r19c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-27"></div> <span class="help-tooltip" id="cb-sub-help-27">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r19c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-29"></div> <span class="help-tooltip" id="tfbs-body-help-29">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
DmdR1
<div class="collapser collapser-target-DmdR1 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Iron(II)-dependent repressor<br>
<span>Score</span>: 23.10 out of 36.53
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r19c1-71020-DmdR1"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r19c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-31"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-31">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r19c1-pfam-details-svg" data-filename="NZ_PHSN01000027.1_r19c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r19c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r19c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-32"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-32">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r19c1-tigrfam-details-svg" data-filename="NZ_PHSN01000027.1_r19c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r19c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r19c1-tfbs-finder" data-name="r19c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r19c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-28"></div> <span class="help-tooltip" id="tfbs-side-help-28">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/DmdR1" target="_blank">DmdR1</a></td>
<td>Iron(II)-dependent repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r29c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_PHSN01000004.1 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-57"></div> <span class="help-tooltip" id="region_svg-help-57">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_PHSN01000004.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r29c1-svg" data-filename="NZ_PHSN01000004.1_r29c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 419,330 - 440,091 nt. (total: 20,762 nt)

<a class="cluster-rules-header" id="r29c1-rules-header" href="#r29c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r29c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r29c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-58"></div> <span class="help-tooltip" id="focus-panel-help-58">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r29c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r29c1-gene-table-details" data-name="r29c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r29c1-MIBiG-cluster-compare" data-name="r29c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r29c1-clusterblast" data-name="r29c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r29c1-knownclusterblast" data-name="r29c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r29c1-subclusterblast" data-name="r29c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r29c1-tfbs-finder" data-name="r29c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r29c1-pfam-details" data-name="r29c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r29c1-tigrfam-details" data-name="r29c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r29c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-39"></div> <span class="help-tooltip" id="gt-help-tooltip-help-39">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r29c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r29c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r29c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r29c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r29c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-33"></div> <span class="help-tooltip" id="MIBiG-help-33">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r29c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r29c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r29c1"></div>
<div id="comparison-MIBiG-r29c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000904: 116-14341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000904/index.html#r1c1" target="_new">BGC0000904</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Other</td>

<td>FR-900098</td>


<td>Streptomyces rubellomurinus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001859: 6609-19144">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001859/index.html#r1c1" target="_new">BGC0001859</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.41, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Other (Phosphonate)</td>

<td>fosfomycin</td>


<td>Pseudomonas syringae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001184: 0-7298">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001184/index.html#r1c1" target="_new">BGC0001184</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.41, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Other</td>

<td>bacilysin</td>


<td>Bacillus velezensis FZB42</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001285: 8-22479">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001285/index.html#r1c1" target="_new">BGC0001285</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.48, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.10</td>
<td>Other (Fatty acid)</td>

<td>pseudopyronine A, pseudopyronine B</td>


<td>Pseudomonas putida</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000395: 101-33405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000395/index.html#r1c1" target="_new">BGC0000395</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.46, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>NRP</td>

<td>nocardicin A</td>


<td>Nocardia uniformis subsp. tsuyamanensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002044: 0-65291">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002044/index.html#r1c1" target="_new">BGC0002044</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.49, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>NRP</td>

<td>empedopeptin</td>


<td>Massilia sp. YMA4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002440: 0-50466">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002440/index.html#r1c1" target="_new">BGC0002440</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.43, order:0.30, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>NRP</td>

<td>longicatenamide B, longicatenamide C, longicatenamide A, longicatenamide D</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001947: 0-23423">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001947/index.html#r1c1" target="_new">BGC0001947</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.47, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.06</td>
<td>Saccharide</td>

<td>K53 capsular polysaccharide</td>


<td>Acinetobacter baumannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000760: 0-26953">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000760/index.html#r1c1" target="_new">BGC0000760</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.47, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>Saccharide</td>

<td>emulsan</td>


<td>Acinetobacter venetianus RAG-1 = CIP 110063</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001457: 0-54671">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001457/index.html#r1c1" target="_new">BGC0001457</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.44, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>NRP</td>

<td>jomthonic acid A, jomthonic acid B, jomthonic acid C</td>


<td>Streptomyces caniferus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r29c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000904: 116-14341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000904/index.html#r1c1" target="_new">BGC0000904</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other</td>
<td>FR-900098</td>
<td>Streptomyces rubellomurinus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001859: 6609-19144">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001859/index.html#r1c1" target="_new">BGC0001859</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.41, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Other (Phosphonate)</td>
<td>fosfomycin</td>
<td>Pseudomonas syringae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001184: 0-7298">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001184/index.html#r1c1" target="_new">BGC0001184</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.41, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Other</td>
<td>bacilysin</td>
<td>Bacillus velezensis FZB42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001285: 8-22479">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001285/index.html#r1c1" target="_new">BGC0001285</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.48, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Other (Fatty acid)</td>
<td>pseudopyronine A, pseudopyronine B</td>
<td>Pseudomonas putida</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000395: 101-33405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000395/index.html#r1c1" target="_new">BGC0000395</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.46, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>NRP</td>
<td>nocardicin A</td>
<td>Nocardia uniformis subsp. tsuyamanensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002044: 0-65291">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002044/index.html#r1c1" target="_new">BGC0002044</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.49, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>NRP</td>
<td>empedopeptin</td>
<td>Massilia sp. YMA4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002440: 0-50466">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002440/index.html#r1c1" target="_new">BGC0002440</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.43, order:0.30, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>NRP</td>
<td>longicatenamide B, longicatenamide C, longicatenamide A, longicatenamide D</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001947: 0-23423">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001947/index.html#r1c1" target="_new">BGC0001947</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.47, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Saccharide</td>
<td>K53 capsular polysaccharide</td>
<td>Acinetobacter baumannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000760: 0-26953">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000760/index.html#r1c1" target="_new">BGC0000760</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.47, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>Saccharide</td>
<td>emulsan</td>
<td>Acinetobacter venetianus RAG-1 = CIP 110063</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001457: 0-54671">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001457/index.html#r1c1" target="_new">BGC0001457</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.44, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>NRP</td>
<td>jomthonic acid A, jomthonic acid B, jomthonic acid C</td>
<td>Streptomyces caniferus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r29c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-34"></div> <span class="help-tooltip" id="cb-general-help-34">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r29c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r29c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r29c1_1.svg">
NZ_CP038492 (1546062-1566833): Boseongicola sp. CCM32 chromosome, complete ge... (60% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r29c1_2.svg">
NZ_VISG01000001 (417908-438679): Rhodobacteraceae bacterium PT6CLA Ga0206392 ... (63% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r29c1_3.svg">
NZ_QGGW01000001 (614688-635459): Roseicyclus mahoneyensis strain DSM 16097 Ga... (63% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r29c1_4.svg">
NZ_QETF01000014 (15171-35930): Rhodobacteraceae bacterium WDS4C29 contig14, w... (66% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r29c1_5.svg">
NZ_GG697169 (457937-478602): Thalassobium sp. R2A62 scf 1112329232034, whole ... (52% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r29c1_6.svg">
NZ_CP010869 (1570899-1591667): Confluentimicrobium sp. EMB200-NS6 strain EMBL... (55% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r29c1_7.svg">
NC_009049 (1429419-1447824): Rhodobacter sphaeroides ATCC 17029 chromosome 1,... (52% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r29c1_8.svg">
NZ_FZOV01000001 (427613-448381): Rhodobacter megalophilus strain DSM 18937, w... (52% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r29c1_9.svg">
NZ_CP033442 (1872607-1893375): Rhodobacter sphaeroides strain AB25 chromosome 1 (52% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r29c1_10.svg">
NZ_CP030271 (1366785-1385319): Rhodobacter sphaeroides 2.4.1 chromosome 1, co... (52% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r29c1-download">Download graphic</button>
</div>
<div id="clusterblast-r29c1-svg"></div>

</div>
</div>

<div class="body-details-section r29c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-35"></div> <span class="help-tooltip" id="cb-known-help-35">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r29c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-36"></div> <span class="help-tooltip" id="cb-sub-help-36">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r29c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-38"></div> <span class="help-tooltip" id="tfbs-body-help-38">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
BldD
<div class="collapser collapser-target-BldD collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Development and antibiotic global regulator<br>
<span>Score</span>: 20.23 out of 23.43
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r29c1-435176-BldD"></div>

</div>


</div>
</div>



<div class="body-details-section r29c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-40"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-40">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r29c1-pfam-details-svg" data-filename="NZ_PHSN01000004.1_r29c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r29c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r29c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-41"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-41">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r29c1-tigrfam-details-svg" data-filename="NZ_PHSN01000004.1_r29c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r29c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r29c1-tfbs-finder" data-name="r29c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r29c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-37"></div> <span class="help-tooltip" id="tfbs-side-help-37">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/BldD" target="_blank">BldD</a></td>
<td>Development and antibiotic global regulator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r38c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_PHSN01000005.1 - Region 1 - hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-59"></div> <span class="help-tooltip" id="region_svg-help-59">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_PHSN01000005.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r38c1-svg" data-filename="NZ_PHSN01000005.1_r38c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 229,956 - 250,771 nt. (total: 20,816 nt)

<a class="cluster-rules-header" id="r38c1-rules-header" href="#r38c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r38c1-rules">hserlactone: Autoind_synth</div>

<div class="region-svg-container">
<div id='r38c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-60"></div> <span class="help-tooltip" id="focus-panel-help-60">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r38c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r38c1-gene-table-details" data-name="r38c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r38c1-MIBiG-cluster-compare" data-name="r38c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r38c1-clusterblast" data-name="r38c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r38c1-knownclusterblast" data-name="r38c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r38c1-subclusterblast" data-name="r38c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r38c1-tfbs-finder" data-name="r38c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r38c1-pfam-details" data-name="r38c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r38c1-tigrfam-details" data-name="r38c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r38c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-48"></div> <span class="help-tooltip" id="gt-help-tooltip-help-48">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r38c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r38c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r38c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r38c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r38c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-42"></div> <span class="help-tooltip" id="MIBiG-help-42">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r38c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r38c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r38c1"></div>
<div id="comparison-MIBiG-r38c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">hserlactone</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000844: 2-13901">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000844/index.html#r1c1" target="_new">BGC0000844</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>Other</td>

<td>clavulanic acid</td>


<td>Streptomyces clavuligerus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001310: 0-2705">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001310/index.html#r1c1" target="_new">BGC0001310</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>naringenin</td>


<td>Streptomyces clavuligerus ATCC 27064</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002127: 147-3013">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002127/index.html#r1c1" target="_new">BGC0002127</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>flaviolin, 1,3,6,8-tetrahydroxynaphthalene</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001224: 0-9834">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001224/index.html#r1c1" target="_new">BGC0001224</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Other</td>

<td>reductasporine</td>


<td>uncultured bacterium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001336: 0-12620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001336/index.html#r1c1" target="_new">BGC0001336</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Alkaloid</td>

<td>erdasporine A, erdasporine B, erdasporine C</td>


<td>uncultured bacterium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001485: 0-10917">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001485/index.html#r1c1" target="_new">BGC0001485</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.44, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Alkaloid</td>

<td>chuangxinmycin</td>


<td>Actinoplanes tsinanensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002144: 0-5515">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002144/index.html#r1c1" target="_new">BGC0002144</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.52, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>phazolicin</td>


<td>Rhizobium sp. Pop5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002406: 621-194635">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002406/index.html#r1c1" target="_new">BGC0002406</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Other</td>

<td>hydroxycinnamoyltyramine</td>


<td>Oryza sativa Japonica Group</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001223: 0-9682">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001223/index.html#r1c1" target="_new">BGC0001223</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Other</td>

<td>hydroxysporine</td>


<td>uncultured bacterium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483: 0-25255">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.45, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Other</td>

<td>5-isoprenylindole-3-carboxylate -D-glycosyl ester</td>


<td>Streptomyces sp. RM-5-8</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r38c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000844: 2-13901">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000844/index.html#r1c1" target="_new">BGC0000844</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Other</td>
<td>clavulanic acid</td>
<td>Streptomyces clavuligerus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001310: 0-2705">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001310/index.html#r1c1" target="_new">BGC0001310</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>naringenin</td>
<td>Streptomyces clavuligerus ATCC 27064</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002127: 147-3013">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002127/index.html#r1c1" target="_new">BGC0002127</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>flaviolin, 1,3,6,8-tetrahydroxynaphthalene</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001224: 0-9834">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001224/index.html#r1c1" target="_new">BGC0001224</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Other</td>
<td>reductasporine</td>
<td>uncultured bacterium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001336: 0-12620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001336/index.html#r1c1" target="_new">BGC0001336</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Alkaloid</td>
<td>erdasporine A, erdasporine B, erdasporine C</td>
<td>uncultured bacterium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001485: 0-10917">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001485/index.html#r1c1" target="_new">BGC0001485</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.44, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Alkaloid</td>
<td>chuangxinmycin</td>
<td>Actinoplanes tsinanensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002144: 0-5515">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002144/index.html#r1c1" target="_new">BGC0002144</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.52, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>RiPP</td>
<td>phazolicin</td>
<td>Rhizobium sp. Pop5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002406: 621-194635">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002406/index.html#r1c1" target="_new">BGC0002406</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Other</td>
<td>hydroxycinnamoyltyramine</td>
<td>Oryza sativa Japonica Group</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001223: 0-9682">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001223/index.html#r1c1" target="_new">BGC0001223</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Other</td>
<td>hydroxysporine</td>
<td>uncultured bacterium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483: 0-25255">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.45, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Other</td>
<td>5-isoprenylindole-3-carboxylate -D-glycosyl ester</td>
<td>Streptomyces sp. RM-5-8</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r38c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-43"></div> <span class="help-tooltip" id="cb-general-help-43">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r38c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r38c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r38c1_1.svg">
NZ_CP031956 (1150551-1169006): Phaeobacter sp. LSS9 chromosome, complete genome (16% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r38c1_2.svg">
NZ_CP053708 (4148169-4190536): Acetobacteraceae bacterium strain PAMC 26569 c... (7% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r38c1_3.svg">
NZ_CP012917 (99385-171835): Azospirillum brasilense strain Sp 7 plasmid ABSP7... (4% of genes show similarity), T2PKS
</option>

<option value = "svg/clusterblast_r38c1_4.svg">
NZ_JACIGJ010000001 (77238-98144): Roseospira marina strain DSM 15113 Ga039428... (15% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r38c1_5.svg">
NZ_CP018076 (418970-479660): Sulfitobacter sp. AM1-D1 chromosome, complete ge... (5% of genes show similarity), acyl_amino_acids
</option>

<option value = "svg/clusterblast_r38c1_6.svg">
NZ_CP044218 (3049700-3093819): Mesorhizobium sp. NIBRBAC000500504 chromosome,... (5% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r38c1_7.svg">
NZ_UCIA01000014 (40506-101274): Rhizobiales bacterium isolate AFS083231, whol... (3% of genes show similarity), acyl_amino_acids
</option>

<option value = "svg/clusterblast_r38c1_8.svg">
NZ_CP015318 (1143469-1204240): Mesorhizobium amorphae CCNWGS0123 chromosome, ... (3% of genes show similarity), acyl_amino_acids
</option>

<option value = "svg/clusterblast_r38c1_9.svg">
NZ_JABBGK010000001 (47220-107982): Rhizobium sp. S-51 contig1, whole genome s... (3% of genes show similarity), acyl_amino_acids
</option>

<option value = "svg/clusterblast_r38c1_10.svg">
NZ_UCAH01000003 (63481-124231): Rhizobiales bacterium isolate AFS059521, whol... (3% of genes show similarity), acyl_amino_acids
</option>

</select>
<button id= "clusterblast-r38c1-download">Download graphic</button>
</div>
<div id="clusterblast-r38c1-svg"></div>

</div>
</div>

<div class="body-details-section r38c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-44"></div> <span class="help-tooltip" id="cb-known-help-44">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r38c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-45"></div> <span class="help-tooltip" id="cb-sub-help-45">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r38c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-47"></div> <span class="help-tooltip" id="tfbs-body-help-47">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
OsdR
<div class="collapser collapser-target-OsdR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Development and stress management regulator<br>
<span>Score</span>: 24.03 out of 30.12
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r38c1-239324-OsdR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r38c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-49"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-49">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r38c1-pfam-details-svg" data-filename="NZ_PHSN01000005.1_r38c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r38c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r38c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-50"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-50">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r38c1-tigrfam-details-svg" data-filename="NZ_PHSN01000005.1_r38c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r38c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r38c1-tfbs-finder" data-name="r38c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r38c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-46"></div> <span class="help-tooltip" id="tfbs-side-help-46">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/OsdR" target="_blank">OsdR</a></td>
<td>Development and stress management regulator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>