
<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>GCF_001294535.1.prokka - 5 region(s) - antiSMASH results</title>
  <style>img {width: 20px; height: 20px;}</style>
  <link rel="stylesheet" type="text/css" href="css/bacteria.css">
  <meta property="og:title" content="GCF_001294535.1.prokka - 5 record(s) - 5 region(s)" />
  <meta property="og:description" content="">
  <meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
  <meta property="og:image:alt" content="antiSMASH logo" />
  <meta property="og:image:width" content="600" />
  <meta property="og:image:height" content="600" />
  <meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/GCF_001294535.1.prokka/index.html" />
  <meta property="og:type" content="website" />
  <meta name="twitter:card" content="summary" />
  <meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
  <div id="header">
      <nav>
    <div class="main-link">
     <div>
        <a href="https://antismash.secondarymetabolites.org/">
          <img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
        </a>
       </div>
       <div>
        <a href="https://antismash.secondarymetabolites.org/">
          antiSMASH version 6.0.0
        </a>
      </div>
    </div>
    <div class="custom-description"></div>
    <div class="ancillary-links">
      <div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
        <ul class="dropdown-options">
        
        <li><a href="GCF_001294535.1.prokka.zip">Download all results</a></li>
        
          <li><a href="GCF_001294535.1.prokka.gbk">Download GenBank summary file</a></li>
        
        </ul>
      </div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
      <div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
    </div>
  </nav>
    <div id="buttons">
      <span id="region-type">Select genomic region:</span>
      <div id="regionbuttons">
        <div class="arrow-left" id="prev-region"></div>
        <div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
        <div style="display: flex; flex-wrap: wrap">
        
          
            
            
                
            
            <div class="regbutton NRPS-like,T1PKS hybrid r31c1">
              <a href="#r31c1">31.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton hserlactone r52c1">
              <a href="#r52c1">52.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton hserlactone r55c1">
              <a href="#r55c1">55.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton hserlactone r65c1">
              <a href="#r65c1">65.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS-like r90c1">
              <a href="#r90c1">90.1</a>
            </div>
          
        
        </div>
        <div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
      </div>
    </div>
  </div>

  <!-- overview page -->
  
  <div class="page" id="overview">
   
   
     <h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>
   
   <div class="overview-layout">
    <div id="record-tables">
     <div id="single-record-tables">
     <div class="record-overview-header">
        <strong>NZ_LGIC01000031</strong> (original name was: NZ_LGIC01000031.1) (Cypionkella psychrotolerans)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-31" data-filename="NZ_LGIC01000031_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-31" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r31c1">
     
     
         
     
     <td class="regbutton NRPS-like,T1PKS hybrid r31c1">
      <a href="#r31c1">Region&nbsp31.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">20,633</td>
     <td class="digits table-split-left">72,527</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>NZ_LGIC01000052</strong> (original name was: NZ_LGIC01000052.1) (Cypionkella psychrotolerans)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-52" data-filename="NZ_LGIC01000052_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-52" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r52c1">
     
     
     <td class="regbutton hserlactone r52c1">
      <a href="#r52c1">Region&nbsp52.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
     </td>
     <td class="digits">477</td>
     <td class="digits table-split-left">21,067</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>NZ_LGIC01000055</strong> (original name was: NZ_LGIC01000055.1) (Cypionkella psychrotolerans)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-55" data-filename="NZ_LGIC01000055_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-55" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r55c1">
     
     
     <td class="regbutton hserlactone r55c1">
      <a href="#r55c1">Region&nbsp55.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">17,033</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>NZ_LGIC01000065</strong> (original name was: NZ_LGIC01000065.1) (Cypionkella psychrotolerans)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-65" data-filename="NZ_LGIC01000065_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-65" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r65c1">
     
     
     <td class="regbutton hserlactone r65c1">
      <a href="#r65c1">Region&nbsp65.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
     </td>
     <td class="digits">192,341</td>
     <td class="digits table-split-left">213,015</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>NZ_LGIC01000090</strong> (original name was: NZ_LGIC01000090.1) (Cypionkella psychrotolerans)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-90" data-filename="NZ_LGIC01000090_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-90" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r90c1">
     
     
     <td class="regbutton NRPS-like r90c1">
      <a href="#r90c1">Region&nbsp90.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">37,514</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div>
     </div>
     
     <div id="compact-record-table" style="display: none;">
      <div class="record-table">
       <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 
 
  <tr class="linked-row odd" data-anchor="#r31c1">
     
     
         
     
     <td class="regbutton NRPS-like,T1PKS hybrid r31c1">
      <a href="#r31c1">Region&nbsp31.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">20,633</td>
     <td class="digits table-split-left">72,527</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r52c1">
     
     
     <td class="regbutton hserlactone r52c1">
      <a href="#r52c1">Region&nbsp52.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
     </td>
     <td class="digits">477</td>
     <td class="digits table-split-left">21,067</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r55c1">
     
     
     <td class="regbutton hserlactone r55c1">
      <a href="#r55c1">Region&nbsp55.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">17,033</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r65c1">
     
     
     <td class="regbutton hserlactone r65c1">
      <a href="#r65c1">Region&nbsp65.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
     </td>
     <td class="digits">192,341</td>
     <td class="digits table-split-left">213,015</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r90c1">
     
     
     <td class="regbutton NRPS-like r90c1">
      <a href="#r90c1">Region&nbsp90.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">37,514</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
 </tbody>
</table>
      </div>
     </div>
     
     
     <dl id="regionless-records">
      <dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>
       
         <dd><strong>NZ_LGIC01000001</strong> (original name was: NZ_LGIC01000001.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000002</strong> (original name was: NZ_LGIC01000002.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000003</strong> (original name was: NZ_LGIC01000003.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000004</strong> (original name was: NZ_LGIC01000004.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000005</strong> (original name was: NZ_LGIC01000005.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000006</strong> (original name was: NZ_LGIC01000006.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000007</strong> (original name was: NZ_LGIC01000007.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000008</strong> (original name was: NZ_LGIC01000008.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000009</strong> (original name was: NZ_LGIC01000009.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000010</strong> (original name was: NZ_LGIC01000010.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000011</strong> (original name was: NZ_LGIC01000011.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000012</strong> (original name was: NZ_LGIC01000012.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000013</strong> (original name was: NZ_LGIC01000013.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000014</strong> (original name was: NZ_LGIC01000014.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000015</strong> (original name was: NZ_LGIC01000015.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000016</strong> (original name was: NZ_LGIC01000016.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000017</strong> (original name was: NZ_LGIC01000017.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000018</strong> (original name was: NZ_LGIC01000018.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000019</strong> (original name was: NZ_LGIC01000019.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000020</strong> (original name was: NZ_LGIC01000020.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000021</strong> (original name was: NZ_LGIC01000021.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000022</strong> (original name was: NZ_LGIC01000022.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000023</strong> (original name was: NZ_LGIC01000023.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000024</strong> (original name was: NZ_LGIC01000024.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000025</strong> (original name was: NZ_LGIC01000025.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000026</strong> (original name was: NZ_LGIC01000026.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000027</strong> (original name was: NZ_LGIC01000027.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000028</strong> (original name was: NZ_LGIC01000028.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000029</strong> (original name was: NZ_LGIC01000029.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000030</strong> (original name was: NZ_LGIC01000030.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000032</strong> (original name was: NZ_LGIC01000032.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000033</strong> (original name was: NZ_LGIC01000033.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000034</strong> (original name was: NZ_LGIC01000034.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000035</strong> (original name was: NZ_LGIC01000035.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000036</strong> (original name was: NZ_LGIC01000036.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000037</strong> (original name was: NZ_LGIC01000037.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000038</strong> (original name was: NZ_LGIC01000038.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000039</strong> (original name was: NZ_LGIC01000039.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000040</strong> (original name was: NZ_LGIC01000040.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000041</strong> (original name was: NZ_LGIC01000041.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000042</strong> (original name was: NZ_LGIC01000042.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000043</strong> (original name was: NZ_LGIC01000043.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000044</strong> (original name was: NZ_LGIC01000044.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000045</strong> (original name was: NZ_LGIC01000045.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000046</strong> (original name was: NZ_LGIC01000046.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000047</strong> (original name was: NZ_LGIC01000047.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000048</strong> (original name was: NZ_LGIC01000048.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000049</strong> (original name was: NZ_LGIC01000049.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000050</strong> (original name was: NZ_LGIC01000050.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000051</strong> (original name was: NZ_LGIC01000051.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000053</strong> (original name was: NZ_LGIC01000053.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000054</strong> (original name was: NZ_LGIC01000054.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000056</strong> (original name was: NZ_LGIC01000056.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000057</strong> (original name was: NZ_LGIC01000057.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000058</strong> (original name was: NZ_LGIC01000058.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000059</strong> (original name was: NZ_LGIC01000059.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000060</strong> (original name was: NZ_LGIC01000060.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000061</strong> (original name was: NZ_LGIC01000061.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000062</strong> (original name was: NZ_LGIC01000062.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000063</strong> (original name was: NZ_LGIC01000063.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000064</strong> (original name was: NZ_LGIC01000064.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000066</strong> (original name was: NZ_LGIC01000066.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000067</strong> (original name was: NZ_LGIC01000067.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000068</strong> (original name was: NZ_LGIC01000068.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000069</strong> (original name was: NZ_LGIC01000069.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000070</strong> (original name was: NZ_LGIC01000070.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000071</strong> (original name was: NZ_LGIC01000071.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000072</strong> (original name was: NZ_LGIC01000072.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000073</strong> (original name was: NZ_LGIC01000073.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000074</strong> (original name was: NZ_LGIC01000074.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000075</strong> (original name was: NZ_LGIC01000075.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000076</strong> (original name was: NZ_LGIC01000076.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000077</strong> (original name was: NZ_LGIC01000077.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000078</strong> (original name was: NZ_LGIC01000078.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000079</strong> (original name was: NZ_LGIC01000079.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000080</strong> (original name was: NZ_LGIC01000080.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000081</strong> (original name was: NZ_LGIC01000081.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000082</strong> (original name was: NZ_LGIC01000082.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000083</strong> (original name was: NZ_LGIC01000083.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000084</strong> (original name was: NZ_LGIC01000084.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000085</strong> (original name was: NZ_LGIC01000085.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000086</strong> (original name was: NZ_LGIC01000086.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000087</strong> (original name was: NZ_LGIC01000087.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000088</strong> (original name was: NZ_LGIC01000088.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000089</strong> (original name was: NZ_LGIC01000089.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000091</strong> (original name was: NZ_LGIC01000091.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000092</strong> (original name was: NZ_LGIC01000092.1) (Cypionkella psychrotolerans)</dd>
       
         <dd><strong>NZ_LGIC01000093</strong> (original name was: NZ_LGIC01000093.1) (Cypionkella psychrotolerans)</dd>
       
     </dl>
     
    </div>
    
    <div class="overview-switches">
     <div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    
   </div>
  </div>
  






<div class="page" id='r31c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00031_NZ_LGIC.. - Region 1 - NRPS-like,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-5"></div> <span class="help-tooltip" id="region_svg-help-5">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = NZ_LGIC01000031.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r31c1-svg" data-filename="NZ_LGIC01000031_r31c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 20,633 - 72,527 nt. (total: 51,895 nt)
        
        <a class="cluster-rules-header" id="r31c1-rules-header" href="#r31c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r31c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r31c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-6"></div> <span class="help-tooltip" id="focus-panel-help-6">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r31c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r31c1-nrps_pks" data-name="r31c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
    </div>
    
     
    <div class="body-details-section r31c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-1"></div> <span class="help-tooltip" id="nrps-domain-help-1">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r31c1-details-svg" data-filename="NZ_LGIC01000031_r31c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r31c1-details-svg"></div>
</div>
    </div>
     
    
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r31c1-nrps_pks_products" data-name="r31c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r31c1-nrps_pks" data-name="r31c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r31c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-2"></div> <span class="help-tooltip" id="nrps-structures-help-2">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 20632 - 72527:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r31c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r31c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 20632 - 65123:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r31c1-2" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r31c1-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 3, location 26143 - 72527:
          <div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r31c1-3" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r31c1-3-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r31c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-3"></div> <span class="help-tooltip" id="nrps-monomers-help-3">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">CDDIDNMG_01075</span></strong>: X
      <div class="collapser collapser-target-CDDIDNMG_01075 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>A-OX (532..1314)</strong>: X
          <div class="collapser collapser-target-CDDIDNMG_01075 collapser-level-candidate"> </div><div class="collapser-content">
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">CDDIDNMG_01077</span></strong>: mal
      <div class="collapser collapser-target-CDDIDNMG_01077 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (539..822)</strong>: mal
          <div class="collapser collapser-target-CDDIDNMG_01077 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-CDDIDNMG_01077-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 62.5%</dd>
<dd>2-Rhyd-Malonyl-CoA: 58.3%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-CDDIDNMG_01077-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 77.1</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 45.5</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 41.9</dt>

  <dd></dd><dt>inactive: 38.5</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 27.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 15.9</dt>

  <dd></dd><dt>fatty_acid: 12.2</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 12.2</dt>

  <dd></dd><dt>Isobutyryl-CoA: 10.0</dt>

  <dd></dd><dt>Propionyl-CoA: 9.1</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
  </div>

 </div>

</div>




<div class="page" id='r52c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00052_NZ_LGIC.. - Region 1 - hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-7"></div> <span class="help-tooltip" id="region_svg-help-7">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = NZ_LGIC01000052.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r52c1-svg" data-filename="NZ_LGIC01000052_r52c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 477 - 21,067 nt. (total: 20,591 nt)
        
        <a class="cluster-rules-header" id="r52c1-rules-header" href="#r52c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r52c1-rules">hserlactone: Autoind_synth</div>
      
      <div class="region-svg-container">
       <div id='r52c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-8"></div> <span class="help-tooltip" id="focus-panel-help-8">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r52c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
    </div>
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
    </div>
    
  </div>

 </div>

</div>




<div class="page" id='r55c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00055_NZ_LGIC.. - Region 1 - hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-9"></div> <span class="help-tooltip" id="region_svg-help-9">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = NZ_LGIC01000055.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r55c1-svg" data-filename="NZ_LGIC01000055_r55c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 17,033 nt. (total: 17,033 nt)
        
        <a class="cluster-rules-header" id="r55c1-rules-header" href="#r55c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r55c1-rules">hserlactone: Autoind_synth</div>
      
      <div class="region-svg-container">
       <div id='r55c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-10"></div> <span class="help-tooltip" id="focus-panel-help-10">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r55c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
    </div>
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
    </div>
    
  </div>

 </div>

</div>




<div class="page" id='r65c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00065_NZ_LGIC.. - Region 1 - hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-11"></div> <span class="help-tooltip" id="region_svg-help-11">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = NZ_LGIC01000065.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r65c1-svg" data-filename="NZ_LGIC01000065_r65c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 192,341 - 213,015 nt. (total: 20,675 nt)
        
        <a class="cluster-rules-header" id="r65c1-rules-header" href="#r65c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r65c1-rules">hserlactone: Autoind_synth</div>
      
      <div class="region-svg-container">
       <div id='r65c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-12"></div> <span class="help-tooltip" id="focus-panel-help-12">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r65c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
    </div>
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
    </div>
    
  </div>

 </div>

</div>




<div class="page" id='r90c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00090_NZ_LGIC.. - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-13"></div> <span class="help-tooltip" id="region_svg-help-13">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = NZ_LGIC01000090.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r90c1-svg" data-filename="NZ_LGIC01000090_r90c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 37,514 nt. (total: 37,514 nt)
        
        <a class="cluster-rules-header" id="r90c1-rules-header" href="#r90c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r90c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r90c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-14"></div> <span class="help-tooltip" id="focus-panel-help-14">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r90c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r90c1-nrps_pks" data-name="r90c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
    </div>
    
     
    <div class="body-details-section r90c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-4"></div> <span class="help-tooltip" id="nrps-domain-help-4">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r90c1-details-svg" data-filename="NZ_LGIC01000090_r90c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox" checked>  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r90c1-details-svg"></div>
</div>
    </div>
     
    
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
    </div>
    
     
    
     
    
  </div>

 </div>

</div>


  <footer class="footer">
    <div class="container">
      <div>
        <img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
      </div>
      <div class="cite-me">
        If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
      </div>
      <div>
        <img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
      </div>
    </div>
  </footer>

  <script src="js/jquery.js"></script>
  <script src="js/antismash.js"></script>
  <script src="regions.js"></script>
  <script>
    $(document).ready(function() {
        viewer["start"](all_regions, details_data, resultsData, recordData);
    })
  </script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink">
 <defs>
  <filter id="inset-shadow">
   <feOffset dx="-2" dy="-2"></feOffset>
   <feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
   <feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
   <feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
   <feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
   <feComponentTransfer in="shadow" result="shadow">
    <feFuncA type="linear" slope=".95"></feFuncA>
   </feComponentTransfer>
   <feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
  </filter>
 </defs>
</svg>
</body>
</html>