
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>GCF_003122215.1_Pararhodobacter_marinus_CIC4N-9 - 4 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/bacteria.css">
<meta property="og:title" content="GCF_003122215.1_Pararhodobacter_marinus_CIC4N-9 - 4 record(s) - 4 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-753d0f25-c070-4543-aa7f-434342d631de/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://antismash.secondarymetabolites.org/">
<img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://antismash.secondarymetabolites.org/">
antiSMASH version 7.0.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="GCF_003122215.1_Pararhodobacter_marinus_CIC4N-9.zip">Download all results</a></li>

<li><a href="GCF_003122215.1_Pararhodobacter_marinus_CIC4N-9.gbk">Download GenBank summary file</a></li>

<li><a href="bacteria-753d0f25-c070-4543-aa7f-434342d631de.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton hybrid r11c1">
<a href="#r11c1">11.1</a>
</div>



<div class="regbutton NRPS NRPS-like r25c1">
<a href="#r25c1">25.1</a>
</div>



<div class="regbutton other ectoine r28c1">
<a href="#r28c1">28.1</a>
</div>



<div class="regbutton other betalactone r30c1">
<a href="#r30c1">30.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>NZ_QEYD01000001.1</strong> (Pararhodobacter marinus)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-11" data-filename="NZ_QEYD01000001.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-11" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r11c1">
<td class="regbutton hybrid r11c1">
<a href="#r11c1">Region&nbsp11.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">390,633</td>
<td class="digits table-split-left">442,429</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000189/1" target="_blank">3-(2&#39;-hydroxy-3&#39;-oxo-4&#39;-methylpentyl)-indole/3-(2&#39;-hydroxy-3&#39;-oxo-4&#39;-methylhexyl)-indole/3-(2&#39;-acetoxy-3&#39;-oxo-4&#39;-methylpentyl)-indole/3-(2&#39;-acetoxy-3&#39;-oxo-4&#39;-methylhexyl)-indole</a></td>
<td>Polyketide:Type III polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 25%, #ffffff00 25%)">25%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_QEYD01000004.1</strong> (Pararhodobacter marinus)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-25" data-filename="NZ_QEYD01000004.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-25" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r25c1">
<td class="regbutton NRPS NRPS-like r25c1">
<a href="#r25c1">Region&nbsp25.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">359,416</td>
<td class="digits table-split-left">403,294</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_QEYD01000007.1</strong> (Pararhodobacter marinus)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-28" data-filename="NZ_QEYD01000007.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-28" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r28c1">
<td class="regbutton other ectoine r28c1">
<a href="#r28c1">Region&nbsp28.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
</td>
<td class="digits">2,204</td>
<td class="digits table-split-left">12,596</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000860/1" target="_blank">ectoine</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_QEYD01000009.1</strong> (Pararhodobacter marinus)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-30" data-filename="NZ_QEYD01000009.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-30" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r30c1">
<td class="regbutton other betalactone r30c1">
<a href="#r30c1">Region&nbsp30.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#betalactone" target="_blank">betalactone</a>
</td>
<td class="digits">148,251</td>
<td class="digits table-split-left">172,562</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000279/1" target="_blank">xantholipin</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r11c1">
<td class="regbutton hybrid r11c1">
<a href="#r11c1">Region&nbsp11.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">390,633</td>
<td class="digits table-split-left">442,429</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000189/1" target="_blank">3-(2&#39;-hydroxy-3&#39;-oxo-4&#39;-methylpentyl)-indole/3-(2&#39;-hydroxy-3&#39;-oxo-4&#39;-methylhexyl)-indole/3-(2&#39;-acetoxy-3&#39;-oxo-4&#39;-methylpentyl)-indole/3-(2&#39;-acetoxy-3&#39;-oxo-4&#39;-methylhexyl)-indole</a></td>
<td>Polyketide:Type III polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 25%, #ffffff00 25%)">25%</td>

</tr>


<tr class="linked-row even" data-anchor="#r25c1">
<td class="regbutton NRPS NRPS-like r25c1">
<a href="#r25c1">Region&nbsp25.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">359,416</td>
<td class="digits table-split-left">403,294</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r28c1">
<td class="regbutton other ectoine r28c1">
<a href="#r28c1">Region&nbsp28.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
</td>
<td class="digits">2,204</td>
<td class="digits table-split-left">12,596</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000860/1" target="_blank">ectoine</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>


<tr class="linked-row even" data-anchor="#r30c1">
<td class="regbutton other betalactone r30c1">
<a href="#r30c1">Region&nbsp30.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#betalactone" target="_blank">betalactone</a>
</td>
<td class="digits">148,251</td>
<td class="digits table-split-left">172,562</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000279/1" target="_blank">xantholipin</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>NZ_QEYD01000010.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000011.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000012.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000013.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000014.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000015.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000016.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000017.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000018.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000019.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000020.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000021.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000022.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000023.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000024.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000025.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000026.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000027.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000028.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000029.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000002.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000030.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000003.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000005.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000006.1</strong> (Pararhodobacter marinus)</dd>

<dd><strong>NZ_QEYD01000008.1</strong> (Pararhodobacter marinus)</dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>







<div class="page" id='r11c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_QEYD01000001.1 - Region 1 - NRPS-like,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-43"></div> <span class="help-tooltip" id="region_svg-help-43">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_QEYD01000001.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r11c1-svg" data-filename="NZ_QEYD01000001.1_r11c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 390,633 - 442,429 nt. (total: 51,797 nt)

<a class="cluster-rules-header" id="r11c1-rules-header" href="#r11c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r11c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r11c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-44"></div> <span class="help-tooltip" id="focus-panel-help-44">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r11c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r11c1-gene-table-details" data-name="r11c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r11c1-nrps_pks" data-name="r11c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r11c1-MIBiG-cluster-compare" data-name="r11c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r11c1-clusterblast" data-name="r11c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r11c1-knownclusterblast" data-name="r11c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r11c1-subclusterblast" data-name="r11c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r11c1-tfbs-finder" data-name="r11c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r11c1-nrps-pks-bubbles" data-name="r11c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r11c1-pfam-details" data-name="r11c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r11c1-tigrfam-details" data-name="r11c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r11c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-11"></div> <span class="help-tooltip" id="gt-help-tooltip-help-11">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r11c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r11c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r11c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r11c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r11c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-1"></div> <span class="help-tooltip" id="nrps-domain-help-1">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r11c1-details-svg" data-filename="NZ_QEYD01000001.1_r11c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r11c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r11c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-2"></div> <span class="help-tooltip" id="MIBiG-help-2">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r11c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r11c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r11c1"></div>
<div id="comparison-MIBiG-r11c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.50</td>
<td>Polyketide</td>

<td>strobilurin A</td>


<td>Strobilurus tenacellus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000189: 0-8123">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000189/index.html#r1c1" target="_new">BGC0000189</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.53, order:0.43, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.53, order:0.43, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.45</td>
<td>Polyketide</td>

<td>3-(2&#39;-hydroxy-3&#39;-oxo-4&#39;-methylpentyl)-indole, 3-(2&#39;-hydroxy-3&#39;-oxo-4&#39;-methylhexyl)-indole, 3-(2&#39;-acetoxy-3&#39;-oxo-4&#39;-methylpentyl)-indole, 3-(2&#39;-acetoxy-3&#39;-oxo-4&#39;-methylhexyl)-indole</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.44</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001591: 66-30223">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001591/index.html#r1c1" target="_new">BGC0001591</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.47, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.47, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.40</td>
<td>Other</td>

<td>fatty acid enol ester, N-tetradecanoyl tyrosine</td>


<td>uncultured bacterium CSLC2</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002138: 0-9694">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002138/index.html#r1c1" target="_new">BGC0002138</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.39</td>
<td>Polyketide</td>

<td>fuelimycin A, fuelimicin B, fuelimicin C</td>


<td>Streptomyces albireticuli NRRL B-1670</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.39</td>
<td>Polyketide</td>

<td>HEx-pks16 polyketide</td>


<td>Punctularia strigosozonata HHB-11173 SS5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000393: 0-47918">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000393/index.html#r1c1" target="_new">BGC0000393</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.47, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>






<td class="cc-heat-cell" title="0.51 (id:0.47, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.38</td>
<td>NRP, Polyketide</td>

<td>myxoprincomide-c506</td>


<td>Myxococcus xanthus DK 1622</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>






<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.38</td>
<td>Polyketide</td>

<td>solanapyrone A</td>


<td>Ascochyta rabiei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001749: 322-54057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001749/index.html#r1c1" target="_new">BGC0001749</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>






<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.38</td>
<td>Polyketide</td>

<td>epipyriculol</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.48, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>






<td class="cc-heat-cell" title="0.51 (id:0.48, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.37</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 10E9</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r11c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>strobilurin A</td>
<td>Strobilurus tenacellus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000189: 0-8123">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000189/index.html#r1c1" target="_new">BGC0000189</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.53, order:0.43, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>3-(2&#39;-hydroxy-3&#39;-oxo-4&#39;-methylpentyl)-indole, 3-(2&#39;-hydroxy-3&#39;-oxo-4&#39;-methylhexyl)-indole, 3-(2&#39;-acetoxy-3&#39;-oxo-4&#39;-methylpentyl)-indole, 3-(2&#39;-acetoxy-3&#39;-oxo-4&#39;-methylhexyl)-indole</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000730: 151-21786">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000730/index.html#r1c1" target="_new">BGC0000730</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.48, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Klebsiella pneumoniae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000731: 151-19654">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000731/index.html#r1c1" target="_new">BGC0000731</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.48, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Klebsiella pneumoniae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001591: 66-30223">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001591/index.html#r1c1" target="_new">BGC0001591</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.47, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other</td>
<td>fatty acid enol ester, N-tetradecanoyl tyrosine</td>
<td>uncultured bacterium CSLC2</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>HEx-pks16 polyketide</td>
<td>Punctularia strigosozonata HHB-11173 SS5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000732: 151-20553">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000732/index.html#r1c1" target="_new">BGC0000732</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.48, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Raoultella planticola</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000763: 750-18450">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000763/index.html#r1c1" target="_new">BGC0000763</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.47, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Saccharide</td>
<td>stewartan</td>
<td>Pantoea stewartii subsp. stewartii DC283</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000791: 0-12875">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000791/index.html#r1c1" target="_new">BGC0000791</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.47, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Saccharide</td>
<td>O-antigen</td>
<td>Cronobacter sakazakii</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r11c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-3"></div> <span class="help-tooltip" id="cb-general-help-3">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r11c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r11c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r11c1_1.svg">
NZ_FNOM01000001 (453882-506024): Roseicitreum antarcticum strain CGMCC 1.8894... (19% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r11c1_2.svg">
NZ_VLKU01000003 (57113-109067): Paracoccus sulfuroxidans strain CGMCC 1.5364 ... (17% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r11c1_3.svg">
NZ_CP018076 (1138633-1217190): Sulfitobacter sp. AM1-D1 chromosome, complete ... (10% of genes show similarity), NRPS-like,T1PKS,T3PKS
</option>

<option value = "svg/clusterblast_r11c1_4.svg">
NZ_JACIEI010000001 (517165-569024): Sulfitobacter undariae strain DSM 102234 ... (17% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r11c1_5.svg">
NZ_CP045372 (3378099-3430157): Sulfitobacter sp. THAF37 chromosome, complete ... (18% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r11c1_6.svg">
NZ_CP016364 (2883881-2936068): Phaeobacter porticola strain P97 chromosome, c... (20% of genes show similarity), NRPS-like,T1PKS,hserlactone
</option>

<option value = "svg/clusterblast_r11c1_7.svg">
NZ_CP025803 (2887359-2939378): Sulfitobacter sp. SK011 chromosome, complete g... (18% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r11c1_8.svg">
NC_023135 (2706603-2758631): Leisingera methylohalidivorans DSM 14336, comple... (20% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r11c1_9.svg">
NZ_FNBP01000002 (288918-340949): Sulfitobacter delicatus strain DSM 16477, wh... (17% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r11c1_10.svg">
NZ_PHJF01000003 (149422-201425): Epibacterium ulvae strain U95 Ga0207336 103,... (20% of genes show similarity), NRPS-like,T1PKS
</option>

</select>
<button id= "clusterblast-r11c1-download">Download graphic</button>
</div>
<div id="clusterblast-r11c1-svg"></div>

</div>
</div>

<div class="body-details-section r11c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-4"></div> <span class="help-tooltip" id="cb-known-help-4">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r11c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r11c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r11c1_1.svg">
3-(2&#39;-hydroxy-3&#39;-oxo-4&#39;-methylpentyl)-indole/3-(2&#39;-hydroxy-3&#39;-oxo-4&#39;-methylhexyl)-indole/3-(2&#39;-acetoxy-3&#39;-oxo-4&#39;-methylpentyl)-indole/3-(2&#39;-acetoxy-3&#39;-oxo-4&#39;-methylhexyl)-indole
</option>

<option value = "svg/knownclusterblast_r11c1_2.svg">
bottromycin A2
</option>

</select>
<button id="knownclusterblast-r11c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r11c1-svg"></div>

</div>
</div>

<div class="body-details-section r11c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-5"></div> <span class="help-tooltip" id="cb-sub-help-5">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r11c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-9"></div> <span class="help-tooltip" id="tfbs-body-help-9">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AfsR
<div class="collapser collapser-target-AfsR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Pleiotropic regulatory for antibiotic production<br>
<span>Score</span>: 19.28 out of 23.05
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r11c1-442267-AfsR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (3):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 18.19 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r11c1-391754-ZuR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 19.49 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r11c1-400566-ZuR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 19.72 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r11c1-405867-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r11c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-10"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-10">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r11c1-domain-bubble-svg" data-filename="NZ_QEYD01000001.1_r11c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r11c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (390632 - 442429): neighbouring NRPS-like-T1PKS
</option>



<option value="CC2">
Candidate 2 (390632 - 435159): single NRPS-like
</option>



<option value="CC3">
Candidate 3 (396102 - 442429): single T1PKS
</option>


</select>

<div class="details-svg" id="r11c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r11c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r11c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-12"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-12">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r11c1-pfam-details-svg" data-filename="NZ_QEYD01000001.1_r11c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r11c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r11c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-13"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-13">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r11c1-tigrfam-details-svg" data-filename="NZ_QEYD01000001.1_r11c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r11c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r11c1-nrps_pks_products" data-name="r11c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r11c1-nrps_pks_monomers" data-name="r11c1-nrps_pks_monomers">
NRPS/PKS monomers
</div>



<div class="sidepanel-details-header r11c1-tfbs-finder" data-name="r11c1-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r11c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-6"></div> <span class="help-tooltip" id="nrps-structures-help-6">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 390632 - 442429:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r11c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r11c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 2, location 390632 - 435159:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r11c1-2" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r11c1-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 3, location 396102 - 442429:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r11c1-3" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r11c1-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r11c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS monomer predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-7"></div> <span class="help-tooltip" id="nrps-monomers-help-7">Shows the predicted monomers for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">BNFEMFIJ_01595</span></strong>: X
<div class="collapser collapser-target-BNFEMFIJ_01595 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>A-OX (537..1310)</strong>: X
<div class="collapser collapser-target-BNFEMFIJ_01595 collapser-level-candidate"> </div><div class="collapser-content">

</div>
</dd>




</div>
<br>

<strong><span class="serif">BNFEMFIJ_01597</span></strong>: mal
<div class="collapser collapser-target-BNFEMFIJ_01597 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (534..828)</strong>: mal
<div class="collapser collapser-target-BNFEMFIJ_01597 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-BNFEMFIJ_01597-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 58.3%</dd>
<dd>2-Rhyd-Malonyl-CoA: 54.2%</dd>
<dd>inactive: 54.2%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-BNFEMFIJ_01597-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 82.6</dt>

<dd></dd><dt>Methylmalonyl-CoA: 49.4</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 45.2</dt>

<dd></dd><dt>inactive: 37.4</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 35.2</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 26.2</dt>

<dd></dd><dt>fatty_acid: 23.2</dt>

<dd></dd><dt>Isobutyryl-CoA: 21.0</dt>

<dd></dd><dt>Benzoyl-CoA: 15.4</dt>

<dd></dd><dt>Propionyl-CoA: 13.9</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 11.7</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>CHC-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r11c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-8"></div> <span class="help-tooltip" id="tfbs-side-help-8">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AfsR" target="_blank">AfsR</a></td>
<td>Pleiotropic regulatory for antibiotic production</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>




<div class="page" id='r25c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_QEYD01000004.1 - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-45"></div> <span class="help-tooltip" id="region_svg-help-45">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_QEYD01000004.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r25c1-svg" data-filename="NZ_QEYD01000004.1_r25c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 359,416 - 403,294 nt. (total: 43,879 nt)

<a class="cluster-rules-header" id="r25c1-rules-header" href="#r25c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r25c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r25c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-46"></div> <span class="help-tooltip" id="focus-panel-help-46">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r25c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r25c1-gene-table-details" data-name="r25c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r25c1-nrps_pks" data-name="r25c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r25c1-MIBiG-cluster-compare" data-name="r25c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r25c1-clusterblast" data-name="r25c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r25c1-knownclusterblast" data-name="r25c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r25c1-subclusterblast" data-name="r25c1-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r25c1-nrps-pks-bubbles" data-name="r25c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r25c1-pfam-details" data-name="r25c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r25c1-tigrfam-details" data-name="r25c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r25c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-22"></div> <span class="help-tooltip" id="gt-help-tooltip-help-22">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r25c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r25c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r25c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r25c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r25c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-14"></div> <span class="help-tooltip" id="nrps-domain-help-14">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r25c1-details-svg" data-filename="NZ_QEYD01000004.1_r25c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r25c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r25c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-15"></div> <span class="help-tooltip" id="MIBiG-help-15">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r25c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r25c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r25c1"></div>
<div id="comparison-MIBiG-r25c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002002: 0-14395">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002002/index.html#r1c1" target="_new">BGC0002002</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>1,6-phenazinedimethanol, fontizine A</td>


<td>Serratia fonticola</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002476: 0-10330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002476/index.html#r1c1" target="_new">BGC0002476</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Escherichia coli str. K-12 substr. MG1655</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001869: 1-11792">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001869/index.html#r1c1" target="_new">BGC0001869</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Other (Phenazine), NRP</td>

<td>griseoluteic acid, demethoxygriseoluteic acid, phenaszentine A, phenaszentine C, phenaszenketide C, phenaszenketide D</td>


<td>Xenorhabdus szentirmaii DSM 16338</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000913: 1-11028">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000913/index.html#r1c1" target="_new">BGC0000913</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Other</td>

<td>MK-8</td>


<td>Enterobacter cloacae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002639: 0-8541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002639/index.html#r1c1" target="_new">BGC0002639</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>phospho-tyrolose, tyrolose, tyrolitol, tyrocitabine-626, acyl-tyrocitabine-668, acyl-tyrocitabine-696, acyl-tyrocitabine-724, acyl-tyrocitabine-752</td>


<td>Lactobacillus iners LEAF 2052A-d</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000597: 0-10846">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000597/index.html#r1c1" target="_new">BGC0000597</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>RiPP</td>

<td>SWA-2138</td>


<td>Streptomyces sp. e14</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>RiPP</td>

<td>SCO-2138</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596: 0-21820">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>RiPP</td>

<td>SLI-2138</td>


<td>Streptomyces lividans TK24</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001343: 61-19132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001343/index.html#r1c1" target="_new">BGC0001343</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>aurachin A, aurachin D</td>


<td>Stigmatella aurantiaca Sg a15</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r25c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002002: 0-14395">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002002/index.html#r1c1" target="_new">BGC0002002</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>1,6-phenazinedimethanol, fontizine A</td>
<td>Serratia fonticola</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002476: 0-10330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002476/index.html#r1c1" target="_new">BGC0002476</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Escherichia coli str. K-12 substr. MG1655</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001869: 1-11792">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001869/index.html#r1c1" target="_new">BGC0001869</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Other (Phenazine), NRP</td>
<td>griseoluteic acid, demethoxygriseoluteic acid, phenaszentine A, phenaszentine C, phenaszenketide C, phenaszenketide D</td>
<td>Xenorhabdus szentirmaii DSM 16338</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000913: 1-11028">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000913/index.html#r1c1" target="_new">BGC0000913</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Other</td>
<td>MK-8</td>
<td>Enterobacter cloacae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002639: 0-8541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002639/index.html#r1c1" target="_new">BGC0002639</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>phospho-tyrolose, tyrolose, tyrolitol, tyrocitabine-626, acyl-tyrocitabine-668, acyl-tyrocitabine-696, acyl-tyrocitabine-724, acyl-tyrocitabine-752</td>
<td>Lactobacillus iners LEAF 2052A-d</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000597: 0-10846">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000597/index.html#r1c1" target="_new">BGC0000597</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>RiPP</td>
<td>SWA-2138</td>
<td>Streptomyces sp. e14</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>RiPP</td>
<td>SCO-2138</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596: 0-21820">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>RiPP</td>
<td>SLI-2138</td>
<td>Streptomyces lividans TK24</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001343: 61-19132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001343/index.html#r1c1" target="_new">BGC0001343</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>aurachin A, aurachin D</td>
<td>Stigmatella aurantiaca Sg a15</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r25c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-16"></div> <span class="help-tooltip" id="cb-general-help-16">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r25c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r25c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r25c1_1.svg">
NC_022049 (230638-327538): Paracoccus aminophilus JCM 7686 plasmid pAMI4, com... (6% of genes show similarity), LAP,NRPS-like,TfuA-related,thiopeptide
</option>

<option value = "svg/clusterblast_r25c1_2.svg">
NZ_SAVA01000002 (127464-171295): Sinirhodobacter hungdaonensis strain CGMCC 1... (6% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r25c1_3.svg">
NZ_FOZW01000018 (17570-48736): Yangia pacifica strain DSM 26894, whole genome... (22% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r25c1_4.svg">
NZ_CP015362 (290184-501485): Streptomyces sp. S8 chromosome, complete genome (4% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r25c1_5.svg">
NZ_SNYO01000001 (572190-669119): Actinomycetospora succinea strain DSM 45775 ... (10% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r25c1_6.svg">
NZ_CP051855 (3444265-3561448): Rhodococcus imtechensis strain XM24D chromosom... (7% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r25c1_7.svg">
NZ_CP009111 (3362353-3480055): Rhodococcus opacus strain 1CP, complete genome (6% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r25c1_8.svg">
NZ_LMPN02000036 (110314-197282): Brevibacillus sp. Leaf182 182 CLC 10 consens... (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r25c1_9.svg">
NZ_CCAZ020000002 (1005998-1049931): Afipia felis strain 76713, whole genome s... (7% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r25c1_10.svg">
NZ_QEKW01000001 (503288-585948): Actinomycetospora cinnamomea strain DSM 4577... (9% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r25c1-download">Download graphic</button>
</div>
<div id="clusterblast-r25c1-svg"></div>

</div>
</div>

<div class="body-details-section r25c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-17"></div> <span class="help-tooltip" id="cb-known-help-17">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r25c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-18"></div> <span class="help-tooltip" id="cb-sub-help-18">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r25c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-21"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-21">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r25c1-domain-bubble-svg" data-filename="NZ_QEYD01000004.1_r25c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r25c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (359415 - 403294): single NRPS-like
</option>


</select>

<div class="details-svg" id="r25c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r25c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r25c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-23"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-23">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r25c1-pfam-details-svg" data-filename="NZ_QEYD01000004.1_r25c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r25c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r25c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-24"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-24">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r25c1-tigrfam-details-svg" data-filename="NZ_QEYD01000004.1_r25c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r25c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r25c1-nrps_pks_products" data-name="r25c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r25c1-nrps_pks_monomers" data-name="r25c1-nrps_pks_monomers">
NRPS/PKS monomers
</div>








</div>










<div class="sidepanel-details-section r25c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-19"></div> <span class="help-tooltip" id="nrps-structures-help-19">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 359415 - 403294:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r25c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r25c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r25c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS monomer predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-20"></div> <span class="help-tooltip" id="nrps-monomers-help-20">Shows the predicted monomers for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">BNFEMFIJ_02939</span></strong>: X
<div class="collapser collapser-target-BNFEMFIJ_02939 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (483..871)</strong>: X
<div class="collapser collapser-target-BNFEMFIJ_02939 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-BNFEMFIJ_02939-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVGSIAAIEK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ala <span class="serif">DVWCIAAIYK</span> (65% 8Å match)</dd>
<dd>Trp <span class="serif">DVSSIGAVEK</span> (62% 8Å match)</dd>
<dd>Arg <span class="serif">DVESIGAIAK</span> (56% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">BNFEMFIJ_02957</span></strong>: X
<div class="collapser collapser-target-BNFEMFIJ_02957 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (27..463)</strong>: X
<div class="collapser collapser-target-BNFEMFIJ_02957 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Asp, Asn, Glu, Gln, Aad
<div class="collapser collapser-target-BNFEMFIJ_02957-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">TAVNIGMPIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

</dl>
</div>
</div>








</div>

</div>

</div>




<div class="page" id='r28c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_QEYD01000007.1 - Region 1 - ectoine <div class="help-container"> <div class="help-icon" data-id="region_svg-help-47"></div> <span class="help-tooltip" id="region_svg-help-47">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_QEYD01000007.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r28c1-svg" data-filename="NZ_QEYD01000007.1_r28c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,204 - 12,596 nt. (total: 10,393 nt)

<a class="cluster-rules-header" id="r28c1-rules-header" href="#r28c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r28c1-rules">ectoine: ectoine_synt</div>

<div class="region-svg-container">
<div id='r28c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-48"></div> <span class="help-tooltip" id="focus-panel-help-48">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r28c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r28c1-gene-table-details" data-name="r28c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r28c1-MIBiG-cluster-compare" data-name="r28c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r28c1-clusterblast" data-name="r28c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r28c1-knownclusterblast" data-name="r28c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r28c1-subclusterblast" data-name="r28c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r28c1-tfbs-finder" data-name="r28c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r28c1-pfam-details" data-name="r28c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r28c1-tigrfam-details" data-name="r28c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r28c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-31"></div> <span class="help-tooltip" id="gt-help-tooltip-help-31">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r28c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r28c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r28c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r28c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r28c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-25"></div> <span class="help-tooltip" id="MIBiG-help-25">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r28c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r28c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r28c1"></div>
<div id="comparison-MIBiG-r28c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">ectoine</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860: 710-5134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860</a></td>





<td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>


<td>0.57</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylarcula marina</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859: 258-3305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859</a></td>





<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>


<td>0.52</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium alcaliphilum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856: 517-4347">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856</a></td>





<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>


<td>0.52</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylophaga thalassica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857: 300-4871">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857</a></td>





<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>


<td>0.51</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylophaga alcalica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855: 245-3455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855</a></td>





<td class="cc-heat-cell" title="0.73 (id:0.50, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>


<td>0.41</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium kenyense</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.48, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.37</td>
<td>Other</td>

<td>ectoine</td>


<td>Sporosarcina pasteurii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854: 838-6879">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.50, order:0.83, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium alcaliphilum 20Z</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.47, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylobacter marinus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853: 8-3345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.48, order:0.76, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>Other</td>

<td>ectoine</td>


<td>Streptomyces anulatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052: 188-3620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.48, order:0.76, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>Other (Ectoine)</td>

<td>ectoine</td>


<td>Streptomyces sp.</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r28c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860: 710-5134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860</a></td>


<td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
<td>0.83</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylarcula marina</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859: 258-3305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859</a></td>


<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
<td>0.79</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium alcaliphilum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856: 517-4347">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856</a></td>


<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
<td>0.79</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylophaga thalassica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857: 300-4871">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857</a></td>


<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
<td>0.79</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylophaga alcalica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855: 245-3455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855</a></td>


<td class="cc-heat-cell" title="0.73 (id:0.50, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
<td>0.73</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium kenyense</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.48, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Other</td>
<td>ectoine</td>
<td>Sporosarcina pasteurii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854: 838-6879">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.50, order:0.83, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium alcaliphilum 20Z</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.47, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylobacter marinus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853: 8-3345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.48, order:0.76, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Other</td>
<td>ectoine</td>
<td>Streptomyces anulatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052: 188-3620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.48, order:0.76, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Other (Ectoine)</td>
<td>ectoine</td>
<td>Streptomyces sp.</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r28c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-26"></div> <span class="help-tooltip" id="cb-general-help-26">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r28c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r28c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r28c1_1.svg">
NZ_WIND01000003 (260429-270822): Rhodobacteraceae bacterium 2CG4 NODE 3 lengt... (41% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r28c1_2.svg">
NZ_KI421504 (1404584-1414977): Leisingera aquimarina DSM 24565 Leiaq neu scaf... (55% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r28c1_3.svg">
NZ_FODS01000016 (8583-18976): Salinihabitans flavidus strain DSM 27842, whole... (41% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r28c1_4.svg">
NZ_CP038236 (54078-64471): Leisingera sp. NJS201 plasmid unnamed2, complete s... (55% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r28c1_5.svg">
NZ_SKCI01000002 (678123-688516): Marimonas lutisali strain GH1-19 contig2, wh... (50% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r28c1_6.svg">
NZ_KI421513 (262489-272882): Leisingera caerulea DSM 24564 PhacaeDRAFT Scaffo... (55% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r28c1_7.svg">
NZ_CP051181 (955170-965572): Thalassobius gelatinovorus strain NEB572 chromos... (45% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r28c1_8.svg">
NZ_FOVR01000018 (61154-71559): Cohaesibacter marisflavi strain CGMCC 1.9157, ... (50% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r28c1_9.svg">
NZ_VANT01000006 (50863-61268): Cohaesibacter sp. CAU 1516 contig6, whole geno... (71% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r28c1_10.svg">
NZ_KN293991 (88361-98754): Roseovarius mucosus DSM 17069 Rosmu scaffold17, wh... (55% of genes show similarity), ectoine
</option>

</select>
<button id= "clusterblast-r28c1-download">Download graphic</button>
</div>
<div id="clusterblast-r28c1-svg"></div>

</div>
</div>

<div class="body-details-section r28c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-27"></div> <span class="help-tooltip" id="cb-known-help-27">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r28c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r28c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r28c1_1.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r28c1_2.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r28c1_3.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r28c1_4.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r28c1_5.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r28c1_6.svg">
ectoine
</option>

</select>
<button id="knownclusterblast-r28c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r28c1-svg"></div>

</div>
</div>

<div class="body-details-section r28c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-28"></div> <span class="help-tooltip" id="cb-sub-help-28">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r28c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-30"></div> <span class="help-tooltip" id="tfbs-body-help-30">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (2):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
LexA
<div class="collapser collapser-target-LexA collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Repressor of DNA damage response<br>
<span>Score</span>: 21.86 out of 29.65
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r28c1-3628-LexA"></div>

<div class="tfbs-hit-detail">
BldD
<div class="collapser collapser-target-BldD collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Development and antibiotic global regulator<br>
<span>Score</span>: 19.47 out of 23.43
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r28c1-12529-BldD"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r28c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-32"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-32">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r28c1-pfam-details-svg" data-filename="NZ_QEYD01000007.1_r28c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r28c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r28c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-33"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-33">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r28c1-tigrfam-details-svg" data-filename="NZ_QEYD01000007.1_r28c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r28c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r28c1-tfbs-finder" data-name="r28c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r28c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-29"></div> <span class="help-tooltip" id="tfbs-side-help-29">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/LexA" target="_blank">LexA</a></td>
<td>Repressor of DNA damage response</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/BldD" target="_blank">BldD</a></td>
<td>Development and antibiotic global regulator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r30c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_QEYD01000009.1 - Region 1 - betalactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-49"></div> <span class="help-tooltip" id="region_svg-help-49">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_QEYD01000009.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r30c1-svg" data-filename="NZ_QEYD01000009.1_r30c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 148,251 - 172,562 nt. (total: 24,312 nt)

<a class="cluster-rules-header" id="r30c1-rules-header" href="#r30c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r30c1-rules">betalactone: (HMGL-like and AMP-binding)</div>

<div class="region-svg-container">
<div id='r30c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-50"></div> <span class="help-tooltip" id="focus-panel-help-50">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r30c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r30c1-gene-table-details" data-name="r30c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r30c1-MIBiG-cluster-compare" data-name="r30c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r30c1-clusterblast" data-name="r30c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r30c1-knownclusterblast" data-name="r30c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r30c1-subclusterblast" data-name="r30c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r30c1-tfbs-finder" data-name="r30c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r30c1-pfam-details" data-name="r30c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r30c1-tigrfam-details" data-name="r30c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r30c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-40"></div> <span class="help-tooltip" id="gt-help-tooltip-help-40">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r30c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r30c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r30c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r30c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r30c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-34"></div> <span class="help-tooltip" id="MIBiG-help-34">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r30c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r30c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r30c1"></div>
<div id="comparison-MIBiG-r30c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">betalactone</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002404: 204-23152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002404/index.html#r1c1" target="_new">BGC0002404</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Other</td>

<td>falcarindiol</td>


<td>Solanum lycopersicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Other</td>

<td>citrinin</td>


<td>Monascus aurantiacus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031: 3-5918">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>Alkaloid</td>

<td>kitasetaline</td>


<td>Kitasatospora setae KM-6054</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000844: 2-13901">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000844/index.html#r1c1" target="_new">BGC0000844</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>Other</td>

<td>clavulanic acid</td>


<td>Streptomyces clavuligerus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002493: 1205-6076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002493/index.html#r1c1" target="_new">BGC0002493</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>parabactin</td>


<td>Paracoccus denitrificans PD1222</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002069: 0-10624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002069/index.html#r1c1" target="_new">BGC0002069</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Other (Aminocoumarin)</td>

<td>styrolide A, styrolide B</td>


<td>Pseudomonas fluorescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001605: 0-9118">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001605/index.html#r1c1" target="_new">BGC0001605</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Terpene</td>

<td>gibberellin</td>


<td>Sinorhizobium fredii NGR234</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002284: 0-17900">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002284/index.html#r1c1" target="_new">BGC0002284</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.48, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>Other</td>

<td>corynecin III, corynecin I, corynecin II</td>


<td>Rhodococcus sp. H-CA8f</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000632: 0-16122">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000632/index.html#r1c1" target="_new">BGC0000632</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Terpene, Saccharide</td>

<td>brasilicardin A</td>


<td>Nocardia terpenica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234: 152-6978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.47, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>jadomycin</td>


<td>Streptomyces venezuelae ATCC 10712</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r30c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002404: 204-23152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002404/index.html#r1c1" target="_new">BGC0002404</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>falcarindiol</td>
<td>Solanum lycopersicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>citrinin</td>
<td>Monascus aurantiacus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031: 3-5918">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Alkaloid</td>
<td>kitasetaline</td>
<td>Kitasatospora setae KM-6054</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000844: 2-13901">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000844/index.html#r1c1" target="_new">BGC0000844</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Other</td>
<td>clavulanic acid</td>
<td>Streptomyces clavuligerus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002493: 1205-6076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002493/index.html#r1c1" target="_new">BGC0002493</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>parabactin</td>
<td>Paracoccus denitrificans PD1222</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002069: 0-10624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002069/index.html#r1c1" target="_new">BGC0002069</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Other (Aminocoumarin)</td>
<td>styrolide A, styrolide B</td>
<td>Pseudomonas fluorescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001605: 0-9118">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001605/index.html#r1c1" target="_new">BGC0001605</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Terpene</td>
<td>gibberellin</td>
<td>Sinorhizobium fredii NGR234</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002284: 0-17900">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002284/index.html#r1c1" target="_new">BGC0002284</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.48, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Other</td>
<td>corynecin III, corynecin I, corynecin II</td>
<td>Rhodococcus sp. H-CA8f</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000632: 0-16122">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000632/index.html#r1c1" target="_new">BGC0000632</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Terpene, Saccharide</td>
<td>brasilicardin A</td>
<td>Nocardia terpenica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234: 152-6978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.47, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>jadomycin</td>
<td>Streptomyces venezuelae ATCC 10712</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r30c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-35"></div> <span class="help-tooltip" id="cb-general-help-35">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r30c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r30c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r30c1_1.svg">
NZ_FNIR01000003 (81289-105771): Klenkia soli strain DSM 45843, whole genome s... (57% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r30c1_2.svg">
NZ_FWXX01000006 (11608-35872): Tropicibacter naphthalenivorans strain DSM 195... (56% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r30c1_3.svg">
NZ_JQMF01000027 (47755-72160): Rhizobium sp. CF097 EW87DRAFT scaffold00027.27... (44% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r30c1_4.svg">
NZ_JHVP01000004 (261145-289455): Geodermatophilaceae bacterium URHA0031 N543D... (39% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r30c1_5.svg">
NZ_QKMH01000002 (143081-167415): Geodermatophilus sp. DSM 44511 Ga0056087 sca... (28% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r30c1_6.svg">
NZ_NQYH01000004 (86546-110861): Pusillimonas maritima strain L52-1-41 NODE 4 ... (38% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r30c1_7.svg">
NZ_NQOU01000002 (156550-180865): Pusillimonas maritima strain 17-4A NODE 2 le... (38% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r30c1_8.svg">
NZ_CP049990 (151528-174923): Pusillimonas sp. DMV24BSW D chromosome, complete... (38% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r30c1_9.svg">
NZ_NIQA01000002 (366034-390340): Pusillimonas sp. T2 scaffold2, whole genome ... (38% of genes show similarity), betalactone
</option>

<option value = "svg/clusterblast_r30c1_10.svg">
NZ_SLZS01000011 (2345-30716): Streptomyces sp. BK335 Ga0307713 111, whole gen... (30% of genes show similarity), betalactone
</option>

</select>
<button id= "clusterblast-r30c1-download">Download graphic</button>
</div>
<div id="clusterblast-r30c1-svg"></div>

</div>
</div>

<div class="body-details-section r30c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-36"></div> <span class="help-tooltip" id="cb-known-help-36">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r30c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r30c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r30c1_1.svg">
xantholipin
</option>

<option value = "svg/knownclusterblast_r30c1_2.svg">
R1128
</option>

<option value = "svg/knownclusterblast_r30c1_3.svg">
persiamycin A
</option>

<option value = "svg/knownclusterblast_r30c1_4.svg">
formicamycins A-M
</option>

</select>
<button id="knownclusterblast-r30c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r30c1-svg"></div>

</div>
</div>

<div class="body-details-section r30c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-37"></div> <span class="help-tooltip" id="cb-sub-help-37">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r30c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-39"></div> <span class="help-tooltip" id="tfbs-body-help-39">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
DasR
<div class="collapser collapser-target-DasR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: N-acetylglucosamine dependent repressor<br>
<span>Score</span>: 21.47 out of 27.04
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r30c1-172394-DasR"></div>

</div>


</div>
</div>



<div class="body-details-section r30c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-41"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-41">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r30c1-pfam-details-svg" data-filename="NZ_QEYD01000009.1_r30c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r30c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r30c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-42"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-42">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r30c1-tigrfam-details-svg" data-filename="NZ_QEYD01000009.1_r30c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r30c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r30c1-tfbs-finder" data-name="r30c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r30c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-38"></div> <span class="help-tooltip" id="tfbs-side-help-38">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/DasR" target="_blank">DasR</a></td>
<td>N-acetylglucosamine dependent repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>