
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>GCF_003034985.1_Cereibacter_changlensis_JA139_JA139 - 10 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/bacteria.css">
<meta property="og:title" content="GCF_003034985.1_Cereibacter_changlensis_JA139_JA139 - 9 record(s) - 10 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-7a3c34cb-c249-457f-b2fa-b41af47c356d/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://antismash.secondarymetabolites.org/">
<img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://antismash.secondarymetabolites.org/">
antiSMASH version 7.0.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="GCF_003034985.1_Cereibacter_changlensis_JA139_JA139.zip">Download all results</a></li>

<li><a href="GCF_003034985.1_Cereibacter_changlensis_JA139_JA139.gbk">Download GenBank summary file</a></li>

<li><a href="bacteria-7a3c34cb-c249-457f-b2fa-b41af47c356d.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton terpene terpene r2c1">
<a href="#r2c1">2.1</a>
</div>

<div class="regbutton other hserlactone r2c2">
<a href="#r2c2">2.2</a>
</div>



<div class="regbutton RiPP RRE-containing r65c1">
<a href="#r65c1">65.1</a>
</div>



<div class="regbutton terpene terpene r79c1">
<a href="#r79c1">79.1</a>
</div>



<div class="regbutton other hserlactone r89c1">
<a href="#r89c1">89.1</a>
</div>



<div class="regbutton NRPS NRPS-like r93c1">
<a href="#r93c1">93.1</a>
</div>



<div class="regbutton hybrid r151c1">
<a href="#r151c1">151.1</a>
</div>



<div class="regbutton RiPP redox-cofactor r187c1">
<a href="#r187c1">187.1</a>
</div>



<div class="regbutton other hserlactone r251c1">
<a href="#r251c1">251.1</a>
</div>



<div class="regbutton other hserlactone r318c1">
<a href="#r318c1">318.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>NZ_PZKG01000002.1</strong> (Cereibacter changlensis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-2" data-filename="NZ_PZKG01000002.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-2" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r2c1">
<td class="regbutton terpene terpene r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">22,977</td>
<td class="digits table-split-left">43,744</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c2">
<td class="regbutton other hserlactone r2c2">
<a href="#r2c2">Region&nbsp2.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">54,155</td>
<td class="digits table-split-left">74,892</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_PZKG01000065.1</strong> (Cereibacter changlensis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-65" data-filename="NZ_PZKG01000065.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-65" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r65c1">
<td class="regbutton RiPP RRE-containing r65c1">
<a href="#r65c1">Region&nbsp65.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#rre-containing" target="_blank">RRE-containing</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">11,175</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001765/1" target="_blank">salecan</a></td>
<td>Saccharide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 10%, #ffffff00 10%)">10%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_PZKG01000079.1</strong> (Cereibacter changlensis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-79" data-filename="NZ_PZKG01000079.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-79" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r79c1">
<td class="regbutton terpene terpene r79c1">
<a href="#r79c1">Region&nbsp79.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">6,624</td>
<td class="digits table-split-left">19,647</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000647/1" target="_blank">carotenoid</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_PZKG01000089.1</strong> (Cereibacter changlensis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-89" data-filename="NZ_PZKG01000089.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-89" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r89c1">
<td class="regbutton other hserlactone r89c1">
<a href="#r89c1">Region&nbsp89.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">17,237</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_PZKG01000093.1</strong> (Cereibacter changlensis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-93" data-filename="NZ_PZKG01000093.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-93" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r93c1">
<td class="regbutton NRPS NRPS-like r93c1">
<a href="#r93c1">Region&nbsp93.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">15,795</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002043/1" target="_blank">cittilin A/cittilin B</a></td>
<td>RiPP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 11%, #ffffff00 11%)">11%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_PZKG01000151.1</strong> (Cereibacter changlensis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-151" data-filename="NZ_PZKG01000151.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-151" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r151c1">
<td class="regbutton hybrid r151c1">
<a href="#r151c1">Region&nbsp151.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">8,432</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_PZKG01000187.1</strong> (Cereibacter changlensis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-187" data-filename="NZ_PZKG01000187.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-187" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r187c1">
<td class="regbutton RiPP redox-cofactor r187c1">
<a href="#r187c1">Region&nbsp187.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#redox-cofactor" target="_blank">redox-cofactor</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">6,255</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_PZKG01000251.1</strong> (Cereibacter changlensis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-251" data-filename="NZ_PZKG01000251.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-251" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r251c1">
<td class="regbutton other hserlactone r251c1">
<a href="#r251c1">Region&nbsp251.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">3,036</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_PZKG01000318.1</strong> (Cereibacter changlensis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-318" data-filename="NZ_PZKG01000318.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-318" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r318c1">
<td class="regbutton other hserlactone r318c1">
<a href="#r318c1">Region&nbsp318.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">1,602</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r2c1">
<td class="regbutton terpene terpene r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">22,977</td>
<td class="digits table-split-left">43,744</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r2c2">
<td class="regbutton other hserlactone r2c2">
<a href="#r2c2">Region&nbsp2.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">54,155</td>
<td class="digits table-split-left">74,892</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r65c1">
<td class="regbutton RiPP RRE-containing r65c1">
<a href="#r65c1">Region&nbsp65.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#rre-containing" target="_blank">RRE-containing</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">11,175</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001765/1" target="_blank">salecan</a></td>
<td>Saccharide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 10%, #ffffff00 10%)">10%</td>

</tr>


<tr class="linked-row even" data-anchor="#r79c1">
<td class="regbutton terpene terpene r79c1">
<a href="#r79c1">Region&nbsp79.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">6,624</td>
<td class="digits table-split-left">19,647</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000647/1" target="_blank">carotenoid</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r89c1">
<td class="regbutton other hserlactone r89c1">
<a href="#r89c1">Region&nbsp89.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">17,237</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r93c1">
<td class="regbutton NRPS NRPS-like r93c1">
<a href="#r93c1">Region&nbsp93.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">15,795</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002043/1" target="_blank">cittilin A/cittilin B</a></td>
<td>RiPP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 11%, #ffffff00 11%)">11%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r151c1">
<td class="regbutton hybrid r151c1">
<a href="#r151c1">Region&nbsp151.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">8,432</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r187c1">
<td class="regbutton RiPP redox-cofactor r187c1">
<a href="#r187c1">Region&nbsp187.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#redox-cofactor" target="_blank">redox-cofactor</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">6,255</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r251c1">
<td class="regbutton other hserlactone r251c1">
<a href="#r251c1">Region&nbsp251.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">3,036</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r318c1">
<td class="regbutton other hserlactone r318c1">
<a href="#r318c1">Region&nbsp318.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">1,602</td>

<td colspan="3"></td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>NZ_PZKG01000001.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000003.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000004.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000005.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000006.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000007.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000008.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000009.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000010.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000011.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000012.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000013.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000014.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000015.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000016.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000017.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000018.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000019.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000020.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000021.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000022.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000023.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000024.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000025.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000026.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000027.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000028.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000029.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000030.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000031.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000032.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000033.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000034.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000035.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000036.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000037.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000038.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000039.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000040.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000041.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000042.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000043.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000044.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000045.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000046.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000047.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000048.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000049.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000050.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000051.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000052.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000053.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000054.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000055.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000056.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000057.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000058.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000059.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000060.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000061.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000062.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000063.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000064.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000066.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000067.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000068.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000069.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000070.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000071.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000072.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000073.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000074.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000075.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000076.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000077.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000078.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000080.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000081.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000082.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000083.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000084.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000085.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000086.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000087.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000088.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000090.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000091.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000092.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000094.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000095.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000096.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000097.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000098.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000099.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000100.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000101.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000102.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000103.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000104.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000105.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000106.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000107.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000108.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000109.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000110.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000111.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000112.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000113.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000114.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000115.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000116.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000117.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000118.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000119.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000120.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000121.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000122.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000123.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000124.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000125.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000126.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000127.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000128.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000129.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000130.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000131.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000132.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000133.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000134.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000135.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000136.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000137.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000138.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000139.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000140.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000141.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000142.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000143.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000144.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000145.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000146.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000147.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000148.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000149.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000150.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000152.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000153.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000154.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000155.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000156.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000157.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000158.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000159.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000160.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000161.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000162.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000163.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000164.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000165.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000166.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000167.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000168.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000169.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000170.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000171.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000172.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000173.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000174.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000175.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000176.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000177.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000178.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000179.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000180.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000181.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000182.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000183.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000184.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000185.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000186.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000188.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000189.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000190.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000191.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000192.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000193.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000194.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000195.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000196.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000197.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000198.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000199.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000200.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000201.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000202.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000203.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000204.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000205.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000206.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000207.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000208.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000209.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000210.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000211.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000212.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000213.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000214.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000215.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000216.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000217.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000218.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000219.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000220.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000221.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000222.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000223.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000224.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000225.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000226.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000227.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000228.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000229.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000230.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000231.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000232.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000233.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000234.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000235.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000236.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000237.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000238.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000239.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000240.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000241.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000242.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000243.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000244.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000245.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000246.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000247.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000248.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000249.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000250.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000252.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000253.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000254.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000255.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000256.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000257.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000258.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000259.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000260.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000261.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000262.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000263.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000264.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000265.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000266.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000267.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000268.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000269.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000270.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000271.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000272.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000273.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000274.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000275.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000276.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000277.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000278.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000279.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000280.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000281.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000282.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000283.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000284.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000285.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000286.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000287.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000288.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000289.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000290.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000291.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000292.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000293.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000294.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000295.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000296.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000297.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000298.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000299.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000300.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000301.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000302.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000303.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000304.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000305.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000306.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000307.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000308.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000309.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000310.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000311.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000312.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000313.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000314.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000315.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000316.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000317.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000319.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000320.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000321.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000322.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000323.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000324.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000325.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000326.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000327.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000328.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000329.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000330.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000331.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000332.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000333.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000334.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000335.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000336.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000337.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000338.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000339.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000340.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000341.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000342.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000343.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000344.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000345.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000346.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000347.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000348.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000349.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000350.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000351.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000352.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000353.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000354.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000355.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000356.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000357.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000358.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000359.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000360.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000361.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000362.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000363.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000364.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000365.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000366.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000367.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000368.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000369.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000370.1</strong> (Cereibacter changlensis)</dd>

<dd><strong>NZ_PZKG01000371.1</strong> (Cereibacter changlensis)</dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>







<div class="page" id='r2c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_PZKG01000002.1 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-90"></div> <span class="help-tooltip" id="region_svg-help-90">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_PZKG01000002.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c1-svg" data-filename="NZ_PZKG01000002.1_r2c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 22,977 - 43,744 nt. (total: 20,768 nt)

<a class="cluster-rules-header" id="r2c1-rules-header" href="#r2c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r2c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-91"></div> <span class="help-tooltip" id="focus-panel-help-91">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c1-gene-table-details" data-name="r2c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c1-MIBiG-cluster-compare" data-name="r2c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c1-clusterblast" data-name="r2c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c1-knownclusterblast" data-name="r2c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c1-subclusterblast" data-name="r2c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r2c1-tfbs-finder" data-name="r2c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r2c1-pfam-details" data-name="r2c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c1-tigrfam-details" data-name="r2c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-7"></div> <span class="help-tooltip" id="gt-help-tooltip-help-7">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-1"></div> <span class="help-tooltip" id="MIBiG-help-1">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c1"></div>
<div id="comparison-MIBiG-r2c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001859: 6609-19144">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001859/index.html#r1c1" target="_new">BGC0001859</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Other (Phosphonate)</td>

<td>fosfomycin</td>


<td>Pseudomonas syringae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000904: 116-14341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000904/index.html#r1c1" target="_new">BGC0000904</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Other</td>

<td>FR-900098</td>


<td>Streptomyces rubellomurinus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Terpene</td>

<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>


<td>Annulohypoxylon truncatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551: 0-6728">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>SapB</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001210: 0-9235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001210/index.html#r1c1" target="_new">BGC0001210</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>pseudomycoicidin</td>


<td>Bacillus pseudomycoides DSM 12442</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000521: 0-12132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000521/index.html#r1c1" target="_new">BGC0000521</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>lacticin 481</td>


<td>Lactococcus lactis subsp. lactis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000616: 0-4173">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000616/index.html#r1c1" target="_new">BGC0000616</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>amylocyclicin</td>


<td>Bacillus velezensis FZB42</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002700: 0-5832">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002700/index.html#r1c1" target="_new">BGC0002700</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>bacinapeptin</td>


<td>Bacillus nakamurai</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002630: 0-4291">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002630/index.html#r1c1" target="_new">BGC0002630</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>thatisin</td>


<td>Lysobacter antibioticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002150: 0-3220">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002150/index.html#r1c1" target="_new">BGC0002150</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>triculamin</td>


<td>Streptomyces triculaminicus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001859: 6609-19144">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001859/index.html#r1c1" target="_new">BGC0001859</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other (Phosphonate)</td>
<td>fosfomycin</td>
<td>Pseudomonas syringae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000904: 116-14341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000904/index.html#r1c1" target="_new">BGC0000904</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Other</td>
<td>FR-900098</td>
<td>Streptomyces rubellomurinus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Terpene</td>
<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>
<td>Annulohypoxylon truncatum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551: 0-6728">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>SapB</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001210: 0-9235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001210/index.html#r1c1" target="_new">BGC0001210</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>pseudomycoicidin</td>
<td>Bacillus pseudomycoides DSM 12442</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000521: 0-12132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000521/index.html#r1c1" target="_new">BGC0000521</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>lacticin 481</td>
<td>Lactococcus lactis subsp. lactis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000616: 0-4173">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000616/index.html#r1c1" target="_new">BGC0000616</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>amylocyclicin</td>
<td>Bacillus velezensis FZB42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002700: 0-5832">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002700/index.html#r1c1" target="_new">BGC0002700</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>bacinapeptin</td>
<td>Bacillus nakamurai</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002630: 0-4291">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002630/index.html#r1c1" target="_new">BGC0002630</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>thatisin</td>
<td>Lysobacter antibioticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002150: 0-3220">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002150/index.html#r1c1" target="_new">BGC0002150</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>triculamin</td>
<td>Streptomyces triculaminicus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-2"></div> <span class="help-tooltip" id="cb-general-help-2">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c1_1.svg">
NZ_CP023548 (1959658-1979193): Rhodobacter sp. CZR27 chromosome, complete genome (52% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c1_2.svg">
NC_009049 (1429419-1447824): Rhodobacter sphaeroides ATCC 17029 chromosome 1,... (52% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c1_3.svg">
NZ_PZZW01000002 (59846-79139): Rhodobacter johrii strain JA192 Ga0191650 102,... (52% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c1_4.svg">
NZ_CP033442 (1872607-1893375): Rhodobacter sphaeroides strain AB25 chromosome 1 (52% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c1_5.svg">
NZ_FZOV01000001 (427613-448381): Rhodobacter megalophilus strain DSM 18937, w... (52% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c1_6.svg">
NZ_CM001161 (791443-809866): Rhodobacter sphaeroides WS8N chromosome I, whole... (52% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c1_7.svg">
NZ_CP030271 (1366785-1385319): Rhodobacter sphaeroides 2.4.1 chromosome 1, co... (52% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c1_8.svg">
NC_011963 (1044327-1063597): Rhodobacter sphaeroides KD131 chromosome 1, comp... (47% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c1_9.svg">
NZ_CP038492 (1546062-1566833): Boseongicola sp. CCM32 chromosome, complete ge... (45% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c1_10.svg">
NZ_FOGU01000004 (180868-209551): Roseivivax roseus strain DSM 23042, whole ge... (32% of genes show similarity), hserlactone,terpene
</option>

</select>
<button id= "clusterblast-r2c1-download">Download graphic</button>
</div>
<div id="clusterblast-r2c1-svg"></div>

</div>
</div>

<div class="body-details-section r2c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-3"></div> <span class="help-tooltip" id="cb-known-help-3">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-4"></div> <span class="help-tooltip" id="cb-sub-help-4">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r2c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-6"></div> <span class="help-tooltip" id="tfbs-body-help-6">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AbrC3
<div class="collapser collapser-target-AbrC3 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Antibiotic production activator<br>
<span>Score</span>: 18.07 out of 18.16
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r2c1-43659-AbrC3"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (2):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
NuR
<div class="collapser collapser-target-NuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Nickel-responsive regulator<br>
<span>Score</span>: 20.70 out of 30.98
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c1-26259-NuR"></div>

<div class="tfbs-hit-detail">
HypR
<div class="collapser collapser-target-HypR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: L-hydroxyproline utilization repressor<br>
<span>Score</span>: 19.14 out of 22.40
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c1-40409-HypR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r2c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-8"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-8">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-pfam-details-svg" data-filename="NZ_PZKG01000002.1_r2c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-9"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-9">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-tigrfam-details-svg" data-filename="NZ_PZKG01000002.1_r2c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r2c1-tfbs-finder" data-name="r2c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r2c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-5"></div> <span class="help-tooltip" id="tfbs-side-help-5">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NuR" target="_blank">NuR</a></td>
<td>Nickel-responsive regulator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/HypR" target="_blank">HypR</a></td>
<td>L-hydroxyproline utilization repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AbrC3" target="_blank">AbrC3</a></td>
<td>Antibiotic production activator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r2c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_PZKG01000002.1 - Region 2 - hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-92"></div> <span class="help-tooltip" id="region_svg-help-92">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_PZKG01000002.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c2-svg" data-filename="NZ_PZKG01000002.1_r2c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 54,155 - 74,892 nt. (total: 20,738 nt)

<a class="cluster-rules-header" id="r2c2-rules-header" href="#r2c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c2-rules">hserlactone: Autoind_synth</div>

<div class="region-svg-container">
<div id='r2c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-93"></div> <span class="help-tooltip" id="focus-panel-help-93">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c2-gene-table-details" data-name="r2c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c2-MIBiG-cluster-compare" data-name="r2c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c2-clusterblast" data-name="r2c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c2-knownclusterblast" data-name="r2c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c2-subclusterblast" data-name="r2c2-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r2c2-tfbs-finder" data-name="r2c2-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r2c2-pfam-details" data-name="r2c2-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c2-tigrfam-details" data-name="r2c2-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-16"></div> <span class="help-tooltip" id="gt-help-tooltip-help-16">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-10"></div> <span class="help-tooltip" id="MIBiG-help-10">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c2"></div>
<div id="comparison-MIBiG-r2c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">hserlactone</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286: 17-17862">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>viguiepinol</td>


<td>Streptomyces sp. KO-3988</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001896: 0-8329">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001896/index.html#r1c1" target="_new">BGC0001896</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Other</td>

<td>carbazomycin B</td>


<td>Streptomyces luteoverticillatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002554: 144-7739">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002554/index.html#r1c1" target="_new">BGC0002554</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.49, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Alkaloid</td>

<td>carquinostatin A, carquinostatin B</td>


<td>Streptomyces exfoliatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001903: 0-10870">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001903/index.html#r1c1" target="_new">BGC0001903</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Other</td>

<td>neocarazostatin A</td>


<td>Streptomyces sp. MA37</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000922: 0-13136">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000922/index.html#r1c1" target="_new">BGC0000922</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Other</td>

<td>pseudomonas quinolone signal (PQS)</td>


<td>Pseudomonas aeruginosa PAO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002711: 0-9402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002711/index.html#r1c1" target="_new">BGC0002711</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.48, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Polyketide</td>

<td>nostovalerolactone, 9-dehydronostovalerolactone</td>


<td>Nostoc punctiforme PCC 73102</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000800: 32-15282">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000800/index.html#r1c1" target="_new">BGC0000800</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.49, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.15</td>
<td>Saccharide</td>

<td>xanthan</td>


<td>Xanthomonas campestris</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001810: 122-16114">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001810/index.html#r1c1" target="_new">BGC0001810</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.47, order:N/A, components:0.41)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>Polyketide</td>

<td>trehangelin</td>


<td>Polymorphospora rubra</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001080: 0-17804">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001080/index.html#r1c1" target="_new">BGC0001080</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.46, order:N/A, components:0.41)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>Other (Phenazine)</td>

<td>endophenazine A, endophenazine B</td>


<td>Streptomyces anulatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001850: 1068-23493">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001850/index.html#r1c1" target="_new">BGC0001850</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.46, order:N/A, components:0.41)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Other (Shikimate-derived)</td>

<td>diazaquinomycin A, diazaquinomycin E, diazaquinomycin F, diazaquinomycin G</td>


<td>Streptomyces sp. F001</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286: 17-17862">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>viguiepinol</td>
<td>Streptomyces sp. KO-3988</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001896: 0-8329">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001896/index.html#r1c1" target="_new">BGC0001896</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other</td>
<td>carbazomycin B</td>
<td>Streptomyces luteoverticillatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002554: 144-7739">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002554/index.html#r1c1" target="_new">BGC0002554</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.49, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Alkaloid</td>
<td>carquinostatin A, carquinostatin B</td>
<td>Streptomyces exfoliatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001903: 0-10870">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001903/index.html#r1c1" target="_new">BGC0001903</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Other</td>
<td>neocarazostatin A</td>
<td>Streptomyces sp. MA37</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000922: 0-13136">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000922/index.html#r1c1" target="_new">BGC0000922</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Other</td>
<td>pseudomonas quinolone signal (PQS)</td>
<td>Pseudomonas aeruginosa PAO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002711: 0-9402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002711/index.html#r1c1" target="_new">BGC0002711</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.48, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Polyketide</td>
<td>nostovalerolactone, 9-dehydronostovalerolactone</td>
<td>Nostoc punctiforme PCC 73102</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000800: 32-15282">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000800/index.html#r1c1" target="_new">BGC0000800</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.49, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Saccharide</td>
<td>xanthan</td>
<td>Xanthomonas campestris</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001810: 122-16114">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001810/index.html#r1c1" target="_new">BGC0001810</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.47, order:N/A, components:0.41)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Polyketide</td>
<td>trehangelin</td>
<td>Polymorphospora rubra</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001080: 0-17804">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001080/index.html#r1c1" target="_new">BGC0001080</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.46, order:N/A, components:0.41)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Other (Phenazine)</td>
<td>endophenazine A, endophenazine B</td>
<td>Streptomyces anulatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001850: 1068-23493">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001850/index.html#r1c1" target="_new">BGC0001850</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.46, order:N/A, components:0.41)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Other (Shikimate-derived)</td>
<td>diazaquinomycin A, diazaquinomycin E, diazaquinomycin F, diazaquinomycin G</td>
<td>Streptomyces sp. F001</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-11"></div> <span class="help-tooltip" id="cb-general-help-11">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c2_1.svg">
NZ_SLXL01000003 (159498-180245): Rhodovulum adriaticum strain DSM 2781 Ga0310... (54% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r2c2_2.svg">
NZ_NSBU01000007 (1036290-1055384): Actibacterium pelagium strain JN33 JN33 S7... (54% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r2c2_3.svg">
NZ_CP046620 (639893-660317): Rhodobacteraceae bacterium 9Alg 56 chromosome, c... (59% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r2c2_4.svg">
NZ_FOGU01000004 (180868-209551): Roseivivax roseus strain DSM 23042, whole ge... (38% of genes show similarity), hserlactone,terpene
</option>

<option value = "svg/clusterblast_r2c2_5.svg">
NZ_CP004393 (2661955-2681136): Celeribacter indicus strain P73 chromosome, co... (52% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r2c2_6.svg">
NZ_FORY01000008 (149907-170660): Celeribacter halophilus strain CGMCC 1.8891,... (52% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r2c2_7.svg">
NZ_CP054599 (2176742-2196501): Sulfitobacter pseudonitzschiae strain H46 chro... (61% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r2c2_8.svg">
NZ_JIBC01000007 (81448-102171): Sulfitobacter sp. 20 GPM-1509m N517DRAFT scaf... (61% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r2c2_9.svg">
NZ_CH902584 (3511904-3532642): Roseovarius sp. 217 1099463000297, whole genom... (60% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r2c2_10.svg">
NZ_QVMW01000013 (44432-65164): Rhodobacteraceae bacterium 63075 63075.63 clea... (54% of genes show similarity), hserlactone
</option>

</select>
<button id= "clusterblast-r2c2-download">Download graphic</button>
</div>
<div id="clusterblast-r2c2-svg"></div>

</div>
</div>

<div class="body-details-section r2c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-12"></div> <span class="help-tooltip" id="cb-known-help-12">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-13"></div> <span class="help-tooltip" id="cb-sub-help-13">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r2c2-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-15"></div> <span class="help-tooltip" id="tfbs-body-help-15">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AfsR
<div class="collapser collapser-target-AfsR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Pleiotropic regulatory for antibiotic production<br>
<span>Score</span>: 18.46 out of 23.05
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r2c2-62961-AfsR"></div>

</div>


</div>
</div>



<div class="body-details-section r2c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-17"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-17">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c2-pfam-details-svg" data-filename="NZ_PZKG01000002.1_r2c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c2-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-18"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-18">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c2-tigrfam-details-svg" data-filename="NZ_PZKG01000002.1_r2c2_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c2-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r2c2-tfbs-finder" data-name="r2c2-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r2c2-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-14"></div> <span class="help-tooltip" id="tfbs-side-help-14">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AfsR" target="_blank">AfsR</a></td>
<td>Pleiotropic regulatory for antibiotic production</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r65c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_PZKG01000065.1 - Region 1 - RRE-containing <div class="help-container"> <div class="help-icon" data-id="region_svg-help-94"></div> <span class="help-tooltip" id="region_svg-help-94">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_PZKG01000065.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r65c1-svg" data-filename="NZ_PZKG01000065.1_r65c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 11,175 nt. (total: 11,175 nt)

<a class="cluster-rules-header" id="r65c1-rules-header" href="#r65c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r65c1-rules">RRE-containing: minimum(1, [Bottromycin_Methyltransferase_RRE, Cyanobactin_D_RRE, Cytolysin_C_RRE, Goadsporin_CD_RRE, Heterocycloanthracin_C_RRE, Lanthipeptide_LanB_RRE, Lasso_Fused_RRE, Mycofactocin_RRE, NHLP_CD_RRE, Other_Sactipeptide_rSAM_RRE, Pantocin_Microcin_RRE, Plantazolicin_C_RRE, PlpY_RRE, PqqD_RRE, Proteusin_Epimerase_RRE, Proteusin_Methyltransferase_RRE, Quinohemoprotein_rSAM_RRE, Ranthipeptide_rSAM_RRE, Stand_Alone_Lasso_RRE, Streptide_RRE, Subtilosin_rSAM_RRE, Thiaglutamate_B_RRE, Thiaglutamate_G_RRE, Thiaglutamate_I_RRE, Thiopeptide_F_RRE, Thuricin_rSAM_RRE, Thurincin_rSAM_RRE, Trifolitoxin_RRE])</div>

<div class="region-svg-container">
<div id='r65c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-95"></div> <span class="help-tooltip" id="focus-panel-help-95">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r65c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r65c1-gene-table-details" data-name="r65c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r65c1-MIBiG-cluster-compare" data-name="r65c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r65c1-clusterblast" data-name="r65c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r65c1-knownclusterblast" data-name="r65c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r65c1-subclusterblast" data-name="r65c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r65c1-tfbs-finder" data-name="r65c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r65c1-pfam-details" data-name="r65c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r65c1-tigrfam-details" data-name="r65c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r65c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-26"></div> <span class="help-tooltip" id="gt-help-tooltip-help-26">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r65c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r65c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r65c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r65c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r65c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-19"></div> <span class="help-tooltip" id="MIBiG-help-19">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r65c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r65c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r65c1"></div>
<div id="comparison-MIBiG-r65c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">RRE-containing</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000740: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000740/index.html#r1c1" target="_new">BGC0000740</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000746: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000746/index.html#r1c1" target="_new">BGC0000746</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000747: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000747/index.html#r1c1" target="_new">BGC0000747</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000748: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000748/index.html#r1c1" target="_new">BGC0000748</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000741: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000741/index.html#r1c1" target="_new">BGC0000741</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000744: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000744/index.html#r1c1" target="_new">BGC0000744</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000745: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000745/index.html#r1c1" target="_new">BGC0000745</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000742: 0-2815">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000742/index.html#r1c1" target="_new">BGC0000742</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000743: 0-2815">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000743/index.html#r1c1" target="_new">BGC0000743</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000800: 32-15282">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000800/index.html#r1c1" target="_new">BGC0000800</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Saccharide</td>

<td>xanthan</td>


<td>Xanthomonas campestris</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r65c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000740: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000740/index.html#r1c1" target="_new">BGC0000740</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000746: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000746/index.html#r1c1" target="_new">BGC0000746</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000747: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000747/index.html#r1c1" target="_new">BGC0000747</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000748: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000748/index.html#r1c1" target="_new">BGC0000748</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000741: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000741/index.html#r1c1" target="_new">BGC0000741</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000744: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000744/index.html#r1c1" target="_new">BGC0000744</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000745: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000745/index.html#r1c1" target="_new">BGC0000745</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000742: 0-2815">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000742/index.html#r1c1" target="_new">BGC0000742</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000743: 0-2815">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000743/index.html#r1c1" target="_new">BGC0000743</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000800: 32-15282">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000800/index.html#r1c1" target="_new">BGC0000800</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Saccharide</td>
<td>xanthan</td>
<td>Xanthomonas campestris</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r65c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-20"></div> <span class="help-tooltip" id="cb-general-help-20">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r65c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r65c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r65c1_1.svg">
NC_009049 (1251632-1273983): Rhodobacter sphaeroides ATCC 17029 chromosome 1,... (50% of genes show similarity), lassopeptide
</option>

<option value = "svg/clusterblast_r65c1_2.svg">
NZ_FZOV01000001 (602633-624984): Rhodobacter megalophilus strain DSM 18937, w... (50% of genes show similarity), lassopeptide
</option>

<option value = "svg/clusterblast_r65c1_3.svg">
NZ_CP030271 (1189092-1211442): Rhodobacter sphaeroides 2.4.1 chromosome 1, co... (50% of genes show similarity), lassopeptide
</option>

<option value = "svg/clusterblast_r65c1_4.svg">
NZ_CM001161 (613317-635668): Rhodobacter sphaeroides WS8N chromosome I, whole... (50% of genes show similarity), lassopeptide
</option>

<option value = "svg/clusterblast_r65c1_5.svg">
NZ_CP033446 (2017187-2039553): Rhodobacter sphaeroides strain AB27 chromosome 1 (50% of genes show similarity), lassopeptide
</option>

<option value = "svg/clusterblast_r65c1_6.svg">
NZ_CP033442 (2047710-2070179): Rhodobacter sphaeroides strain AB25 chromosome 1 (50% of genes show similarity), lassopeptide
</option>

<option value = "svg/clusterblast_r65c1_7.svg">
NC_011963 (866202-888553): Rhodobacter sphaeroides KD131 chromosome 1, comple... (52% of genes show similarity), lassopeptide
</option>

<option value = "svg/clusterblast_r65c1_8.svg">
NZ_CP023548 (2135891-2158236): Rhodobacter sp. CZR27 chromosome, complete genome (47% of genes show similarity), lassopeptide
</option>

<option value = "svg/clusterblast_r65c1_9.svg">
NZ_PZZW01000001 (897999-920350): Rhodobacter johrii strain JA192 Ga0191650 10... (50% of genes show similarity), lassopeptide
</option>

<option value = "svg/clusterblast_r65c1_10.svg">
NZ_CP031750 (1736658-1759003): Rhodobacter sphaeroides strain EBL0706 chromos... (47% of genes show similarity), lassopeptide
</option>

</select>
<button id= "clusterblast-r65c1-download">Download graphic</button>
</div>
<div id="clusterblast-r65c1-svg"></div>

</div>
</div>

<div class="body-details-section r65c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-21"></div> <span class="help-tooltip" id="cb-known-help-21">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r65c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r65c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r65c1_1.svg">
salecan
</option>

<option value = "svg/knownclusterblast_r65c1_2.svg">
salecan
</option>

</select>
<button id="knownclusterblast-r65c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r65c1-svg"></div>

</div>
</div>

<div class="body-details-section r65c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-22"></div> <span class="help-tooltip" id="cb-sub-help-22">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r65c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-25"></div> <span class="help-tooltip" id="tfbs-body-help-25">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 17.32 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r65c1-9484-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r65c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-27"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-27">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r65c1-pfam-details-svg" data-filename="NZ_PZKG01000065.1_r65c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r65c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r65c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-28"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-28">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r65c1-tigrfam-details-svg" data-filename="NZ_PZKG01000065.1_r65c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r65c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r65c1-RREfinder" data-name="r65c1-RREfinder">
RREFinder
</div>



<div class="sidepanel-details-header r65c1-tfbs-finder" data-name="r65c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r65c1-RREfinder">
<div class="more-details">
<div class="heading">
<span>RRE predictions</span>
<div class="help-container"> <div class="help-icon" data-id="rre-side-help-23"></div> <span class="help-tooltip" id="rre-side-help-23">RREfinder results sidepanel.</span></div>
</div>
<dl class = 'prediction-text'>

<span><strong>RRE-containing protocluster</strong> (0...11175)</span>
<dt></dt>




<strong>EOMLKCNM_02755</strong> <div class="collapser collapser-target-EOMLKCNM_02755 collapser-level-cds"> </div><div class="collapser-content">

<dd>RRE type: Stand_Alone_Lasso_RRE</dd>
<dd>Location: 9 - 91</dd>
<dd>Score: 81.3</dd>

</div><br>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r65c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-24"></div> <span class="help-tooltip" id="tfbs-side-help-24">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r79c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_PZKG01000079.1 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-96"></div> <span class="help-tooltip" id="region_svg-help-96">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_PZKG01000079.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r79c1-svg" data-filename="NZ_PZKG01000079.1_r79c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 6,624 - 19,647 nt. (total: 13,024 nt)

<a class="cluster-rules-header" id="r79c1-rules-header" href="#r79c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r79c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r79c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-97"></div> <span class="help-tooltip" id="focus-panel-help-97">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r79c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r79c1-gene-table-details" data-name="r79c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r79c1-MIBiG-cluster-compare" data-name="r79c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r79c1-clusterblast" data-name="r79c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r79c1-knownclusterblast" data-name="r79c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r79c1-subclusterblast" data-name="r79c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r79c1-pfam-details" data-name="r79c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r79c1-tigrfam-details" data-name="r79c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r79c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-33"></div> <span class="help-tooltip" id="gt-help-tooltip-help-33">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r79c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r79c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r79c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r79c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r79c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-29"></div> <span class="help-tooltip" id="MIBiG-help-29">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r79c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r79c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r79c1"></div>
<div id="comparison-MIBiG-r79c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647: 251-2872">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647</a></td>





<td class="cc-heat-cell" title="0.85 (id:0.62, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.85), rgb(0, 0, 0, 0.85) 85%, white 85%)"></td>


<td>0.62</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Rhodobacter sphaeroides</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656: 169-3770">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.47, order:0.69, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Terpene</td>

<td>zeaxanthin</td>


<td>Xanthobacter autotrophicus Py2</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648: 1057-11443">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.44, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Myxococcus xanthus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002216: 305-18235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002216/index.html#r1c1" target="_new">BGC0002216</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Terpene</td>

<td>HEx-tc1 terpene</td>


<td>Trichoderma virens Gv29-8</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645: 0-7475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Halobacillus halophilus DSM 2266</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664: 0-8391">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.44, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Terpene</td>

<td>isorenieratene</td>


<td>Streptomyces griseus subsp. griseus NBRC 13350</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646: 60-6380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.47, order:0.48, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Terpene</td>

<td>-carotein</td>


<td>uncultured bacterium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630: 389-6051">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.47, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Terpene</td>

<td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>


<td>Paracoccus haeundaensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640: 2399-8466">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.46, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.19</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Enterobacteriaceae bacterium DC404</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000641: 1208-7454">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000641/index.html#r1c1" target="_new">BGC0000641</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.46, order:0.48, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Enterobacteriaceae bacterium DC416</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r79c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647: 251-2872">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647</a></td>


<td class="cc-heat-cell" title="0.85 (id:0.62, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.85), rgb(0, 0, 0, 0.85) 85%, white 85%)"></td>
<td>0.85</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Rhodobacter sphaeroides</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656: 169-3770">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.47, order:0.69, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Terpene</td>
<td>zeaxanthin</td>
<td>Xanthobacter autotrophicus Py2</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648: 1057-11443">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.44, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Myxococcus xanthus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002216: 305-18235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002216/index.html#r1c1" target="_new">BGC0002216</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Terpene</td>
<td>HEx-tc1 terpene</td>
<td>Trichoderma virens Gv29-8</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645: 0-7475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Halobacillus halophilus DSM 2266</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664: 0-8391">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.44, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Terpene</td>
<td>isorenieratene</td>
<td>Streptomyces griseus subsp. griseus NBRC 13350</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646: 60-6380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.47, order:0.48, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Terpene</td>
<td>-carotein</td>
<td>uncultured bacterium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630: 389-6051">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.47, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Terpene</td>
<td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>
<td>Paracoccus haeundaensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640: 2399-8466">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.46, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Enterobacteriaceae bacterium DC404</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000641: 1208-7454">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000641/index.html#r1c1" target="_new">BGC0000641</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.46, order:0.48, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Enterobacteriaceae bacterium DC416</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r79c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-30"></div> <span class="help-tooltip" id="cb-general-help-30">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r79c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r79c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r79c1_1.svg">
NZ_JAAIKE010000001 (123967-144083): Rhodobacter sp. ETT8 contig1, whole genom... (72% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r79c1_2.svg">
NZ_SNYP01000010 (18408-39440): Rhodobacter veldkampii DSM 11550 Ga0310482 110... (72% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r79c1_3.svg">
NZ_JACIEJ010000015 (21148-42186): Sagittula marina strain DSM 102235 Ga037326... (72% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r79c1_4.svg">
NZ_CP030271 (1983677-2004745): Rhodobacter sphaeroides 2.4.1 chromosome 1, co... (70% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r79c1_5.svg">
NZ_CM001161 (1388242-1409310): Rhodobacter sphaeroides WS8N chromosome I, who... (70% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r79c1_6.svg">
NC_011963 (1615872-1636940): Rhodobacter sphaeroides KD131 chromosome 1, comp... (70% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r79c1_7.svg">
NZ_FZOV01000005 (171484-192552): Rhodobacter megalophilus strain DSM 18937, w... (70% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r79c1_8.svg">
NC_009049 (2016043-2037111): Rhodobacter sphaeroides ATCC 17029 chromosome 1,... (70% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r79c1_9.svg">
NZ_PZZW01000002 (622436-643504): Rhodobacter johrii strain JA192 Ga0191650 10... (70% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r79c1_10.svg">
NZ_CP023548 (2233263-2254328): Rhodobacter sp. CZR27 chromosome, complete genome (70% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r79c1-download">Download graphic</button>
</div>
<div id="clusterblast-r79c1-svg"></div>

</div>
</div>

<div class="body-details-section r79c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-31"></div> <span class="help-tooltip" id="cb-known-help-31">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r79c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r79c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r79c1_1.svg">
carotenoid
</option>

</select>
<button id="knownclusterblast-r79c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r79c1-svg"></div>

</div>
</div>

<div class="body-details-section r79c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-32"></div> <span class="help-tooltip" id="cb-sub-help-32">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r79c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-34"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-34">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r79c1-pfam-details-svg" data-filename="NZ_PZKG01000079.1_r79c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r79c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r79c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-35"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-35">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r79c1-tigrfam-details-svg" data-filename="NZ_PZKG01000079.1_r79c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r79c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>




<div class="page" id='r89c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_PZKG01000089.1 - Region 1 - hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-98"></div> <span class="help-tooltip" id="region_svg-help-98">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_PZKG01000089.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r89c1-svg" data-filename="NZ_PZKG01000089.1_r89c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 17,237 nt. (total: 17,237 nt)

<a class="cluster-rules-header" id="r89c1-rules-header" href="#r89c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r89c1-rules">hserlactone: Autoind_synth</div>

<div class="region-svg-container">
<div id='r89c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-99"></div> <span class="help-tooltip" id="focus-panel-help-99">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r89c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r89c1-gene-table-details" data-name="r89c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r89c1-MIBiG-cluster-compare" data-name="r89c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r89c1-clusterblast" data-name="r89c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r89c1-knownclusterblast" data-name="r89c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r89c1-subclusterblast" data-name="r89c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r89c1-tfbs-finder" data-name="r89c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r89c1-pfam-details" data-name="r89c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r89c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-42"></div> <span class="help-tooltip" id="gt-help-tooltip-help-42">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r89c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r89c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r89c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r89c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r89c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-36"></div> <span class="help-tooltip" id="MIBiG-help-36">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r89c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r89c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r89c1"></div>
<div id="comparison-MIBiG-r89c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">hserlactone</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001684: 0-4789">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001684/index.html#r1c1" target="_new">BGC0001684</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.49, order:0.57, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Other</td>

<td>N-acyloxyacyl glutamine</td>


<td>Klebsiella variicola</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001579: 0-8838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001579/index.html#r1c1" target="_new">BGC0001579</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>RiPP</td>

<td>duramycin</td>


<td>Streptomyces cinnamoneus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563: 185-5330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>venezuelin</td>


<td>Streptomyces venezuelae ATCC 10712</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001226: 0-6018">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001226/index.html#r1c1" target="_new">BGC0001226</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>streptocollin</td>


<td>Streptomyces collinus Tu 365</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001779: 0-6292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001779/index.html#r1c1" target="_new">BGC0001779</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>siamycin</td>


<td>Streptomyces nodosus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001660: 0-7887">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001660/index.html#r1c1" target="_new">BGC0001660</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>mathermycin</td>


<td>Marinactinospora thermotolerans</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001210: 0-9235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001210/index.html#r1c1" target="_new">BGC0001210</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>pseudomycoicidin</td>


<td>Bacillus pseudomycoides DSM 12442</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000547: 618-22455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000547/index.html#r1c1" target="_new">BGC0000547</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>salivaricin 9</td>


<td>Streptococcus salivarius</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000528: 0-8183">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000528/index.html#r1c1" target="_new">BGC0000528</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>michiganin A</td>


<td>Clavibacter michiganensis subsp. michiganensis NCPPB 382</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000527: 0-12324">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000527/index.html#r1c1" target="_new">BGC0000527</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>mersacidin</td>


<td>Bacillus sp. HIL-Y85/54728</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r89c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001684: 0-4789">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001684/index.html#r1c1" target="_new">BGC0001684</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.49, order:0.57, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other</td>
<td>N-acyloxyacyl glutamine</td>
<td>Klebsiella variicola</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001579: 0-8838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001579/index.html#r1c1" target="_new">BGC0001579</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>RiPP</td>
<td>duramycin</td>
<td>Streptomyces cinnamoneus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563: 185-5330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>RiPP</td>
<td>venezuelin</td>
<td>Streptomyces venezuelae ATCC 10712</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001226: 0-6018">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001226/index.html#r1c1" target="_new">BGC0001226</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>RiPP</td>
<td>streptocollin</td>
<td>Streptomyces collinus Tu 365</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001779: 0-6292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001779/index.html#r1c1" target="_new">BGC0001779</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>RiPP</td>
<td>siamycin</td>
<td>Streptomyces nodosus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001660: 0-7887">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001660/index.html#r1c1" target="_new">BGC0001660</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>RiPP</td>
<td>mathermycin</td>
<td>Marinactinospora thermotolerans</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001210: 0-9235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001210/index.html#r1c1" target="_new">BGC0001210</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>RiPP</td>
<td>pseudomycoicidin</td>
<td>Bacillus pseudomycoides DSM 12442</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000547: 618-22455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000547/index.html#r1c1" target="_new">BGC0000547</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>RiPP</td>
<td>salivaricin 9</td>
<td>Streptococcus salivarius</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000528: 0-8183">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000528/index.html#r1c1" target="_new">BGC0000528</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>RiPP</td>
<td>michiganin A</td>
<td>Clavibacter michiganensis subsp. michiganensis NCPPB 382</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000527: 0-12324">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000527/index.html#r1c1" target="_new">BGC0000527</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>mersacidin</td>
<td>Bacillus sp. HIL-Y85/54728</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r89c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-37"></div> <span class="help-tooltip" id="cb-general-help-37">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r89c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r89c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r89c1_1.svg">
NZ_OAOQ01000001 (323718-364255): Rhodobacter ovatus strain JA234, whole genom... (32% of genes show similarity), hserlactone,terpene
</option>

<option value = "svg/clusterblast_r89c1_2.svg">
NZ_CP023548 (1437657-1457513): Rhodobacter sp. CZR27 chromosome, complete genome (63% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r89c1_3.svg">
NZ_PZZW01000002 (473590-494223): Rhodobacter johrii strain JA192 Ga0191650 10... (50% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r89c1_4.svg">
NZ_CP033446 (1400560-1421193): Rhodobacter sphaeroides strain AB27 chromosome 1 (47% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r89c1_5.svg">
NZ_FZOV01000001 (7995-28628): Rhodobacter megalophilus strain DSM 18937, whol... (50% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r89c1_6.svg">
NZ_CP030271 (1827287-1847920): Rhodobacter sphaeroides 2.4.1 chromosome 1, co... (50% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r89c1_7.svg">
NZ_CP033442 (1410091-1430724): Rhodobacter sphaeroides strain AB25 chromosome 1 (50% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r89c1_8.svg">
NC_011963 (1467095-1487728): Rhodobacter sphaeroides KD131 chromosome 1, comp... (50% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r89c1_9.svg">
NC_009049 (1847444-1868077): Rhodobacter sphaeroides ATCC 17029 chromosome 1,... (50% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r89c1_10.svg">
NZ_CP031750 (1545756-1564250): Rhodobacter sphaeroides strain EBL0706 chromos... (52% of genes show similarity), hserlactone
</option>

</select>
<button id= "clusterblast-r89c1-download">Download graphic</button>
</div>
<div id="clusterblast-r89c1-svg"></div>

</div>
</div>

<div class="body-details-section r89c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-38"></div> <span class="help-tooltip" id="cb-known-help-38">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r89c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-39"></div> <span class="help-tooltip" id="cb-sub-help-39">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r89c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-41"></div> <span class="help-tooltip" id="tfbs-body-help-41">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (2):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
NrdR
<div class="collapser collapser-target-NrdR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Represses ribonucleotide reductase encoding genes<br>
<span>Score</span>: 20.23 out of 27.18
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r89c1-11706-NrdR"></div>

<div class="tfbs-hit-detail">
LexA
<div class="collapser collapser-target-LexA collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Repressor of DNA damage response<br>
<span>Score</span>: 20.12 out of 29.65
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r89c1-17002-LexA"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r89c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-43"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-43">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r89c1-pfam-details-svg" data-filename="NZ_PZKG01000089.1_r89c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r89c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r89c1-tfbs-finder" data-name="r89c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r89c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-40"></div> <span class="help-tooltip" id="tfbs-side-help-40">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrdR" target="_blank">NrdR</a></td>
<td>Represses ribonucleotide reductase encoding genes</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/LexA" target="_blank">LexA</a></td>
<td>Repressor of DNA damage response</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r93c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_PZKG01000093.1 - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-100"></div> <span class="help-tooltip" id="region_svg-help-100">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_PZKG01000093.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r93c1-svg" data-filename="NZ_PZKG01000093.1_r93c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 15,795 nt. (total: 15,795 nt)

<a class="cluster-rules-header" id="r93c1-rules-header" href="#r93c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r93c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r93c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-101"></div> <span class="help-tooltip" id="focus-panel-help-101">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r93c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r93c1-gene-table-details" data-name="r93c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r93c1-nrps_pks" data-name="r93c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r93c1-MIBiG-cluster-compare" data-name="r93c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r93c1-clusterblast" data-name="r93c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r93c1-knownclusterblast" data-name="r93c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r93c1-subclusterblast" data-name="r93c1-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r93c1-nrps-pks-bubbles" data-name="r93c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r93c1-pfam-details" data-name="r93c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r93c1-tigrfam-details" data-name="r93c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r93c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-52"></div> <span class="help-tooltip" id="gt-help-tooltip-help-52">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r93c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r93c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r93c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r93c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r93c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-44"></div> <span class="help-tooltip" id="nrps-domain-help-44">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r93c1-details-svg" data-filename="NZ_PZKG01000093.1_r93c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r93c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r93c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-45"></div> <span class="help-tooltip" id="MIBiG-help-45">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r93c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r93c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r93c1"></div>
<div id="comparison-MIBiG-r93c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>NRP</td>

<td>indigoidine</td>


<td>Streptomyces chromofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002476: 0-10330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002476/index.html#r1c1" target="_new">BGC0002476</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Escherichia coli str. K-12 substr. MG1655</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>ergovaline</td>


<td>Epichloe festucae var. lolii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561: 0-17578">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>curacomycin</td>


<td>Streptomyces curacoi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000876: 71-9235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000876/index.html#r1c1" target="_new">BGC0000876</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.46, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>Other</td>

<td>neopolyoxin C</td>


<td>Streptomyces tendae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 10E9</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 22D11</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r93c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>indigoidine</td>
<td>Streptomyces chromofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002476: 0-10330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002476/index.html#r1c1" target="_new">BGC0002476</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Escherichia coli str. K-12 substr. MG1655</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>ergovaline</td>
<td>Epichloe festucae var. lolii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561: 0-17578">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>curacomycin</td>
<td>Streptomyces curacoi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000876: 71-9235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000876/index.html#r1c1" target="_new">BGC0000876</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.46, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Other</td>
<td>neopolyoxin C</td>
<td>Streptomyces tendae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 10E9</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 22D11</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r93c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-46"></div> <span class="help-tooltip" id="cb-general-help-46">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r93c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r93c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r93c1_1.svg">
NZ_FMZV01000001 (40589-96265): Ruegeria marina strain CGMCC 1.9108, whole gen... (13% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r93c1_2.svg">
NZ_CP054599 (3489938-3545474): Sulfitobacter pseudonitzschiae strain H46 chro... (16% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r93c1_3.svg">
NZ_FNOB01000009 (14393-66889): Acidimangrovimonas indica strain DSM 24802, wh... (11% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r93c1_4.svg">
NZ_CP025583 (1662544-1715115): Paracoccus jeotgali strain CBA4604 chromosome,... (9% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r93c1_5.svg">
NZ_VNKR01000088 (260711-313238): Pseudoruegeria sp. M32A2M Ps Sc 9, whole gen... (11% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r93c1_6.svg">
NC_009654 (1791384-1863995): Marinomonas sp. MWYL1, complete genome (7% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r93c1_7.svg">
NZ_LS992241 (3827968-3908875): Paenibacillus alvei isolate Paenibacillus B-LR... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r93c1_8.svg">
NZ_CP010431 (1902365-1983361): Pandoraea sputorum strain DSM 21091 chromosome... (9% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r93c1_9.svg">
NZ_CP011253 (3450654-3531612): Pandoraea oxalativorans strain DSM 23570 chrom... (9% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r93c1_10.svg">
NC_022657 (5089153-5154637): Actinoplanes friuliensis DSM 7358, complete sequ... (8% of genes show similarity), NRPS,betalactone
</option>

</select>
<button id= "clusterblast-r93c1-download">Download graphic</button>
</div>
<div id="clusterblast-r93c1-svg"></div>

</div>
</div>

<div class="body-details-section r93c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-47"></div> <span class="help-tooltip" id="cb-known-help-47">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r93c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r93c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r93c1_1.svg">
cittilin A/cittilin B
</option>

<option value = "svg/knownclusterblast_r93c1_2.svg">
roseobactin
</option>

<option value = "svg/knownclusterblast_r93c1_3.svg">
acinetobactin
</option>

</select>
<button id="knownclusterblast-r93c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r93c1-svg"></div>

</div>
</div>

<div class="body-details-section r93c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-48"></div> <span class="help-tooltip" id="cb-sub-help-48">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r93c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-51"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-51">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r93c1-domain-bubble-svg" data-filename="NZ_PZKG01000093.1_r93c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r93c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (0 - 15795): single NRPS-like
</option>


</select>

<div class="details-svg" id="r93c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r93c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r93c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-53"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-53">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r93c1-pfam-details-svg" data-filename="NZ_PZKG01000093.1_r93c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r93c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r93c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-54"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-54">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r93c1-tigrfam-details-svg" data-filename="NZ_PZKG01000093.1_r93c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r93c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r93c1-nrps_pks_products" data-name="r93c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r93c1-nrps_pks_monomers" data-name="r93c1-nrps_pks_monomers">
NRPS/PKS monomers
</div>








</div>










<div class="sidepanel-details-section r93c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-49"></div> <span class="help-tooltip" id="nrps-structures-help-49">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 0 - 15795:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r93c1-1" class="smiles-canvas" data-smiles="NCC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Gly)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r93c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NCC(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r93c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS monomer predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-50"></div> <span class="help-tooltip" id="nrps-monomers-help-50">Shows the predicted monomers for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">EOMLKCNM_03272</span></strong>: Gly
<div class="collapser collapser-target-EOMLKCNM_03272 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (131..518)</strong>: Gly
<div class="collapser collapser-target-EOMLKCNM_03272 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly
<div class="collapser collapser-target-EOMLKCNM_03272-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Tiny, hydrophilic, transition to aliphatic (Gly, Ala)</dd>
<dt>Single AA prediction:</dt>
<dd>gly (Gly)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DILQLGLIWK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Gly <span class="serif">DILQLGLIWK</span> (91% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>








</div>

</div>

</div>




<div class="page" id='r151c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_PZKG01000151.1 - Region 1 - NRPS-like,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-102"></div> <span class="help-tooltip" id="region_svg-help-102">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_PZKG01000151.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r151c1-svg" data-filename="NZ_PZKG01000151.1_r151c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 8,432 nt. (total: 8,432 nt)

<a class="cluster-rules-header" id="r151c1-rules-header" href="#r151c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r151c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r151c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-103"></div> <span class="help-tooltip" id="focus-panel-help-103">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r151c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r151c1-gene-table-details" data-name="r151c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r151c1-nrps_pks" data-name="r151c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r151c1-MIBiG-cluster-compare" data-name="r151c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r151c1-clusterblast" data-name="r151c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r151c1-knownclusterblast" data-name="r151c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r151c1-subclusterblast" data-name="r151c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r151c1-tfbs-finder" data-name="r151c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r151c1-nrps-pks-bubbles" data-name="r151c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r151c1-pfam-details" data-name="r151c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r151c1-tigrfam-details" data-name="r151c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r151c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-65"></div> <span class="help-tooltip" id="gt-help-tooltip-help-65">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r151c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r151c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r151c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r151c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r151c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-55"></div> <span class="help-tooltip" id="nrps-domain-help-55">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r151c1-details-svg" data-filename="NZ_PZKG01000151.1_r151c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r151c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r151c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-56"></div> <span class="help-tooltip" id="MIBiG-help-56">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r151c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r151c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r151c1"></div>
<div id="comparison-MIBiG-r151c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.59</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 10E9</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.59</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 22D11</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.58</td>
<td>NRP</td>

<td>indigoidine</td>


<td>Streptomyces chromofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.55</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.54</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002171: 8-7317">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002171/index.html#r1c1" target="_new">BGC0002171</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.48</td>
<td>NRP, Polyketide</td>

<td>pyrophen, campyrone B</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002275: 0-10320">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002275/index.html#r1c1" target="_new">BGC0002275</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.48</td>
<td>NRP, Polyketide</td>

<td>pyrophen, campyrone B</td>


<td>Aspergillus niger CBS 101883</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056: 2-6689">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.48</td>
<td>Polyketide</td>

<td>esperamicin</td>


<td>Actinomadura verrucosospora</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002238: 216-10949">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002238/index.html#r1c1" target="_new">BGC0002238</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.42</td>
<td>Polyketide</td>

<td>2,4&#39;-dihydroxy-3&#39;-methoxypropiophenone</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.42</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r151c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 10E9</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 22D11</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>indigoidine</td>
<td>Streptomyces chromofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002171: 8-7317">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002171/index.html#r1c1" target="_new">BGC0002171</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP, Polyketide</td>
<td>pyrophen, campyrone B</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002275: 0-10320">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002275/index.html#r1c1" target="_new">BGC0002275</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP, Polyketide</td>
<td>pyrophen, campyrone B</td>
<td>Aspergillus niger CBS 101883</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056: 2-6689">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>esperamicin</td>
<td>Actinomadura verrucosospora</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002238: 216-10949">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002238/index.html#r1c1" target="_new">BGC0002238</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>2,4&#39;-dihydroxy-3&#39;-methoxypropiophenone</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r151c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-57"></div> <span class="help-tooltip" id="cb-general-help-57">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r151c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r151c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r151c1_1.svg">
NZ_VNKR01000088 (260711-313238): Pseudoruegeria sp. M32A2M Ps Sc 9, whole gen... (8% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r151c1_2.svg">
NZ_CP025583 (1662544-1715115): Paracoccus jeotgali strain CBA4604 chromosome,... (11% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r151c1_3.svg">
NZ_JACIIL010000026 (12773-65266): Rhizobium leucaenae strain SEMIA 4015 Ga039... (14% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r151c1_4.svg">
NZ_CP024310 (992021-1044775): Sinorhizobium fredii strain NXT3 plasmid pSfreN... (16% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r151c1_5.svg">
NZ_CP023064 (810848-863602): Sinorhizobium sp. CCBAU 05631 plasmid pSS05631b,... (16% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r151c1_6.svg">
NZ_JACIIG010000002 (396970-449463): Rhizobium leucaenae strain SEMIA 492 Ga03... (14% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r151c1_7.svg">
NZ_FMZV01000001 (40589-96265): Ruegeria marina strain CGMCC 1.9108, whole gen... (9% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r151c1_8.svg">
NZ_CP054599 (3489938-3545474): Sulfitobacter pseudonitzschiae strain H46 chro... (11% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r151c1_9.svg">
NZ_UBRI01000008 (236851-289326): Rhizobiales bacterium isolate AFS027653, who... (9% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r151c1_10.svg">
NZ_UBKY01000031 (90466-142959): Rhizobiales bacterium isolate AFS004248, whol... (9% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r151c1-download">Download graphic</button>
</div>
<div id="clusterblast-r151c1-svg"></div>

</div>
</div>

<div class="body-details-section r151c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-58"></div> <span class="help-tooltip" id="cb-known-help-58">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r151c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-59"></div> <span class="help-tooltip" id="cb-sub-help-59">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r151c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-63"></div> <span class="help-tooltip" id="tfbs-body-help-63">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 18.00 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r151c1-4539-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r151c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-64"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-64">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r151c1-domain-bubble-svg" data-filename="NZ_PZKG01000151.1_r151c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r151c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (0 - 8432): chemical hybrid NRPS-like-T1PKS
</option>


</select>

<div class="details-svg" id="r151c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r151c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r151c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-66"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-66">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r151c1-pfam-details-svg" data-filename="NZ_PZKG01000151.1_r151c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r151c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r151c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-67"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-67">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r151c1-tigrfam-details-svg" data-filename="NZ_PZKG01000151.1_r151c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r151c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r151c1-nrps_pks_products" data-name="r151c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r151c1-nrps_pks_monomers" data-name="r151c1-nrps_pks_monomers">
NRPS/PKS monomers
</div>



<div class="sidepanel-details-header r151c1-tfbs-finder" data-name="r151c1-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r151c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-60"></div> <span class="help-tooltip" id="nrps-structures-help-60">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 0 - 8432:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r151c1-1" class="smiles-canvas" data-smiles="NC(C(C)CC)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Ile)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ile" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ile*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r151c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(C(C)CC)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r151c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS monomer predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-61"></div> <span class="help-tooltip" id="nrps-monomers-help-61">Shows the predicted monomers for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">EOMLKCNM_03922</span></strong>: Ile
<div class="collapser collapser-target-EOMLKCNM_03922 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ile" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ile*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (25..422)</strong>: Ile
<div class="collapser collapser-target-EOMLKCNM_03922 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ile
<div class="collapser collapser-target-EOMLKCNM_03922-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aromatic side chain (Phe, Trp, Ph-Gly, Tyr, bOH-Tyr)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>ile (Ile)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DGLVVGAVIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Phe <span class="serif">DALVMGAVIK</span> (50% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r151c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-62"></div> <span class="help-tooltip" id="tfbs-side-help-62">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>




<div class="page" id='r187c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_PZKG01000187.1 - Region 1 - redox-cofactor <div class="help-container"> <div class="help-icon" data-id="region_svg-help-104"></div> <span class="help-tooltip" id="region_svg-help-104">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_PZKG01000187.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r187c1-svg" data-filename="NZ_PZKG01000187.1_r187c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 6,255 nt. (total: 6,255 nt)

<a class="cluster-rules-header" id="r187c1-rules-header" href="#r187c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r187c1-rules">redox-cofactor: (cds(PF04055 and (TIGR04085 or SPASM)) and (TIGR02111 or TIGR03966) and minimum(1, [Bottromycin_Methyltransferase_RRE, Cyanobactin_D_RRE, Cytolysin_C_RRE, Goadsporin_CD_RRE, Heterocycloanthracin_C_RRE, Lanthipeptide_LanB_RRE, Lasso_Fused_RRE, Mycofactocin_RRE, NHLP_CD_RRE, Other_Sactipeptide_rSAM_RRE, Pantocin_Microcin_RRE, Plantazolicin_C_RRE, PlpY_RRE, PqqD_RRE, Proteusin_Epimerase_RRE, Proteusin_Methyltransferase_RRE, Quinohemoprotein_rSAM_RRE, Ranthipeptide_rSAM_RRE, Stand_Alone_Lasso_RRE, Streptide_RRE, Subtilosin_rSAM_RRE, Thiaglutamate_B_RRE, Thiaglutamate_G_RRE, Thiaglutamate_I_RRE, Thiopeptide_F_RRE, Thuricin_rSAM_RRE, Thurincin_rSAM_RRE, Trifolitoxin_RRE]))</div>

<div class="region-svg-container">
<div id='r187c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-105"></div> <span class="help-tooltip" id="focus-panel-help-105">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r187c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r187c1-gene-table-details" data-name="r187c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r187c1-MIBiG-cluster-compare" data-name="r187c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r187c1-clusterblast" data-name="r187c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r187c1-knownclusterblast" data-name="r187c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r187c1-subclusterblast" data-name="r187c1-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r187c1-pfam-details" data-name="r187c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r187c1-tigrfam-details" data-name="r187c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r187c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-73"></div> <span class="help-tooltip" id="gt-help-tooltip-help-73">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r187c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r187c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r187c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r187c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r187c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-68"></div> <span class="help-tooltip" id="MIBiG-help-68">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r187c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r187c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r187c1"></div>
<div id="comparison-MIBiG-r187c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">redox-cofactor</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002436: 0-19978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002436/index.html#r1c1" target="_new">BGC0002436</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Other</td>

<td>coformycin</td>


<td>Streptomyces citricolor</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001100: 0-30960">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001100/index.html#r1c1" target="_new">BGC0001100</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.47, order:0.46, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>NRP, Polyketide</td>

<td>lankacidin C</td>


<td>Streptomyces rochei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000934: 146-25006">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000934/index.html#r1c1" target="_new">BGC0000934</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.42, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Other (Phenazine)</td>

<td>5-acetyl-5,10-dihydrophenazine-1-carboxylic acid, 5-(2-hydroxyacetyl)-5,10-dihydrophenazine-1-carboxylic acid, endophenazine A1, endophenazine F, endophenazine G</td>


<td>Kitasatospora sp. HKI 714</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000905: 1-16359">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000905/index.html#r1c1" target="_new">BGC0000905</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.41, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Other</td>

<td>heme D1</td>


<td>Heliobacillus mobilis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000906: 424-16949">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000906/index.html#r1c1" target="_new">BGC0000906</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.41, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Other</td>

<td>heme D1</td>


<td>Heliophilum fasciatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000148: 1134-79984">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000148/index.html#r1c1" target="_new">BGC0000148</a></td>





<td class="cc-heat-cell" title="0.19 (id:0.45, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>


<td>0.04</td>
<td>Polyketide</td>

<td>A83543A</td>


<td>Saccharopolyspora spinosa</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521: 1747-26349">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521</a></td>





<td class="cc-heat-cell" title="0.19 (id:0.41, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>


<td>0.04</td>
<td>NRP</td>

<td>auriculamide</td>


<td>Herpetosiphon aurantiacus DSM 785</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002314: 0-67305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002314/index.html#r1c1" target="_new">BGC0002314</a></td>





<td class="cc-heat-cell" title="0.17 (id:0.41, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>


<td>0.04</td>
<td>NRP</td>

<td>corbomycin</td>


<td>Streptomyces sp. WAC 01529</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001236: 0-29057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001236/index.html#r1c1" target="_new">BGC0001236</a></td>





<td class="cc-heat-cell" title="0.17 (id:0.41, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>


<td>0.03</td>
<td>Polyketide</td>

<td>thiotetroamide</td>


<td>Streptomyces afghaniensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002496: 894-14801">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002496/index.html#r1c1" target="_new">BGC0002496</a></td>





<td class="cc-heat-cell" title="0.17 (id:0.44, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>


<td>0.03</td>
<td>NRP</td>

<td>serratiochelin A, serratiochelin C, serratiochelin B</td>


<td>Serratia plymuthica</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r187c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002436: 0-19978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002436/index.html#r1c1" target="_new">BGC0002436</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Other</td>
<td>coformycin</td>
<td>Streptomyces citricolor</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001100: 0-30960">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001100/index.html#r1c1" target="_new">BGC0001100</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.47, order:0.46, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>NRP, Polyketide</td>
<td>lankacidin C</td>
<td>Streptomyces rochei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000934: 146-25006">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000934/index.html#r1c1" target="_new">BGC0000934</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.42, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Other (Phenazine)</td>
<td>5-acetyl-5,10-dihydrophenazine-1-carboxylic acid, 5-(2-hydroxyacetyl)-5,10-dihydrophenazine-1-carboxylic acid, endophenazine A1, endophenazine F, endophenazine G</td>
<td>Kitasatospora sp. HKI 714</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000905: 1-16359">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000905/index.html#r1c1" target="_new">BGC0000905</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.41, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Other</td>
<td>heme D1</td>
<td>Heliobacillus mobilis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000906: 424-16949">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000906/index.html#r1c1" target="_new">BGC0000906</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.41, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Other</td>
<td>heme D1</td>
<td>Heliophilum fasciatum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000148: 1134-79984">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000148/index.html#r1c1" target="_new">BGC0000148</a></td>


<td class="cc-heat-cell" title="0.19 (id:0.45, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
<td>0.19</td>
<td>Polyketide</td>
<td>A83543A</td>
<td>Saccharopolyspora spinosa</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521: 1747-26349">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521</a></td>


<td class="cc-heat-cell" title="0.19 (id:0.41, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
<td>0.19</td>
<td>NRP</td>
<td>auriculamide</td>
<td>Herpetosiphon aurantiacus DSM 785</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002314: 0-67305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002314/index.html#r1c1" target="_new">BGC0002314</a></td>


<td class="cc-heat-cell" title="0.17 (id:0.41, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>
<td>0.17</td>
<td>NRP</td>
<td>corbomycin</td>
<td>Streptomyces sp. WAC 01529</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001236: 0-29057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001236/index.html#r1c1" target="_new">BGC0001236</a></td>


<td class="cc-heat-cell" title="0.17 (id:0.41, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>
<td>0.17</td>
<td>Polyketide</td>
<td>thiotetroamide</td>
<td>Streptomyces afghaniensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002496: 894-14801">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002496/index.html#r1c1" target="_new">BGC0002496</a></td>


<td class="cc-heat-cell" title="0.17 (id:0.44, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>
<td>0.17</td>
<td>NRP</td>
<td>serratiochelin A, serratiochelin C, serratiochelin B</td>
<td>Serratia plymuthica</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r187c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-69"></div> <span class="help-tooltip" id="cb-general-help-69">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r187c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r187c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r187c1_1.svg">
NZ_CP023454 (2004845-2047476): Rhodobacteraceae bacterium QY30 chromosome, co... (12% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r187c1_2.svg">
NZ_VIVN01000022 (51990-81104): Neobacillus bataviensis strain 2482 Ga0310713 ... (12% of genes show similarity), LAP,bacteriocin,thiopeptide
</option>

<option value = "svg/clusterblast_r187c1_3.svg">
NC_014664 (759687-776610): Rhodomicrobium vannielii ATCC 17100, complete sequ... (14% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r187c1_4.svg">
NZ_LR723668 (88411-110073): Rhizobium sp. Khangiran2 chromosome 1 (13% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r187c1_5.svg">
NZ_CP007155 (3951723-4011898): Kutzneria albida DSM 43870 chromosome, complet... (6% of genes show similarity), LAP,T1PKS,lassopeptide,thiopeptide
</option>

<option value = "svg/clusterblast_r187c1_6.svg">
NZ_CP034086 (1804866-1813270): Methylocystis rosea strain GW6 chromosome, com... (30% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r187c1_7.svg">
NZ_JABVWW010000014 (44099-54942): Methylocystis sp. FS NODE 14 length 90404 c... (23% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r187c1_8.svg">
NZ_CP044328 (3136004-3146847): Methylocystis rosea strain BRCS1 chromosome, c... (30% of genes show similarity), bacteriocin
</option>

<option value = "svg/clusterblast_r187c1_9.svg">
NZ_FOAI01000014 (37385-58423): Roseivivax marinus strain DSM 27511, whole gen... (13% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r187c1_10.svg">
NZ_VIWP01000001 (228115-262562): Neorhizobium alkalisoli strain 1225 Ga036542... (10% of genes show similarity), TfuA-related,thiopeptide
</option>

</select>
<button id= "clusterblast-r187c1-download">Download graphic</button>
</div>
<div id="clusterblast-r187c1-svg"></div>

</div>
</div>

<div class="body-details-section r187c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-70"></div> <span class="help-tooltip" id="cb-known-help-70">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r187c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-71"></div> <span class="help-tooltip" id="cb-sub-help-71">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r187c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-74"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-74">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r187c1-pfam-details-svg" data-filename="NZ_PZKG01000187.1_r187c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r187c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r187c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-75"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-75">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r187c1-tigrfam-details-svg" data-filename="NZ_PZKG01000187.1_r187c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r187c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r187c1-RREfinder" data-name="r187c1-RREfinder">
RREFinder
</div>






</div>








<div class="sidepanel-details-section r187c1-RREfinder">
<div class="more-details">
<div class="heading">
<span>RRE predictions</span>
<div class="help-container"> <div class="help-icon" data-id="rre-side-help-72"></div> <span class="help-tooltip" id="rre-side-help-72">RREfinder results sidepanel.</span></div>
</div>
<dl class = 'prediction-text'>

<span><strong>redox-cofactor protocluster</strong> (0...6255)</span>
<dt></dt>




<strong>EOMLKCNM_04158</strong> <div class="collapser collapser-target-EOMLKCNM_04158 collapser-level-cds"> </div><div class="collapser-content">

<dd>RRE type: PqqD_RRE</dd>
<dd>Location: 11 - 97</dd>
<dd>Score: 95.6</dd>

</div><br>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>




<div class="page" id='r251c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_PZKG01000251.1 - Region 1 - hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-106"></div> <span class="help-tooltip" id="region_svg-help-106">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_PZKG01000251.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r251c1-svg" data-filename="NZ_PZKG01000251.1_r251c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 3,036 nt. (total: 3,036 nt)

<a class="cluster-rules-header" id="r251c1-rules-header" href="#r251c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r251c1-rules">hserlactone: Autoind_synth</div>

<div class="region-svg-container">
<div id='r251c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-107"></div> <span class="help-tooltip" id="focus-panel-help-107">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r251c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r251c1-gene-table-details" data-name="r251c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r251c1-MIBiG-cluster-compare" data-name="r251c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r251c1-clusterblast" data-name="r251c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r251c1-knownclusterblast" data-name="r251c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r251c1-subclusterblast" data-name="r251c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r251c1-tfbs-finder" data-name="r251c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r251c1-pfam-details" data-name="r251c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r251c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-82"></div> <span class="help-tooltip" id="gt-help-tooltip-help-82">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r251c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r251c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r251c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r251c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r251c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-76"></div> <span class="help-tooltip" id="MIBiG-help-76">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<div><p>No matches found.</p></div>

</div>
</div>



<div class="body-details-section r251c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-77"></div> <span class="help-tooltip" id="cb-general-help-77">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r251c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r251c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r251c1_1.svg">
NZ_LR701542 (238215-258881): Roseovarius sp. EC-HK134 isolate RV134, whole ge... (9% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r251c1_2.svg">
NZ_CH902585 (37070-57736): Roseovarius sp. 217 1099463000295, whole genome sh... (9% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r251c1_3.svg">
NZ_KE557320 (345870-366548): Rubellimicrobium thermophilum DSM 16684 R1 scaff... (9% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r251c1_4.svg">
NZ_RWGU01000038 (6838-27429): Rhodovulum robiginosum strain DSM 12329 NODE 14... (8% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r251c1_5.svg">
NZ_VKKX01000011 (366573-387161): Gemmobacter caeruleus strain N8 NODE 1, whol... (7% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r251c1_6.svg">
NZ_FNAT01000014 (24516-45107): Limimaricola pyoseonensis strain DSM 21424, wh... (7% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r251c1_7.svg">
NZ_VYQE01000001 (1019078-1039669): Histidinibacterium sp. 176SS1-4 contig1, w... (8% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r251c1_8.svg">
NZ_CP019437 (1554382-1575048): Thioclava nitratireducens strain 25B10 4 chrom... (11% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r251c1_9.svg">
NZ_SLXP01000001 (184445-205075): Rhodovulum marinum strain DSM 18063 Ga031048... (9% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r251c1_10.svg">
NZ_CP032125 (79679-99251): Rhodobacteraceae bacterium BAR1 chromosome, comple... (11% of genes show similarity), hserlactone
</option>

</select>
<button id= "clusterblast-r251c1-download">Download graphic</button>
</div>
<div id="clusterblast-r251c1-svg"></div>

</div>
</div>

<div class="body-details-section r251c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-78"></div> <span class="help-tooltip" id="cb-known-help-78">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r251c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-79"></div> <span class="help-tooltip" id="cb-sub-help-79">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r251c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-81"></div> <span class="help-tooltip" id="tfbs-body-help-81">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 20.16 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r251c1-498-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r251c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-83"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-83">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r251c1-pfam-details-svg" data-filename="NZ_PZKG01000251.1_r251c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r251c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r251c1-tfbs-finder" data-name="r251c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r251c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-80"></div> <span class="help-tooltip" id="tfbs-side-help-80">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r318c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_PZKG01000318.1 - Region 1 - hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-108"></div> <span class="help-tooltip" id="region_svg-help-108">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_PZKG01000318.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r318c1-svg" data-filename="NZ_PZKG01000318.1_r318c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 1,602 nt. (total: 1,602 nt)

<a class="cluster-rules-header" id="r318c1-rules-header" href="#r318c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r318c1-rules">hserlactone: Autoind_synth</div>

<div class="region-svg-container">
<div id='r318c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-109"></div> <span class="help-tooltip" id="focus-panel-help-109">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r318c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r318c1-gene-table-details" data-name="r318c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r318c1-MIBiG-cluster-compare" data-name="r318c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r318c1-clusterblast" data-name="r318c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r318c1-knownclusterblast" data-name="r318c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r318c1-subclusterblast" data-name="r318c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r318c1-pfam-details" data-name="r318c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r318c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-88"></div> <span class="help-tooltip" id="gt-help-tooltip-help-88">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r318c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r318c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r318c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r318c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r318c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-84"></div> <span class="help-tooltip" id="MIBiG-help-84">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<div><p>No matches found.</p></div>

</div>
</div>



<div class="body-details-section r318c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-85"></div> <span class="help-tooltip" id="cb-general-help-85">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<p>No significant ClusterBlast hits found.</p>
</div>

</div>
</div>

<div class="body-details-section r318c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-86"></div> <span class="help-tooltip" id="cb-known-help-86">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r318c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-87"></div> <span class="help-tooltip" id="cb-sub-help-87">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r318c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-89"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-89">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r318c1-pfam-details-svg" data-filename="NZ_PZKG01000318.1_r318c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r318c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>