
<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>GCF_003034985.1.prokka - 10 region(s) - antiSMASH results</title>
  <style>img {width: 20px; height: 20px;}</style>
  <link rel="stylesheet" type="text/css" href="css/bacteria.css">
  <meta property="og:title" content="GCF_003034985.1.prokka - 9 record(s) - 10 region(s)" />
  <meta property="og:description" content="">
  <meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
  <meta property="og:image:alt" content="antiSMASH logo" />
  <meta property="og:image:width" content="600" />
  <meta property="og:image:height" content="600" />
  <meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/GCF_003034985.1.prokka/index.html" />
  <meta property="og:type" content="website" />
  <meta name="twitter:card" content="summary" />
  <meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
  <div id="header">
      <nav>
    <div class="main-link">
     <div>
        <a href="https://antismash.secondarymetabolites.org/">
          <img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
        </a>
       </div>
       <div>
        <a href="https://antismash.secondarymetabolites.org/">
          antiSMASH version 6.0.0
        </a>
      </div>
    </div>
    <div class="custom-description"></div>
    <div class="ancillary-links">
      <div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
        <ul class="dropdown-options">
        
        <li><a href="GCF_003034985.1.prokka.zip">Download all results</a></li>
        
          <li><a href="GCF_003034985.1.prokka.gbk">Download GenBank summary file</a></li>
        
        </ul>
      </div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
      <div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
    </div>
  </nav>
    <div id="buttons">
      <span id="region-type">Select genomic region:</span>
      <div id="regionbuttons">
        <div class="arrow-left" id="prev-region"></div>
        <div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
        <div style="display: flex; flex-wrap: wrap">
        
          
            
            
            <div class="regbutton terpene r2c1">
              <a href="#r2c1">2.1</a>
            </div>
          
            
            
            <div class="regbutton hserlactone r2c2">
              <a href="#r2c2">2.2</a>
            </div>
          
        
          
            
            
            <div class="regbutton RRE-containing r65c1">
              <a href="#r65c1">65.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r79c1">
              <a href="#r79c1">79.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton hserlactone r89c1">
              <a href="#r89c1">89.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS-like r93c1">
              <a href="#r93c1">93.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS-like,T1PKS hybrid r151c1">
              <a href="#r151c1">151.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton redox-cofactor r187c1">
              <a href="#r187c1">187.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton hserlactone r251c1">
              <a href="#r251c1">251.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton hserlactone r318c1">
              <a href="#r318c1">318.1</a>
            </div>
          
        
        </div>
        <div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
      </div>
    </div>
  </div>

  <!-- overview page -->
  
  <div class="page" id="overview">
   
   
     <h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>
   
   <div class="overview-layout">
    <div id="record-tables">
     <div id="single-record-tables">
     <div class="record-overview-header">
        <strong>NZ_PZKG01000002</strong> (original name was: NZ_PZKG01000002.1) (Cereibacter changlensis)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-2" data-filename="NZ_PZKG01000002_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-2" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r2c1">
     
     
     <td class="regbutton terpene r2c1">
      <a href="#r2c1">Region&nbsp2.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">22,977</td>
     <td class="digits table-split-left">43,744</td>
     
      <td colspan="3"></td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r2c2">
     
     
     <td class="regbutton hserlactone r2c2">
      <a href="#r2c2">Region&nbsp2.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
     </td>
     <td class="digits">54,155</td>
     <td class="digits table-split-left">74,892</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>NZ_PZKG01000065</strong> (original name was: NZ_PZKG01000065.1) (Cereibacter changlensis)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-65" data-filename="NZ_PZKG01000065_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-65" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r65c1">
     
     
     <td class="regbutton RRE-containing r65c1">
      <a href="#r65c1">Region&nbsp65.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#rre-containing" target="_blank">RRE-containing</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">11,175</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>NZ_PZKG01000079</strong> (original name was: NZ_PZKG01000079.1) (Cereibacter changlensis)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-79" data-filename="NZ_PZKG01000079_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-79" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r79c1">
     
     
     <td class="regbutton terpene r79c1">
      <a href="#r79c1">Region&nbsp79.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">6,624</td>
     <td class="digits table-split-left">19,647</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>NZ_PZKG01000089</strong> (original name was: NZ_PZKG01000089.1) (Cereibacter changlensis)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-89" data-filename="NZ_PZKG01000089_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-89" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r89c1">
     
     
     <td class="regbutton hserlactone r89c1">
      <a href="#r89c1">Region&nbsp89.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">17,237</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>NZ_PZKG01000093</strong> (original name was: NZ_PZKG01000093.1) (Cereibacter changlensis)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-93" data-filename="NZ_PZKG01000093_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-93" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r93c1">
     
     
     <td class="regbutton NRPS-like r93c1">
      <a href="#r93c1">Region&nbsp93.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">15,795</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>NZ_PZKG01000151</strong> (original name was: NZ_PZKG01000151.1) (Cereibacter changlensis)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-151" data-filename="NZ_PZKG01000151_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-151" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r151c1">
     
     
         
     
     <td class="regbutton NRPS-like,T1PKS hybrid r151c1">
      <a href="#r151c1">Region&nbsp151.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">8,432</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>NZ_PZKG01000187</strong> (original name was: NZ_PZKG01000187.1) (Cereibacter changlensis)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-187" data-filename="NZ_PZKG01000187_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-187" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r187c1">
     
     
     <td class="regbutton redox-cofactor r187c1">
      <a href="#r187c1">Region&nbsp187.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#redox-cofactor" target="_blank">redox-cofactor</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">6,255</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>NZ_PZKG01000251</strong> (original name was: NZ_PZKG01000251.1) (Cereibacter changlensis)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-251" data-filename="NZ_PZKG01000251_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-251" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r251c1">
     
     
     <td class="regbutton hserlactone r251c1">
      <a href="#r251c1">Region&nbsp251.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">3,036</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>NZ_PZKG01000318</strong> (original name was: NZ_PZKG01000318.1) (Cereibacter changlensis)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-318" data-filename="NZ_PZKG01000318_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-318" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r318c1">
     
     
     <td class="regbutton hserlactone r318c1">
      <a href="#r318c1">Region&nbsp318.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">1,602</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div>
     </div>
     
     <div id="compact-record-table" style="display: none;">
      <div class="record-table">
       <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 
 
  <tr class="linked-row odd" data-anchor="#r2c1">
     
     
     <td class="regbutton terpene r2c1">
      <a href="#r2c1">Region&nbsp2.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">22,977</td>
     <td class="digits table-split-left">43,744</td>
     
      <td colspan="3"></td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r2c2">
     
     
     <td class="regbutton hserlactone r2c2">
      <a href="#r2c2">Region&nbsp2.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
     </td>
     <td class="digits">54,155</td>
     <td class="digits table-split-left">74,892</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r65c1">
     
     
     <td class="regbutton RRE-containing r65c1">
      <a href="#r65c1">Region&nbsp65.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#rre-containing" target="_blank">RRE-containing</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">11,175</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r79c1">
     
     
     <td class="regbutton terpene r79c1">
      <a href="#r79c1">Region&nbsp79.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">6,624</td>
     <td class="digits table-split-left">19,647</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r89c1">
     
     
     <td class="regbutton hserlactone r89c1">
      <a href="#r89c1">Region&nbsp89.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">17,237</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r93c1">
     
     
     <td class="regbutton NRPS-like r93c1">
      <a href="#r93c1">Region&nbsp93.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">15,795</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r151c1">
     
     
         
     
     <td class="regbutton NRPS-like,T1PKS hybrid r151c1">
      <a href="#r151c1">Region&nbsp151.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">8,432</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r187c1">
     
     
     <td class="regbutton redox-cofactor r187c1">
      <a href="#r187c1">Region&nbsp187.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#redox-cofactor" target="_blank">redox-cofactor</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">6,255</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r251c1">
     
     
     <td class="regbutton hserlactone r251c1">
      <a href="#r251c1">Region&nbsp251.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">3,036</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r318c1">
     
     
     <td class="regbutton hserlactone r318c1">
      <a href="#r318c1">Region&nbsp318.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">1,602</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
 </tbody>
</table>
      </div>
     </div>
     
     
     <dl id="regionless-records">
      <dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>
       
         <dd><strong>NZ_PZKG01000001</strong> (original name was: NZ_PZKG01000001.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000003</strong> (original name was: NZ_PZKG01000003.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000004</strong> (original name was: NZ_PZKG01000004.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000005</strong> (original name was: NZ_PZKG01000005.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000006</strong> (original name was: NZ_PZKG01000006.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000007</strong> (original name was: NZ_PZKG01000007.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000008</strong> (original name was: NZ_PZKG01000008.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000009</strong> (original name was: NZ_PZKG01000009.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000010</strong> (original name was: NZ_PZKG01000010.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000011</strong> (original name was: NZ_PZKG01000011.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000012</strong> (original name was: NZ_PZKG01000012.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000013</strong> (original name was: NZ_PZKG01000013.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000014</strong> (original name was: NZ_PZKG01000014.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000015</strong> (original name was: NZ_PZKG01000015.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000016</strong> (original name was: NZ_PZKG01000016.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000017</strong> (original name was: NZ_PZKG01000017.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000018</strong> (original name was: NZ_PZKG01000018.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000019</strong> (original name was: NZ_PZKG01000019.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000020</strong> (original name was: NZ_PZKG01000020.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000021</strong> (original name was: NZ_PZKG01000021.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000022</strong> (original name was: NZ_PZKG01000022.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000023</strong> (original name was: NZ_PZKG01000023.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000024</strong> (original name was: NZ_PZKG01000024.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000025</strong> (original name was: NZ_PZKG01000025.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000026</strong> (original name was: NZ_PZKG01000026.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000027</strong> (original name was: NZ_PZKG01000027.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000028</strong> (original name was: NZ_PZKG01000028.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000029</strong> (original name was: NZ_PZKG01000029.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000030</strong> (original name was: NZ_PZKG01000030.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000031</strong> (original name was: NZ_PZKG01000031.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000032</strong> (original name was: NZ_PZKG01000032.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000033</strong> (original name was: NZ_PZKG01000033.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000034</strong> (original name was: NZ_PZKG01000034.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000035</strong> (original name was: NZ_PZKG01000035.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000036</strong> (original name was: NZ_PZKG01000036.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000037</strong> (original name was: NZ_PZKG01000037.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000038</strong> (original name was: NZ_PZKG01000038.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000039</strong> (original name was: NZ_PZKG01000039.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000040</strong> (original name was: NZ_PZKG01000040.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000041</strong> (original name was: NZ_PZKG01000041.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000042</strong> (original name was: NZ_PZKG01000042.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000043</strong> (original name was: NZ_PZKG01000043.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000044</strong> (original name was: NZ_PZKG01000044.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000045</strong> (original name was: NZ_PZKG01000045.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000046</strong> (original name was: NZ_PZKG01000046.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000047</strong> (original name was: NZ_PZKG01000047.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000048</strong> (original name was: NZ_PZKG01000048.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000049</strong> (original name was: NZ_PZKG01000049.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000050</strong> (original name was: NZ_PZKG01000050.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000051</strong> (original name was: NZ_PZKG01000051.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000052</strong> (original name was: NZ_PZKG01000052.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000053</strong> (original name was: NZ_PZKG01000053.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000054</strong> (original name was: NZ_PZKG01000054.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000055</strong> (original name was: NZ_PZKG01000055.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000056</strong> (original name was: NZ_PZKG01000056.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000057</strong> (original name was: NZ_PZKG01000057.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000058</strong> (original name was: NZ_PZKG01000058.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000059</strong> (original name was: NZ_PZKG01000059.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000060</strong> (original name was: NZ_PZKG01000060.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000061</strong> (original name was: NZ_PZKG01000061.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000062</strong> (original name was: NZ_PZKG01000062.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000063</strong> (original name was: NZ_PZKG01000063.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000064</strong> (original name was: NZ_PZKG01000064.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000066</strong> (original name was: NZ_PZKG01000066.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000067</strong> (original name was: NZ_PZKG01000067.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000068</strong> (original name was: NZ_PZKG01000068.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000069</strong> (original name was: NZ_PZKG01000069.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000070</strong> (original name was: NZ_PZKG01000070.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000071</strong> (original name was: NZ_PZKG01000071.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000072</strong> (original name was: NZ_PZKG01000072.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000073</strong> (original name was: NZ_PZKG01000073.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000074</strong> (original name was: NZ_PZKG01000074.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000075</strong> (original name was: NZ_PZKG01000075.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000076</strong> (original name was: NZ_PZKG01000076.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000077</strong> (original name was: NZ_PZKG01000077.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000078</strong> (original name was: NZ_PZKG01000078.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000080</strong> (original name was: NZ_PZKG01000080.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000081</strong> (original name was: NZ_PZKG01000081.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000082</strong> (original name was: NZ_PZKG01000082.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000083</strong> (original name was: NZ_PZKG01000083.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000084</strong> (original name was: NZ_PZKG01000084.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000085</strong> (original name was: NZ_PZKG01000085.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000086</strong> (original name was: NZ_PZKG01000086.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000087</strong> (original name was: NZ_PZKG01000087.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000088</strong> (original name was: NZ_PZKG01000088.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000090</strong> (original name was: NZ_PZKG01000090.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000091</strong> (original name was: NZ_PZKG01000091.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000092</strong> (original name was: NZ_PZKG01000092.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000094</strong> (original name was: NZ_PZKG01000094.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000095</strong> (original name was: NZ_PZKG01000095.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000096</strong> (original name was: NZ_PZKG01000096.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000097</strong> (original name was: NZ_PZKG01000097.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000098</strong> (original name was: NZ_PZKG01000098.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000099</strong> (original name was: NZ_PZKG01000099.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000100</strong> (original name was: NZ_PZKG01000100.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000101</strong> (original name was: NZ_PZKG01000101.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000102</strong> (original name was: NZ_PZKG01000102.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000103</strong> (original name was: NZ_PZKG01000103.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000104</strong> (original name was: NZ_PZKG01000104.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000105</strong> (original name was: NZ_PZKG01000105.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000106</strong> (original name was: NZ_PZKG01000106.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000107</strong> (original name was: NZ_PZKG01000107.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000108</strong> (original name was: NZ_PZKG01000108.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000109</strong> (original name was: NZ_PZKG01000109.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000110</strong> (original name was: NZ_PZKG01000110.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000111</strong> (original name was: NZ_PZKG01000111.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000112</strong> (original name was: NZ_PZKG01000112.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000113</strong> (original name was: NZ_PZKG01000113.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000114</strong> (original name was: NZ_PZKG01000114.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000115</strong> (original name was: NZ_PZKG01000115.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000116</strong> (original name was: NZ_PZKG01000116.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000117</strong> (original name was: NZ_PZKG01000117.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000118</strong> (original name was: NZ_PZKG01000118.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000119</strong> (original name was: NZ_PZKG01000119.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000120</strong> (original name was: NZ_PZKG01000120.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000121</strong> (original name was: NZ_PZKG01000121.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000122</strong> (original name was: NZ_PZKG01000122.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000123</strong> (original name was: NZ_PZKG01000123.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000124</strong> (original name was: NZ_PZKG01000124.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000125</strong> (original name was: NZ_PZKG01000125.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000126</strong> (original name was: NZ_PZKG01000126.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000127</strong> (original name was: NZ_PZKG01000127.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000128</strong> (original name was: NZ_PZKG01000128.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000129</strong> (original name was: NZ_PZKG01000129.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000130</strong> (original name was: NZ_PZKG01000130.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000131</strong> (original name was: NZ_PZKG01000131.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000132</strong> (original name was: NZ_PZKG01000132.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000133</strong> (original name was: NZ_PZKG01000133.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000134</strong> (original name was: NZ_PZKG01000134.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000135</strong> (original name was: NZ_PZKG01000135.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000136</strong> (original name was: NZ_PZKG01000136.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000137</strong> (original name was: NZ_PZKG01000137.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000138</strong> (original name was: NZ_PZKG01000138.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000139</strong> (original name was: NZ_PZKG01000139.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000140</strong> (original name was: NZ_PZKG01000140.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000141</strong> (original name was: NZ_PZKG01000141.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000142</strong> (original name was: NZ_PZKG01000142.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000143</strong> (original name was: NZ_PZKG01000143.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000144</strong> (original name was: NZ_PZKG01000144.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000145</strong> (original name was: NZ_PZKG01000145.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000146</strong> (original name was: NZ_PZKG01000146.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000147</strong> (original name was: NZ_PZKG01000147.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000148</strong> (original name was: NZ_PZKG01000148.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000149</strong> (original name was: NZ_PZKG01000149.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000150</strong> (original name was: NZ_PZKG01000150.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000152</strong> (original name was: NZ_PZKG01000152.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000153</strong> (original name was: NZ_PZKG01000153.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000154</strong> (original name was: NZ_PZKG01000154.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000155</strong> (original name was: NZ_PZKG01000155.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000156</strong> (original name was: NZ_PZKG01000156.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000157</strong> (original name was: NZ_PZKG01000157.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000158</strong> (original name was: NZ_PZKG01000158.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000159</strong> (original name was: NZ_PZKG01000159.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000160</strong> (original name was: NZ_PZKG01000160.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000161</strong> (original name was: NZ_PZKG01000161.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000162</strong> (original name was: NZ_PZKG01000162.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000163</strong> (original name was: NZ_PZKG01000163.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000164</strong> (original name was: NZ_PZKG01000164.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000165</strong> (original name was: NZ_PZKG01000165.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000166</strong> (original name was: NZ_PZKG01000166.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000167</strong> (original name was: NZ_PZKG01000167.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000168</strong> (original name was: NZ_PZKG01000168.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000169</strong> (original name was: NZ_PZKG01000169.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000170</strong> (original name was: NZ_PZKG01000170.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000171</strong> (original name was: NZ_PZKG01000171.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000172</strong> (original name was: NZ_PZKG01000172.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000173</strong> (original name was: NZ_PZKG01000173.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000174</strong> (original name was: NZ_PZKG01000174.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000175</strong> (original name was: NZ_PZKG01000175.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000176</strong> (original name was: NZ_PZKG01000176.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000177</strong> (original name was: NZ_PZKG01000177.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000178</strong> (original name was: NZ_PZKG01000178.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000179</strong> (original name was: NZ_PZKG01000179.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000180</strong> (original name was: NZ_PZKG01000180.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000181</strong> (original name was: NZ_PZKG01000181.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000182</strong> (original name was: NZ_PZKG01000182.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000183</strong> (original name was: NZ_PZKG01000183.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000184</strong> (original name was: NZ_PZKG01000184.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000185</strong> (original name was: NZ_PZKG01000185.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000186</strong> (original name was: NZ_PZKG01000186.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000188</strong> (original name was: NZ_PZKG01000188.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000189</strong> (original name was: NZ_PZKG01000189.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000190</strong> (original name was: NZ_PZKG01000190.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000191</strong> (original name was: NZ_PZKG01000191.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000192</strong> (original name was: NZ_PZKG01000192.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000193</strong> (original name was: NZ_PZKG01000193.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000194</strong> (original name was: NZ_PZKG01000194.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000195</strong> (original name was: NZ_PZKG01000195.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000196</strong> (original name was: NZ_PZKG01000196.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000197</strong> (original name was: NZ_PZKG01000197.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000198</strong> (original name was: NZ_PZKG01000198.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000199</strong> (original name was: NZ_PZKG01000199.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000200</strong> (original name was: NZ_PZKG01000200.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000201</strong> (original name was: NZ_PZKG01000201.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000202</strong> (original name was: NZ_PZKG01000202.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000203</strong> (original name was: NZ_PZKG01000203.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000204</strong> (original name was: NZ_PZKG01000204.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000205</strong> (original name was: NZ_PZKG01000205.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000206</strong> (original name was: NZ_PZKG01000206.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000207</strong> (original name was: NZ_PZKG01000207.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000208</strong> (original name was: NZ_PZKG01000208.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000209</strong> (original name was: NZ_PZKG01000209.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000210</strong> (original name was: NZ_PZKG01000210.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000211</strong> (original name was: NZ_PZKG01000211.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000212</strong> (original name was: NZ_PZKG01000212.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000213</strong> (original name was: NZ_PZKG01000213.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000214</strong> (original name was: NZ_PZKG01000214.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000215</strong> (original name was: NZ_PZKG01000215.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000216</strong> (original name was: NZ_PZKG01000216.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000217</strong> (original name was: NZ_PZKG01000217.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000218</strong> (original name was: NZ_PZKG01000218.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000219</strong> (original name was: NZ_PZKG01000219.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000220</strong> (original name was: NZ_PZKG01000220.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000221</strong> (original name was: NZ_PZKG01000221.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000222</strong> (original name was: NZ_PZKG01000222.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000223</strong> (original name was: NZ_PZKG01000223.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000224</strong> (original name was: NZ_PZKG01000224.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000225</strong> (original name was: NZ_PZKG01000225.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000226</strong> (original name was: NZ_PZKG01000226.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000227</strong> (original name was: NZ_PZKG01000227.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000228</strong> (original name was: NZ_PZKG01000228.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000229</strong> (original name was: NZ_PZKG01000229.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000230</strong> (original name was: NZ_PZKG01000230.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000231</strong> (original name was: NZ_PZKG01000231.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000232</strong> (original name was: NZ_PZKG01000232.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000233</strong> (original name was: NZ_PZKG01000233.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000234</strong> (original name was: NZ_PZKG01000234.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000235</strong> (original name was: NZ_PZKG01000235.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000236</strong> (original name was: NZ_PZKG01000236.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000237</strong> (original name was: NZ_PZKG01000237.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000238</strong> (original name was: NZ_PZKG01000238.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000239</strong> (original name was: NZ_PZKG01000239.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000240</strong> (original name was: NZ_PZKG01000240.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000241</strong> (original name was: NZ_PZKG01000241.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000242</strong> (original name was: NZ_PZKG01000242.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000243</strong> (original name was: NZ_PZKG01000243.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000244</strong> (original name was: NZ_PZKG01000244.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000245</strong> (original name was: NZ_PZKG01000245.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000246</strong> (original name was: NZ_PZKG01000246.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000247</strong> (original name was: NZ_PZKG01000247.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000248</strong> (original name was: NZ_PZKG01000248.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000249</strong> (original name was: NZ_PZKG01000249.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000250</strong> (original name was: NZ_PZKG01000250.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000252</strong> (original name was: NZ_PZKG01000252.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000253</strong> (original name was: NZ_PZKG01000253.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000254</strong> (original name was: NZ_PZKG01000254.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000255</strong> (original name was: NZ_PZKG01000255.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000256</strong> (original name was: NZ_PZKG01000256.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000257</strong> (original name was: NZ_PZKG01000257.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000258</strong> (original name was: NZ_PZKG01000258.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000259</strong> (original name was: NZ_PZKG01000259.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000260</strong> (original name was: NZ_PZKG01000260.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000261</strong> (original name was: NZ_PZKG01000261.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000262</strong> (original name was: NZ_PZKG01000262.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000263</strong> (original name was: NZ_PZKG01000263.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000264</strong> (original name was: NZ_PZKG01000264.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000265</strong> (original name was: NZ_PZKG01000265.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000266</strong> (original name was: NZ_PZKG01000266.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000267</strong> (original name was: NZ_PZKG01000267.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000268</strong> (original name was: NZ_PZKG01000268.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000269</strong> (original name was: NZ_PZKG01000269.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000270</strong> (original name was: NZ_PZKG01000270.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000271</strong> (original name was: NZ_PZKG01000271.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000272</strong> (original name was: NZ_PZKG01000272.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000273</strong> (original name was: NZ_PZKG01000273.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000274</strong> (original name was: NZ_PZKG01000274.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000275</strong> (original name was: NZ_PZKG01000275.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000276</strong> (original name was: NZ_PZKG01000276.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000277</strong> (original name was: NZ_PZKG01000277.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000278</strong> (original name was: NZ_PZKG01000278.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000279</strong> (original name was: NZ_PZKG01000279.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000280</strong> (original name was: NZ_PZKG01000280.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000281</strong> (original name was: NZ_PZKG01000281.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000282</strong> (original name was: NZ_PZKG01000282.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000283</strong> (original name was: NZ_PZKG01000283.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000284</strong> (original name was: NZ_PZKG01000284.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000285</strong> (original name was: NZ_PZKG01000285.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000286</strong> (original name was: NZ_PZKG01000286.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000287</strong> (original name was: NZ_PZKG01000287.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000288</strong> (original name was: NZ_PZKG01000288.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000289</strong> (original name was: NZ_PZKG01000289.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000290</strong> (original name was: NZ_PZKG01000290.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000291</strong> (original name was: NZ_PZKG01000291.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000292</strong> (original name was: NZ_PZKG01000292.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000293</strong> (original name was: NZ_PZKG01000293.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000294</strong> (original name was: NZ_PZKG01000294.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000295</strong> (original name was: NZ_PZKG01000295.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000296</strong> (original name was: NZ_PZKG01000296.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000297</strong> (original name was: NZ_PZKG01000297.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000298</strong> (original name was: NZ_PZKG01000298.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000299</strong> (original name was: NZ_PZKG01000299.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000300</strong> (original name was: NZ_PZKG01000300.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000301</strong> (original name was: NZ_PZKG01000301.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000302</strong> (original name was: NZ_PZKG01000302.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000303</strong> (original name was: NZ_PZKG01000303.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000304</strong> (original name was: NZ_PZKG01000304.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000305</strong> (original name was: NZ_PZKG01000305.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000306</strong> (original name was: NZ_PZKG01000306.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000307</strong> (original name was: NZ_PZKG01000307.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000308</strong> (original name was: NZ_PZKG01000308.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000309</strong> (original name was: NZ_PZKG01000309.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000310</strong> (original name was: NZ_PZKG01000310.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000311</strong> (original name was: NZ_PZKG01000311.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000312</strong> (original name was: NZ_PZKG01000312.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000313</strong> (original name was: NZ_PZKG01000313.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000314</strong> (original name was: NZ_PZKG01000314.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000315</strong> (original name was: NZ_PZKG01000315.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000316</strong> (original name was: NZ_PZKG01000316.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000317</strong> (original name was: NZ_PZKG01000317.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000319</strong> (original name was: NZ_PZKG01000319.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000320</strong> (original name was: NZ_PZKG01000320.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000321</strong> (original name was: NZ_PZKG01000321.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000322</strong> (original name was: NZ_PZKG01000322.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000323</strong> (original name was: NZ_PZKG01000323.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000324</strong> (original name was: NZ_PZKG01000324.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000325</strong> (original name was: NZ_PZKG01000325.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000326</strong> (original name was: NZ_PZKG01000326.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000327</strong> (original name was: NZ_PZKG01000327.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000328</strong> (original name was: NZ_PZKG01000328.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000329</strong> (original name was: NZ_PZKG01000329.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000330</strong> (original name was: NZ_PZKG01000330.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000331</strong> (original name was: NZ_PZKG01000331.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000332</strong> (original name was: NZ_PZKG01000332.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000333</strong> (original name was: NZ_PZKG01000333.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000334</strong> (original name was: NZ_PZKG01000334.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000335</strong> (original name was: NZ_PZKG01000335.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000336</strong> (original name was: NZ_PZKG01000336.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000337</strong> (original name was: NZ_PZKG01000337.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000338</strong> (original name was: NZ_PZKG01000338.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000339</strong> (original name was: NZ_PZKG01000339.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000340</strong> (original name was: NZ_PZKG01000340.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000341</strong> (original name was: NZ_PZKG01000341.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000342</strong> (original name was: NZ_PZKG01000342.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000343</strong> (original name was: NZ_PZKG01000343.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000344</strong> (original name was: NZ_PZKG01000344.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000345</strong> (original name was: NZ_PZKG01000345.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000346</strong> (original name was: NZ_PZKG01000346.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000347</strong> (original name was: NZ_PZKG01000347.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000348</strong> (original name was: NZ_PZKG01000348.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000349</strong> (original name was: NZ_PZKG01000349.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000350</strong> (original name was: NZ_PZKG01000350.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000351</strong> (original name was: NZ_PZKG01000351.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000352</strong> (original name was: NZ_PZKG01000352.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000353</strong> (original name was: NZ_PZKG01000353.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000354</strong> (original name was: NZ_PZKG01000354.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000355</strong> (original name was: NZ_PZKG01000355.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000356</strong> (original name was: NZ_PZKG01000356.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000357</strong> (original name was: NZ_PZKG01000357.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000358</strong> (original name was: NZ_PZKG01000358.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000359</strong> (original name was: NZ_PZKG01000359.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000360</strong> (original name was: NZ_PZKG01000360.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000361</strong> (original name was: NZ_PZKG01000361.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000362</strong> (original name was: NZ_PZKG01000362.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000363</strong> (original name was: NZ_PZKG01000363.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000364</strong> (original name was: NZ_PZKG01000364.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000365</strong> (original name was: NZ_PZKG01000365.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000366</strong> (original name was: NZ_PZKG01000366.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000367</strong> (original name was: NZ_PZKG01000367.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000368</strong> (original name was: NZ_PZKG01000368.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000369</strong> (original name was: NZ_PZKG01000369.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000370</strong> (original name was: NZ_PZKG01000370.1) (Cereibacter changlensis)</dd>
       
         <dd><strong>NZ_PZKG01000371</strong> (original name was: NZ_PZKG01000371.1) (Cereibacter changlensis)</dd>
       
     </dl>
     
    </div>
    
    <div class="overview-switches">
     <div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    
   </div>
  </div>
  






<div class="page" id='r2c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00002_NZ_PZKG.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-7"></div> <span class="help-tooltip" id="region_svg-help-7">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = NZ_PZKG01000002.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r2c1-svg" data-filename="NZ_PZKG01000002_r2c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 22,977 - 43,744 nt. (total: 20,768 nt)
        
        <a class="cluster-rules-header" id="r2c1-rules-header" href="#r2c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r2c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r2c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-8"></div> <span class="help-tooltip" id="focus-panel-help-8">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r2c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
    </div>
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
    </div>
    
  </div>

 </div>

</div>

<div class="page" id='r2c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00002_NZ_PZKG.. - Region 2 - hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-9"></div> <span class="help-tooltip" id="region_svg-help-9">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = NZ_PZKG01000002.region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r2c2-svg" data-filename="NZ_PZKG01000002_r2c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 54,155 - 74,892 nt. (total: 20,738 nt)
        
        <a class="cluster-rules-header" id="r2c2-rules-header" href="#r2c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r2c2-rules">hserlactone: Autoind_synth</div>
      
      <div class="region-svg-container">
       <div id='r2c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-10"></div> <span class="help-tooltip" id="focus-panel-help-10">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r2c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
    </div>
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
    </div>
    
  </div>

 </div>

</div>




<div class="page" id='r65c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00065_NZ_PZKG.. - Region 1 - RRE-containing <div class="help-container"> <div class="help-icon" data-id="region_svg-help-11"></div> <span class="help-tooltip" id="region_svg-help-11">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = NZ_PZKG01000065.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r65c1-svg" data-filename="NZ_PZKG01000065_r65c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 11,175 nt. (total: 11,175 nt)
        
        <a class="cluster-rules-header" id="r65c1-rules-header" href="#r65c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r65c1-rules">RRE-containing: minimum(1, [Bottromycin_Methyltransferase_RRE, Cyanobactin_D_RRE, Cytolysin_C_RRE, Goadsporin_CD_RRE, Heterocycloanthracin_C_RRE, Lanthipeptide_LanB_RRE, Lasso_Fused_RRE, Mycofactocin_RRE, NHLP_CD_RRE, Other_Sactipeptide_rSAM_RRE, Pantocin_Microcin_RRE, Plantazolicin_C_RRE, PlpY_RRE, PqqD_RRE, Proteusin_Epimerase_RRE, Proteusin_Methyltransferase_RRE, Quinohemoprotein_rSAM_RRE, Ranthipeptide_rSAM_RRE, Stand_Alone_Lasso_RRE, Streptide_RRE, Subtilosin_rSAM_RRE, Thiaglutamate_B_RRE, Thiaglutamate_G_RRE, Thiaglutamate_I_RRE, Thiopeptide_F_RRE, Thuricin_rSAM_RRE, Thurincin_rSAM_RRE, Trifolitoxin_RRE])</div>
      
      <div class="region-svg-container">
       <div id='r65c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-12"></div> <span class="help-tooltip" id="focus-panel-help-12">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r65c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
    </div>
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
    </div>
    
  </div>

 </div>

</div>




<div class="page" id='r79c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00079_NZ_PZKG.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-13"></div> <span class="help-tooltip" id="region_svg-help-13">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = NZ_PZKG01000079.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r79c1-svg" data-filename="NZ_PZKG01000079_r79c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 6,624 - 19,647 nt. (total: 13,024 nt)
        
        <a class="cluster-rules-header" id="r79c1-rules-header" href="#r79c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r79c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r79c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-14"></div> <span class="help-tooltip" id="focus-panel-help-14">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r79c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
    </div>
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
    </div>
    
  </div>

 </div>

</div>




<div class="page" id='r89c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00089_NZ_PZKG.. - Region 1 - hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-15"></div> <span class="help-tooltip" id="region_svg-help-15">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = NZ_PZKG01000089.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r89c1-svg" data-filename="NZ_PZKG01000089_r89c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 17,237 nt. (total: 17,237 nt)
        
        <a class="cluster-rules-header" id="r89c1-rules-header" href="#r89c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r89c1-rules">hserlactone: Autoind_synth</div>
      
      <div class="region-svg-container">
       <div id='r89c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-16"></div> <span class="help-tooltip" id="focus-panel-help-16">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r89c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
    </div>
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
    </div>
    
  </div>

 </div>

</div>




<div class="page" id='r93c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00093_NZ_PZKG.. - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-17"></div> <span class="help-tooltip" id="region_svg-help-17">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = NZ_PZKG01000093.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r93c1-svg" data-filename="NZ_PZKG01000093_r93c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 15,795 nt. (total: 15,795 nt)
        
        <a class="cluster-rules-header" id="r93c1-rules-header" href="#r93c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r93c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r93c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-18"></div> <span class="help-tooltip" id="focus-panel-help-18">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r93c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r93c1-nrps_pks" data-name="r93c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
    </div>
    
     
    <div class="body-details-section r93c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-1"></div> <span class="help-tooltip" id="nrps-domain-help-1">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r93c1-details-svg" data-filename="NZ_PZKG01000093_r93c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r93c1-details-svg"></div>
</div>
    </div>
     
    
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r93c1-nrps_pks_products" data-name="r93c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r93c1-nrps_pks" data-name="r93c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r93c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-2"></div> <span class="help-tooltip" id="nrps-structures-help-2">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 15795:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r93c1-1" class="smiles-canvas" data-smiles="NCC(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(gly)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r93c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NCC(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r93c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-3"></div> <span class="help-tooltip" id="nrps-monomers-help-3">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">EOMLKCNM_03272</span></strong>: gly
      <div class="collapser collapser-target-EOMLKCNM_03272 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (131..518)</strong>: gly
          <div class="collapser collapser-target-EOMLKCNM_03272 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly
              <div class="collapser collapser-target-EOMLKCNM_03272-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>gly</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DILQLGLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
  </div>

 </div>

</div>




<div class="page" id='r151c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00151_NZ_PZKG.. - Region 1 - NRPS-like,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-19"></div> <span class="help-tooltip" id="region_svg-help-19">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = NZ_PZKG01000151.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r151c1-svg" data-filename="NZ_PZKG01000151_r151c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 8,432 nt. (total: 8,432 nt)
        
        <a class="cluster-rules-header" id="r151c1-rules-header" href="#r151c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r151c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r151c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-20"></div> <span class="help-tooltip" id="focus-panel-help-20">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r151c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r151c1-nrps_pks" data-name="r151c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
    </div>
    
     
    <div class="body-details-section r151c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-4"></div> <span class="help-tooltip" id="nrps-domain-help-4">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r151c1-details-svg" data-filename="NZ_PZKG01000151_r151c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r151c1-details-svg"></div>
</div>
    </div>
     
    
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r151c1-nrps_pks_products" data-name="r151c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r151c1-nrps_pks" data-name="r151c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r151c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-5"></div> <span class="help-tooltip" id="nrps-structures-help-5">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 8432:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r151c1-1" class="smiles-canvas" data-smiles="NC(C(C)CC)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ile)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ile" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ile*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r151c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(C(C)CC)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r151c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-6"></div> <span class="help-tooltip" id="nrps-monomers-help-6">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">EOMLKCNM_03922</span></strong>: ile
      <div class="collapser collapser-target-EOMLKCNM_03922 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ile" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ile*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (25..422)</strong>: ile
          <div class="collapser collapser-target-EOMLKCNM_03922 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ile
              <div class="collapser collapser-target-EOMLKCNM_03922-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>phe, trp, phg, tyr, bht</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>ile</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DgLVvGAViK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
  </div>

 </div>

</div>




<div class="page" id='r187c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00187_NZ_PZKG.. - Region 1 - redox-cofactor <div class="help-container"> <div class="help-icon" data-id="region_svg-help-21"></div> <span class="help-tooltip" id="region_svg-help-21">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = NZ_PZKG01000187.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r187c1-svg" data-filename="NZ_PZKG01000187_r187c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 6,255 nt. (total: 6,255 nt)
        
        <a class="cluster-rules-header" id="r187c1-rules-header" href="#r187c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r187c1-rules">redox-cofactor: (cds(PF04055 and (TIGR04085 or SPASM)) and (TIGR02111 or TIGR03966) and minimum(1, [Bottromycin_Methyltransferase_RRE, Cyanobactin_D_RRE, Cytolysin_C_RRE, Goadsporin_CD_RRE, Heterocycloanthracin_C_RRE, Lanthipeptide_LanB_RRE, Lasso_Fused_RRE, Mycofactocin_RRE, NHLP_CD_RRE, Other_Sactipeptide_rSAM_RRE, Pantocin_Microcin_RRE, Plantazolicin_C_RRE, PlpY_RRE, PqqD_RRE, Proteusin_Epimerase_RRE, Proteusin_Methyltransferase_RRE, Quinohemoprotein_rSAM_RRE, Ranthipeptide_rSAM_RRE, Stand_Alone_Lasso_RRE, Streptide_RRE, Subtilosin_rSAM_RRE, Thiaglutamate_B_RRE, Thiaglutamate_G_RRE, Thiaglutamate_I_RRE, Thiopeptide_F_RRE, Thuricin_rSAM_RRE, Thurincin_rSAM_RRE, Trifolitoxin_RRE]))</div>
      
      <div class="region-svg-container">
       <div id='r187c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-22"></div> <span class="help-tooltip" id="focus-panel-help-22">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r187c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
    </div>
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
    </div>
    
  </div>

 </div>

</div>




<div class="page" id='r251c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00251_NZ_PZKG.. - Region 1 - hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-23"></div> <span class="help-tooltip" id="region_svg-help-23">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = NZ_PZKG01000251.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r251c1-svg" data-filename="NZ_PZKG01000251_r251c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 3,036 nt. (total: 3,036 nt)
        
        <a class="cluster-rules-header" id="r251c1-rules-header" href="#r251c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r251c1-rules">hserlactone: Autoind_synth</div>
      
      <div class="region-svg-container">
       <div id='r251c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-24"></div> <span class="help-tooltip" id="focus-panel-help-24">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r251c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
    </div>
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
    </div>
    
  </div>

 </div>

</div>




<div class="page" id='r318c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00318_NZ_PZKG.. - Region 1 - hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-25"></div> <span class="help-tooltip" id="region_svg-help-25">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = NZ_PZKG01000318.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r318c1-svg" data-filename="NZ_PZKG01000318_r318c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 1,602 nt. (total: 1,602 nt)
        
        <a class="cluster-rules-header" id="r318c1-rules-header" href="#r318c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r318c1-rules">hserlactone: Autoind_synth</div>
      
      <div class="region-svg-container">
       <div id='r318c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-26"></div> <span class="help-tooltip" id="focus-panel-help-26">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r318c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
    </div>
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
    </div>
    
  </div>

 </div>

</div>


  <footer class="footer">
    <div class="container">
      <div>
        <img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
      </div>
      <div class="cite-me">
        If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
      </div>
      <div>
        <img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
      </div>
    </div>
  </footer>

  <script src="js/jquery.js"></script>
  <script src="js/antismash.js"></script>
  <script src="regions.js"></script>
  <script>
    $(document).ready(function() {
        viewer["start"](all_regions, details_data, resultsData, recordData);
    })
  </script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink">
 <defs>
  <filter id="inset-shadow">
   <feOffset dx="-2" dy="-2"></feOffset>
   <feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
   <feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
   <feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
   <feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
   <feComponentTransfer in="shadow" result="shadow">
    <feFuncA type="linear" slope=".95"></feFuncA>
   </feComponentTransfer>
   <feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
  </filter>
 </defs>
</svg>
</body>
</html>