
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>GCF_900499075.1_Roseibaca_ekhonensis_CECT_7235 - 5 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/bacteria.css">
<meta property="og:title" content="GCF_900499075.1_Roseibaca_ekhonensis_CECT_7235 - 5 record(s) - 5 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-e9487b7e-95fc-4a5b-afa0-a45d087fd15b/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://antismash.secondarymetabolites.org/">
<img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://antismash.secondarymetabolites.org/">
antiSMASH version 7.0.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="GCF_900499075.1_Roseibaca_ekhonensis_CECT_7235.zip">Download all results</a></li>

<li><a href="GCF_900499075.1_Roseibaca_ekhonensis_CECT_7235.gbk">Download GenBank summary file</a></li>

<li><a href="bacteria-e9487b7e-95fc-4a5b-afa0-a45d087fd15b.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton terpene terpene r57c1">
<a href="#r57c1">57.1</a>
</div>



<div class="regbutton RiPP RRE-containing r72c1">
<a href="#r72c1">72.1</a>
</div>



<div class="regbutton PKS T3PKS r90c1">
<a href="#r90c1">90.1</a>
</div>



<div class="regbutton other ectoine r103c1">
<a href="#r103c1">103.1</a>
</div>



<div class="regbutton hybrid r158c1">
<a href="#r158c1">158.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>NZ_UIHC01000012.1</strong> (Roseibaca ekhonensis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-57" data-filename="NZ_UIHC01000012.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-57" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r57c1">
<td class="regbutton terpene terpene r57c1">
<a href="#r57c1">Region&nbsp57.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">31,185</td>
<td class="digits table-split-left">52,219</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000647/1" target="_blank">carotenoid</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_UIHC01000019.1</strong> (Roseibaca ekhonensis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-72" data-filename="NZ_UIHC01000019.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-72" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r72c1">
<td class="regbutton RiPP RRE-containing r72c1">
<a href="#r72c1">Region&nbsp72.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#rre-containing" target="_blank">RRE-containing</a>
</td>
<td class="digits">2,395</td>
<td class="digits table-split-left">22,670</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_UIHC01000006.1</strong> (Roseibaca ekhonensis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-90" data-filename="NZ_UIHC01000006.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-90" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r90c1">
<td class="regbutton PKS T3PKS r90c1">
<a href="#r90c1">Region&nbsp90.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
</td>
<td class="digits">62,330</td>
<td class="digits table-split-left">103,370</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_UIHC01000001.1</strong> (Roseibaca ekhonensis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-103" data-filename="NZ_UIHC01000001.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-103" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r103c1">
<td class="regbutton other ectoine r103c1">
<a href="#r103c1">Region&nbsp103.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
</td>
<td class="digits">174,739</td>
<td class="digits table-split-left">185,125</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000860/1" target="_blank">ectoine</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 80%, #ffffff00 80%)">80%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_UIHC01000007.1</strong> (Roseibaca ekhonensis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-158" data-filename="NZ_UIHC01000007.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-158" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r158c1">
<td class="regbutton hybrid r158c1">
<a href="#r158c1">Region&nbsp158.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">5,329</td>
<td class="digits table-split-left">57,183</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001941/1" target="_blank">fulvuthiacene A/fulvuthiacene B</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 8%, #ffffff00 8%)">8%</td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r57c1">
<td class="regbutton terpene terpene r57c1">
<a href="#r57c1">Region&nbsp57.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">31,185</td>
<td class="digits table-split-left">52,219</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000647/1" target="_blank">carotenoid</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>


<tr class="linked-row even" data-anchor="#r72c1">
<td class="regbutton RiPP RRE-containing r72c1">
<a href="#r72c1">Region&nbsp72.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#rre-containing" target="_blank">RRE-containing</a>
</td>
<td class="digits">2,395</td>
<td class="digits table-split-left">22,670</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r90c1">
<td class="regbutton PKS T3PKS r90c1">
<a href="#r90c1">Region&nbsp90.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
</td>
<td class="digits">62,330</td>
<td class="digits table-split-left">103,370</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r103c1">
<td class="regbutton other ectoine r103c1">
<a href="#r103c1">Region&nbsp103.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
</td>
<td class="digits">174,739</td>
<td class="digits table-split-left">185,125</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000860/1" target="_blank">ectoine</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 80%, #ffffff00 80%)">80%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r158c1">
<td class="regbutton hybrid r158c1">
<a href="#r158c1">Region&nbsp158.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">5,329</td>
<td class="digits table-split-left">57,183</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001941/1" target="_blank">fulvuthiacene A/fulvuthiacene B</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 8%, #ffffff00 8%)">8%</td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>NZ_UIHC01000171.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000170.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000169.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000167.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000166.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000164.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000163.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000162.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000161.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000160.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000158.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000156.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000152.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000151.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000150.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000149.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000146.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000143.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000140.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000139.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000138.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000154.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000137.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000132.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000129.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000122.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000121.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000120.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000117.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000116.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000118.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000112.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000111.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000110.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000109.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000119.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000108.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000105.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000102.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000101.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000172.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000100.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000099.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000128.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000124.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000046.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000054.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000047.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000039.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000004.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000035.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000133.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000113.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000037.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000168.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000034.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000028.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000136.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000027.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000104.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000088.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000033.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000005.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000045.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000081.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000031.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000114.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000066.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000098.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000024.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000115.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000070.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000155.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000029.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000008.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000148.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000013.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000159.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000020.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000090.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000036.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000040.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000022.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000145.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000107.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000106.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000085.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000125.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000094.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000026.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000042.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000003.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000126.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000055.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000077.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000173.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000009.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000071.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000010.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000021.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000025.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000030.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000043.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000011.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000147.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000015.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000014.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000067.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000044.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000016.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000017.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000041.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000002.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000165.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000018.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000144.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000130.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000049.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000134.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000050.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000051.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000153.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000060.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000086.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000052.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000135.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000038.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000063.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000053.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000057.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000142.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000058.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000059.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000062.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000064.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000076.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000056.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000068.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000103.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000069.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000093.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000072.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000141.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000023.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000089.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000061.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000073.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000157.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000074.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000048.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000075.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000097.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000078.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000065.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000079.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000127.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000080.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000082.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000083.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000123.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000095.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000084.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000087.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000131.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000091.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000032.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000092.1</strong> (Roseibaca ekhonensis)</dd>

<dd><strong>NZ_UIHC01000096.1</strong> (Roseibaca ekhonensis)</dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>







<div class="page" id='r57c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_UIHC01000012.1 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-50"></div> <span class="help-tooltip" id="region_svg-help-50">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_UIHC01000012.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r57c1-svg" data-filename="NZ_UIHC01000012.1_r57c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 31,185 - 52,219 nt. (total: 21,035 nt)

<a class="cluster-rules-header" id="r57c1-rules-header" href="#r57c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r57c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r57c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-51"></div> <span class="help-tooltip" id="focus-panel-help-51">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r57c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r57c1-gene-table-details" data-name="r57c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r57c1-MIBiG-cluster-compare" data-name="r57c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r57c1-clusterblast" data-name="r57c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r57c1-knownclusterblast" data-name="r57c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r57c1-subclusterblast" data-name="r57c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r57c1-tfbs-finder" data-name="r57c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r57c1-pfam-details" data-name="r57c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r57c1-tigrfam-details" data-name="r57c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r57c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-7"></div> <span class="help-tooltip" id="gt-help-tooltip-help-7">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r57c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r57c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r57c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r57c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r57c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-1"></div> <span class="help-tooltip" id="MIBiG-help-1">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r57c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r57c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r57c1"></div>
<div id="comparison-MIBiG-r57c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647: 251-2872">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647</a></td>





<td class="cc-heat-cell" title="0.82 (id:0.56, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.82), rgb(0, 0, 0, 0.82) 82%, white 82%)"></td>


<td>0.57</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Rhodobacter sphaeroides</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000637: 540-9054">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000637/index.html#r1c1" target="_new">BGC0000637</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Corynebacterium glutamicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656: 169-3770">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.48, order:0.69, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Terpene</td>

<td>zeaxanthin</td>


<td>Xanthobacter autotrophicus Py2</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645: 0-7475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Halobacillus halophilus DSM 2266</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648: 1057-11443">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.44, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Myxococcus xanthus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227: 0-12144">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Terpene</td>

<td>isorenieratene</td>


<td>Streptomyces collinus Tu 365</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664: 0-8391">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Terpene</td>

<td>isorenieratene</td>


<td>Streptomyces griseus subsp. griseus NBRC 13350</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646: 60-6380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.47, order:0.48, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Terpene</td>

<td>β-carotein</td>


<td>uncultured bacterium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630: 389-6051">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.48, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Terpene</td>

<td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>


<td>Paracoccus haeundaensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640: 2399-8466">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.46, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.19</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Enterobacteriaceae bacterium DC404</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r57c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647: 251-2872">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647</a></td>


<td class="cc-heat-cell" title="0.82 (id:0.56, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.82), rgb(0, 0, 0, 0.82) 82%, white 82%)"></td>
<td>0.82</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Rhodobacter sphaeroides</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000637: 540-9054">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000637/index.html#r1c1" target="_new">BGC0000637</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Corynebacterium glutamicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656: 169-3770">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.48, order:0.69, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Terpene</td>
<td>zeaxanthin</td>
<td>Xanthobacter autotrophicus Py2</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645: 0-7475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Halobacillus halophilus DSM 2266</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648: 1057-11443">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.44, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Myxococcus xanthus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227: 0-12144">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Terpene</td>
<td>isorenieratene</td>
<td>Streptomyces collinus Tu 365</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664: 0-8391">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Terpene</td>
<td>isorenieratene</td>
<td>Streptomyces griseus subsp. griseus NBRC 13350</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646: 60-6380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.47, order:0.48, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Terpene</td>
<td>β-carotein</td>
<td>uncultured bacterium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630: 389-6051">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.48, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Terpene</td>
<td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>
<td>Paracoccus haeundaensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640: 2399-8466">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.46, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Enterobacteriaceae bacterium DC404</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r57c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-2"></div> <span class="help-tooltip" id="cb-general-help-2">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r57c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r57c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r57c1_1.svg">
NZ_MEHT01000018 (61223-82261): Roseinatronobacter thiooxidans strain ALG1 ALG... (78% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r57c1_2.svg">
NZ_CP024899 (619066-640104): Rhodobaca barguzinensis strain alga05 chromosome... (78% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r57c1_3.svg">
NZ_SJZH01000001 (193776-214545): Tabrizicola sp. YIM 78059 Scaffold1, whole g... (78% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r57c1_4.svg">
NZ_JAAIKE010000001 (123967-144083): Rhodobacter sp. ETT8 contig1, whole genom... (61% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r57c1_5.svg">
NZ_SNYP01000010 (18408-39440): Rhodobacter veldkampii DSM 11550 Ga0310482 110... (61% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r57c1_6.svg">
NZ_CP028918 (1124240-1143571): Gemmobacter sp. HYN0069 chromosome, complete g... (58% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r57c1_7.svg">
NZ_CP034328 (3886230-3907250): Tabrizicola piscis strain K13M18 chromosome, c... (58% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r57c1_8.svg">
NZ_FZOV01000005 (171484-192552): Rhodobacter megalophilus strain DSM 18937, w... (58% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r57c1_9.svg">
NC_011963 (1615872-1636940): Rhodobacter sphaeroides KD131 chromosome 1, comp... (58% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r57c1_10.svg">
NZ_CP023548 (2233263-2254328): Rhodobacter sp. CZR27 chromosome, complete genome (58% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r57c1-download">Download graphic</button>
</div>
<div id="clusterblast-r57c1-svg"></div>

</div>
</div>

<div class="body-details-section r57c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-3"></div> <span class="help-tooltip" id="cb-known-help-3">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r57c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r57c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r57c1_1.svg">
carotenoid
</option>

</select>
<button id="knownclusterblast-r57c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r57c1-svg"></div>

</div>
</div>

<div class="body-details-section r57c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-4"></div> <span class="help-tooltip" id="cb-sub-help-4">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r57c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-6"></div> <span class="help-tooltip" id="tfbs-body-help-6">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
CatR
<div class="collapser collapser-target-CatR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: H2O2-responsive repressor<br>
<span>Score</span>: 19.02 out of 25.31
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r57c1-42302-CatR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 18.31 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r57c1-49563-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r57c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-8"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-8">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r57c1-pfam-details-svg" data-filename="NZ_UIHC01000012.1_r57c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r57c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r57c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-9"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-9">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r57c1-tigrfam-details-svg" data-filename="NZ_UIHC01000012.1_r57c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r57c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r57c1-tfbs-finder" data-name="r57c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r57c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-5"></div> <span class="help-tooltip" id="tfbs-side-help-5">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/CatR" target="_blank">CatR</a></td>
<td>H2O2-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r72c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_UIHC01000019.1 - Region 1 - RRE-containing <div class="help-container"> <div class="help-icon" data-id="region_svg-help-52"></div> <span class="help-tooltip" id="region_svg-help-52">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_UIHC01000019.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r72c1-svg" data-filename="NZ_UIHC01000019.1_r72c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,395 - 22,670 nt. (total: 20,276 nt)

<a class="cluster-rules-header" id="r72c1-rules-header" href="#r72c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r72c1-rules">RRE-containing: minimum(1, [Bottromycin_Methyltransferase_RRE, Cyanobactin_D_RRE, Cytolysin_C_RRE, Goadsporin_CD_RRE, Heterocycloanthracin_C_RRE, Lanthipeptide_LanB_RRE, Lasso_Fused_RRE, Mycofactocin_RRE, NHLP_CD_RRE, Other_Sactipeptide_rSAM_RRE, Pantocin_Microcin_RRE, Plantazolicin_C_RRE, PlpY_RRE, PqqD_RRE, Proteusin_Epimerase_RRE, Proteusin_Methyltransferase_RRE, Quinohemoprotein_rSAM_RRE, Ranthipeptide_rSAM_RRE, Stand_Alone_Lasso_RRE, Streptide_RRE, Subtilosin_rSAM_RRE, Thiaglutamate_B_RRE, Thiaglutamate_G_RRE, Thiaglutamate_I_RRE, Thiopeptide_F_RRE, Thuricin_rSAM_RRE, Thurincin_rSAM_RRE, Trifolitoxin_RRE])</div>

<div class="region-svg-container">
<div id='r72c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-53"></div> <span class="help-tooltip" id="focus-panel-help-53">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r72c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r72c1-gene-table-details" data-name="r72c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r72c1-MIBiG-cluster-compare" data-name="r72c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r72c1-clusterblast" data-name="r72c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r72c1-knownclusterblast" data-name="r72c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r72c1-subclusterblast" data-name="r72c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r72c1-tfbs-finder" data-name="r72c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r72c1-pfam-details" data-name="r72c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r72c1-tigrfam-details" data-name="r72c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r72c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-17"></div> <span class="help-tooltip" id="gt-help-tooltip-help-17">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r72c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r72c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r72c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r72c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r72c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-10"></div> <span class="help-tooltip" id="MIBiG-help-10">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r72c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r72c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r72c1"></div>
<div id="comparison-MIBiG-r72c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">RRE-containing</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001356: 0-5701">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001356/index.html#r1c1" target="_new">BGC0001356</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>paeninodin</td>


<td>Paenibacillus dendritiformis C454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001610: 410-17543">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001610/index.html#r1c1" target="_new">BGC0001610</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.43, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>Other</td>

<td>gougerotin</td>


<td>Streptomyces ahygroscopicus subsp. wuzhouensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001654: 0-19803">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001654/index.html#r1c1" target="_new">BGC0001654</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.16)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>Saccharide</td>

<td>lipooligosaccharide</td>


<td>Bacteroides thetaiotaomicron VPI-5482</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001010: 0-51855">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001010/index.html#r1c1" target="_new">BGC0001010</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.44, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.06</td>
<td>NRP, Polyketide</td>

<td>melithiazol A</td>


<td>Melittangium lichenicola</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r72c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001356: 0-5701">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001356/index.html#r1c1" target="_new">BGC0001356</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>RiPP</td>
<td>paeninodin</td>
<td>Paenibacillus dendritiformis C454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001610: 410-17543">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001610/index.html#r1c1" target="_new">BGC0001610</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.43, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>Other</td>
<td>gougerotin</td>
<td>Streptomyces ahygroscopicus subsp. wuzhouensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001654: 0-19803">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001654/index.html#r1c1" target="_new">BGC0001654</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.16)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>Saccharide</td>
<td>lipooligosaccharide</td>
<td>Bacteroides thetaiotaomicron VPI-5482</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001010: 0-51855">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001010/index.html#r1c1" target="_new">BGC0001010</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.44, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>NRP, Polyketide</td>
<td>melithiazol A</td>
<td>Melittangium lichenicola</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r72c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-11"></div> <span class="help-tooltip" id="cb-general-help-11">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r72c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r72c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r72c1_1.svg">
NZ_CP022336 (1708328-1731183): Sphingorhabdus sp. SMR4y chromosome, complete ... (18% of genes show similarity), lassopeptide
</option>

<option value = "svg/clusterblast_r72c1_2.svg">
NZ_SWKR01000002 (2370250-2392595): Sphingomonas sp. L-1-4w-11 NODE 4 length 2... (22% of genes show similarity), lassopeptide
</option>

<option value = "svg/clusterblast_r72c1_3.svg">
NZ_VLKC01000002 (1107790-1133192): Sporomusa sp. KB1 SalpaDRAFT Scaffold1.2, ... (17% of genes show similarity), lassopeptide
</option>

<option value = "svg/clusterblast_r72c1_4.svg">
NZ_LMXG01000001 (785567-809610): Bacillus sp. FJAT-28573 scaffold1, whole gen... (10% of genes show similarity), lassopeptide
</option>

<option value = "svg/clusterblast_r72c1_5.svg">
NZ_KE383843 (584302-608161): Paenibacillus harenae DSM 16969 H581DRAFT scaffo... (9% of genes show similarity), lassopeptide
</option>

<option value = "svg/clusterblast_r72c1_6.svg">
NZ_QKLR01000006 (504576-528798): Cohnella sp. OV312 Ga0215740 106, whole geno... (11% of genes show similarity), lassopeptide
</option>

<option value = "svg/clusterblast_r72c1_7.svg">
NZ_LN881722 (216413-240418): Gorillibacterium timonense strain SN4, whole gen... (11% of genes show similarity), lassopeptide
</option>

<option value = "svg/clusterblast_r72c1_8.svg">
NZ_JACHXK010000001 (666578-693913): Paenibacillus phyllosphaerae strain CECT ... (8% of genes show similarity), lassopeptide
</option>

<option value = "svg/clusterblast_r72c1_9.svg">
NZ_QPJD01000015 (139824-163686): Paenibacillus prosopidis strain CECT 7506 Ga... (9% of genes show similarity), lassopeptide
</option>

<option value = "svg/clusterblast_r72c1_10.svg">
NZ_RYZZ01000007 (161019-185196): Peribacillus cavernae strain L5 18849 L5 1  ... (8% of genes show similarity), lassopeptide
</option>

</select>
<button id= "clusterblast-r72c1-download">Download graphic</button>
</div>
<div id="clusterblast-r72c1-svg"></div>

</div>
</div>

<div class="body-details-section r72c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-12"></div> <span class="help-tooltip" id="cb-known-help-12">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r72c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-13"></div> <span class="help-tooltip" id="cb-sub-help-13">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r72c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-16"></div> <span class="help-tooltip" id="tfbs-body-help-16">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AfsR
<div class="collapser collapser-target-AfsR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Pleiotropic regulatory for antibiotic production<br>
<span>Score</span>: 20.11 out of 23.05
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r72c1-5940-AfsR"></div>

</div>


</div>
</div>



<div class="body-details-section r72c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-18"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-18">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r72c1-pfam-details-svg" data-filename="NZ_UIHC01000019.1_r72c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r72c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r72c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-19"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-19">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r72c1-tigrfam-details-svg" data-filename="NZ_UIHC01000019.1_r72c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r72c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r72c1-RREfinder" data-name="r72c1-RREfinder">
RREFinder
</div>



<div class="sidepanel-details-header r72c1-tfbs-finder" data-name="r72c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r72c1-RREfinder">
<div class="more-details">
<div class="heading">
<span>RRE predictions</span>
<div class="help-container"> <div class="help-icon" data-id="rre-side-help-14"></div> <span class="help-tooltip" id="rre-side-help-14">RREfinder results sidepanel.</span></div>
</div>
<dl class = 'prediction-text'>

<span><strong>RRE-containing protocluster</strong> (2394...22670)</span>
<dt></dt>




<strong>JDHOFBHF_00977</strong> <div class="collapser collapser-target-JDHOFBHF_00977 collapser-level-cds"> </div><div class="collapser-content">

<dd>RRE type: Stand_Alone_Lasso_RRE</dd>
<dd>Location: 8 - 89</dd>
<dd>Score: 84.0</dd>

</div><br>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r72c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-15"></div> <span class="help-tooltip" id="tfbs-side-help-15">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AfsR" target="_blank">AfsR</a></td>
<td>Pleiotropic regulatory for antibiotic production</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r90c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_UIHC01000006.1 - Region 1 - T3PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-54"></div> <span class="help-tooltip" id="region_svg-help-54">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_UIHC01000006.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r90c1-svg" data-filename="NZ_UIHC01000006.1_r90c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 62,330 - 103,370 nt. (total: 41,041 nt)

<a class="cluster-rules-header" id="r90c1-rules-header" href="#r90c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r90c1-rules">T3PKS: (Chal_sti_synt_C or Chal_sti_synt_N)</div>

<div class="region-svg-container">
<div id='r90c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-55"></div> <span class="help-tooltip" id="focus-panel-help-55">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r90c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r90c1-gene-table-details" data-name="r90c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r90c1-nrps_pks" data-name="r90c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r90c1-MIBiG-cluster-compare" data-name="r90c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r90c1-clusterblast" data-name="r90c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r90c1-knownclusterblast" data-name="r90c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r90c1-subclusterblast" data-name="r90c1-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r90c1-pfam-details" data-name="r90c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r90c1-tigrfam-details" data-name="r90c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r90c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-25"></div> <span class="help-tooltip" id="gt-help-tooltip-help-25">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r90c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r90c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r90c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r90c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r90c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-20"></div> <span class="help-tooltip" id="nrps-domain-help-20">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r90c1-details-svg" data-filename="NZ_UIHC01000006.1_r90c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r90c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r90c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-21"></div> <span class="help-tooltip" id="MIBiG-help-21">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r90c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r90c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r90c1"></div>
<div id="comparison-MIBiG-r90c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T3PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000282: 0-2613">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000282/index.html#r1c1" target="_new">BGC0000282</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.48, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.36</td>
<td>Polyketide</td>

<td>alkylresorcinol</td>


<td>Streptomyces griseus subsp. griseus NBRC 13350</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001938: 0-1819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001938/index.html#r1c1" target="_new">BGC0001938</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>entotheonella</td>


<td>Candidatus Entotheonella serta</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001310: 0-2705">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001310/index.html#r1c1" target="_new">BGC0001310</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>naringenin</td>


<td>Streptomyces clavuligerus ATCC 27064</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002332: 137-13618">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002332/index.html#r1c1" target="_new">BGC0002332</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>tetraketide resorcinols</td>


<td>Candidatus Entotheonella sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002656: 1710-32172">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002656/index.html#r1c1" target="_new">BGC0002656</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>oryzanaphthopyran A, oryzanaphthopyran B, oryzanaphthopyran C, oryzanthrone A, oryzanthrone B, chlororyzanthrone A, chlororyzanthrone B</td>


<td>Streptacidiphilus oryzae TH49</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000589: 0-8421">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000589/index.html#r1c1" target="_new">BGC0000589</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>RiPP</td>

<td>microcin M</td>


<td>Escherichia coli Nissle 1917</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000783: 0-13186">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000783/index.html#r1c1" target="_new">BGC0000783</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Saccharide</td>

<td>O-antigen</td>


<td>Xanthomonas oryzae pv. oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831: 1-6152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.48, order:0.48, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>alkylpyrone-407, alkylpyrone-393</td>


<td>Myxococcus xanthus DK 1622</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000586: 254-13092">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000586/index.html#r1c1" target="_new">BGC0000586</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>RiPP</td>

<td>microcin E492</td>


<td>Klebsiella pneumoniae RYC492</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000888: 0-4342">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000888/index.html#r1c1" target="_new">BGC0000888</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Other</td>

<td>bacilysin</td>


<td>Bacillus sp. CS93</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r90c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000282: 0-2613">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000282/index.html#r1c1" target="_new">BGC0000282</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.48, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Polyketide</td>
<td>alkylresorcinol</td>
<td>Streptomyces griseus subsp. griseus NBRC 13350</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001938: 0-1819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001938/index.html#r1c1" target="_new">BGC0001938</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>entotheonella</td>
<td>Candidatus Entotheonella serta</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001310: 0-2705">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001310/index.html#r1c1" target="_new">BGC0001310</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>naringenin</td>
<td>Streptomyces clavuligerus ATCC 27064</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002332: 137-13618">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002332/index.html#r1c1" target="_new">BGC0002332</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>tetraketide resorcinols</td>
<td>Candidatus Entotheonella sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002656: 1710-32172">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002656/index.html#r1c1" target="_new">BGC0002656</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>oryzanaphthopyran A, oryzanaphthopyran B, oryzanaphthopyran C, oryzanthrone A, oryzanthrone B, chlororyzanthrone A, chlororyzanthrone B</td>
<td>Streptacidiphilus oryzae TH49</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000589: 0-8421">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000589/index.html#r1c1" target="_new">BGC0000589</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>RiPP</td>
<td>microcin M</td>
<td>Escherichia coli Nissle 1917</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000783: 0-13186">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000783/index.html#r1c1" target="_new">BGC0000783</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Saccharide</td>
<td>O-antigen</td>
<td>Xanthomonas oryzae pv. oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831: 1-6152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.48, order:0.48, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>alkylpyrone-407, alkylpyrone-393</td>
<td>Myxococcus xanthus DK 1622</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000586: 254-13092">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000586/index.html#r1c1" target="_new">BGC0000586</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>RiPP</td>
<td>microcin E492</td>
<td>Klebsiella pneumoniae RYC492</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000888: 0-4342">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000888/index.html#r1c1" target="_new">BGC0000888</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other</td>
<td>bacilysin</td>
<td>Bacillus sp. CS93</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r90c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-22"></div> <span class="help-tooltip" id="cb-general-help-22">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r90c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r90c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r90c1_1.svg">
NZ_FQUP01000003 (42124-83237): Kaistia soli DSM 19436, whole genome shotgun s... (16% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r90c1_2.svg">
NZ_AUHO01000006 (278063-319143): Kaistia adipata DSM 17808 G539DRAFT scaffold... (16% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r90c1_3.svg">
NC_009445 (145168-186221): Bradyrhizobium sp. ORS 278, complete genome (10% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r90c1_4.svg">
NZ_LT859959 (7419867-7460920): Bradyrhizobium sp. ORS 285 chromosome BRAD285 (10% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r90c1_5.svg">
NC_009485 (187463-228516): Bradyrhizobium sp. BTAi1, complete sequence (10% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r90c1_6.svg">
NZ_LT629693 (3841297-3882350): Bradyrhizobium ottawaense strain GAS524 chromo... (10% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r90c1_7.svg">
NZ_LT670845 (2374737-2415790): Bradyrhizobium lablabi strain MT34 chromosome I (10% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r90c1_8.svg">
NZ_CP049289 (6224601-6265663): Bradyrhizobium sp. 6(2017) strain 1S3 chromoso... (11% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r90c1_9.svg">
NZ_KB900701 (1497164-1538217): Bradyrhizobium elkanii USDA 76 BraelDRAFT scaf... (10% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r90c1_10.svg">
NC_020453 (152728-193781): Bradyrhizobium oligotrophicum S58, complete genome (10% of genes show similarity), T3PKS
</option>

</select>
<button id= "clusterblast-r90c1-download">Download graphic</button>
</div>
<div id="clusterblast-r90c1-svg"></div>

</div>
</div>

<div class="body-details-section r90c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-23"></div> <span class="help-tooltip" id="cb-known-help-23">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r90c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-24"></div> <span class="help-tooltip" id="cb-sub-help-24">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r90c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-26"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-26">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r90c1-pfam-details-svg" data-filename="NZ_UIHC01000006.1_r90c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r90c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r90c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-27"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-27">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r90c1-tigrfam-details-svg" data-filename="NZ_UIHC01000006.1_r90c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r90c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">















</div>















</div>

</div>

</div>




<div class="page" id='r103c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_UIHC01000001.1 - Region 1 - ectoine <div class="help-container"> <div class="help-icon" data-id="region_svg-help-56"></div> <span class="help-tooltip" id="region_svg-help-56">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_UIHC01000001.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r103c1-svg" data-filename="NZ_UIHC01000001.1_r103c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 174,739 - 185,125 nt. (total: 10,387 nt)

<a class="cluster-rules-header" id="r103c1-rules-header" href="#r103c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r103c1-rules">ectoine: ectoine_synt</div>

<div class="region-svg-container">
<div id='r103c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-57"></div> <span class="help-tooltip" id="focus-panel-help-57">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r103c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r103c1-gene-table-details" data-name="r103c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r103c1-MIBiG-cluster-compare" data-name="r103c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r103c1-clusterblast" data-name="r103c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r103c1-knownclusterblast" data-name="r103c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r103c1-subclusterblast" data-name="r103c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r103c1-tfbs-finder" data-name="r103c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r103c1-pfam-details" data-name="r103c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r103c1-tigrfam-details" data-name="r103c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r103c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-34"></div> <span class="help-tooltip" id="gt-help-tooltip-help-34">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r103c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r103c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r103c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r103c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r103c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-28"></div> <span class="help-tooltip" id="MIBiG-help-28">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r103c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r103c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r103c1"></div>
<div id="comparison-MIBiG-r103c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">ectoine</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860: 710-5134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860</a></td>





<td class="cc-heat-cell" title="0.81 (id:0.53, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>


<td>0.54</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylarcula marina</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859: 258-3305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859</a></td>





<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>


<td>0.51</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium alcaliphilum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856: 517-4347">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856</a></td>





<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>


<td>0.51</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylophaga thalassica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857: 300-4871">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857</a></td>





<td class="cc-heat-cell" title="0.79 (id:0.49, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>


<td>0.51</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylophaga alcalica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855: 245-3455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855</a></td>





<td class="cc-heat-cell" title="0.72 (id:0.50, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>


<td>0.41</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium kenyense</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.50, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>Other</td>

<td>ectoine</td>


<td>Sporosarcina pasteurii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854: 838-6879">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.49, order:0.83, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium alcaliphilum 20Z</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052: 188-3620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.48, order:0.76, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Other (Ectoine)</td>

<td>ectoine</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853: 8-3345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.48, order:0.76, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Other</td>

<td>ectoine</td>


<td>Streptomyces anulatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.47, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylobacter marinus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r103c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860: 710-5134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860</a></td>


<td class="cc-heat-cell" title="0.81 (id:0.53, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>
<td>0.81</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylarcula marina</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859: 258-3305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859</a></td>


<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
<td>0.79</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium alcaliphilum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856: 517-4347">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856</a></td>


<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
<td>0.79</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylophaga thalassica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857: 300-4871">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857</a></td>


<td class="cc-heat-cell" title="0.79 (id:0.49, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
<td>0.79</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylophaga alcalica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855: 245-3455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855</a></td>


<td class="cc-heat-cell" title="0.72 (id:0.50, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
<td>0.72</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium kenyense</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.50, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Other</td>
<td>ectoine</td>
<td>Sporosarcina pasteurii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854: 838-6879">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.49, order:0.83, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium alcaliphilum 20Z</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052: 188-3620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.48, order:0.76, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other (Ectoine)</td>
<td>ectoine</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853: 8-3345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.48, order:0.76, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other</td>
<td>ectoine</td>
<td>Streptomyces anulatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.47, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylobacter marinus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r103c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-29"></div> <span class="help-tooltip" id="cb-general-help-29">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r103c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r103c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r103c1_1.svg">
NZ_CP024899 (2098634-2109021): Rhodobaca barguzinensis strain alga05 chromoso... (50% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r103c1_2.svg">
NZ_MEHT01000009 (25734-53535): Roseinatronobacter thiooxidans strain ALG1 ALG... (17% of genes show similarity), ectoine,terpene
</option>

<option value = "svg/clusterblast_r103c1_3.svg">
NZ_LXYH01000056 (3698-14082): Rhodobacteraceae bacterium EhC02 A5 Contig6, wh... (50% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r103c1_4.svg">
NZ_CP049056 (3924672-3935059): Rhodobacteraceae bacterium RR4-56 chromosome, ... (71% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r103c1_5.svg">
NZ_FOQH01000008 (123921-134320): Albimonas pacifica strain CGMCC 1.11030, who... (62% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r103c1_6.svg">
NZ_CP022191 (349630-360023): Yangia pacifica strain YSBP01 plasmid unnamed1, ... (55% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r103c1_7.svg">
NZ_ML119082 (150587-160980): Histidinibacterium lentulum strain B17 Scaffold2... (71% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r103c1_8.svg">
NZ_FOZW01000005 (263225-273618): Yangia pacifica strain DSM 26894, whole geno... (55% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r103c1_9.svg">
NZ_CP031841 (2803713-2894138): Breoghania sp. L-A4 chromosome, complete genome (8% of genes show similarity), NRPS,T3PKS,ectoine,transAT-PKS
</option>

<option value = "svg/clusterblast_r103c1_10.svg">
NZ_CP014598 (134091-144484): Yangia sp. CCB-MM3 plasmid unnamed2, complete se... (71% of genes show similarity), ectoine
</option>

</select>
<button id= "clusterblast-r103c1-download">Download graphic</button>
</div>
<div id="clusterblast-r103c1-svg"></div>

</div>
</div>

<div class="body-details-section r103c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-30"></div> <span class="help-tooltip" id="cb-known-help-30">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r103c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r103c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r103c1_1.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r103c1_2.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r103c1_3.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r103c1_4.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r103c1_5.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r103c1_6.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r103c1_7.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r103c1_8.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r103c1_9.svg">
ectoine
</option>

</select>
<button id="knownclusterblast-r103c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r103c1-svg"></div>

</div>
</div>

<div class="body-details-section r103c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-31"></div> <span class="help-tooltip" id="cb-sub-help-31">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r103c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-33"></div> <span class="help-tooltip" id="tfbs-body-help-33">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ArgR
<div class="collapser collapser-target-ArgR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Regulator of arginine biosynthesis genes<br>
<span>Score</span>: 16.91 out of 20.01
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r103c1-180114-ArgR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r103c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-35"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-35">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r103c1-pfam-details-svg" data-filename="NZ_UIHC01000001.1_r103c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r103c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r103c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-36"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-36">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r103c1-tigrfam-details-svg" data-filename="NZ_UIHC01000001.1_r103c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r103c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r103c1-tfbs-finder" data-name="r103c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r103c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-32"></div> <span class="help-tooltip" id="tfbs-side-help-32">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ArgR" target="_blank">ArgR</a></td>
<td>Regulator of arginine biosynthesis genes</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r158c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_UIHC01000007.1 - Region 1 - NRPS-like,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-58"></div> <span class="help-tooltip" id="region_svg-help-58">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_UIHC01000007.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r158c1-svg" data-filename="NZ_UIHC01000007.1_r158c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 5,329 - 57,183 nt. (total: 51,855 nt)

<a class="cluster-rules-header" id="r158c1-rules-header" href="#r158c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r158c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r158c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-59"></div> <span class="help-tooltip" id="focus-panel-help-59">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r158c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r158c1-gene-table-details" data-name="r158c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r158c1-nrps_pks" data-name="r158c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r158c1-MIBiG-cluster-compare" data-name="r158c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r158c1-clusterblast" data-name="r158c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r158c1-knownclusterblast" data-name="r158c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r158c1-subclusterblast" data-name="r158c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r158c1-tfbs-finder" data-name="r158c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r158c1-nrps-pks-bubbles" data-name="r158c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r158c1-pfam-details" data-name="r158c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r158c1-tigrfam-details" data-name="r158c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r158c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-47"></div> <span class="help-tooltip" id="gt-help-tooltip-help-47">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r158c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r158c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r158c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r158c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r158c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-37"></div> <span class="help-tooltip" id="nrps-domain-help-37">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r158c1-details-svg" data-filename="NZ_UIHC01000007.1_r158c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r158c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r158c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-38"></div> <span class="help-tooltip" id="MIBiG-help-38">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r158c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r158c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r158c1"></div>
<div id="comparison-MIBiG-r158c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002076: 6-3180">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002076/index.html#r1c1" target="_new">BGC0002076</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.49</td>
<td>Other</td>

<td>4,5-didehydroacaterin, acaterin</td>


<td>Pseudomonas jessenii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001723: 128-10733">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001723/index.html#r1c1" target="_new">BGC0001723</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.48</td>
<td>Polyketide</td>

<td>oviedomycin</td>


<td>Streptomyces antibioticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000836: 0-15022">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000836/index.html#r1c1" target="_new">BGC0000836</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.47</td>
<td>Other</td>

<td>APE Ec</td>


<td>Escherichia coli CFT073</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.44</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000903: 126-11373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000903/index.html#r1c1" target="_new">BGC0000903</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.54, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.54, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.43</td>
<td>Other</td>

<td>fluorometabolite</td>


<td>Streptomyces cattleya</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001177: 10040-36741">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001177/index.html#r1c1" target="_new">BGC0001177</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.54, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.54, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.43</td>
<td>Polyketide</td>

<td>mensacaricin</td>


<td>Streptomyces bottropensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001651: 0-10711">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001651/index.html#r1c1" target="_new">BGC0001651</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.39</td>
<td>Other</td>

<td>legionaminic acid</td>


<td>Tannerella forsythia 92A2</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002218: 3-22615">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002218/index.html#r1c1" target="_new">BGC0002218</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.39</td>
<td>Terpene</td>

<td>(-)-Cyatha-3,12-diene</td>


<td>Dentipellis sp. KUC8613</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000393: 0-47918">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000393/index.html#r1c1" target="_new">BGC0000393</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.48, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.48, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.39</td>
<td>NRP, Polyketide</td>

<td>myxoprincomide-c506</td>


<td>Myxococcus xanthus DK 1622</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000043: 0-13097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000043/index.html#r1c1" target="_new">BGC0000043</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>






<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.38</td>
<td>Polyketide</td>

<td>curacin A</td>


<td>Moorea producens 19L</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r158c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002076: 6-3180">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002076/index.html#r1c1" target="_new">BGC0002076</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Other</td>
<td>4,5-didehydroacaterin, acaterin</td>
<td>Pseudomonas jessenii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001723: 128-10733">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001723/index.html#r1c1" target="_new">BGC0001723</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>oviedomycin</td>
<td>Streptomyces antibioticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000836: 0-15022">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000836/index.html#r1c1" target="_new">BGC0000836</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>APE Ec</td>
<td>Escherichia coli CFT073</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000903: 126-11373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000903/index.html#r1c1" target="_new">BGC0000903</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.54, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Other</td>
<td>fluorometabolite</td>
<td>Streptomyces cattleya</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001177: 10040-36741">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001177/index.html#r1c1" target="_new">BGC0001177</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.54, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>mensacaricin</td>
<td>Streptomyces bottropensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678: 0-13437">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.48, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Terpene</td>
<td>pentalenolactone</td>
<td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001651: 0-10711">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001651/index.html#r1c1" target="_new">BGC0001651</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Other</td>
<td>legionaminic acid</td>
<td>Tannerella forsythia 92A2</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002218: 3-22615">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002218/index.html#r1c1" target="_new">BGC0002218</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Terpene</td>
<td>(-)-Cyatha-3,12-diene</td>
<td>Dentipellis sp. KUC8613</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000393: 0-47918">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000393/index.html#r1c1" target="_new">BGC0000393</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.48, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP, Polyketide</td>
<td>myxoprincomide-c506</td>
<td>Myxococcus xanthus DK 1622</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r158c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-39"></div> <span class="help-tooltip" id="cb-general-help-39">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r158c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r158c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r158c1_1.svg">
NZ_KI421507 (432640-484872): Pseudophaeobacter arcticus DSM 23566 Phaar neu s... (30% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r158c1_2.svg">
NZ_CH902583 (1373600-1425694): Roseobacter sp. MED193 1099517003978, whole ge... (30% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r158c1_3.svg">
NZ_VAUA01000003 (389505-441614): Pelagicola sp. DSW4-44 contig3, whole genome... (26% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r158c1_4.svg">
NZ_KE386893 (29718-81879): Pseudodonghicola xiamenensis DSM 18339 G455DRAFT s... (25% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r158c1_5.svg">
NZ_FNVD01000002 (221290-273384): Jhaorihella thermophila strain DSM 23413, wh... (26% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r158c1_6.svg">
NZ_CP015124 (3864180-3916338): Phaeobacter gallaeciensis strain JL2886, compl... (26% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r158c1_7.svg">
NZ_CP010725 (542207-594394): Phaeobacter inhibens strain P88 chromosome, comp... (25% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r158c1_8.svg">
NC_018286 (517435-569622): Phaeobacter inhibens 2.10, complete sequence (26% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r158c1_9.svg">
NZ_CP010741 (562259-614446): Phaeobacter inhibens strain P59 chromosome, comp... (26% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r158c1_10.svg">
NZ_CP010610 (537911-590098): Phaeobacter inhibens strain P92 chromosome, comp... (26% of genes show similarity), NRPS-like,T1PKS
</option>

</select>
<button id= "clusterblast-r158c1-download">Download graphic</button>
</div>
<div id="clusterblast-r158c1-svg"></div>

</div>
</div>

<div class="body-details-section r158c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-40"></div> <span class="help-tooltip" id="cb-known-help-40">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r158c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r158c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r158c1_1.svg">
fulvuthiacene A/fulvuthiacene B
</option>

</select>
<button id="knownclusterblast-r158c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r158c1-svg"></div>

</div>
</div>

<div class="body-details-section r158c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-41"></div> <span class="help-tooltip" id="cb-sub-help-41">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r158c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-45"></div> <span class="help-tooltip" id="tfbs-body-help-45">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AfsQ1
<div class="collapser collapser-target-AfsQ1 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Two-component system AfsQ1-Q2, activator of antibiotic production<br>
<span>Score</span>: 18.42 out of 19.52
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r158c1-48033-AfsQ1"></div>

</div>


</div>
</div>



<div class="body-details-section r158c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-46"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-46">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r158c1-domain-bubble-svg" data-filename="NZ_UIHC01000007.1_r158c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r158c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (5328 - 57183): neighbouring T1PKS-NRPS-like
</option>



<option value="CC2">
Candidate 2 (5328 - 51661): single T1PKS
</option>



<option value="CC3">
Candidate 3 (12653 - 57183): single NRPS-like
</option>


</select>

<div class="details-svg" id="r158c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r158c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r158c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-48"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-48">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r158c1-pfam-details-svg" data-filename="NZ_UIHC01000007.1_r158c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r158c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r158c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-49"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-49">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r158c1-tigrfam-details-svg" data-filename="NZ_UIHC01000007.1_r158c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r158c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r158c1-nrps_pks_products" data-name="r158c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r158c1-nrps_pks_monomers" data-name="r158c1-nrps_pks_monomers">
NRPS/PKS monomers
</div>



<div class="sidepanel-details-header r158c1-tfbs-finder" data-name="r158c1-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r158c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-42"></div> <span class="help-tooltip" id="nrps-structures-help-42">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 5328 - 57183:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r158c1-1" class="smiles-canvas" data-smiles="C=CNC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(ccmal) + (X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal*&amp;nrps2=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r158c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C=CNC([*])C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 2, location 5328 - 51661:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r158c1-2" class="smiles-canvas" data-smiles="C=CNC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(ccmal) + (X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal*&amp;nrps2=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r158c1-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C=CNC([*])C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 3, location 12653 - 57183:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r158c1-3" class="smiles-canvas" data-smiles="C=CNC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(ccmal) + (X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal*&amp;nrps2=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r158c1-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C=CNC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r158c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS monomer predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-43"></div> <span class="help-tooltip" id="nrps-monomers-help-43">Shows the predicted monomers for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">JDHOFBHF_03715</span></strong>: mal
<div class="collapser collapser-target-JDHOFBHF_03715 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (533..828)</strong>: mal
<div class="collapser collapser-target-JDHOFBHF_03715 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-JDHOFBHF_03715-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 62.5%</dd>
<dd>2-Rhyd-Malonyl-CoA: 58.3%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-JDHOFBHF_03715-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 77.2</dt>

<dd></dd><dt>inactive: 49.5</dt>

<dd></dd><dt>Methylmalonyl-CoA: 41.9</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 40.3</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 29.6</dt>

<dd></dd><dt>Propionyl-CoA: 24.5</dt>

<dd></dd><dt>Benzoyl-CoA: 17.4</dt>

<dd></dd><dt>fatty_acid: 15.9</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 14.6</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>Isobutyryl-CoA: 0.0</dt>

<dd></dd><dt>CHC-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

<strong><span class="serif">JDHOFBHF_03717</span></strong>: X
<div class="collapser collapser-target-JDHOFBHF_03717 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>A-OX (537..1312)</strong>: X
<div class="collapser collapser-target-JDHOFBHF_03717 collapser-level-candidate"> </div><div class="collapser-content">

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r158c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-44"></div> <span class="help-tooltip" id="tfbs-side-help-44">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AfsQ1" target="_blank">AfsQ1</a></td>
<td>Two-component system AfsQ1-Q2, activator of antibiotic production</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>