
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>GCF_003034965.1_Fuscovulum_blasticum_DSM_2131 - 4 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/bacteria.css">
<meta property="og:title" content="GCF_003034965.1_Fuscovulum_blasticum_DSM_2131 - 4 record(s) - 4 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-8d193117-49db-403f-9273-e1acba53b201/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://antismash.secondarymetabolites.org/">
<img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://antismash.secondarymetabolites.org/">
antiSMASH version 7.0.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="GCF_003034965.1_Fuscovulum_blasticum_DSM_2131.zip">Download all results</a></li>

<li><a href="GCF_003034965.1_Fuscovulum_blasticum_DSM_2131.gbk">Download GenBank summary file</a></li>

<li><a href="bacteria-8d193117-49db-403f-9273-e1acba53b201.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton RiPP RRE-containing r2c1">
<a href="#r2c1">2.1</a>
</div>



<div class="regbutton other hserlactone r12c1">
<a href="#r12c1">12.1</a>
</div>



<div class="regbutton terpene terpene r18c1">
<a href="#r18c1">18.1</a>
</div>



<div class="regbutton terpene terpene r22c1">
<a href="#r22c1">22.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>NZ_PZKE01000002.1</strong> (Fuscovulum blasticum)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-2" data-filename="NZ_PZKE01000002.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-2" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r2c1">
<td class="regbutton RiPP RRE-containing r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#rre-containing" target="_blank">RRE-containing</a>
</td>
<td class="digits">24,246</td>
<td class="digits table-split-left">44,650</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_PZKE01000012.1</strong> (Fuscovulum blasticum)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-12" data-filename="NZ_PZKE01000012.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-12" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r12c1">
<td class="regbutton other hserlactone r12c1">
<a href="#r12c1">Region&nbsp12.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">26,514</td>
<td class="digits table-split-left">47,101</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_PZKE01000018.1</strong> (Fuscovulum blasticum)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-18" data-filename="NZ_PZKE01000018.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-18" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r18c1">
<td class="regbutton terpene terpene r18c1">
<a href="#r18c1">Region&nbsp18.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">20,722</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_PZKE01000022.1</strong> (Fuscovulum blasticum)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-22" data-filename="NZ_PZKE01000022.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-22" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r22c1">
<td class="regbutton terpene terpene r22c1">
<a href="#r22c1">Region&nbsp22.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">14,632</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000647/1" target="_blank">carotenoid</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r2c1">
<td class="regbutton RiPP RRE-containing r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#rre-containing" target="_blank">RRE-containing</a>
</td>
<td class="digits">24,246</td>
<td class="digits table-split-left">44,650</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r12c1">
<td class="regbutton other hserlactone r12c1">
<a href="#r12c1">Region&nbsp12.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">26,514</td>
<td class="digits table-split-left">47,101</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r18c1">
<td class="regbutton terpene terpene r18c1">
<a href="#r18c1">Region&nbsp18.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">20,722</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r22c1">
<td class="regbutton terpene terpene r22c1">
<a href="#r22c1">Region&nbsp22.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">14,632</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000647/1" target="_blank">carotenoid</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>NZ_PZKE01000001.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000003.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000004.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000005.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000006.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000007.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000008.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000009.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000010.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000011.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000013.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000014.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000015.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000016.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000017.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000019.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000020.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000021.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000023.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000024.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000025.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000026.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000027.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000028.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000029.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000030.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000031.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000032.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000033.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000034.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000035.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000036.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000037.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000038.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000039.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000040.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000041.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000042.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000043.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000044.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000045.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000046.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000047.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000048.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000049.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000050.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000051.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000052.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000053.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000054.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000055.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000056.1</strong> (Fuscovulum blasticum)</dd>

<dd><strong>NZ_PZKE01000057.1</strong> (Fuscovulum blasticum)</dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>







<div class="page" id='r2c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_PZKE01000002.1 - Region 1 - RRE-containing <div class="help-container"> <div class="help-icon" data-id="region_svg-help-38"></div> <span class="help-tooltip" id="region_svg-help-38">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_PZKE01000002.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c1-svg" data-filename="NZ_PZKE01000002.1_r2c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 24,246 - 44,650 nt. (total: 20,405 nt)

<a class="cluster-rules-header" id="r2c1-rules-header" href="#r2c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c1-rules">RRE-containing: minimum(1, [Bottromycin_Methyltransferase_RRE, Cyanobactin_D_RRE, Cytolysin_C_RRE, Goadsporin_CD_RRE, Heterocycloanthracin_C_RRE, Lanthipeptide_LanB_RRE, Lasso_Fused_RRE, Mycofactocin_RRE, NHLP_CD_RRE, Other_Sactipeptide_rSAM_RRE, Pantocin_Microcin_RRE, Plantazolicin_C_RRE, PlpY_RRE, PqqD_RRE, Proteusin_Epimerase_RRE, Proteusin_Methyltransferase_RRE, Quinohemoprotein_rSAM_RRE, Ranthipeptide_rSAM_RRE, Stand_Alone_Lasso_RRE, Streptide_RRE, Subtilosin_rSAM_RRE, Thiaglutamate_B_RRE, Thiaglutamate_G_RRE, Thiaglutamate_I_RRE, Thiopeptide_F_RRE, Thuricin_rSAM_RRE, Thurincin_rSAM_RRE, Trifolitoxin_RRE])</div>

<div class="region-svg-container">
<div id='r2c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-39"></div> <span class="help-tooltip" id="focus-panel-help-39">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c1-gene-table-details" data-name="r2c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c1-MIBiG-cluster-compare" data-name="r2c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c1-clusterblast" data-name="r2c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c1-knownclusterblast" data-name="r2c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c1-subclusterblast" data-name="r2c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c1-tfbs-finder" data-name="r2c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r2c1-pfam-details" data-name="r2c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c1-tigrfam-details" data-name="r2c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-8"></div> <span class="help-tooltip" id="gt-help-tooltip-help-8">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-1"></div> <span class="help-tooltip" id="MIBiG-help-1">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c1"></div>
<div id="comparison-MIBiG-r2c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">RRE-containing</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000754: 271-8587">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000754/index.html#r1c1" target="_new">BGC0000754</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Avibacterium paragallinarum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000755: 272-8587">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000755/index.html#r1c1" target="_new">BGC0000755</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Avibacterium paragallinarum 221</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000741: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000741/index.html#r1c1" target="_new">BGC0000741</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000744: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000744/index.html#r1c1" target="_new">BGC0000744</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000745: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000745/index.html#r1c1" target="_new">BGC0000745</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001779: 0-6292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001779/index.html#r1c1" target="_new">BGC0001779</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>RiPP</td>

<td>siamycin</td>


<td>Streptomyces nodosus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000644: 97-6276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000644/index.html#r1c1" target="_new">BGC0000644</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Dietzia sp. CQ4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001387: 150-29437">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001387/index.html#r1c1" target="_new">BGC0001387</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.48, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Other</td>

<td>nucleocidin</td>


<td>Streptomyces calvus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000740: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000740/index.html#r1c1" target="_new">BGC0000740</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000746: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000746/index.html#r1c1" target="_new">BGC0000746</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Saccharide</td>

<td>capsular polysaccharide</td>


<td>Streptococcus agalactiae</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000754: 271-8587">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000754/index.html#r1c1" target="_new">BGC0000754</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Avibacterium paragallinarum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000755: 272-8587">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000755/index.html#r1c1" target="_new">BGC0000755</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Avibacterium paragallinarum 221</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000741: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000741/index.html#r1c1" target="_new">BGC0000741</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000744: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000744/index.html#r1c1" target="_new">BGC0000744</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000745: 0-4768">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000745/index.html#r1c1" target="_new">BGC0000745</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001779: 0-6292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001779/index.html#r1c1" target="_new">BGC0001779</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>RiPP</td>
<td>siamycin</td>
<td>Streptomyces nodosus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000644: 97-6276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000644/index.html#r1c1" target="_new">BGC0000644</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Dietzia sp. CQ4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001387: 150-29437">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001387/index.html#r1c1" target="_new">BGC0001387</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.48, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other</td>
<td>nucleocidin</td>
<td>Streptomyces calvus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000740: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000740/index.html#r1c1" target="_new">BGC0000740</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000746: 0-4888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000746/index.html#r1c1" target="_new">BGC0000746</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Saccharide</td>
<td>capsular polysaccharide</td>
<td>Streptococcus agalactiae</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-2"></div> <span class="help-tooltip" id="cb-general-help-2">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c1_1.svg">
NZ_CP034446 (1128544-1149368): Mesorhizobium sp. M2A.F.Ca.ET.043.05.1.1 chrom... (23% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c1_2.svg">
NZ_CP034445 (412629-433448): Mesorhizobium sp. M2A.F.Ca.ET.043.02.1.1 chromos... (21% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c1_3.svg">
NZ_CP034449 (5957886-5978711): Mesorhizobium sp. M2A.F.Ca.ET.046.03.2.1 chrom... (21% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c1_4.svg">
NZ_VLJP01000001 (170651-211839): Mesorhizobium sp. J18 MesJ18 scaffold 0, who... (15% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r2c1_5.svg">
NZ_CP021431 (1900732-1943543): Yoonia vestfoldensis strain SMR4r chromosome, ... (9% of genes show similarity), NRPS-like,bacteriocin
</option>

<option value = "svg/clusterblast_r2c1_6.svg">
NZ_KB907979 (151755-194560): Yoonia vestfoldensis DSM 16212 H147DRAFT scaffol... (9% of genes show similarity), NRPS-like,bacteriocin
</option>

<option value = "svg/clusterblast_r2c1_7.svg">
NZ_PPGA01000001 (423189-453474): Paracoccus denitrificans strain ISTOD1 IIFCA... (15% of genes show similarity), LAP,bacteriocin,thiopeptide
</option>

<option value = "svg/clusterblast_r2c1_8.svg">
NC_008687 (1337178-1367277): Paracoccus denitrificans PD1222 chromosome 2, co... (14% of genes show similarity), LAP,bacteriocin,thiopeptide
</option>

<option value = "svg/clusterblast_r2c1_9.svg">
NZ_CP029208 (2831087-2872275): Nitratireductor sp. OM-1 chromosome, complete ... (13% of genes show similarity), arylpolyene
</option>

<option value = "svg/clusterblast_r2c1_10.svg">
NZ_FOXZ01000001 (535280-576468): Nitratireductor indicus strain CGMCC 1.10953... (10% of genes show similarity), arylpolyene
</option>

</select>
<button id= "clusterblast-r2c1-download">Download graphic</button>
</div>
<div id="clusterblast-r2c1-svg"></div>

</div>
</div>

<div class="body-details-section r2c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-3"></div> <span class="help-tooltip" id="cb-known-help-3">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-4"></div> <span class="help-tooltip" id="cb-sub-help-4">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-7"></div> <span class="help-tooltip" id="tfbs-body-help-7">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (5):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 17.18 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c1-24617-ZuR"></div>

<div class="tfbs-hit-detail">
DasR
<div class="collapser collapser-target-DasR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: N-acetylglucosamine dependent repressor<br>
<span>Score</span>: 20.75 out of 27.04
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c1-24918-DasR"></div>

<div class="tfbs-hit-detail">
DmdR1
<div class="collapser collapser-target-DmdR1 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Iron(II)-dependent repressor<br>
<span>Score</span>: 23.19 out of 36.53
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c1-34693-DmdR1"></div>

<div class="tfbs-hit-detail">
NuR
<div class="collapser collapser-target-NuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Nickel-responsive regulator<br>
<span>Score</span>: 20.97 out of 30.98
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c1-43737-NuR"></div>

<div class="tfbs-hit-detail">
LexA
<div class="collapser collapser-target-LexA collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Repressor of DNA damage response<br>
<span>Score</span>: 20.27 out of 29.65
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c1-44623-LexA"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r2c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-9"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-9">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-pfam-details-svg" data-filename="NZ_PZKE01000002.1_r2c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-10"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-10">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-tigrfam-details-svg" data-filename="NZ_PZKE01000002.1_r2c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r2c1-RREfinder" data-name="r2c1-RREfinder">
RREFinder
</div>



<div class="sidepanel-details-header r2c1-tfbs-finder" data-name="r2c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r2c1-RREfinder">
<div class="more-details">
<div class="heading">
<span>RRE predictions</span>
<div class="help-container"> <div class="help-icon" data-id="rre-side-help-5"></div> <span class="help-tooltip" id="rre-side-help-5">RREfinder results sidepanel.</span></div>
</div>
<dl class = 'prediction-text'>

<span><strong>RRE-containing protocluster</strong> (24245...44650)</span>
<dt></dt>




<strong>OCIILLLM_00442</strong> <div class="collapser collapser-target-OCIILLLM_00442 collapser-level-cds"> </div><div class="collapser-content">

<dd>RRE type: Lasso_Fused_RRE</dd>
<dd>Location: 7 - 80</dd>
<dd>Score: 30.8</dd>

</div><br>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r2c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-6"></div> <span class="help-tooltip" id="tfbs-side-help-6">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/DasR" target="_blank">DasR</a></td>
<td>N-acetylglucosamine dependent repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/DmdR1" target="_blank">DmdR1</a></td>
<td>Iron(II)-dependent repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NuR" target="_blank">NuR</a></td>
<td>Nickel-responsive regulator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/LexA" target="_blank">LexA</a></td>
<td>Repressor of DNA damage response</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r12c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_PZKE01000012.1 - Region 1 - hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-40"></div> <span class="help-tooltip" id="region_svg-help-40">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_PZKE01000012.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r12c1-svg" data-filename="NZ_PZKE01000012.1_r12c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 26,514 - 47,101 nt. (total: 20,588 nt)

<a class="cluster-rules-header" id="r12c1-rules-header" href="#r12c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r12c1-rules">hserlactone: Autoind_synth</div>

<div class="region-svg-container">
<div id='r12c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-41"></div> <span class="help-tooltip" id="focus-panel-help-41">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r12c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r12c1-gene-table-details" data-name="r12c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r12c1-MIBiG-cluster-compare" data-name="r12c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r12c1-clusterblast" data-name="r12c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r12c1-knownclusterblast" data-name="r12c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r12c1-subclusterblast" data-name="r12c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r12c1-tfbs-finder" data-name="r12c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r12c1-pfam-details" data-name="r12c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r12c1-tigrfam-details" data-name="r12c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r12c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-17"></div> <span class="help-tooltip" id="gt-help-tooltip-help-17">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r12c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r12c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r12c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r12c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r12c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-11"></div> <span class="help-tooltip" id="MIBiG-help-11">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r12c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r12c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r12c1"></div>
<div id="comparison-MIBiG-r12c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">hserlactone</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001992: 0-4749">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001992/index.html#r1c1" target="_new">BGC0001992</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>Other</td>

<td>thioguanine</td>


<td>Erwinia amylovora CFBP1430</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001671: 5106-76272">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001671/index.html#r1c1" target="_new">BGC0001671</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.47, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.06</td>
<td>NRP</td>

<td>monobactam</td>


<td>Agrobacterium tumefaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001686: 0-15938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001686/index.html#r1c1" target="_new">BGC0001686</a></td>





<td class="cc-heat-cell" title="0.17 (id:0.45, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>


<td>0.04</td>
<td>NRP</td>

<td>N-octanoyl-Met-Phe-H</td>


<td>Clostridium sp. CAG:567</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r12c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001992: 0-4749">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001992/index.html#r1c1" target="_new">BGC0001992</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Other</td>
<td>thioguanine</td>
<td>Erwinia amylovora CFBP1430</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001671: 5106-76272">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001671/index.html#r1c1" target="_new">BGC0001671</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.47, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>NRP</td>
<td>monobactam</td>
<td>Agrobacterium tumefaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001686: 0-15938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001686/index.html#r1c1" target="_new">BGC0001686</a></td>


<td class="cc-heat-cell" title="0.17 (id:0.45, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>
<td>0.17</td>
<td>NRP</td>
<td>N-octanoyl-Met-Phe-H</td>
<td>Clostridium sp. CAG:567</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r12c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-12"></div> <span class="help-tooltip" id="cb-general-help-12">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r12c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r12c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r12c1_1.svg">
NZ_CP020470 (1663241-1683829): Rhodobacter blasticus strain 28/5 chromosome, ... (100% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r12c1_2.svg">
NZ_SJZH01000004 (247745-269960): Tabrizicola sp. YIM 78059 Scaffold4, whole g... (36% of genes show similarity), ectoine,hserlactone
</option>

<option value = "svg/clusterblast_r12c1_3.svg">
NZ_CP028918 (419188-439689): Gemmobacter sp. HYN0069 chromosome, complete genome (42% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r12c1_4.svg">
NZ_VKKX01000011 (366573-387161): Gemmobacter caeruleus strain N8 NODE 1, whol... (42% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r12c1_5.svg">
NZ_CP015421 (206539-226527): Rhodovulum sulfidophilum strain SNK001, complete... (44% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r12c1_6.svg">
NZ_MSYP01000001 (757788-777790): Rhodovulum sulfidophilum strain AB14 Scaffol... (44% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r12c1_7.svg">
NZ_MSYR01000001 (3772350-3792941): Rhodovulum sulfidophilum strain AB30 Scaff... (44% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r12c1_8.svg">
NZ_CP035510 (1833833-1854421): Haematobacter massiliensis strain OT1 chromoso... (42% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r12c1_9.svg">
NZ_SLXU01000010 (72163-92754): Rhodovulum bhavnagarense strain DSM 24766 Ga03... (55% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r12c1_10.svg">
NZ_CP015039 (1237337-1257928): Rhodovulum sp. P5 chromosome, complete genome (57% of genes show similarity), hserlactone
</option>

</select>
<button id= "clusterblast-r12c1-download">Download graphic</button>
</div>
<div id="clusterblast-r12c1-svg"></div>

</div>
</div>

<div class="body-details-section r12c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-13"></div> <span class="help-tooltip" id="cb-known-help-13">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r12c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-14"></div> <span class="help-tooltip" id="cb-sub-help-14">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r12c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-16"></div> <span class="help-tooltip" id="tfbs-body-help-16">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AbrC3
<div class="collapser collapser-target-AbrC3 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Antibiotic production activator<br>
<span>Score</span>: 18.07 out of 18.16
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r12c1-26621-AbrC3"></div>

</div>


</div>
</div>



<div class="body-details-section r12c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-18"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-18">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r12c1-pfam-details-svg" data-filename="NZ_PZKE01000012.1_r12c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r12c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r12c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-19"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-19">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r12c1-tigrfam-details-svg" data-filename="NZ_PZKE01000012.1_r12c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r12c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r12c1-tfbs-finder" data-name="r12c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r12c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-15"></div> <span class="help-tooltip" id="tfbs-side-help-15">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AbrC3" target="_blank">AbrC3</a></td>
<td>Antibiotic production activator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r18c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_PZKE01000018.1 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-42"></div> <span class="help-tooltip" id="region_svg-help-42">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_PZKE01000018.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r18c1-svg" data-filename="NZ_PZKE01000018.1_r18c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 20,722 nt. (total: 20,722 nt)

<a class="cluster-rules-header" id="r18c1-rules-header" href="#r18c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r18c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r18c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-43"></div> <span class="help-tooltip" id="focus-panel-help-43">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r18c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r18c1-gene-table-details" data-name="r18c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r18c1-MIBiG-cluster-compare" data-name="r18c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r18c1-clusterblast" data-name="r18c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r18c1-knownclusterblast" data-name="r18c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r18c1-subclusterblast" data-name="r18c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r18c1-tfbs-finder" data-name="r18c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r18c1-pfam-details" data-name="r18c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r18c1-tigrfam-details" data-name="r18c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r18c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-26"></div> <span class="help-tooltip" id="gt-help-tooltip-help-26">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r18c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r18c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r18c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r18c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r18c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-20"></div> <span class="help-tooltip" id="MIBiG-help-20">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r18c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r18c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r18c1"></div>
<div id="comparison-MIBiG-r18c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001859: 6609-19144">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001859/index.html#r1c1" target="_new">BGC0001859</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Other (Phosphonate)</td>

<td>fosfomycin</td>


<td>Pseudomonas syringae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000904: 116-14341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000904/index.html#r1c1" target="_new">BGC0000904</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Other</td>

<td>FR-900098</td>


<td>Streptomyces rubellomurinus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002406: 621-194635">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002406/index.html#r1c1" target="_new">BGC0002406</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Other</td>

<td>hydroxycinnamoyltyramine</td>


<td>Oryza sativa Japonica Group</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001210: 0-9235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001210/index.html#r1c1" target="_new">BGC0001210</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>pseudomycoicidin</td>


<td>Bacillus pseudomycoides DSM 12442</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551: 0-6728">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>SapB</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000521: 0-12132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000521/index.html#r1c1" target="_new">BGC0000521</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>lacticin 481</td>


<td>Lactococcus lactis subsp. lactis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519: 0-6400">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>


<td>Actinomadura namibiensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000626: 0-7622">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000626/index.html#r1c1" target="_new">BGC0000626</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>thuricin</td>


<td>Bacillus thuringiensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000498: 1-14995">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000498/index.html#r1c1" target="_new">BGC0000498</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>bovicin HJ50</td>


<td>Streptococcus equinus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000561: 0-16402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000561/index.html#r1c1" target="_new">BGC0000561</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>thermophilin 1277</td>


<td>Streptococcus thermophilus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r18c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001859: 6609-19144">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001859/index.html#r1c1" target="_new">BGC0001859</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other (Phosphonate)</td>
<td>fosfomycin</td>
<td>Pseudomonas syringae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000904: 116-14341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000904/index.html#r1c1" target="_new">BGC0000904</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other</td>
<td>FR-900098</td>
<td>Streptomyces rubellomurinus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002406: 621-194635">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002406/index.html#r1c1" target="_new">BGC0002406</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Other</td>
<td>hydroxycinnamoyltyramine</td>
<td>Oryza sativa Japonica Group</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001210: 0-9235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001210/index.html#r1c1" target="_new">BGC0001210</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>pseudomycoicidin</td>
<td>Bacillus pseudomycoides DSM 12442</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551: 0-6728">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>SapB</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000521: 0-12132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000521/index.html#r1c1" target="_new">BGC0000521</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>lacticin 481</td>
<td>Lactococcus lactis subsp. lactis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519: 0-6400">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
<td>Actinomadura namibiensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000626: 0-7622">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000626/index.html#r1c1" target="_new">BGC0000626</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>thuricin</td>
<td>Bacillus thuringiensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000498: 1-14995">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000498/index.html#r1c1" target="_new">BGC0000498</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>bovicin HJ50</td>
<td>Streptococcus equinus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000561: 0-16402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000561/index.html#r1c1" target="_new">BGC0000561</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>RiPP</td>
<td>thermophilin 1277</td>
<td>Streptococcus thermophilus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r18c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-21"></div> <span class="help-tooltip" id="cb-general-help-21">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r18c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r18c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r18c1_1.svg">
NZ_CP020470 (502132-520697): Rhodobacter blasticus strain 28/5 chromosome, co... (94% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r18c1_2.svg">
NZ_CP034328 (73281-92755): Tabrizicola piscis strain K13M18 chromosome, compl... (88% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r18c1_3.svg">
NZ_CP028918 (1661226-1682000): Gemmobacter sp. HYN0069 chromosome, complete g... (82% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r18c1_4.svg">
NZ_JAAIKE010000001 (485895-506654): Rhodobacter sp. ETT8 contig1, whole genom... (70% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r18c1_5.svg">
NZ_OAOQ01000001 (323718-364255): Rhodobacter ovatus strain JA234, whole genom... (20% of genes show similarity), hserlactone,terpene
</option>

<option value = "svg/clusterblast_r18c1_6.svg">
NZ_CP023548 (1959658-1979193): Rhodobacter sp. CZR27 chromosome, complete genome (47% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r18c1_7.svg">
NZ_CP015421 (3154326-3175088): Rhodovulum sulfidophilum strain SNK001, comple... (47% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r18c1_8.svg">
NZ_MSYR01000001 (842201-861459): Rhodovulum sulfidophilum strain AB30 Scaffol... (50% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r18c1_9.svg">
NZ_AP014800 (1507906-1528668): Rhodovulum sulfidophilum DNA, complete genome,... (47% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r18c1_10.svg">
NZ_PZZW01000002 (59846-79139): Rhodobacter johrii strain JA192 Ga0191650 102,... (42% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r18c1-download">Download graphic</button>
</div>
<div id="clusterblast-r18c1-svg"></div>

</div>
</div>

<div class="body-details-section r18c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-22"></div> <span class="help-tooltip" id="cb-known-help-22">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r18c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-23"></div> <span class="help-tooltip" id="cb-sub-help-23">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r18c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-25"></div> <span class="help-tooltip" id="tfbs-body-help-25">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AbrC3
<div class="collapser collapser-target-AbrC3 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Antibiotic production activator<br>
<span>Score</span>: 18.16 out of 18.16
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r18c1-20257-AbrC3"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
LexA
<div class="collapser collapser-target-LexA collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Repressor of DNA damage response<br>
<span>Score</span>: 20.20 out of 29.65
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r18c1-5857-LexA"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r18c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-27"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-27">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r18c1-pfam-details-svg" data-filename="NZ_PZKE01000018.1_r18c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r18c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r18c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-28"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-28">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r18c1-tigrfam-details-svg" data-filename="NZ_PZKE01000018.1_r18c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r18c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r18c1-tfbs-finder" data-name="r18c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r18c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-24"></div> <span class="help-tooltip" id="tfbs-side-help-24">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/LexA" target="_blank">LexA</a></td>
<td>Repressor of DNA damage response</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AbrC3" target="_blank">AbrC3</a></td>
<td>Antibiotic production activator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r22c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_PZKE01000022.1 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-44"></div> <span class="help-tooltip" id="region_svg-help-44">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_PZKE01000022.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r22c1-svg" data-filename="NZ_PZKE01000022.1_r22c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 14,632 nt. (total: 14,632 nt)

<a class="cluster-rules-header" id="r22c1-rules-header" href="#r22c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r22c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r22c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-45"></div> <span class="help-tooltip" id="focus-panel-help-45">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r22c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r22c1-gene-table-details" data-name="r22c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r22c1-MIBiG-cluster-compare" data-name="r22c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r22c1-clusterblast" data-name="r22c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r22c1-knownclusterblast" data-name="r22c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r22c1-subclusterblast" data-name="r22c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r22c1-tfbs-finder" data-name="r22c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r22c1-pfam-details" data-name="r22c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r22c1-tigrfam-details" data-name="r22c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r22c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-35"></div> <span class="help-tooltip" id="gt-help-tooltip-help-35">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r22c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r22c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r22c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r22c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r22c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-29"></div> <span class="help-tooltip" id="MIBiG-help-29">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r22c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r22c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r22c1"></div>
<div id="comparison-MIBiG-r22c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647: 251-2872">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647</a></td>





<td class="cc-heat-cell" title="0.82 (id:0.56, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.82), rgb(0, 0, 0, 0.82) 82%, white 82%)"></td>


<td>0.57</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Rhodobacter sphaeroides</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656: 169-3770">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.47, order:0.69, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Terpene</td>

<td>zeaxanthin</td>


<td>Xanthobacter autotrophicus Py2</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000637: 540-9054">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000637/index.html#r1c1" target="_new">BGC0000637</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Corynebacterium glutamicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645: 0-7475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Halobacillus halophilus DSM 2266</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648: 1057-11443">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.44, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Myxococcus xanthus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646: 60-6380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.48, order:0.48, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Terpene</td>

<td>-carotein</td>


<td>uncultured bacterium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630: 389-6051">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.47, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Terpene</td>

<td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>


<td>Paracoccus haeundaensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640: 2399-8466">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.47, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Enterobacteriaceae bacterium DC404</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664: 0-8391">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:0.45, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>isorenieratene</td>


<td>Streptomyces griseus subsp. griseus NBRC 13350</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000641: 1208-7454">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000641/index.html#r1c1" target="_new">BGC0000641</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.47, order:0.48, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Enterobacteriaceae bacterium DC416</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r22c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647: 251-2872">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647</a></td>


<td class="cc-heat-cell" title="0.82 (id:0.56, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.82), rgb(0, 0, 0, 0.82) 82%, white 82%)"></td>
<td>0.82</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Rhodobacter sphaeroides</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656: 169-3770">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.47, order:0.69, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Terpene</td>
<td>zeaxanthin</td>
<td>Xanthobacter autotrophicus Py2</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000637: 540-9054">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000637/index.html#r1c1" target="_new">BGC0000637</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Corynebacterium glutamicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645: 0-7475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Halobacillus halophilus DSM 2266</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648: 1057-11443">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.44, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Myxococcus xanthus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646: 60-6380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.48, order:0.48, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Terpene</td>
<td>-carotein</td>
<td>uncultured bacterium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630: 389-6051">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.47, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Terpene</td>
<td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>
<td>Paracoccus haeundaensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640: 2399-8466">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.47, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Enterobacteriaceae bacterium DC404</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664: 0-8391">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:0.45, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>isorenieratene</td>
<td>Streptomyces griseus subsp. griseus NBRC 13350</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000641: 1208-7454">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000641/index.html#r1c1" target="_new">BGC0000641</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.47, order:0.48, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Enterobacteriaceae bacterium DC416</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r22c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-30"></div> <span class="help-tooltip" id="cb-general-help-30">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r22c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r22c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r22c1_1.svg">
NZ_CP020470 (698384-719398): Rhodobacter blasticus strain 28/5 chromosome, co... (75% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r22c1_2.svg">
NZ_JAAIKE010000001 (123967-144083): Rhodobacter sp. ETT8 contig1, whole genom... (55% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r22c1_3.svg">
NZ_CP028918 (1124240-1143571): Gemmobacter sp. HYN0069 chromosome, complete g... (58% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r22c1_4.svg">
NZ_CP034328 (3886230-3907250): Tabrizicola piscis strain K13M18 chromosome, c... (58% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r22c1_5.svg">
NC_011963 (1615872-1636940): Rhodobacter sphaeroides KD131 chromosome 1, comp... (58% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r22c1_6.svg">
NZ_CP030271 (1983677-2004745): Rhodobacter sphaeroides 2.4.1 chromosome 1, co... (58% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r22c1_7.svg">
NZ_FZOV01000005 (171484-192552): Rhodobacter megalophilus strain DSM 18937, w... (58% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r22c1_8.svg">
NC_009049 (2016043-2037111): Rhodobacter sphaeroides ATCC 17029 chromosome 1,... (58% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r22c1_9.svg">
NZ_CM001161 (1388242-1409310): Rhodobacter sphaeroides WS8N chromosome I, who... (58% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r22c1_10.svg">
NZ_PZZW01000002 (622436-643504): Rhodobacter johrii strain JA192 Ga0191650 10... (58% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r22c1-download">Download graphic</button>
</div>
<div id="clusterblast-r22c1-svg"></div>

</div>
</div>

<div class="body-details-section r22c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-31"></div> <span class="help-tooltip" id="cb-known-help-31">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r22c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r22c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r22c1_1.svg">
carotenoid
</option>

</select>
<button id="knownclusterblast-r22c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r22c1-svg"></div>

</div>
</div>

<div class="body-details-section r22c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-32"></div> <span class="help-tooltip" id="cb-sub-help-32">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r22c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-34"></div> <span class="help-tooltip" id="tfbs-body-help-34">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AfsR
<div class="collapser collapser-target-AfsR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Pleiotropic regulatory for antibiotic production<br>
<span>Score</span>: 18.46 out of 23.05
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r22c1-14327-AfsR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 17.80 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r22c1-6271-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r22c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-36"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-36">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r22c1-pfam-details-svg" data-filename="NZ_PZKE01000022.1_r22c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r22c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r22c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-37"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-37">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r22c1-tigrfam-details-svg" data-filename="NZ_PZKE01000022.1_r22c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r22c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r22c1-tfbs-finder" data-name="r22c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r22c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-33"></div> <span class="help-tooltip" id="tfbs-side-help-33">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AfsR" target="_blank">AfsR</a></td>
<td>Pleiotropic regulatory for antibiotic production</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>