
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>GCF_003990445.1_Pararhodobacter_zhoushanensis_ZQ420 - 2 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/bacteria.css">
<meta property="og:title" content="GCF_003990445.1_Pararhodobacter_zhoushanensis_ZQ420 - 2 record(s) - 2 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-0bbeb631-b566-4607-b56f-6381ed9a36c9/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://antismash.secondarymetabolites.org/">
<img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://antismash.secondarymetabolites.org/">
antiSMASH version 7.0.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="GCF_003990445.1_Pararhodobacter_zhoushanensis_ZQ420.zip">Download all results</a></li>

<li><a href="GCF_003990445.1_Pararhodobacter_zhoushanensis_ZQ420.gbk">Download GenBank summary file</a></li>

<li><a href="bacteria-0bbeb631-b566-4607-b56f-6381ed9a36c9.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton NRPS NRPS-like r23c1">
<a href="#r23c1">23.1</a>
</div>



<div class="regbutton hybrid r26c1">
<a href="#r26c1">26.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>NZ_REEZ01000003.1</strong> (Pararhodobacter zhoushanensis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-23" data-filename="NZ_REEZ01000003.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-23" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r23c1">
<td class="regbutton NRPS NRPS-like r23c1">
<a href="#r23c1">Region&nbsp23.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">78,624</td>
<td class="digits table-split-left">120,573</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_REEZ01000004.1</strong> (Pararhodobacter zhoushanensis)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-26" data-filename="NZ_REEZ01000004.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-26" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r26c1">
<td class="regbutton hybrid r26c1">
<a href="#r26c1">Region&nbsp26.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">162,202</td>
<td class="digits table-split-left">213,985</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000469/1" target="_blank">bottromycin A2</a></td>
<td>RiPP:Bottromycin</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r23c1">
<td class="regbutton NRPS NRPS-like r23c1">
<a href="#r23c1">Region&nbsp23.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">78,624</td>
<td class="digits table-split-left">120,573</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r26c1">
<td class="regbutton hybrid r26c1">
<a href="#r26c1">Region&nbsp26.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">162,202</td>
<td class="digits table-split-left">213,985</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000469/1" target="_blank">bottromycin A2</a></td>
<td>RiPP:Bottromycin</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>NZ_REEZ01000001.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000010.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000011.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000012.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000013.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000014.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000015.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000016.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000017.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000018.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000019.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000002.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000020.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000021.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000022.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000023.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000024.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000025.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000026.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000027.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000028.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000029.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000030.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000031.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000005.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000006.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000007.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000008.1</strong> (Pararhodobacter zhoushanensis)</dd>

<dd><strong>NZ_REEZ01000009.1</strong> (Pararhodobacter zhoushanensis)</dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>







<div class="page" id='r23c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_REEZ01000003.1 - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-25"></div> <span class="help-tooltip" id="region_svg-help-25">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_REEZ01000003.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r23c1-svg" data-filename="NZ_REEZ01000003.1_r23c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 78,624 - 120,573 nt. (total: 41,950 nt)

<a class="cluster-rules-header" id="r23c1-rules-header" href="#r23c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r23c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r23c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-26"></div> <span class="help-tooltip" id="focus-panel-help-26">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r23c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r23c1-gene-table-details" data-name="r23c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r23c1-nrps_pks" data-name="r23c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r23c1-MIBiG-cluster-compare" data-name="r23c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r23c1-clusterblast" data-name="r23c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r23c1-knownclusterblast" data-name="r23c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r23c1-subclusterblast" data-name="r23c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r23c1-tfbs-finder" data-name="r23c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r23c1-pfam-details" data-name="r23c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r23c1-tigrfam-details" data-name="r23c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r23c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-9"></div> <span class="help-tooltip" id="gt-help-tooltip-help-9">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r23c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r23c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r23c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r23c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r23c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-1"></div> <span class="help-tooltip" id="nrps-domain-help-1">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r23c1-details-svg" data-filename="NZ_REEZ01000003.1_r23c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r23c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r23c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-2"></div> <span class="help-tooltip" id="MIBiG-help-2">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r23c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r23c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r23c1"></div>
<div id="comparison-MIBiG-r23c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001897: 3-16597">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001897/index.html#r1c1" target="_new">BGC0001897</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>cepacin A</td>


<td>Burkholderia ambifaria IOP40-10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831: 1-6152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>alkylpyrone-407, alkylpyrone-393</td>


<td>Myxococcus xanthus DK 1622</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000866: 467-5165">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000866/index.html#r1c1" target="_new">BGC0000866</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Other</td>

<td>polyhydroxyalkanoate</td>


<td>Burkholderia sp. DSM 9242</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000892: 0-13187">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000892/index.html#r1c1" target="_new">BGC0000892</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.47, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>Other</td>

<td>caryoynencin</td>


<td>Burkholderia gladioli BSR3</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001458: 100-11195">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001458/index.html#r1c1" target="_new">BGC0001458</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Other (Aminocoumarin)</td>

<td>salinipostin G</td>


<td>Salinispora tropica CNB-440</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001917: 148-16415">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001917/index.html#r1c1" target="_new">BGC0001917</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>5-alkyl-1,2,3,4-tetrahydroquinolines, streptoaminals</td>


<td>Streptomyces nigrescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000131: 0-6280">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000131/index.html#r1c1" target="_new">BGC0000131</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.41, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>pyrrolomycin</td>


<td>Streptomyces sp. UC 11065</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002591: 0-20232">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002591/index.html#r1c1" target="_new">BGC0002591</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>Polyketide, Terpene</td>

<td>aurachin C, aurachin D, aurachin SS</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>holrhizin</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r23c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001897: 3-16597">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001897/index.html#r1c1" target="_new">BGC0001897</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>cepacin A</td>
<td>Burkholderia ambifaria IOP40-10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831: 1-6152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>alkylpyrone-407, alkylpyrone-393</td>
<td>Myxococcus xanthus DK 1622</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000866: 467-5165">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000866/index.html#r1c1" target="_new">BGC0000866</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>polyhydroxyalkanoate</td>
<td>Burkholderia sp. DSM 9242</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000892: 0-13187">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000892/index.html#r1c1" target="_new">BGC0000892</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.47, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>caryoynencin</td>
<td>Burkholderia gladioli BSR3</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001458: 100-11195">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001458/index.html#r1c1" target="_new">BGC0001458</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Other (Aminocoumarin)</td>
<td>salinipostin G</td>
<td>Salinispora tropica CNB-440</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001917: 148-16415">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001917/index.html#r1c1" target="_new">BGC0001917</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>5-alkyl-1,2,3,4-tetrahydroquinolines, streptoaminals</td>
<td>Streptomyces nigrescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000131: 0-6280">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000131/index.html#r1c1" target="_new">BGC0000131</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.41, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>pyrrolomycin</td>
<td>Streptomyces sp. UC 11065</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002591: 0-20232">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002591/index.html#r1c1" target="_new">BGC0002591</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Polyketide, Terpene</td>
<td>aurachin C, aurachin D, aurachin SS</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>holrhizin</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r23c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-3"></div> <span class="help-tooltip" id="cb-general-help-3">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r23c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r23c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r23c1_1.svg">
NZ_CP031585 (4751632-4793705): Rhodobacteraceae bacterium strain La 6 chromos... (38% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r23c1_2.svg">
NZ_QBFD01000012 (122195-164274): Sulfitobacter sp. DFL-14 JP42 contig000012, ... (29% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r23c1_3.svg">
NZ_CP018076 (2876345-2918424): Sulfitobacter sp. AM1-D1 chromosome, complete ... (31% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r23c1_4.svg">
NZ_CP015421 (1424556-1471549): Rhodovulum sulfidophilum strain SNK001, comple... (17% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r23c1_5.svg">
NZ_CP020384 (3196493-3243495): Rhodovulum sp. MB263 chromosome, complete genome (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r23c1_6.svg">
NZ_MSYR01000001 (2582639-2629578): Rhodovulum sulfidophilum strain AB30 Scaff... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r23c1_7.svg">
NZ_AP014800 (580242-625857): Rhodovulum sulfidophilum DNA, complete genome, s... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r23c1_8.svg">
NZ_MSYP01000001 (3510761-3557700): Rhodovulum sulfidophilum strain AB14 Scaff... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r23c1_9.svg">
NZ_LR701544 (3139194-3183142): Roseovarius sp. EC-HK134 isolate RV134, whole ... (7% of genes show similarity), NRPS-like,hserlactone
</option>

<option value = "svg/clusterblast_r23c1_10.svg">
NZ_CP020474 (283000-325918): Roseovarius mucosus strain SMR3 chromosome, comp... (7% of genes show similarity), NRPS-like,hserlactone
</option>

</select>
<button id= "clusterblast-r23c1-download">Download graphic</button>
</div>
<div id="clusterblast-r23c1-svg"></div>

</div>
</div>

<div class="body-details-section r23c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-4"></div> <span class="help-tooltip" id="cb-known-help-4">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r23c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-5"></div> <span class="help-tooltip" id="cb-sub-help-5">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r23c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-8"></div> <span class="help-tooltip" id="tfbs-body-help-8">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (3):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 20.02 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r23c1-104501-NrtR"></div>

<div class="tfbs-hit-detail">
LexA
<div class="collapser collapser-target-LexA collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Repressor of DNA damage response<br>
<span>Score</span>: 23.72 out of 29.65
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r23c1-107513-LexA"></div>

<div class="tfbs-hit-detail">
NuR
<div class="collapser collapser-target-NuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Nickel-responsive regulator<br>
<span>Score</span>: 20.97 out of 30.98
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r23c1-116706-NuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r23c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-10"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-10">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r23c1-pfam-details-svg" data-filename="NZ_REEZ01000003.1_r23c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r23c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r23c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-11"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-11">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r23c1-tigrfam-details-svg" data-filename="NZ_REEZ01000003.1_r23c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r23c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r23c1-nrps_pks_monomers" data-name="r23c1-nrps_pks_monomers">
NRPS/PKS monomers
</div>



<div class="sidepanel-details-header r23c1-tfbs-finder" data-name="r23c1-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r23c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS monomer predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-6"></div> <span class="help-tooltip" id="nrps-monomers-help-6">Shows the predicted monomers for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">PADFOAJF_02840</span></strong>: pk
<div class="collapser collapser-target-PADFOAJF_02840 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>CAL_domain (21..456)</strong>: pk
<div class="collapser collapser-target-PADFOAJF_02840 collapser-level-candidate"> </div><div class="collapser-content">

minowa:  Acetyl-CoA
<div class="collapser collapser-target-PADFOAJF_02840-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Acetyl-CoA: 34.1</dt>

<dd></dd><dt>fatty_acid: 24.7</dt>

<dd></dd><dt>shikimic_acid: 0.0</dt>

<dd></dd><dt>AHBA: -4.4</dt>

<dd></dd><dt>NH2: -7.3</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r23c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-7"></div> <span class="help-tooltip" id="tfbs-side-help-7">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrtR" target="_blank">NrtR</a></td>
<td>NAD synthesis repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/LexA" target="_blank">LexA</a></td>
<td>Repressor of DNA damage response</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NuR" target="_blank">NuR</a></td>
<td>Nickel-responsive regulator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r26c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_REEZ01000004.1 - Region 1 - NRPS-like,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-27"></div> <span class="help-tooltip" id="region_svg-help-27">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_REEZ01000004.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r26c1-svg" data-filename="NZ_REEZ01000004.1_r26c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 162,202 - 213,985 nt. (total: 51,784 nt)

<a class="cluster-rules-header" id="r26c1-rules-header" href="#r26c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r26c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r26c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-28"></div> <span class="help-tooltip" id="focus-panel-help-28">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r26c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r26c1-gene-table-details" data-name="r26c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r26c1-nrps_pks" data-name="r26c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r26c1-MIBiG-cluster-compare" data-name="r26c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r26c1-clusterblast" data-name="r26c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r26c1-knownclusterblast" data-name="r26c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r26c1-subclusterblast" data-name="r26c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r26c1-tfbs-finder" data-name="r26c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r26c1-nrps-pks-bubbles" data-name="r26c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r26c1-pfam-details" data-name="r26c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r26c1-tigrfam-details" data-name="r26c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r26c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-22"></div> <span class="help-tooltip" id="gt-help-tooltip-help-22">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r26c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r26c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r26c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r26c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r26c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-12"></div> <span class="help-tooltip" id="nrps-domain-help-12">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r26c1-details-svg" data-filename="NZ_REEZ01000004.1_r26c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r26c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r26c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-13"></div> <span class="help-tooltip" id="MIBiG-help-13">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r26c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r26c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r26c1"></div>
<div id="comparison-MIBiG-r26c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002474: 103-7254">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002474/index.html#r1c1" target="_new">BGC0002474</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.48, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.48, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.62</td>
<td>NRP</td>

<td>agrobactin</td>


<td>Agrobacterium tumefaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001137: 0-18476">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001137/index.html#r1c1" target="_new">BGC0001137</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.59</td>
<td>Alkaloid</td>

<td>marinacarboline A, marinacarboline B, marinacarboline C, marinacarboline D</td>


<td>Marinactinospora thermotolerans</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000597: 0-10846">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000597/index.html#r1c1" target="_new">BGC0000597</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>






<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.51</td>
<td>RiPP</td>

<td>SWA-2138</td>


<td>Streptomyces sp. e14</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444: 3281-14045">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.50</td>
<td>Other (shikimate derived)</td>

<td>caboxamycin</td>


<td>Streptomyces sp. NTK 937</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002711: 0-9402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002711/index.html#r1c1" target="_new">BGC0002711</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.45</td>
<td>Polyketide</td>

<td>nostovalerolactone, 9-dehydronostovalerolactone</td>


<td>Nostoc punctiforme PCC 73102</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.45</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001161: 0-9161">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001161/index.html#r1c1" target="_new">BGC0001161</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.43, order:0.68, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.43, order:0.68, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.44</td>
<td>Polyketide</td>

<td>1-heptadecene</td>


<td>Moorea bouillonii PNG</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001163: 0-9299">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001163/index.html#r1c1" target="_new">BGC0001163</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.43, order:0.68, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.43, order:0.68, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.44</td>
<td>Polyketide</td>

<td>1-heptadecene</td>


<td>Moorea producens 3L</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001162: 0-9348">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001162/index.html#r1c1" target="_new">BGC0001162</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.42, order:0.68, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.42, order:0.68, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.44</td>
<td>Polyketide</td>

<td>1-heptadecene</td>


<td>Moorea producens JHB</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.43</td>
<td>Polyketide</td>

<td>strobilurin A</td>


<td>Strobilurus tenacellus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r26c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002474: 103-7254">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002474/index.html#r1c1" target="_new">BGC0002474</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.48, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>agrobactin</td>
<td>Agrobacterium tumefaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001137: 0-18476">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001137/index.html#r1c1" target="_new">BGC0001137</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Alkaloid</td>
<td>marinacarboline A, marinacarboline B, marinacarboline C, marinacarboline D</td>
<td>Marinactinospora thermotolerans</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000597: 0-10846">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000597/index.html#r1c1" target="_new">BGC0000597</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>RiPP</td>
<td>SWA-2138</td>
<td>Streptomyces sp. e14</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444: 3281-14045">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Other (shikimate derived)</td>
<td>caboxamycin</td>
<td>Streptomyces sp. NTK 937</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000868: 230-9248">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000868/index.html#r1c1" target="_new">BGC0000868</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>teichuronic acid</td>
<td>Bacillus subtilis subsp. subtilis str. 168</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000763: 750-18450">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000763/index.html#r1c1" target="_new">BGC0000763</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Saccharide</td>
<td>stewartan</td>
<td>Pantoea stewartii subsp. stewartii DC283</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001075: 2658-14492">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001075/index.html#r1c1" target="_new">BGC0001075</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.46, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene, Polyketide</td>
<td>aurachin RE</td>
<td>Rhodococcus erythropolis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002711: 0-9402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002711/index.html#r1c1" target="_new">BGC0002711</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>nostovalerolactone, 9-dehydronostovalerolactone</td>
<td>Nostoc punctiforme PCC 73102</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002366: 952-22278">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002366/index.html#r1c1" target="_new">BGC0002366</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>enterocin, neoenterocin A, neoenterocin B</td>
<td>Streptomyces sp. SCSIO 11863</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r26c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-14"></div> <span class="help-tooltip" id="cb-general-help-14">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r26c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r26c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r26c1_1.svg">
NZ_KE386893 (29718-81879): Pseudodonghicola xiamenensis DSM 18339 G455DRAFT s... (33% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r26c1_2.svg">
NZ_KI421500 (36467-88446): Leisingera daeponensis DSM 23529 Phada neu scaffol... (35% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r26c1_3.svg">
NZ_CP041159 (438023-490057): Leisingera aquaemixtae strain R2C4 chromosome, c... (35% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r26c1_4.svg">
NZ_DS995281 (598475-650509): Rhodobacterales bacterium Y4I scf 1106129324773,... (34% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r26c1_5.svg">
NZ_KI421513 (2223863-2275891): Leisingera caerulea DSM 24564 PhacaeDRAFT Scaf... (33% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r26c1_6.svg">
NZ_JWLC01000011 (38003-90033): Leisingera sp. ANG-M1 contig 11, whole genome ... (34% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r26c1_7.svg">
NZ_CP035417 (3025329-3077348): Leisingera sp. NJS204 chromosome, complete genome (35% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r26c1_8.svg">
NZ_KI421504 (4111057-4163085): Leisingera aquimarina DSM 24565 Leiaq neu scaf... (34% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r26c1_9.svg">
NZ_CP018076 (1138633-1217190): Sulfitobacter sp. AM1-D1 chromosome, complete ... (17% of genes show similarity), NRPS-like,T1PKS,T3PKS
</option>

<option value = "svg/clusterblast_r26c1_10.svg">
NZ_JWLE01000021 (34729-86763): Leisingera sp. ANG-DT contig 21, whole genome ... (33% of genes show similarity), NRPS-like,T1PKS
</option>

</select>
<button id= "clusterblast-r26c1-download">Download graphic</button>
</div>
<div id="clusterblast-r26c1-svg"></div>

</div>
</div>

<div class="body-details-section r26c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-15"></div> <span class="help-tooltip" id="cb-known-help-15">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r26c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r26c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r26c1_1.svg">
bottromycin A2
</option>

</select>
<button id="knownclusterblast-r26c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r26c1-svg"></div>

</div>
</div>

<div class="body-details-section r26c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-16"></div> <span class="help-tooltip" id="cb-sub-help-16">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r26c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-20"></div> <span class="help-tooltip" id="tfbs-body-help-20">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (3):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 20.23 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r26c1-172165-ZuR"></div>

<div class="tfbs-hit-detail">
NuR
<div class="collapser collapser-target-NuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Nickel-responsive regulator<br>
<span>Score</span>: 20.72 out of 30.98
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r26c1-197569-NuR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 18.84 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r26c1-203337-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r26c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-21"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-21">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r26c1-domain-bubble-svg" data-filename="NZ_REEZ01000004.1_r26c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r26c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (162201 - 213985): neighbouring NRPS-like-T1PKS
</option>



<option value="CC2">
Candidate 2 (162201 - 206719): single NRPS-like
</option>



<option value="CC3">
Candidate 3 (167655 - 213985): single T1PKS
</option>


</select>

<div class="details-svg" id="r26c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r26c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r26c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-23"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-23">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r26c1-pfam-details-svg" data-filename="NZ_REEZ01000004.1_r26c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r26c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r26c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-24"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-24">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r26c1-tigrfam-details-svg" data-filename="NZ_REEZ01000004.1_r26c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r26c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r26c1-nrps_pks_products" data-name="r26c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r26c1-nrps_pks_monomers" data-name="r26c1-nrps_pks_monomers">
NRPS/PKS monomers
</div>



<div class="sidepanel-details-header r26c1-tfbs-finder" data-name="r26c1-tfbs-finder">
TFBS Finder
</div>






</div>










<div class="sidepanel-details-section r26c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-17"></div> <span class="help-tooltip" id="nrps-structures-help-17">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 162201 - 213985:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r26c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r26c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 2, location 162201 - 206719:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r26c1-2" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r26c1-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 3, location 167655 - 213985:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r26c1-3" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r26c1-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r26c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS monomer predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-18"></div> <span class="help-tooltip" id="nrps-monomers-help-18">Shows the predicted monomers for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">PADFOAJF_03481</span></strong>: X
<div class="collapser collapser-target-PADFOAJF_03481 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>A-OX (538..1311)</strong>: X
<div class="collapser collapser-target-PADFOAJF_03481 collapser-level-candidate"> </div><div class="collapser-content">

</div>
</dd>




</div>
<br>

<strong><span class="serif">PADFOAJF_03483</span></strong>: mal
<div class="collapser collapser-target-PADFOAJF_03483 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (534..828)</strong>: mal
<div class="collapser collapser-target-PADFOAJF_03483 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-PADFOAJF_03483-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 62.5%</dd>
<dd>2-Rhyd-Malonyl-CoA: 58.3%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-PADFOAJF_03483-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 78.1</dt>

<dd></dd><dt>Methylmalonyl-CoA: 46.0</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 43.8</dt>

<dd></dd><dt>inactive: 35.6</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 34.7</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 22.3</dt>

<dd></dd><dt>fatty_acid: 22.1</dt>

<dd></dd><dt>Isobutyryl-CoA: 15.8</dt>

<dd></dd><dt>Propionyl-CoA: 15.2</dt>

<dd></dd><dt>Benzoyl-CoA: 12.1</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 11.0</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>CHC-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

<strong><span class="serif">PADFOAJF_03488</span></strong>: X
<div class="collapser collapser-target-PADFOAJF_03488 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (25..430)</strong>: X
<div class="collapser collapser-target-PADFOAJF_03488 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-PADFOAJF_03488-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">GAHFIAYGNK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>clAnt <span class="serif">GAFFIGAGVK</span> (38% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

<strong><span class="serif">PADFOAJF_03492</span></strong>: X
<div class="collapser collapser-target-PADFOAJF_03492 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (16..400)</strong>: X
<div class="collapser collapser-target-PADFOAJF_03492 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Asp, Asn, Glu, Gln, Aad
<div class="collapser collapser-target-PADFOAJF_03492-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">GLAMVGRMNK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r26c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-19"></div> <span class="help-tooltip" id="tfbs-side-help-19">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NuR" target="_blank">NuR</a></td>
<td>Nickel-responsive regulator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>






</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>