
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>GCF_001884735.1_Natronohydrobacter_thiooxidans_AH01 - 3 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/bacteria.css">
<meta property="og:title" content="GCF_001884735.1_Natronohydrobacter_thiooxidans_AH01 - 3 record(s) - 3 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-80216b44-3bef-4830-8f52-56cdad60a54f/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://antismash.secondarymetabolites.org/">
<img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://antismash.secondarymetabolites.org/">
antiSMASH version 7.0.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="GCF_001884735.1_Natronohydrobacter_thiooxidans_AH01.zip">Download all results</a></li>

<li><a href="GCF_001884735.1_Natronohydrobacter_thiooxidans_AH01.gbk">Download GenBank summary file</a></li>

<li><a href="bacteria-80216b44-3bef-4830-8f52-56cdad60a54f.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton hybrid r2c1">
<a href="#r2c1">2.1</a>
</div>



<div class="regbutton other hserlactone r3c1">
<a href="#r3c1">3.1</a>
</div>



<div class="regbutton other ectoine r13c1">
<a href="#r13c1">13.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>NZ_MJAP01000001.1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-2" data-filename="NZ_MJAP01000001.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-2" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r2c1">
<td class="regbutton hybrid r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">60,669</td>
<td class="digits table-split-left">112,456</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001941/1" target="_blank">fulvuthiacene A/fulvuthiacene B</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 8%, #ffffff00 8%)">8%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_MJAP01000010.1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-3" data-filename="NZ_MJAP01000010.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-3" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r3c1">
<td class="regbutton other hserlactone r3c1">
<a href="#r3c1">Region&nbsp3.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">87,997</td>
<td class="digits table-split-left">108,824</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NZ_MJAP01000002.1</strong>
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-13" data-filename="NZ_MJAP01000002.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-13" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r13c1">
<td class="regbutton other ectoine r13c1">
<a href="#r13c1">Region&nbsp13.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
</td>
<td class="digits">244,544</td>
<td class="digits table-split-left">254,930</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000860/1" target="_blank">ectoine</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 80%, #ffffff00 80%)">80%</td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r2c1">
<td class="regbutton hybrid r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">60,669</td>
<td class="digits table-split-left">112,456</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001941/1" target="_blank">fulvuthiacene A/fulvuthiacene B</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 8%, #ffffff00 8%)">8%</td>

</tr>


<tr class="linked-row even" data-anchor="#r3c1">
<td class="regbutton other hserlactone r3c1">
<a href="#r3c1">Region&nbsp3.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hserlactone" target="_blank">hserlactone</a>
</td>
<td class="digits">87,997</td>
<td class="digits table-split-left">108,824</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r13c1">
<td class="regbutton other ectoine r13c1">
<a href="#r13c1">Region&nbsp13.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
</td>
<td class="digits">244,544</td>
<td class="digits table-split-left">254,930</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000860/1" target="_blank">ectoine</a></td>
<td>Other</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 80%, #ffffff00 80%)">80%</td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>NZ_MJAP01000039.1</strong></dd>

<dd><strong>NZ_MJAP01000011.1</strong></dd>

<dd><strong>NZ_MJAP01000012.1</strong></dd>

<dd><strong>NZ_MJAP01000013.1</strong></dd>

<dd><strong>NZ_MJAP01000014.1</strong></dd>

<dd><strong>NZ_MJAP01000015.1</strong></dd>

<dd><strong>NZ_MJAP01000016.1</strong></dd>

<dd><strong>NZ_MJAP01000017.1</strong></dd>

<dd><strong>NZ_MJAP01000018.1</strong></dd>

<dd><strong>NZ_MJAP01000019.1</strong></dd>

<dd><strong>NZ_MJAP01000020.1</strong></dd>

<dd><strong>NZ_MJAP01000021.1</strong></dd>

<dd><strong>NZ_MJAP01000022.1</strong></dd>

<dd><strong>NZ_MJAP01000023.1</strong></dd>

<dd><strong>NZ_MJAP01000024.1</strong></dd>

<dd><strong>NZ_MJAP01000025.1</strong></dd>

<dd><strong>NZ_MJAP01000026.1</strong></dd>

<dd><strong>NZ_MJAP01000027.1</strong></dd>

<dd><strong>NZ_MJAP01000028.1</strong></dd>

<dd><strong>NZ_MJAP01000029.1</strong></dd>

<dd><strong>NZ_MJAP01000003.1</strong></dd>

<dd><strong>NZ_MJAP01000030.1</strong></dd>

<dd><strong>NZ_MJAP01000031.1</strong></dd>

<dd><strong>NZ_MJAP01000032.1</strong></dd>

<dd><strong>NZ_MJAP01000033.1</strong></dd>

<dd><strong>NZ_MJAP01000034.1</strong></dd>

<dd><strong>NZ_MJAP01000035.1</strong></dd>

<dd><strong>NZ_MJAP01000036.1</strong></dd>

<dd><strong>NZ_MJAP01000037.1</strong></dd>

<dd><strong>NZ_MJAP01000038.1</strong></dd>

<dd><strong>NZ_MJAP01000004.1</strong></dd>

<dd><strong>NZ_MJAP01000005.1</strong></dd>

<dd><strong>NZ_MJAP01000006.1</strong></dd>

<dd><strong>NZ_MJAP01000007.1</strong></dd>

<dd><strong>NZ_MJAP01000008.1</strong></dd>

<dd><strong>NZ_MJAP01000009.1</strong></dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>







<div class="page" id='r2c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_MJAP01000001.1 - Region 1 - NRPS-like,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-30"></div> <span class="help-tooltip" id="region_svg-help-30">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_MJAP01000001.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c1-svg" data-filename="NZ_MJAP01000001.1_r2c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 60,669 - 112,456 nt. (total: 51,788 nt)

<a class="cluster-rules-header" id="r2c1-rules-header" href="#r2c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r2c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-31"></div> <span class="help-tooltip" id="focus-panel-help-31">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c1-gene-table-details" data-name="r2c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c1-nrps_pks" data-name="r2c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c1-MIBiG-cluster-compare" data-name="r2c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c1-clusterblast" data-name="r2c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c1-knownclusterblast" data-name="r2c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c1-subclusterblast" data-name="r2c1-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r2c1-nrps-pks-bubbles" data-name="r2c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r2c1-pfam-details" data-name="r2c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r2c1-tigrfam-details" data-name="r2c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r2c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-9"></div> <span class="help-tooltip" id="gt-help-tooltip-help-9">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-1"></div> <span class="help-tooltip" id="nrps-domain-help-1">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-details-svg" data-filename="NZ_MJAP01000001.1_r2c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-2"></div> <span class="help-tooltip" id="MIBiG-help-2">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c1"></div>
<div id="comparison-MIBiG-r2c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002474: 103-7254">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002474/index.html#r1c1" target="_new">BGC0002474</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.49</td>
<td>NRP</td>

<td>agrobactin</td>


<td>Agrobacterium tumefaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.44</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000903: 126-11373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000903/index.html#r1c1" target="_new">BGC0000903</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.54, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.54, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.43</td>
<td>Other</td>

<td>fluorometabolite</td>


<td>Streptomyces cattleya</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.43</td>
<td>Polyketide</td>

<td>strobilurin A</td>


<td>Strobilurus tenacellus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001177: 10040-36741">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001177/index.html#r1c1" target="_new">BGC0001177</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.54, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.54, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.43</td>
<td>Polyketide</td>

<td>mensacaricin</td>


<td>Streptomyces bottropensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276: 1101-6575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.42</td>
<td>Polyketide</td>

<td>6-methylsalicyclic acid</td>


<td>Aspergillus terreus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.42</td>
<td>Polyketide</td>

<td>citreoviridin</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275: 1063-6614">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.42</td>
<td>Polyketide</td>

<td>6-methylsalicyclic acid</td>


<td>Glarea lozoyensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273: 0-5301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.42</td>
<td>Polyketide</td>

<td>asperlactone</td>


<td>Aspergillus ochraceus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.42</td>
<td>Polyketide</td>

<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002474: 103-7254">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002474/index.html#r1c1" target="_new">BGC0002474</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>agrobactin</td>
<td>Agrobacterium tumefaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001177: 10040-36741">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001177/index.html#r1c1" target="_new">BGC0001177</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.54, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>mensacaricin</td>
<td>Streptomyces bottropensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000903: 126-11373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000903/index.html#r1c1" target="_new">BGC0000903</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.54, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Other</td>
<td>fluorometabolite</td>
<td>Streptomyces cattleya</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>strobilurin A</td>
<td>Strobilurus tenacellus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276: 1101-6575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>6-methylsalicyclic acid</td>
<td>Aspergillus terreus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>citreoviridin</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275: 1063-6614">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>6-methylsalicyclic acid</td>
<td>Glarea lozoyensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273: 0-5301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>asperlactone</td>
<td>Aspergillus ochraceus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-3"></div> <span class="help-tooltip" id="cb-general-help-3">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c1_1.svg">
NZ_FNVD01000002 (221290-273384): Jhaorihella thermophila strain DSM 23413, wh... (29% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r2c1_2.svg">
NZ_QJJL01000001 (340047-392033): Loktanella sp. PT4BL Ga0206391 101, whole ge... (27% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r2c1_3.svg">
NZ_CP015124 (3864180-3916338): Phaeobacter gallaeciensis strain JL2886, compl... (26% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r2c1_4.svg">
NZ_PHJF01000003 (149422-201425): Epibacterium ulvae strain U95 Ga0207336 103,... (26% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r2c1_5.svg">
NZ_KE386893 (29718-81879): Pseudodonghicola xiamenensis DSM 18339 G455DRAFT s... (25% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r2c1_6.svg">
NZ_FOCE01000008 (86958-138943): Gemmobacter aquatilis strain DSM 3857, whole ... (20% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r2c1_7.svg">
NZ_FNOM01000001 (453882-506024): Roseicitreum antarcticum strain CGMCC 1.8894... (17% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r2c1_8.svg">
NZ_JWLC01000011 (38003-90033): Leisingera sp. ANG-M1 contig 11, whole genome ... (22% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r2c1_9.svg">
NZ_CP041159 (438023-490057): Leisingera aquaemixtae strain R2C4 chromosome, c... (23% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r2c1_10.svg">
NZ_LR027553 (2588310-2640345): Epibacterium mobile isolate EPIB1 chromosome 1 (26% of genes show similarity), NRPS-like,T1PKS
</option>

</select>
<button id= "clusterblast-r2c1-download">Download graphic</button>
</div>
<div id="clusterblast-r2c1-svg"></div>

</div>
</div>

<div class="body-details-section r2c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-4"></div> <span class="help-tooltip" id="cb-known-help-4">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r2c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r2c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r2c1_1.svg">
fulvuthiacene A/fulvuthiacene B
</option>

</select>
<button id="knownclusterblast-r2c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r2c1-svg"></div>

</div>
</div>

<div class="body-details-section r2c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-5"></div> <span class="help-tooltip" id="cb-sub-help-5">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r2c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-8"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-8">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-domain-bubble-svg" data-filename="NZ_MJAP01000001.1_r2c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (60668 - 112456): neighbouring NRPS-like-T1PKS
</option>



<option value="CC2">
Candidate 2 (60668 - 105156): single NRPS-like
</option>



<option value="CC3">
Candidate 3 (66150 - 112456): single T1PKS
</option>


</select>

<div class="details-svg" id="r2c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r2c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-10"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-10">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-pfam-details-svg" data-filename="NZ_MJAP01000001.1_r2c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r2c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-11"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-11">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-tigrfam-details-svg" data-filename="NZ_MJAP01000001.1_r2c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r2c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c1-nrps_pks_products" data-name="r2c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c1-nrps_pks_monomers" data-name="r2c1-nrps_pks_monomers">
NRPS/PKS monomers
</div>








</div>










<div class="sidepanel-details-section r2c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-6"></div> <span class="help-tooltip" id="nrps-structures-help-6">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 60668 - 112456:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 2, location 60668 - 105156:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c1-2" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c1-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 3, location 66150 - 112456:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c1-3" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c1-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CC(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS monomer predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-7"></div> <span class="help-tooltip" id="nrps-monomers-help-7">Shows the predicted monomers for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">ctg2_80</span></strong>: X
<div class="collapser collapser-target-ctg2_80 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>A-OX (518..1293)</strong>: X
<div class="collapser collapser-target-ctg2_80 collapser-level-candidate"> </div><div class="collapser-content">

</div>
</dd>




</div>
<br>

<strong><span class="serif">ctg2_82</span></strong>: mal
<div class="collapser collapser-target-ctg2_82 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (528..823)</strong>: mal
<div class="collapser collapser-target-ctg2_82 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-ctg2_82-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 62.5%</dd>
<dd>2-Rhyd-Malonyl-CoA: 58.3%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-ctg2_82-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 66.8</dt>

<dd></dd><dt>Methylmalonyl-CoA: 39.1</dt>

<dd></dd><dt>inactive: 36.8</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 36.5</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 15.9</dt>

<dd></dd><dt>Benzoyl-CoA: 12.7</dt>

<dd></dd><dt>Propionyl-CoA: 9.4</dt>

<dd></dd><dt>Isobutyryl-CoA: 6.7</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 5.5</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>fatty_acid: 0.0</dt>

<dd></dd><dt>CHC-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>








</div>

</div>

</div>




<div class="page" id='r3c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_MJAP01000010.1 - Region 1 - hserlactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-32"></div> <span class="help-tooltip" id="region_svg-help-32">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_MJAP01000010.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c1-svg" data-filename="NZ_MJAP01000010.1_r3c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 87,997 - 108,824 nt. (total: 20,828 nt)

<a class="cluster-rules-header" id="r3c1-rules-header" href="#r3c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c1-rules">hserlactone: Autoind_synth</div>

<div class="region-svg-container">
<div id='r3c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-33"></div> <span class="help-tooltip" id="focus-panel-help-33">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c1-gene-table-details" data-name="r3c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c1-MIBiG-cluster-compare" data-name="r3c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c1-clusterblast" data-name="r3c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c1-knownclusterblast" data-name="r3c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c1-subclusterblast" data-name="r3c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r3c1-tfbs-finder" data-name="r3c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r3c1-pfam-details" data-name="r3c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r3c1-tigrfam-details" data-name="r3c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r3c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-18"></div> <span class="help-tooltip" id="gt-help-tooltip-help-18">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-12"></div> <span class="help-tooltip" id="MIBiG-help-12">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c1"></div>
<div id="comparison-MIBiG-r3c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">hserlactone</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000616: 0-4173">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000616/index.html#r1c1" target="_new">BGC0000616</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>amylocyclicin</td>


<td>Bacillus velezensis FZB42</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002531: 214-11704">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002531/index.html#r1c1" target="_new">BGC0002531</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>NRP</td>

<td>cepaciachelin</td>


<td>Burkholderia ambifaria AMMD</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002442: 0-8391">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002442/index.html#r1c1" target="_new">BGC0002442</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.44, order:0.40, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>toyoncin</td>


<td>Bacillus toyonensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002490: 0-5965">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002490/index.html#r1c1" target="_new">BGC0002490</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>Other</td>

<td>yersinopine</td>


<td>Yersinia pestis CO92</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551: 0-6728">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>SapB</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000571: 0-4574">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000571/index.html#r1c1" target="_new">BGC0000571</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>burhizin</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002325: 814-4534">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002325/index.html#r1c1" target="_new">BGC0002325</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>mycetohabin-16</td>


<td>Mycetohabitans rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001226: 0-6018">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001226/index.html#r1c1" target="_new">BGC0001226</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>streptocollin</td>


<td>Streptomyces collinus Tu 365</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563: 185-5330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>venezuelin</td>


<td>Streptomyces venezuelae ATCC 10712</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002312: 3-4377">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002312/index.html#r1c1" target="_new">BGC0002312</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>ubonodin</td>


<td>Burkholderia ubonensis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000616: 0-4173">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000616/index.html#r1c1" target="_new">BGC0000616</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>amylocyclicin</td>
<td>Bacillus velezensis FZB42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002531: 214-11704">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002531/index.html#r1c1" target="_new">BGC0002531</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>NRP</td>
<td>cepaciachelin</td>
<td>Burkholderia ambifaria AMMD</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002442: 0-8391">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002442/index.html#r1c1" target="_new">BGC0002442</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.44, order:0.40, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>RiPP</td>
<td>toyoncin</td>
<td>Bacillus toyonensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002490: 0-5965">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002490/index.html#r1c1" target="_new">BGC0002490</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Other</td>
<td>yersinopine</td>
<td>Yersinia pestis CO92</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551: 0-6728">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>RiPP</td>
<td>SapB</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000571: 0-4574">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000571/index.html#r1c1" target="_new">BGC0000571</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>RiPP</td>
<td>burhizin</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002325: 814-4534">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002325/index.html#r1c1" target="_new">BGC0002325</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>RiPP</td>
<td>mycetohabin-16</td>
<td>Mycetohabitans rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001226: 0-6018">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001226/index.html#r1c1" target="_new">BGC0001226</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>RiPP</td>
<td>streptocollin</td>
<td>Streptomyces collinus Tu 365</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563: 185-5330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>RiPP</td>
<td>venezuelin</td>
<td>Streptomyces venezuelae ATCC 10712</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002312: 3-4377">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002312/index.html#r1c1" target="_new">BGC0002312</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>RiPP</td>
<td>ubonodin</td>
<td>Burkholderia ubonensis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-13"></div> <span class="help-tooltip" id="cb-general-help-13">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c1_1.svg">
NZ_MEHT01000009 (229666-250488): Roseinatronobacter thiooxidans strain ALG1 A... (50% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r3c1_2.svg">
NZ_CP024899 (2485912-2506734): Rhodobaca barguzinensis strain alga05 chromoso... (50% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r3c1_3.svg">
NZ_CP033442 (1410091-1430724): Rhodobacter sphaeroides strain AB25 chromosome 1 (16% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r3c1_4.svg">
NC_011963 (1467095-1487728): Rhodobacter sphaeroides KD131 chromosome 1, comp... (16% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r3c1_5.svg">
NZ_CM001161 (1223753-1244386): Rhodobacter sphaeroides WS8N chromosome I, who... (16% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r3c1_6.svg">
NZ_CP033446 (1400560-1421193): Rhodobacter sphaeroides strain AB27 chromosome 1 (15% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r3c1_7.svg">
NC_009049 (1847444-1868077): Rhodobacter sphaeroides ATCC 17029 chromosome 1,... (16% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r3c1_8.svg">
NZ_FZOV01000001 (7995-28628): Rhodobacter megalophilus strain DSM 18937, whol... (16% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r3c1_9.svg">
NZ_CP030271 (1827287-1847920): Rhodobacter sphaeroides 2.4.1 chromosome 1, co... (16% of genes show similarity), hserlactone
</option>

<option value = "svg/clusterblast_r3c1_10.svg">
NZ_PZZW01000002 (473590-494223): Rhodobacter johrii strain JA192 Ga0191650 10... (16% of genes show similarity), hserlactone
</option>

</select>
<button id= "clusterblast-r3c1-download">Download graphic</button>
</div>
<div id="clusterblast-r3c1-svg"></div>

</div>
</div>

<div class="body-details-section r3c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-14"></div> <span class="help-tooltip" id="cb-known-help-14">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-15"></div> <span class="help-tooltip" id="cb-sub-help-15">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r3c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-17"></div> <span class="help-tooltip" id="tfbs-body-help-17">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (3):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
DasR
<div class="collapser collapser-target-DasR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: N-acetylglucosamine dependent repressor<br>
<span>Score</span>: 20.59 out of 27.04
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r3c1-89179-DasR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 18.44 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r3c1-97991-ZuR"></div>

<div class="tfbs-hit-detail">
CsoR
<div class="collapser collapser-target-CsoR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Copper-responsive repressor<br>
<span>Score</span>: 22.24 out of 32.38
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r3c1-104543-CsoR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r3c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-19"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-19">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c1-pfam-details-svg" data-filename="NZ_MJAP01000010.1_r3c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r3c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-20"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-20">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c1-tigrfam-details-svg" data-filename="NZ_MJAP01000010.1_r3c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r3c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r3c1-tfbs-finder" data-name="r3c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r3c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-16"></div> <span class="help-tooltip" id="tfbs-side-help-16">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/DasR" target="_blank">DasR</a></td>
<td>N-acetylglucosamine dependent repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/CsoR" target="_blank">CsoR</a></td>
<td>Copper-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>




<div class="page" id='r13c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NZ_MJAP01000002.1 - Region 1 - ectoine <div class="help-container"> <div class="help-icon" data-id="region_svg-help-34"></div> <span class="help-tooltip" id="region_svg-help-34">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NZ_MJAP01000002.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r13c1-svg" data-filename="NZ_MJAP01000002.1_r13c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 244,544 - 254,930 nt. (total: 10,387 nt)

<a class="cluster-rules-header" id="r13c1-rules-header" href="#r13c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r13c1-rules">ectoine: ectoine_synt</div>

<div class="region-svg-container">
<div id='r13c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-35"></div> <span class="help-tooltip" id="focus-panel-help-35">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r13c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r13c1-gene-table-details" data-name="r13c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r13c1-MIBiG-cluster-compare" data-name="r13c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r13c1-clusterblast" data-name="r13c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r13c1-knownclusterblast" data-name="r13c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r13c1-subclusterblast" data-name="r13c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r13c1-tfbs-finder" data-name="r13c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r13c1-pfam-details" data-name="r13c1-pfam-details">
Pfam domains
</div>

<div class="body-details-header r13c1-tigrfam-details" data-name="r13c1-tigrfam-details">
TIGRFAM domains
</div>


</div>


<div class="body-details-section r13c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-27"></div> <span class="help-tooltip" id="gt-help-tooltip-help-27">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r13c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r13c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r13c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r13c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r13c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-21"></div> <span class="help-tooltip" id="MIBiG-help-21">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r13c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r13c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r13c1"></div>
<div id="comparison-MIBiG-r13c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">ectoine</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860: 710-5134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860</a></td>





<td class="cc-heat-cell" title="0.81 (id:0.53, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>


<td>0.55</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylarcula marina</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859: 258-3305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859</a></td>





<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>


<td>0.51</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium alcaliphilum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856: 517-4347">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856</a></td>





<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>


<td>0.51</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylophaga thalassica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857: 300-4871">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857</a></td>





<td class="cc-heat-cell" title="0.79 (id:0.49, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>


<td>0.51</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylophaga alcalica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855: 245-3455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855</a></td>





<td class="cc-heat-cell" title="0.73 (id:0.50, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>


<td>0.41</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium kenyense</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.49, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>Other</td>

<td>ectoine</td>


<td>Sporosarcina pasteurii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854: 838-6879">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.49, order:0.83, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium alcaliphilum 20Z</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.47, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylobacter marinus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052: 188-3620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.48, order:0.57, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Other (Ectoine)</td>

<td>ectoine</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853: 8-3345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.48, order:0.57, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>Other</td>

<td>ectoine</td>


<td>Streptomyces anulatus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r13c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860: 710-5134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860</a></td>


<td class="cc-heat-cell" title="0.81 (id:0.53, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>
<td>0.81</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylarcula marina</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859: 258-3305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859</a></td>


<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
<td>0.79</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium alcaliphilum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856: 517-4347">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856</a></td>


<td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
<td>0.79</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylophaga thalassica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857: 300-4871">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857</a></td>


<td class="cc-heat-cell" title="0.79 (id:0.49, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
<td>0.79</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylophaga alcalica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855: 245-3455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855</a></td>


<td class="cc-heat-cell" title="0.73 (id:0.50, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
<td>0.73</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium kenyense</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852: 415-2854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.49, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Other</td>
<td>ectoine</td>
<td>Sporosarcina pasteurii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854: 838-6879">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.49, order:0.83, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium alcaliphilum 20Z</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858: 213-3827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.47, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylobacter marinus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052: 188-3620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.48, order:0.57, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other (Ectoine)</td>
<td>ectoine</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853: 8-3345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.48, order:0.57, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>ectoine</td>
<td>Streptomyces anulatus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r13c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-22"></div> <span class="help-tooltip" id="cb-general-help-22">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r13c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r13c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r13c1_1.svg">
NZ_CP032509 (1081147-1091555): Georhizobium profundi strain WS11 chromosome, ... (63% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r13c1_2.svg">
NZ_CP049030 (25308-35707): Rhodobacteraceae bacterium SC52 plasmid pSC52 2, c... (60% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r13c1_3.svg">
NZ_CP010855 (74978-85374): Marinovum algicola DG 898 chromosome, complete genome (60% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r13c1_4.svg">
NZ_RAPP01000004 (318704-372727): Thalassospira sp. DSM 26710 Ga0180962 104, w... (11% of genes show similarity), arylpolyene,ectoine
</option>

<option value = "svg/clusterblast_r13c1_5.svg">
NZ_KZ826488 (5974-44251): Rhodobacteraceae bacterium FSX-11 Scaffold8, whole ... (19% of genes show similarity), betalactone,ectoine
</option>

<option value = "svg/clusterblast_r13c1_6.svg">
NZ_AXBF01000005 (12445-50698): Pseudophaeobacter arcticus DSM 23566 Phaar neu... (19% of genes show similarity), betalactone,ectoine
</option>

<option value = "svg/clusterblast_r13c1_7.svg">
NZ_SWKO01000011 (644250-654652): Marivita sp. LZ-15-2 NODE 1, whole genome sh... (54% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r13c1_8.svg">
NZ_CP024899 (2098634-2109021): Rhodobaca barguzinensis strain alga05 chromoso... (50% of genes show similarity), ectoine
</option>

<option value = "svg/clusterblast_r13c1_9.svg">
NZ_MEHT01000009 (25734-53535): Roseinatronobacter thiooxidans strain ALG1 ALG... (17% of genes show similarity), ectoine,terpene
</option>

<option value = "svg/clusterblast_r13c1_10.svg">
NZ_LXYH01000056 (3698-14082): Rhodobacteraceae bacterium EhC02 A5 Contig6, wh... (50% of genes show similarity), ectoine
</option>

</select>
<button id= "clusterblast-r13c1-download">Download graphic</button>
</div>
<div id="clusterblast-r13c1-svg"></div>

</div>
</div>

<div class="body-details-section r13c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-23"></div> <span class="help-tooltip" id="cb-known-help-23">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r13c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r13c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r13c1_1.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r13c1_2.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r13c1_3.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r13c1_4.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r13c1_5.svg">
ectoine
</option>

<option value = "svg/knownclusterblast_r13c1_6.svg">
ectoine
</option>

</select>
<button id="knownclusterblast-r13c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r13c1-svg"></div>

</div>
</div>

<div class="body-details-section r13c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-24"></div> <span class="help-tooltip" id="cb-sub-help-24">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r13c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-26"></div> <span class="help-tooltip" id="tfbs-body-help-26">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 21.26 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r13c1-254545-ZuR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (2):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 21.13 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r13c1-251825-NrtR"></div>

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 23.18 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r13c1-251878-NrtR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r13c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-28"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-28">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r13c1-pfam-details-svg" data-filename="NZ_MJAP01000002.1_r13c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r13c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>

<div class="body-details-section r13c1-tigrfam-details">

<div class="details">
<div class="heading">
<span>Detailed TIGRFAM domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-29"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-29">Shows TIGRFAM domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r13c1-tigrfam-details-svg" data-filename="NZ_MJAP01000002.1_r13c1_tigrfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="tigrfam-details-svg" id="r13c1-tigrfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r13c1-tfbs-finder" data-name="r13c1-tfbs-finder">
TFBS Finder
</div>




</div>








<div class="sidepanel-details-section r13c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-25"></div> <span class="help-tooltip" id="tfbs-side-help-25">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrtR" target="_blank">NrtR</a></td>
<td>NAD synthesis repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>